<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html lang="en-GB" xml:lang="en-GB" xmlns:og="http://ogp.me/ns#" xmlns:m="http://www.w3.org/1998/Math/MathML" xmlns="http://www.w3.org/1999/xhtml" id="js">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">


<meta name="wpd_version" content="0.2">
<meta name="wpd_baseurl" content="http://www.retrovirology.com/content/7/1/23">
<meta name="wpd_url" content="http://www.retrovirology.com/content/7/1/23">
<meta name="wpd_date" content="2013-04-28T10:43Z">


 
    
    
                                                                                                                                                                                                                                
                                                        
    
            

            
        <script async="" src="cb=gapi.loaded_0"></script><script src="widgets.js" id="twitter-wjs"></script><script gapi_processed="true" src="plusone.js" async="" type="text/javascript"></script><script src="all.js" id="facebook-jssdk"></script><script src="ga.js" async="" type="text/javascript"></script><script>window.bmcIsMobile = "classic";</script>
        
        
        <title>Retrovirology | Full text | Frequent CXCR4 tropism of HIV-1 subtype A and CRF02_AG during late-stage disease - indication of an evolving epidemic in West Africa</title>
<link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
<meta http-equiv="X-UA-Compatible" content="IE=7">


                
    
		
		
		
<!-- <?xml version="1.0" encoding="UTF-8"?>
<rdf:RDF xmlns:cc="http://web.resource.org/cc/" xmlns="http://purl.org/rss/1.0/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:prism="http://prismstandard.org/namespaces/1.2/basic/">
<cc:Work rdf:about="http://www.retrovirology.com/content/7/1/23">
<cc:license rdf:resource="http://creativecommons.org/licenses/by/2.0/"/>
</cc:Work>
<cc:License rdf:about="http://creativecommons.org/licenses/by/2.0/">
<cc:permits rdf:resource="http://web.resource.org/cc/Reproduction"/>
<cc:permits rdf:resource="http://web.resource.org/cc/Distribution"/>
<cc:requires rdf:resource="http://web.resource.org/cc/Notice"/>
<cc:requires rdf:resource="http://web.resource.org/cc/Attribution"/>
<cc:permits rdf:resource="http://web.resource.org/cc/DerivativeWorks"/>
</cc:License>
<item rdf:about="http://www.retrovirology.com/content/7/1/23">
<title>Frequent CXCR4 tropism of HIV-1 subtype A and CRF02_AG during late-stage disease - indication of an evolving epidemic in West Africa</title>
<dc:title>Frequent CXCR4 tropism of HIV-1 subtype A and CRF02_AG during late-stage disease - indication of an evolving epidemic in West Africa</dc:title>
<dc:creator>Esbjörnsson, Joakim</dc:creator>
<dc:creator>Månsson, Fredrik</dc:creator>
<dc:creator>Martínez-Arias, Wilma</dc:creator>
<dc:creator>Vincic, Elzbieta</dc:creator>
<dc:creator>Biague, Antonio J</dc:creator>
<dc:creator>da Silva, Zacarias J</dc:creator>
<dc:creator>Fenyö, Eva M</dc:creator>
<dc:creator>Norrgren, Hans</dc:creator>
<dc:creator>Medstrand, Patrik</dc:creator>
<dc:identifier>info:doi/10.1186/1742-4690-7-23</dc:identifier>
<dc:identifier>info:pmid/20307309</dc:identifier>
<dc:source>Retrovirology 2010, 7:23</dc:source>
<dc:date>2010-03-22</dc:date>

<prism:publicationName>Retrovirology</prism:publicationName>
<prism:publicationDate>2010-03-22</prism:publicationDate>
<prism:volume>7</prism:volume>
<prism:number>1</prism:number>
<prism:section>Research</prism:section>
<prism:startingPage>23</prism:startingPage>
<prism:copyright>2010 Esbjörnsson et al; licensee BioMed Central Ltd.</prism:copyright>
</item>
</rdf:RDF>
 -->
 
 
<meta name="citation_journal_title" content="Retrovirology">
<meta name="citation_publisher" content="BioMed Central Ltd">


<meta name="citation_author" content="Joakim Esbjörnsson">
<meta name="citation_author_institution" content="Department of Experimental Medical Science, Section of Molecular Virology, Lund University, Lund, Sweden">
<meta name="citation_author" content="Fredrik Månsson">
<meta name="citation_author_institution" content="Department of Clinical Sciences, Malmö, Infectious Diseases Research Unit, Lund University, Sweden">
<meta name="citation_author" content="Wilma Martínez-Arias">
<meta name="citation_author_institution" content="Department of Experimental Medical Science, Section of Molecular Virology, Lund University, Lund, Sweden">
<meta name="citation_author" content="Elzbieta Vincic">
<meta name="citation_author_institution" content="Department of Laboratory Medicine Lund, Section of Virology, Lund University, Lund, Sweden">
<meta name="citation_author" content="Antonio J Biague">
<meta name="citation_author_institution" content="National Public Health Laboratory, Bissau, Guinea-Bissau">
<meta name="citation_author" content="Zacarias J da Silva">
<meta name="citation_author_institution" content="National Public Health Laboratory, Bissau, Guinea-Bissau">
<meta name="citation_author" content="Eva M Fenyö">
<meta name="citation_author_institution" content="Department of Laboratory Medicine Lund, Section of Virology, Lund University, Lund, Sweden">
<meta name="citation_author" content="Hans Norrgren">
<meta name="citation_author_institution" content="Department of Clinical Sciences, Division of Infection Medicine, Lund University, Lund, Sweden">
<meta name="citation_author" content="Patrik Medstrand">
<meta name="citation_author_institution" content="Department of Experimental Medical Science, Section of Molecular Virology, Lund University, Lund, Sweden">


<meta name="citation_issn" content="1742-4690">
<meta name="citation_title" content="Frequent CXCR4 tropism of HIV-1 subtype A and CRF02_AG during late-stage disease - indication of an evolving epidemic in West Africa">
<meta name="citation_volume" content="7">
<meta name="citation_issue" content="1">
<meta name="citation_date" content="2010-03-22">
<meta name="citation_firstpage" content="23">
<meta name="citation_doi" content="10.1186/1742-4690-7-23">
<meta name="citation_pdf_url" content="http://www.retrovirology.com/content/pdf/1742-4690-7-23.pdf">
<link rel="alternate" type="application/pdf" title="PDF" href="http://www.retrovirology.com/content/pdf/1742-4690-7-23.pdf">
<meta name="title" content="Frequent CXCR4 tropism of HIV-1 subtype A and CRF02_AG during late-stage disease - indication of an evolving epidemic in West Africa">
<meta name="description" content="HIV-1 is one of the fastest evolving pathogens, and is distinguished by geographic and genetic variants that have been classified into different subtypes and circulating recombinant forms (CRFs). Early in infection the primary coreceptor is CCR5, but during disease course CXCR4-using HIV-1 populations may emerge. This has been correlated with accelerated disease progression in HIV-1 subtype B. Basic knowledge of HIV-1 coreceptor tropism is important due to the recent introduction of coreceptor antagonists in antiretroviral therapy, and subtype-specific differences regarding how frequently HIV-1 CXCR4-using populations appear in late-stage disease need to be further investigated. To study how frequently CXCR4-using populations appear in late-stage disease among HIV-1 subtype A and CRF02_AG, we evaluated the accuracy of a recombinant virus phenotypic assay for these subtypes, and used it to determine the HIV-1 coreceptor tropism of plasma samples collected during late-stage disease in Guinea-Bissau. We also performed a genotypic analysis and investigated subtype-specific differences in the appearance of CXCR4 tropism late in disease.">

<meta name="dc.title" content="Frequent CXCR4 tropism of HIV-1 subtype A and CRF02_AG during late-stage disease - indication of an evolving epidemic in West Africa">
<meta name="dc.creator" content="Esbjörnsson, Joakim">
<meta name="dc.creator" content="Månsson, Fredrik">
<meta name="dc.creator" content="Martínez-Arias, Wilma">
<meta name="dc.creator" content="Vincic, Elzbieta">
<meta name="dc.creator" content="Biague, Antonio J">
<meta name="dc.creator" content="da Silva, Zacarias J">
<meta name="dc.creator" content="Fenyö, Eva M">
<meta name="dc.creator" content="Norrgren, Hans">
<meta name="dc.creator" content="Medstrand, Patrik">
<meta name="dc.description" content="HIV-1 is one of the fastest evolving pathogens, and is distinguished by geographic and genetic variants that have been classified into different subtypes and circulating recombinant forms (CRFs). Early in infection the primary coreceptor is CCR5, but during disease course CXCR4-using HIV-1 populations may emerge. This has been correlated with accelerated disease progression in HIV-1 subtype B. Basic knowledge of HIV-1 coreceptor tropism is important due to the recent introduction of coreceptor antagonists in antiretroviral therapy, and subtype-specific differences regarding how frequently HIV-1 CXCR4-using populations appear in late-stage disease need to be further investigated. To study how frequently CXCR4-using populations appear in late-stage disease among HIV-1 subtype A and CRF02_AG, we evaluated the accuracy of a recombinant virus phenotypic assay for these subtypes, and used it to determine the HIV-1 coreceptor tropism of plasma samples collected during late-stage disease in Guinea-Bissau. We also performed a genotypic analysis and investigated subtype-specific differences in the appearance of CXCR4 tropism late in disease.">
<meta name="dc.source" content="Retrovirology 2010 7:23">
<meta name="dc.format" content="text/html">
<meta name="dc.publisher" content="BioMed Central Ltd">
<meta name="dc.date" content="2010-03-22">
<meta name="dc.type" content="Research">
<meta name="dc.identifier" content="10.1186/1742-4690-7-23">
<meta name="dc.language" content="en">
<meta name="dc.copyright" content="2010 Esbjörnsson et al; licensee BioMed Central Ltd.">
<meta name="dc.rights" content="http://creativecommons.org/licenses/by/2.0/">
<meta name="dc.rightsAgent" content="reprints@biomedcentral.com">

<meta name="prism.issn" content="1742-4690">
<meta name="prism.publicationName" content="Retrovirology">
<meta name="prism.publicationDate" content="2010-03-22">
<meta name="prism.volume" content="7">
<meta name="prism.number" content="1">
<meta name="prism.section" content="Research">
<meta name="prism.startingPage" content="23">
<meta name="prism.copyright" content="2010 Esbjörnsson et al; licensee BioMed Central Ltd.">
<meta name="prism.rightsAgent" content="reprints@biomedcentral.com">


        <meta name="citation_abstract_html_url" content="http://www.retrovirology.com/content/7/1/23/abstract">
            <meta name="citation_fulltext_html_url" content="http://www.retrovirology.com/content/7/1/23">
            <link rel="alternate" type="text/xml" title="XML version" href="http://www.retrovirology.com/content/download/xml/1742-4690-7-23.xml">
            <link rel="image_src" href="http://www.retrovirology.com/content/figures/1742-4690-7-23-toc.gif">
    
     <link rel="section" href="http://retrovirology.com/content/download/supplementary/1742-4690-7-23-s1.doc" title="Additional file 1"> <link rel="section" href="http://retrovirology.com/content/download/supplementary/1742-4690-7-23-s2.doc" title="Additional file 2"> <link rel="section" href="http://retrovirology.com/content/download/supplementary/1742-4690-7-23-s3.doc" title="Additional file 3"> <link rel="section" href="http://retrovirology.com/content/download/supplementary/1742-4690-7-23-s4.doc" title="Additional file 4"> <link rel="section" href="http://retrovirology.com/content/download/supplementary/1742-4690-7-23-s5.doc" title="Additional file 5"> <link rel="section" href="http://retrovirology.com/content/download/supplementary/1742-4690-7-23-s6.doc" title="Additional file 6"> <link rel="section" href="http://retrovirology.com/content/download/supplementary/1742-4690-7-23-s7.doc" title="Additional file 7">
    
    
                    
    
<script type="text/javascript">

// must run before css
document.documentElement.id = "js";
</script>


 
    
    
                                                                                                                                                                                                                                
                                                        
    
            

            
        <script>window.bmcIsMobile = "classic";</script>
        
        
    




	                
    
                
    



    
			                    
    			


    

    


                
    


<!--[if IE]>
                    <link rel="stylesheet" type="text/css" href="/css/hacks/ie-0.css"/>
    <![endif]-->

<!--[if IE 7]>
                    <link rel="stylesheet" type="text/css" href="/css/hacks/ie7-0.css"/>
    <![endif]-->
			
<script type="text/javascript">

    //configuration
    OAS_url = 'http://oas.biomedcentral.com/RealMedia/ads/';

	// control of advert's position on page [AMC]
    
    	
	
    OAS_sitepage = "retrovirology.com/article/10.1186/1742/4690/7/23";
    	
		OAS_listpos = 'Top,x96,Bottom';
	
	
			
			OAS_query = refererTerms("http://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&ved=0CDAQFjAA&url=http%3A%2F%2Fwww.retrovirology.com%2Fcontent%2F7%2F1%2F23&ei=KYqkUZmOKo7d7Qbxz4DoDQ&usg=AFQjCNF3iGmPXR1K1SYThQcX7WHsnpsSsw&sig2=AqQKcgnLirvCjVSrBmMLdg&bvm=bv.47008514,d.ZGU");
		
	

    OAS_target = '_top';
    //end of configuration
    OAS_version = 10;
    OAS_rn = '001234567890'; OAS_rns = '1234567890';
    OAS_rn = new String (Math.random()); OAS_rns = OAS_rn.substring (2, 11);
    function OAS_NORMAL(pos) {
		document.write('<a href="' + OAS_url + 'click_nx.ads/' + OAS_sitepage + '/1' + OAS_rns + '@' + OAS_listpos + '!' + pos + '?' + OAS_query + '" target="' + OAS_target + '">');
		document.write('<img src="' + OAS_url + 'adstream_nx.ads/' + OAS_sitepage + '/1' + OAS_rns + '@' + OAS_listpos + '!' + pos + '?' + OAS_query + '" border="0"/></a>');
    }
    
	function refererTerms(referer) {

	if(referer.match("q=") != null) {
		referer = referer.split("q=");
		referer = referer[1].split("&");
		referer = referer[0];
	} else {
		referer = referer;
	}

		if(referer.match("%20") != null && referer.match("%20").length >= 1) {
			referer =  referer.replace(/%20/gi,"+");
			return referer;
		} else { 
			return referer; 
		}
	}
 </script>

 
 
<link rel="stylesheet" type="text/css" href="23.css" media="all">
</head>
<body data-twttr-rendered="true" class="theme"><div id="dic_bubble" class="selection_bubble fontSize13 noSelect" style="z-index: 9999; border: 1px solid rgb(74, 174, 222); visibility: hidden;"></div><div id="oas-campaign" style="display: none;">retrovirology.com/article/10.1186/1742/4690/7/23</div>
 <div id="oas-positions" style="display: none;">Bottom,Top</div> 


 <script type="text/javascript">

	// Set the version of JavaScript to 10 for a Browser 'Mozilla/3' on webTV 
    OAS_version = 11;
    if ((navigator.userAgent.indexOf('Mozilla/3') != -1) || (navigator.userAgent.indexOf('Mozilla/4.0 WebTV') != -1)) {
     OAS_version = 10; }
    if (OAS_version >= 11) {
    document.write('<scr' + 'ipt type="text/javascript" src="' + OAS_url + 'adstream_mjx.ads/' + OAS_sitepage + '/1' + OAS_rns + '@' + OAS_listpos + '?' + OAS_query + '"><\/script>'); }
</script><script type="text/javascript" src="1562689705@Top,x96,Bottom"></script>
	
<script type="text/javascript">

    document.write('');
	
    function OAS_AD(pos) {
        if (OAS_version >= 11)
         OAS_RICH(pos);
        else
         OAS_NORMAL(pos);
    }
 </script>
	




<script type="text/javascript">

    jsonLoaders = function(){
        var _loaders = [];
        return {
            register: function(config){
                config.prefix = config.prefix || '/webapi/1.0/blogs/';
                config.feedLength = config.feedLength || 3;

                _loaders.push(config);
            },
            list: function(){return _loaders}
        };
    }();

    function totext() {
	    document.write('<a href="javascript:returnToText();">Return to text</a>');
    }

    function retrieve_doi(){
        var metas = document.getElementsByTagName('meta');
        var i;
        for (i = 0; i < metas.length; i++)
            if (metas[i].getAttribute('name') == "citation_doi")
                break;
        var citation_doi = metas[i].getAttribute('content');
        return citation_doi;
    }

</script>




        <link title="Latest articles" rel="alternate" href="http://www.retrovirology.com/latest/rss" type="application/rss+xml" exposetoheader="true">
    <link title="Most viewed" rel="alternate" href="http://www.retrovirology.com/mostviewed/rss/" type="application/rss+xml" exposetoheader="true">
    <link title="Latest comments" rel="alternate" href="http://www.retrovirology.com/latestcomments/rss" type="application/rss+xml" exposetoheader="true">
    


	<script type="text/javascript">

	var _gaq = _gaq || [];
	_gaq.push(['_setAccount', 'UA-9663679-5']);
	_gaq.push(['_gat._anonymizeIp']);
	_gaq.push(['_setLocalRemoteServerMode']);

		_gaq.push(['_trackPageview']); 

	(function() {
		var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
		ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
		var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
	})();
	</script>

	
        








 
    
    
                                                                                                                                                                                                                                
                                                        
    
            

            
        <script>window.bmcIsMobile = "classic";</script>
        
        
    
<script type="text/javascript">


function createCookie(name,value,days,path) {
	if (days) {
		var date = new Date();
		date.setTime(date.getTime()+(days*24*60*60*1000));
		var expires = "; expires="+date.toGMTString();
	}
	else{ var expires = "";}
	if (path!=null)
	{
	    document.cookie = name+"="+value+expires+"; path=/";
	}
	else
	{
	    document.cookie = name+"="+value+expires+";";
	}
}
function createCookie1(name,value,days) {
	if (days) {
		var date = new Date();
		date.setTime(date.getTime()+(days*24*60*60*1000));
		var expires = "; expires="+date.toGMTString();
	}
	else{ var expires = "";}
		    document.cookie = name+"="+value+expires;
	}

function eraseCookie(name) {
    // path describes the location of the cookie
    var path= "/";
	createCookie("database_statistics","",-1,path);
	// null value as a parameter tells that cookie existing in current application context like cookie in  http://biomedcentral.com.local/about/tandc
	createCookie("database_statistics_request_id","",-1,null);
	hidediv();
}
function hidediv() 
{ 
	if (document.getElementById) 
	{
	    document.getElementById('dbstats_main').style.visibility = 'hidden';
	}
}

function getCookieExists(c_name)
{
    var i,x,y,ARRcookies=document.cookie.split(";");
    for (i=0;i<ARRcookies.length;i++)
    {
          x=ARRcookies[i].substr(0,ARRcookies[i].indexOf("="));
          y=ARRcookies[i].substr(ARRcookies[i].indexOf("=")+1);
          x=x.replace(/^\s+|\s+$/g,"");
          if (x==c_name)
            {
                return unescape(y);
            }
     }
}

var request_Id;
var search_cookie = "database_statistics_request_id" + "="
var more_dbstats_link;
//readingDBStatscookie() will be called from JQuery function jQuery(document).ready(function() in behaviour.js which is called on "onload()" event
// in order to read the data from the lates cookie

function readingDBStatsCookie()
{
 if ( getCookieExists("database_statistics")!=null)
   {	
	if ( getCookieExists("database_statistics_request_id") != null)
   {
        if ( getCookieExists("database_statistics_request_id") != null)
        {
            if (document.cookie.length > 0)
            {
                // Search for a cookie.
                offset = document.cookie.indexOf(search_cookie)
                if (offset != -1)
                {
                        offset += search_cookie.length
                        // set index of beginning of value
                        end = document.cookie.indexOf(";",offset)
                        if (end == -1)
                        {
                            end = document.cookie.length
                        }
                       request_Id=document.cookie.substring(offset, end);
                       more_dbstats_link = "http://www.retrovirology.com:80/newjournal/dbstats?content_type=html&request_id="+request_Id;
			  
			  createCookie("database_statistics_request_id","",-1);
				
                       var link = document.createElement('a');
                       link.setAttribute('href', more_dbstats_link);
                       link.innerHTML="<b>More DBStats</b>";
                       link.title="More dbstats";
                       //var text = "<a href=\""+more_dbstats_link+"\" >More DB Stats</a>";
                    document.getElementById('more_dbstats_link').appendChild(link);
                }
            }
        }
	}
 }
}
  if (getCookieExists("database_statistics")!=null)
     {
           document.write( "<div id =\"dbstats_main\" style=\"background-color:white\">"   );
             document.write("<div id =\"dbstats\" style=\"background-color:lightgray;margin-left:300px;margin-right:300px;margin-top:-10px\">");
             document.write("
");
                    document.write("<b>DB Stats</b>");
                    document.write("- consistent gets:");
                    document.write("<span id = \"consistent_gets\">NA</span> ");
                    document.write(", physical reads:");
                    document.write("<span id = \"physical_reads\">NA</span>");
                    document.write(", bytes sent via SQL*Net to client: ");
                    document.write("<span id = \"SQL_NET\">NA</span>");
                    document.write("&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp");
                    document.write("<span id=\"more_dbstats_link\">");
                    document.write("</span>	");
                    document.write("<span id=\"close_dbstats_link\">");
                    document.write("[<a href='javascript:eraseCookie()' >x</a>]");
                    document.write("</span>	");
             document.write("</div>");
           document.write("</div> ");
     }
 </script>
<div id="branding" role="banner">
            			
		<dl class="google-ad wide  advert-text">
	<dt style="display: block;" class="hide">
					<a class="banner-ad" href="http://www.biomedcentral.com/advertisers/digital_advertising"></a>
				
			</dt>
		<dd>

		<!-- OAS AD 'Top' begin -->
		<script type="text/javascript">

		OAS_AD('Top');

		</script>
		<!-- OAS AD 'Top' end -->
		</dd>

		</dl>

		<noscript>

			<dl class="google-ad wide noscript">  
    <dt class="hide"></dt>
    
    <dd>
        <a href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/retrovirology.com/article/10.1186/1742/4690/7/23/17151467985@Top?">
            <img src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/retrovirology.com/article/10.1186/1742/4690/7/23/11554922471@Top?" alt="advert"/>
        </a>
    </dd>
			</dl>
		</noscript>
			        <div class="sup-panel-outer">
        <div class="sup-panel-inner">
            <ul id="login" class="login">
            <li id="loginLink"><a href="http://www.retrovirology.com/logon">Log on</a></li>
    </ul>            <ul class="nav-sup">
    
        <li id="BMC" class="current">
            <a href="http://www.biomedcentral.com/"><span class="img-title">&nbsp;</span>
            <span class="text-title">BioMed Central</span></a>
        </li>
		
        		<li><a href="http://www.biomedcentral.com/journals"><span>Journals</span></a></li>
		
								<li>
				<a href="http://www.biomedcentral.com/gateways"><span>Gateways</span></a>
			</li>
					    	
</ul>
        </div>
    </div>
    <div class="branding-inner">
					<div class="logo">
			<a href="http://www.retrovirology.com/"><img alt="Retrovirology" title="Retrovirology Logo" src="logo.gif"></a>			
			</div>		
				
                    

        <div id="impact-factor" class="official">
    <img src="official.gif" alt="official impact factor" title="6.47">
    <span id="impact-factor-value">6.47</span>
</div>
    
                            

<div class="module search gray small  " role="search">
    <div class="module-inner padded">
        <form method="get" class="searchForm" action="http://www.retrovirology.com/quickSearchRedirectURL" onsubmit="window.location.href=this['url'].value + '?terms=' + encodeURIComponent(this['terms'].value); return false;">
            <fieldset class="search">
			                <span>

                    Search <select name="url">
                                <option value="http://www.retrovirology.com/search/results" selected="">Retrovirology</option>
                                <option value="http://www.biomedcentral.com/search/results">BioMed Central</option>
                           </select> for
                </span>
                            <input style="width: 143px;" id="searchTerms" name="terms" alt="Search terms" class="text " type="text">
                 <button class="w37" value="Go" type="submit">Go</button>
            </fieldset>
        </form>
    </div>
	<a class="advanced-search" href="http://www.retrovirology.com/search">Advanced search</a>
</div> 

                        <div id="mobile-nav-list">

	<ul class="primary-nav" role="navigation">
		<li><a href="http://www.retrovirology.com/"><span id="home-tab">Home</span></a></li>
														<li class="current"><a href="http://www.retrovirology.com/content"><span id="articles-tab">Articles</span></a></li>
										
	

	
            <li><a href="http://www.retrovirology.com/authors/instructions"><span id="authors-tab">Authors</span></a></li>
        <li><a href="http://www.retrovirology.com/about/reviewers"><span id="reviewers-tab">Reviewers</span></a></li>
    		<li><a href="http://www.retrovirology.com/about"><span id="about-tab">About this journal</span></a></li>
	<li><a href="http://www.retrovirology.com/my"><span id="my-tab">My Retrovirology</span></a></li>
		
    		</ul>
</div>
            </div>
</div>
<hr class="hide">
<div class="container">
	<div id="content" class="overflow-visible" role="main">
			    
	

<!--div class="branding-mobile">
		<h1 class="logo">
	    <a href="/">
		    <img alt="Retrovirology" title="Retrovirology Logo" src="/sites/10056/images/logo.gif" />
		</a>
		</h1>
	</div><br/-->



			
	



<div style="position: absolute;" class="outline-wrapper block mobile-hidden" id="left-article-box">
<ul id="box-outline" class="box">
<ul id="outline" class="outline">
   <li>
      <a href="#">Top</a>
   </li>
   <li>
      <a href="#abs">Abstract</a>
   </li>
   <li>
      <a href="#sec1">Background</a>
   </li>
   <li>
      <a href="#sec2">Methods</a>
   </li>
   <li>
      <a href="#sec3">Results</a>
   </li>
   <li>
      <a href="#sec4">Discussion</a>
   </li>
   <li>
      <a href="#sec5">Conclusion</a>
   </li>
   <li>
      <a href="#sec6">Competing interests</a>
   </li>
   <li>
      <a href="#sec7">Authors' contributions</a>
   </li>
   <li>
      <a href="#ack">Acknowledgements</a>
   </li>
   <li class="lowest">
      <a href="#refs">References</a>
   </li>
</ul>
</ul>
<dl id="box-outline1">
</dl>
</div>

<div class="mobile-hidden" id="right-panel">
 	
<div id="mobile-sidebar">
	<div id="mobile-sidebar-tab">
		<span>
			<a href="#" id="mobile-sidebar-tab-link"><img src="sidebar-hidden.png"><img style="display: none;" src="sidebar-shown.png"></a>
		</span>
	</div>	<div id="mobile-sidebar-list">

<div id="article-navigation-bar">
    <div class="issue-information block">
        <h5><a href="http://www.retrovirology.com/"><strong>Retrovirology</strong></a></h5>
        <ul class="square normal">
            <li><a href="http://www.retrovirology.com/content/7">Volume 7</a></li>
				
                    </ul>
    </div>

    <div class="article-information block" id="article-info">
        


<div id="viewing-options-links">
	<h5>Viewing options</h5>
	<ul class="square normal">
        
            <li>
                             <a href="http://www.retrovirology.com/content/7/1/23/abstract">Abstract</a>
                        </li>
        				<li>
							<strong>Full text</strong>
					</li>
				
		
		
		<li class="pdfFileSize">			<a onclick="_gaq.push(['_trackEvent', 'PDF download', 'Article Sidebar', '/content/7/1/23', 1, true]);" href="http://www.retrovirology.com/content/pdf/1742-4690-7-23.pdf">PDF</a>
			<span>(544KB)</span>		</li>
		
		<li>
                    </li>
		
				<li>
						<a href="http://www.retrovirology.com/content/7/1/23/additional">Additional files</a>
					</li>
			</ul>
</div>
		
<div id="associated-material-links">
    <h5>Associated material</h5>
    <ul class="square normal">
                        <li>
            <a href="http://www.retrovirology.com/pubmed/20307309">PubMed record</a>
        </li>
                            <li><a href="http://www.retrovirology.com/content/7/1/23/about">About this article</a></li>
                <li>
                        <a href="http://www.retrovirology.com/content/7/1/23/comments">Readers' comments </a>
                    </li>
        <li>
                    </li>
    </ul>
</div>
		




<div id="related-literature-links">
    <h5>Related literature</h5>
    <ul class="normal">
                    <li> <a href="http://www.retrovirology.com/content/7/1/23/about#citations"><h6>Cited by</h6></a></li>
                <li id="google-blog-search"><a href="http://www.google.com/search?tbm=blg&amp;hl=en&amp;source=hp&amp;biw=1280&amp;bih=899&amp;q=Frequent+CXCR4+tropism+of+HIV-1+subtype+A+and+CRF02_AG+during+late-stage+disease+-+indication+of+an+evolving+epidemic+in+West+Africa&amp;btnG=Search">Google blog search</a></li>
        <h6>Other articles by authors</h6>
        <li>
            <a class="collapser"><i>&nbsp;</i>on Google Scholar</a>
<ul id="authg" class="hidebeforeload">                         <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22J+Esbj%C3%B6rnsson%22">Esbjörnsson J</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22F+M%C3%A5nsson%22">Månsson F</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22W+Mart%C3%ADnez-Arias%22">Martínez-Arias W</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22E+Vincic%22">Vincic E</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22AJ+Biague%22">Biague AJ</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22ZJ+da%20Silva%22">da Silva ZJ</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22EM+Feny%C3%B6%22">Fenyö EM</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22H+Norrgren%22">Norrgren H</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22P+Medstrand%22">Medstrand P</a>
            </li>
            </ul>          </li>
                <li>
            		<a class="collapser"><i>&nbsp;</i>on PubMed</a>
	<ul id="authpm" class="hidebeforeload"> 									<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Esbj%C3%B6rnsson_J%20[Author]">Esbjörnsson J</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=M%C3%A5nsson_F%20[Author]">Månsson F</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Mart%C3%ADnez-Arias_W%20[Author]">Martínez-Arias W</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Vincic_E%20[Author]">Vincic E</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Biague_AJ%20[Author]">Biague AJ</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=da_Silva_ZJ%20[Author]">da Silva ZJ</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Feny%C3%B6_EM%20[Author]">Fenyö EM</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Norrgren_H%20[Author]">Norrgren H</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Medstrand_P%20[Author]">Medstrand P</a>
				</li>
						</ul> 	        </li>
    </ul>     <h6>Related articles/pages</h6>
    <ul class="square normal">
        <li><a href="http://www.google.co.uk/search?hl=en&amp;q=related:http://www.retrovirology.com/content/7/1/23">on Google</a></li>
        <li><a href="http://scholar.google.com/scholar?q=related:http://www.retrovirology.com/content/7/1/23">on Google Scholar</a></li>
                    <li><a href="http://www.retrovirology.com/pubmed/related/20307309">on PubMed</a></li>
            </ul>

</div>		
        
    
    
        

    	





<h5>Tools</h5>
	<ul class="square normal">
									<li>
						<a href="http://www.retrovirology.com/content/7/1/23/citation">Download references</a>
					</li>
				<li><a href="http://www.retrovirology.com/content/download/xml/1742-4690-7-23.xml">Download XML</a></li>
				<li>
						<a href="http://www.retrovirology.com/content/7/1/23/email">Email to a friend</a>
					</li>
		<li><a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186/1742-4690-7-23&amp;volume=7&amp;issue=1&amp;title=Frequent+CXCR4+tropism+of+HIV-1+subtype+A+and+CRF02_AG+during+late-stage+disease+-+indication+of+an+evolving+epidemic+in+West+Africa&amp;author_name=Joakim%20Esbj%C3%B6rnsson&amp;start_page=1&amp;end_page=13">Order reprints</a></li>
		<li>
						<a href="http://www.retrovirology.com/content/7/1/23/postcomment">Post a comment</a>
					</li>
	</ul>
		



<h5>Share this article</h5>
<ul id="social-networking-links">
	<li>
	<div fb-xfbml-state="rendered" class="fb-like fb_edge_widget_with_comment fb_iframe_widget" data-href="http://www.retrovirology.com/content/7/1/23" data-send="false" data-layout="button_count" data-width="100" data-show-faces="false" data-action="recommend"><span style="height: 20px; width: 119px;"><iframe src="23_1.html" class="fb_ltr" title="Like this content on Facebook." style="border: medium none; overflow: hidden; height: 20px; width: 119px;" name="f34eb8f2c18b042" id="f367af97759a0ba" scrolling="no"></iframe></span></div>
	</li>
	
	<li>
		<iframe data-twttr-rendered="true" title="Twitter Tweet Button" style="width: 107px; height: 20px;" class="twitter-share-button twitter-count-horizontal" src="23_2.html" allowtransparency="true" frameborder="0" scrolling="no"></iframe>
	</li>
	<li>
			<div id="___plusone_0" style="text-indent: 0px; margin: 0px; padding: 0px; background: none repeat ; 0% transparent; border-style: none; float: none; line-height: normal; font-size: 1px; vertical-align: baseline; display: inline-block; width: 90px; height: 20px;"><iframe title="+1" data-gapiattached="true" allowtransparency="true" src="23_3.html" name="I0_1369737788463" id="I0_1369737788463" vspace="0" tabindex="0" style="position: static; top: 0px; width: 90px; margin: 0px; border-style: none; left: 0px; visibility: visible; height: 20px;" marginwidth="0" marginheight="0" hspace="0" frameborder="0" scrolling="no" width="100%"></iframe></div>
	</li>
	<li>
		<a class="sharethis_title btn" href="#"><span class="icon">&nbsp;</span>More options...</a>
	<ul style="display: none;" class="iconlist hidebeforeload" id="sharethis">
			<li>
				<a onclick="largepopup(this.href,'citeulike_popup_post',800,600);return false" href="http://www.citeulike.org/posturl?url=http://www.retrovirology.com/content/7/1/23"><span class="share-icons citeulike"></span>Citeulike</a>
			</li>
				<li>
				<a onclick="largepopup(this.href,'linkedin',800,600);return false" href="http://www.linkedin.com/shareArticle?mini=true&amp;url=%23retrovirology%0D%0AFrequent%20CXCR4%20tropism%20of%20HIV-1%20subtype%20A%20and%20CRF02_AG%20during%20late-stage%20disease%20-%20indication%20of%20an%20evolving%20epidemic%20in%20West%20Africa%20http%3A%2F%2Fwww.retrovirology.com%2Fcontent%2F7%2F1%2F23&amp;title=Frequent%20CXCR4%20tropism%20of%20HIV-1%20subtype%20A%20and%20CRF02_AG%20during%20late-stage%20disease%20-%20indication%20of%20an%20evolving%20epidemic%20in%20West%20Africa"><span class="share-icons linkedin"></span>LinkedIn</a>  
	</li>
			<li>
				<a onclick="largepopup(this.href,'Del.icio.us',800,600);return false" href="http://del.icio.us/post?url=http://www.retrovirology.com/content/7/1/23&amp;title=Frequent%20CXCR4%20tropism%20of%20HIV-1%20subtype%20A%20and%20CRF02_AG%20during%20late-stage%20disease%20-%20indication%20of%20an%20evolving%20epidemic%20in%20West%20Africa"><span class="share-icons delicious"></span>Del.icio.us</a>
	</li>
			<li>
				<a href="http://www.retrovirology.com/content/7/1/23/email"><span class="share-icons email"></span>Email</a>
	</li>
			<li>
				<a onclick="largepopup(this.href,'facebook',800,600);return false" href="http://www.facebook.com/sharer.php?u=http://www.retrovirology.com/content/7/1/23"><span class="share-icons facebook"></span>Facebook</a>
	</li>
	<li class="gp">
			<div class="googlehider mobile-hide"> 
				<div id="___plusone_1" style="text-indent: 0px; margin: 0px; padding: 0px; background: none repeat ; 0% transparent; border-style: none; float: none; line-height: normal; font-size: 1px; vertical-align: baseline; display: inline-block; width: 38px; height: 24px;"><iframe title="+1" data-gapiattached="true" allowtransparency="true" src="23_4.html" name="I1_1369737788602" id="I1_1369737788602" vspace="0" tabindex="0" style="position: static; top: 0px; width: 38px; margin: 0px; border-style: none; left: 0px; visibility: visible; height: 24px;" marginwidth="0" marginheight="0" hspace="0" frameborder="0" scrolling="no" width="100%"></iframe></div>
			</div>
				<span class="share-icons googleplus mobile-hide"></span>
				<div class="googlehider2 mobile-hide"> 
				<div id="___plusone_2" style="text-indent: 0px; margin: 0px; padding: 0px; background: none repeat ; 0% transparent; border-style: none; float: none; line-height: normal; font-size: 1px; vertical-align: baseline; display: inline-block; width: 38px; height: 24px;"><iframe title="+1" data-gapiattached="true" allowtransparency="true" src="23_5.html" name="I2_1369737788708" id="I2_1369737788708" vspace="0" tabindex="0" style="position: static; top: 0px; width: 38px; margin: 0px; border-style: none; left: 0px; visibility: visible; height: 24px;" marginwidth="0" marginheight="0" hspace="0" frameborder="0" scrolling="no" width="100%"></iframe></div>
			</div>
				<span class="share-icons googleplus"></span>
				<span class="gp-link"><a href="#">Google+</a></span>
	</li>
			<li>
				<a onclick="largepopup(this.href,'mendeley',800,600);return false" href="http://www.mendeley.com/import/?url=http://www.retrovirology.com/content/7/1/23"><span class="share-icons mendeley"></span>Mendeley</a>
			</li>
			<li>
				<a onclick="largepopup(this.href,'twitter',800,600);return false" href="http://twitter.com/?status=%23retrovirology%0D%0AFrequent%20CXCR4%20tropism%20of%20HIV-1%20subtype%20A%20and%20CRF02_AG%20during%20late-stage%20disease%20-%20indication%20of%20an%20evolving%20epidemic%20in%20West%20Africa%20http%3A%2F%2Fwww.retrovirology.com%2Fcontent%2F7%2F1%2F23"><span class="share-icons twitter"></span>Twitter</a>  
			</li>
			
		</ul>
</li></ul>

    </div>

    	
		
    				  		














		                                    		                                    		
	
	
</div>
		<div class="mobile-sidebar-gradient top"></div>
		<div class="mobile-sidebar-gradient bottom"></div>
	</div>

    

</div>








 </div>
<div class="rounded custom white article full-text">
<div class="wrap-inner content">
    <div class="padded-inner">            


    
 
    
    
                                                                                                                                                                                                                                
                                                        
    
            

            
        <script>window.bmcIsMobile = "classic";</script>
        
        
    
<div id="topmatter">
	


	   








				<a href="http://www.retrovirology.com/about/access"><img alt="Open Access" src="openaccess-large.png" class="access mr15"></a>
					<span class="articletype">Research</span>
<h1>Frequent CXCR4 tropism of HIV-1 subtype A and CRF02_AG during late-stage disease - indication of an evolving epidemic in West Africa</h1>

        
			<div class="multipleins">
		
	<p class="authors">
        																																																																																																																																																																																																																																																																																																		        <strong>Joakim  Esbjörnsson</strong><sup><a href="http://www.retrovirology.com/content/7/1/23/#ins1">1</a></sup><sup>*</sup>, <strong>Fredrik  Månsson</strong><sup><a href="http://www.retrovirology.com/content/7/1/23/#ins2">2</a></sup>, <strong>Wilma  Martínez-Arias</strong><sup><a href="http://www.retrovirology.com/content/7/1/23/#ins1">1</a></sup>, <strong>Elzbieta  Vincic</strong><sup><a href="http://www.retrovirology.com/content/7/1/23/#ins3">3</a></sup>, <strong>Antonio J Biague</strong><sup><a href="http://www.retrovirology.com/content/7/1/23/#ins5">5</a></sup>, <strong>Zacarias J da Silva</strong><sup><a href="http://www.retrovirology.com/content/7/1/23/#ins5">5</a></sup>, <strong>Eva M Fenyö</strong><sup><a href="http://www.retrovirology.com/content/7/1/23/#ins3">3</a></sup>, <strong>Hans  Norrgren</strong><sup><a href="http://www.retrovirology.com/content/7/1/23/#ins4">4</a></sup> and <strong>Patrik  Medstrand</strong><sup><a href="http://www.retrovirology.com/content/7/1/23/#ins1">1</a></sup>	</p>

			<div id="affiliations">
		<div class="module gray  inner">
<div class="module-inner padded-inner">
		<ul>
			<li>                        
                                                                                                                            		<p class="authors">
							<span>*</span>
									 Corresponding author:										            Joakim  Esbjörnsson <a href="mailto:joakim.esbjornsson@med.lu.se">joakim.esbjornsson@med.lu.se</a>    					</p>
      
</li>
			
                                                                                                                                                                                                                                                                            		</ul>

		
		
            <section> 
						<div class="collapsible-content">
							 <p class="options"><a class="affiliations-toggle" href="#"><i class="arrow active"></i>Author Affiliations</a></p>				<div id="ins_container" style="display: block;">
										    						    <p>
						        <sup id="ins1">1</sup>                                Department of Experimental Medical Science, Section of Molecular Virology, Lund University, Lund, Sweden
                            </p>
																    						    <p>
						        <sup id="ins2">2</sup>                                Department of Clinical Sciences, Malmö, Infectious Diseases Research Unit, Lund University, Sweden
                            </p>
																    						    <p>
						        <sup id="ins3">3</sup>                                Department of Laboratory Medicine Lund, Section of Virology, Lund University, Lund, Sweden
                            </p>
																    						    <p>
						        <sup id="ins4">4</sup>                                Department of Clinical Sciences, Division of Infection Medicine, Lund University, Lund, Sweden
                            </p>
																    						    <p>
						        <sup id="ins5">5</sup>                                National Public Health Laboratory, Bissau, Guinea-Bissau
                            </p>
																
				</div>
										<p id="authoremails">            For all author emails, please <a href="http://www.retrovirology.com/logon">log on</a>.
    </p>
						</div>
			</section>
				</div>
</div>
		</div>
	  </div>

            <section class="cit">
                        <div class="collapsible-content">
        <p><em>Retrovirology</em> 2010, <strong>7</strong>:23&nbsp;<span class="pseudotab">doi:10.1186/1742-4690-7-23</span></p>
    	
    
    
        		<br>
		<p>The electronic version of this article is the complete one and can be found online at: <a href="http://www.retrovirology.com/content/7/1/23">http://www.retrovirology.com/content/7/1/23</a></p>
		<br>
                    <table cellpadding="0" cellspacing="0">
        <tbody>
        <tr><td>Received:</td><td>12 January 2010</td></tr>                <tr><td>Accepted:</td><td>22 March 2010</td></tr>        <tr><td>Published:</td><td>22 March 2010</td></tr>        </tbody>
        </table>
    		<!--<br/>-->
		<div style="line-height: 140%;">
        <p>

            © 2010 Esbjörnsson et al; licensee BioMed Central Ltd. <br>
        </p>
					<p>

				This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/2.0">http://creativecommons.org/licenses/by/2.0</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.
			</p>
		    	</div>
	</div></section>

</div>
<div id="article-body">
    <section><a name="abs"></a><h3><span class="arrow"></span>Abstract</h3>
   <div class="collapsible-content">
      <h4>Background</h4>
      <p style="line-height: 160%;">HIV-1 is one of the fastest evolving pathogens, and is distinguished by geographic
         and genetic variants that have been classified into different subtypes and circulating
         recombinant forms (CRFs). Early in infection the primary coreceptor is CCR5, but during
         disease course CXCR4-using HIV-1 populations may emerge. This has been correlated
         with accelerated disease progression in HIV-1 subtype B. Basic knowledge of HIV-1
         coreceptor tropism is important due to the recent introduction of coreceptor antagonists
         in antiretroviral therapy, and subtype-specific differences regarding how frequently
         HIV-1 CXCR4-using populations appear in late-stage disease need to be further investigated.
         To study how frequently CXCR4-using populations appear in late-stage disease among
         HIV-1 subtype A and CRF02_AG, we evaluated the accuracy of a recombinant virus phenotypic
         assay for these subtypes, and used it to determine the HIV-1 coreceptor tropism of
         plasma samples collected during late-stage disease in Guinea-Bissau. We also performed
         a genotypic analysis and investigated subtype-specific differences in the appearance
         of CXCR4 tropism late in disease.
      </p>
      <h4>Results</h4>
      <p style="line-height: 160%;">We found that the recombinant virus phenotypic assay accurately predicted HIV-1 coreceptor
         tropism of subtype A and CRF02_AG. Over the study period (1997-2007), we found an
         increasing and generally high frequency of CXCR4 tropism (86%) in CRF02_AG. By sequence
         analysis of the V3 region of our samples we developed a novel genotypic rule for predicting
         CXCR4 tropism in CRF02_AG, based on the combined criteria of the total number of charged
         amino acids and net charge. This rule had higher sensitivity than previously described
         genotypic rules and may be useful for development of future genotypic tools for this
         CRF. Finally, we conducted a literature analysis, combining data of 498 individuals
         in late-stage disease, and found high amounts of CXCR4 tropism for all major HIV-1
         subtypes (60-77%), except for subtype C (15%).
      </p>
      <h4>Conclusions</h4>
      <p style="line-height: 160%;">The increase in CXCR4 tropism over time suggests an evolving epidemic of CRF02_AG.
         The results of the literature analysis demonstrate the need for further studies investigating
         subtype-specific emergence for CXCR4-tropism; this may be particularly important due
         to the introduction of CCR5-antagonists in HIV treatment regimens.
      </p>
   </div>
</section>
<section><a name="sec1"></a><h3><span class="arrow"></span>Background</h3>
   <div class="collapsible-content">
      <p style="line-height: 160%;">Human immunodeficiency virus type 1 (HIV-1) evolves at an extremely high rate, primarily
         due to a combination of high viral turn-over, an error prone viral reverse transcriptase
         and frequent recombination. This high level of molecular evolution has led to diversification
         of HIV-1 into genetically distinct subtypes (A-D, F-H, J-K), subsubtypes (A1-A3, F1-F2)
         and circulating recombinants forms (CRFs), usually defined by geographical location
         <a name="d31808e197"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>]. The most common subtypes are subtype A (12.3% of the global prevalence), B (10.2%),
         C (49.9%) and G (6.3%), and the CRF01_AE (4.7%) and CRF02_AG (4.8%) <a name="d31808e201"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>].
      </p>
      <p style="line-height: 160%;">HIV-1 enters target cells via interactions with CD4 and a coreceptor, usually one
         of the chemokine receptors CCR5 or CXCR4. Different HIV strains have been classified
         based on coreceptor tropism: CCR5-tropic strains are referred to as R5, CXCR4-tropic
         strains as X4, and dual tropic strains as R5X4 <a name="d31808e207"></a>[<a onclick="LoadInParent('#B2'); return false;" href="#B2">2</a>]. Coreceptor use has been studied extensively in HIV-1 subtype B and C, but needs
         further investigation for other subtypes <a name="d31808e211"></a><a name="d31808e213"></a><a name="d31808e215"></a><a name="d31808e217"></a><a name="d31808e219"></a><a name="d31808e221"></a><a name="d31808e223"></a><a name="d31808e225"></a><a name="d31808e227"></a><a name="d31808e229"></a><a name="d31808e231"></a>[<a onclick="LoadInParent('#B3'); return false;" href="#B3">3</a>-<a onclick="LoadInParent('#B13'); return false;" href="#B13">13</a>]. In subtype B, R5 populations are generally present over the entire course of infection
         whereas R5X4 or X4 populations emerge late in infection. This coreceptor switch has
         been associated with faster CD4+ T cell decline and the development of AIDS, although
         studies describing the opposite, or no difference in CD4+ T cell decline have also
         been observed <a name="d31808e235"></a><a name="d31808e237"></a>[<a onclick="LoadInParent('#B5'); return false;" href="#B5">5</a>,<a onclick="LoadInParent('#B6'); return false;" href="#B6">6</a>].
      </p>
      <p style="line-height: 160%;">Little is known about subtype-specific differences regarding how frequently CXCR4-using
         populations appear in late-stage disease. Most studies investigating HIV-1 subtype
         B coreceptor tropism have focused on either the relation between the detection of
         X4 viruses and disease progression rate, or molecular properties that differ between
         the R5 and X4 viruses <a name="d31808e243"></a><a name="d31808e245"></a><a name="d31808e247"></a><a name="d31808e249"></a>[<a onclick="LoadInParent('#B5'); return false;" href="#B5">5</a>,<a onclick="LoadInParent('#B6'); return false;" href="#B6">6</a>,<a onclick="LoadInParent('#B14'); return false;" href="#B14">14</a>,<a onclick="LoadInParent('#B15'); return false;" href="#B15">15</a>]. HIV-1 CXCR4-using populations are thought to appear in approximately 50% of the
         patients infected with subtype B <a name="d31808e253"></a><a name="d31808e255"></a><a name="d31808e257"></a>[<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a>-<a onclick="LoadInParent('#B18'); return false;" href="#B18">18</a>]. The fraction of subtype C-infected individuals that have CXCR4-using populations
         appear to be less frequent (0-30%) <a name="d31808e261"></a><a name="d31808e263"></a><a name="d31808e265"></a><a name="d31808e267"></a>[<a onclick="LoadInParent('#B8'); return false;" href="#B8">8</a>,<a onclick="LoadInParent('#B9'); return false;" href="#B9">9</a>,<a onclick="LoadInParent('#B11'); return false;" href="#B11">11</a>,<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>]. Moreover, a study of HIV-1 CRF01_AE in 22 AIDS patients showed that 16 subjects
         (73%) had X4 populations <a name="d31808e271"></a>[<a onclick="LoadInParent('#B12'); return false;" href="#B12">12</a>]. Comparing the A and D subtypes, Kaleebu et al. found no significant difference in
         patients with low CD4 counts (≤ 200) <a name="d31808e275"></a>[<a onclick="LoadInParent('#B3'); return false;" href="#B3">3</a>]. When a comparison was done at an earlier stage of HIV-1 infection (CD4 counts &gt;
         200), CXCR4 use was more frequent among patients with subtype D infection, probably
         due to an earlier coreceptor switch than in patients infected with subtype A <a name="d31808e280"></a>[<a onclick="LoadInParent('#B3'); return false;" href="#B3">3</a>]. Other cross-sectional studies do not allow estimation of the emergence of HIV-1
         with X4 phenotype since CD4 counts or clinical statuses were not considered together
         <a name="d31808e284"></a><a name="d31808e286"></a><a name="d31808e288"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>-<a onclick="LoadInParent('#B22'); return false;" href="#B22">22</a>].
      </p>
      <p style="line-height: 160%;">The viral envelope glycoprotein (gp) 120 is organized in five hypervariable regions
         (V1-V5), interspersed within five conserved regions (C1-C5). The major viral determinants
         of the interaction between gp120 and the coreceptors CCR5 or CXCR4 are located in
         the V3 region, even though other regions, such as the V1/V2 and the C4 regions have
         been shown to influence coreceptor use <a name="d31808e294"></a><a name="d31808e296"></a>[<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a>,<a onclick="LoadInParent('#B24'); return false;" href="#B24">24</a>]. To date, most studies have focused on HIV-1 subtype B and C, and there is no clear
         evidence that the V3 region has the same impact on coreceptor interaction among other
         subtypes.
      </p>
      <p style="line-height: 160%;">Basic knowledge of HIV-1 coreceptor evolution has become increasingly important due
         to the recent introduction of CCR5 antagonists as part of antiretroviral therapy against
         HIV-1 <a name="d31808e302"></a><a name="d31808e304"></a>[<a onclick="LoadInParent('#B25'); return false;" href="#B25">25</a>,<a onclick="LoadInParent('#B26'); return false;" href="#B26">26</a>]. Since these drugs have no effect on X4 populations, HIV-1 coreceptor tropism must
         be identified before the initiation of treatment <a name="d31808e308"></a><a name="d31808e310"></a>[<a onclick="LoadInParent('#B27'); return false;" href="#B27">27</a>,<a onclick="LoadInParent('#B28'); return false;" href="#B28">28</a>]. The gold standard for clinical samples is coreceptor determination by recombinant
         phenotypic entry assays <a name="d31808e314"></a><a name="d31808e316"></a>[<a onclick="LoadInParent('#B29'); return false;" href="#B29">29</a>,<a onclick="LoadInParent('#B30'); return false;" href="#B30">30</a>]. Reliable bioinformatic tools based on viral genotype may be a faster and more cost-effective
         way to predict coreceptor tropism. Present genotypic predictors are based on V3 sequences
         from subtype B or C, and have been shown to perform poorly on other subtypes, especially
         in detecting CXCR4-using variants <a name="d31808e320"></a><a name="d31808e322"></a>[<a onclick="LoadInParent('#B31'); return false;" href="#B31">31</a>,<a onclick="LoadInParent('#B32'); return false;" href="#B32">32</a>]. This indicates that molecular differences connected to coreceptor use may be subtype-specific,
         and that specific predictors likely have to be constructed for each major subtype
         and CRF.
      </p>
      <p style="line-height: 160%;">In view of this, we set out to determine the frequency of emergence of X4 phenotype
         in HIV-1 subtype A or CRF02_AG infected individuals in late-stage disease by evaluating
         the performance of a recombinant virus phenotypic assay for subtype A and CRF02_AG.
         Using this tool, we found an increasing and generally high frequency of CXCR4 tropism
         (86%) in CRF02_AG, and developed a novel genotypic rule for predicting CXCR4 tropism,
         based on combined criteria of the total number of charged amino acids and net charge
         of the V3 region. Finally, we compared our results to other HIV-1 subtypes by analyzing
         HIV-1 coreceptor phenotype in individuals in late-stage disease and found high amounts
         of CXCR4 tropism for all major HIV-1 subtypes (60-77%), except for subtype C (15%).
      </p>
   </div>
</section>
<section><a name="sec2"></a><h3><span class="arrow"></span>Methods</h3>
   <div class="collapsible-content">
      <h4>Sample sets</h4>
      <p style="line-height: 160%;">Two sample sets were used in the present study. The first sample set consisted of
         a control panel of 11 HIV-1 isolates with predetermined subtype and coreceptor tropism.
         Subtype was determined by sequencing of the <em>env </em>V3 region, and coreceptor tropism was determined using a phenotypic infection assay
         with either the U87.CD4-CCR5/U87.CD4-CXCR4 cell system or the MT-2 cell system <a name="d31808e341"></a>[<a onclick="LoadInParent('#B22'); return false;" href="#B22">22</a>]. Data were generously provided by Professor Jan Albert, Swedish Institute for Infectious
         Disease Control, Stockholm, Sweden. All isolates were amplified, sequenced, and used
         for evaluation of the recombinant virus phenotypic assay. Three isolates (22480, 22627,
         and 30405) were also used in the genotypic analysis of HIV-1 CRF02_AG. Details of
         the control panel can be found in Table <a name="d31808e344"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/table/T1','',800,470); return false;" href="http://www.retrovirology.com/content/7/1/23/table/T1">1</a>. The second sample set consisted of 33 plasma samples from 33 HIV-1 infected individuals
         and was selected from a cohort of police officers from Guinea-Bissau, West Africa,
         based on sample availability and disease status. The cohort has been described in
         detail elsewhere <a name="d31808e348"></a><a name="d31808e350"></a><a name="d31808e352"></a>[<a onclick="LoadInParent('#B33'); return false;" href="#B33">33</a>-<a onclick="LoadInParent('#B35'); return false;" href="#B35">35</a>]. Twenty-nine of the samples were successfully amplified and subjected to further
         analyses. All of the individuals were treatment naïve and classified to be in late-stage
         disease, as defined by CD4+ T cell count (≤ 200 cells/μl or ≤ 14%) or clinical AIDS
         (CDC: C or WHO: 4) <a name="d31808e356"></a><a name="d31808e358"></a>[<a onclick="LoadInParent('#B36'); return false;" href="#B36">36</a>,<a onclick="LoadInParent('#B37'); return false;" href="#B37">37</a>]. In cases where more than one sample from late-stage disease was available, the last
         sample was chosen. Individuals diagnosed with tuberculosis and clinically categorized
         as CDC: C, but without other AIDS-defining symptoms, were not included in the study.
         For patient samples DL2713H, DL2846I and DL3018H, there were no recorded CD4+ T cell
         counts. These samples were included in the study based on previous observations of
         CD4+ T cell counts of the same patient, according to the described criterions. Details
         of the plasma samples from Guinea-Bissau can be found in Table <a name="d31808e362"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/table/T2','',800,470); return false;" href="http://www.retrovirology.com/content/7/1/23/table/T2">2</a>.
      </p><div class="figs">
      <div class="table"><p><a onclick="popup('http://www.retrovirology.com/content/7/1/23/table/T1','T1',800,470); return false;" href="http://www.retrovirology.com/content/7/1/23/table/T1"><strong>Table 1.</strong></a> Evaluation of the TRT assay using a control panel of HIV-1 subtype A and CRF02_AG
         isolates.
      </p></div>
      <div class="table"><p><a onclick="popup('http://www.retrovirology.com/content/7/1/23/table/T2','T2',800,470); return false;" href="http://www.retrovirology.com/content/7/1/23/table/T2"><strong>Table 2.</strong></a> Clinical parameters, HIV-1 subtype and HIV-1 tropism of the 29 analyzed study subjects.
      </p></div></div>
      <h4>Amplification and sequencing</h4>
      <p style="line-height: 160%;">Viral RNA was extracted and purified from blood plasma samples, using RNeasy Lipid
         Tissue Mini Kit (Qiagen, Stockholm, Sweden) with minor modifications from the manufacturer's
         instructions. Briefly, 200 μl of blood plasma were disrupted in 2000 μl Qiazol and
         10 μg Carrier RNA (Qiagen). The aqueous phase was loaded onto a spin column by multiple
         loading steps. RNA was eluted in 40 μl of RNase-free water and treated with DNase
         I (Fermentas, Helsingborg, Sweden). Viral RNA was reverse transcribed using gene-specific
         primers, and the V1-V3 region amplified using a nested PCR approach (The SuperScript™
         III One-Step RT-PCR System with Platinum<sup>® </sup><em>Taq </em>DNA Polymerase and Platinum<sup>® </sup><em>Taq </em>DNA Polymerase High Fidelity, Invitrogen, Copenhagen, Denmark) according to the manufacturer's
         instructions using primers JE12F (5'-AAAGAGCAGAAGATAGTGGCAATGA-3') and V3A_R2 (5'-TTACAATAGAAAAATTCTCCTCYACA-3')
         for one-step RT-PCR and E20A_F (5'-GGGCTACACATGCCTGTGTACCYACAG-3') and JA169 for nested
         PCR <a name="d31808e1496"></a>[<a onclick="LoadInParent('#B38'); return false;" href="#B38">38</a>]. The V3 region with flanking regions (nucleotides 6847 to 7374 in HXB2; GenBank accession
         number <a target="_top" href="http://www.ncbi.nih.gov/entrez/query.fcgi?db=Nucleotide&amp;cmd=search&amp;term=K03455">K03455</a>) were then directly sequenced using BigDye Terminator v1.1 Cycle Sequencing Kit (Applied
         Biosystems, Stockholm, Sweden) according to the manufacturer's instructions using
         primers JA167 and JA169 <a name="d31808e1504"></a>[<a onclick="LoadInParent('#B38'); return false;" href="#B38">38</a>]. The V3 region was chosen for sequencing due to prevalence of length variations in
         the V1-V2 region. Sequences were determined using ABI Prism 3100 (Applied Biosystems).
         Sample DL2846I had to be cloned as a result of too mixed chromatograms. The amplified
         V1-V3 region of approximately 940 base pairs (nucleotides 6430 to 7374 in HXB2; GenBank
         accession number <a target="_top" href="http://www.ncbi.nih.gov/entrez/query.fcgi?db=Nucleotide&amp;cmd=search&amp;term=K03455">K03455</a>) were cloned using the InsTAclone cloning system (Fermentas) and TOP10 cells (Invitrogen).
         Twelve colonies were picked and the cloned fragments were amplified with Platinum<sup>® </sup><em>Taq </em>DNA Polymerase High Fidelity (Invitrogen) using conventional M13 primers (-20 and
         -24). The amplifications were successful for eight colonies, and clones were named
         with the patient identification number and a clone number. Individual clones were
         purified and sequenced as described.
      </p>
      <h4>Phylogenetic analysis</h4>
      <p style="line-height: 160%;">Sequences were assembled, and contigs were analyzed with CodonCode Aligner version
         1.5.2 (CodonCode Corporation, Dedham, USA) blinded to the phenotype. True permutated
         positions were detected by the software, and manually inspected. All sequences had
         open reading frames and were subjected for further analysis. A multiple alignment
         of our sequences with a reference sequence data set of all major subtypes, sub-subtypes
         and CRFs (downloaded from Los Alamos Sequence Database) was performed in MEGA4 using
         the Clustal algorithm <a name="d31808e1522"></a><a name="d31808e1524"></a><a name="d31808e1526"></a>[<a onclick="LoadInParent('#B39'); return false;" href="#B39">39</a>-<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>]. Nucleotide sequences were aligned via protein sequences and major gap positions
         were removed to a final sequence length of 463 base pairs. A neighbor-joining (N-J)
         tree was constructed in MEGA4 using pair-wise deletion in a maximum composite likelihood
         substitution model with heterogeneous pattern among lineages with a gamma distribution
         of 0.8338 (Akaike Information Criterion (AIC), calculated by Modeltest <a name="d31808e1530"></a>[<a onclick="LoadInParent('#B42'); return false;" href="#B42">42</a>]). The phylogenetic reconstruction was bootstrapped 1000 times to separate sequences
         of different subtypes. The gp120 V3 region of CRF02_AG is subtype A-derived, and separation
         of subclusters belonging to either CRF02_AG or subtype A were not possible in this
         tree. To further characterize these sequences, we constructed a reference data set
         consisting of sequences characterized as subtype A or CRF02_AG. To avoid any bias
         of previously incorrect subtyping due to similarities in the V3 region between subtype
         A and CRF02_AG, only HIV-1 full genome (&gt; 8000 bp) sequences were allowed. We reconstructed
         an N-J tree with our Guinea-Bissau-derived sequences and the described reference sequences
         as outlined above. Reference sequences that formed separate monophyletic clusters
         were removed from the data set to obtain a final reconstruction distinguishing clusters
         of subsubtypes (A1-A3) and CRF02_AG (AIC, gamma distribution 0.9136) (Fig. <a name="d31808e1533"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/figure/F1','',800,470); return false;" href="http://www.retrovirology.com/content/7/1/23/figure/F1">1</a>). Since Felsenstein's bootstrap test can be too conservative, we used the bootstrap
         interior branch test, with 1000 bootstraps, which is a mathematically more rigorous
         statistical method for phylogenetic reconstructions of closely related sequences <a name="d31808e1537"></a><a name="d31808e1539"></a><a name="d31808e1541"></a>[<a onclick="LoadInParent('#B43'); return false;" href="#B43">43</a>-<a onclick="LoadInParent('#B45'); return false;" href="#B45">45</a>]. Details and accession numbers of the constructed reference sequence data set can
         be found in Additional file <a name="d31808e1544"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/suppl/S1','',630,470); return false;" href="http://www.retrovirology.com/content/7/1/23/suppl/S1">1</a>.
      </p><div class="figs">
      <div class="addfile"><p><a onclick="popup('http://www.retrovirology.com/content/7/1/23/suppl/S1','S1',630,470); return false;" href="http://www.retrovirology.com/content/7/1/23/suppl/S1"></a><strong>Additional file 1.</strong> <strong>Table S1 - Reference dataset of subsubtype A1-A3 and CRF02_AG sequences</strong>. HIV-1 reference dataset of subsubtype A1-A3 and CRF02_AG sequences used in the final
         phylogenetic reconstruction for subsubtype and CRF02_AG determination of the sample
         sequences.</p><p>

         	Format: DOC 
         	Size: 40KB <a href="http://www.retrovirology.com/content/supplementary/1742-4690-7-23-s1.doc">Download file</a></p><p>This file can be viewed with: <a onclick="largepopup('http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=3657ce88-7cfa-457a-9aec-f4f827f20cac,'Help');return false;" href="http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=3657ce88-7cfa-457a-9aec-f4f827f20cac">Microsoft Word Viewer</a><span class="open-data-icon"><a href="http://www.retrovirology.com/about/access/#opendata"><img title="Open Data" alt="Open Data" src="opendata.gif"></a></span></p></div>
      <div class="fig"><p><a onclick="popup('http://www.retrovirology.com/content/7/1/23/figure/F1','F1',800,470); return false;" href="http://www.retrovirology.com/content/7/1/23/figure/F1"><img src="1742-4690-7-23-1.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 1.</strong></a> <strong>Classification of subtype A-like sequences</strong>. A neighbour joining tree showing detailed analysis of subtype A-like sequences.
         Interior branch test likelihoods (1000 bootstraps) are shown on branches distinguishing
         sub-subtypes and CRF02_AG. All sample sequences cluster within either the CRF02_AG
         or the A3 cluster. For visual clarity, subsubtype A1 and A2 clusters are represented
         by triangles.
      </p></div></div>
      <h4>Determination of coreceptor tropism and evaluation of phenotypic method</h4>
      <p style="line-height: 160%;">Human kidney embryonic 293T cells and human glioma U87.CD4 cells, stably expressing
         CD4 and one of the chemokine receptors (CCR5 or CXCR4) were maintained as previously
         described <a name="d31808e1579"></a><a name="d31808e1581"></a>[<a onclick="LoadInParent('#B46'); return false;" href="#B46">46</a>,<a onclick="LoadInParent('#B47'); return false;" href="#B47">47</a>]. Chimeric viruses with patient-specific V1-V3 regions were generated based on the
         protocol from the Tropism Recombinant Test (TRT) with minor modifications <a name="d31808e1585"></a><a name="d31808e1587"></a>[<a onclick="LoadInParent('#B15'); return false;" href="#B15">15</a>,<a onclick="LoadInParent('#B29'); return false;" href="#B29">29</a>]. Briefly, 500 ng of amplified V1-V3 fragments from each plasma sample and 3 μg of
         43XCΔV, a NheI-linearized vector containing a full-length pNL4-3 genome with the V1-V3
         region deleted, were transfected into 293T cells using the calcium phosphate precipitation
         method. Chimeric viruses were harvested and stored at -80°C. Twenty-four hours before
         infection, 10<sup>5 </sup>U87.CD4 cells/well were seeded in 48 well plates. For infection, 500 μl of chimeric
         viruses were added in duplicate wells. Cells were washed three times with Dulbecco
         Modified Eagle Medium 16 hours post infection. Cultures were analyzed at day one,
         seven and nine for p24 antigen production by ELISA (Biomérieux, Boxtel, The Netherlands).
      </p>
      <p style="line-height: 160%;">To evaluate the performance of the TRT on subtype A and CRF02_AG (which is A-like
         in the V1-V3 region of gp120), we used a panel of selected isolates (generously provided
         by Professor Jan Albert, Swedish Institute for Infectious Disease Control) (Table
         <a name="d31808e1595"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/table/T1','',800,470); return false;" href="http://www.retrovirology.com/content/7/1/23/table/T1">1</a>). Subtype and phenotype of the isolates had been determined previously <a name="d31808e1599"></a>[<a onclick="LoadInParent('#B21'); return false;" href="#B21">21</a>]. To confirm the phenotype results, we used 200 μl of each isolate for infection of
         U87.CD4-CCR5 and U87.CD4-CXCR4 in duplicate wells. The cells were washed 16 hours
         post infection and then analyzed for p24 production as described. From each isolate
         the V1-V3 region was amplified and used to produce recombinant viruses for infection
         and analysis according to the described procedure. Infections of U87.CD4-CXCR4 with
         chimeric viruses derived from R5 isolates were used as negative controls to calculate
         the p24 background value of the assay. Infections with a significant increase (mean
         value +/- 3 S.D.) in p24 antigen production over time, compared to day one of the
         infection, were considered as positive infections.
      </p>
      <h4>Genotypic characterization of CRF02_AG coreceptor use</h4>
      <p style="line-height: 160%;">The aim was to characterize genetic properties in HIV-1 V3 <em>env </em>distinguishing CRF02_AG with different coreceptor tropism. Only two sequences of the
         control panel (22480 and 22627) and three sequences of the plasma samples (DL4477D,
         DL2391G and DL22853E) were of HIV-1 CRF02_AG with pure CCR5 tropism. Since this number
         was too low for an appropriate comparison between the R5 and R5/X4 groups, we added
         all available V3 sequences, with known coreceptor tropism, from Los Alamos sequence
         data base, to a final dataset of 111 sequences (Additional files <a name="d31808e1611"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/suppl/S2','',630,470); return false;" href="http://www.retrovirology.com/content/7/1/23/suppl/S2">2</a>, <a name="d31808e1614"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/suppl/S3','',630,470); return false;" href="http://www.retrovirology.com/content/7/1/23/suppl/S3">3</a>, <a name="d31808e1617"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/suppl/S4','',630,470); return false;" href="http://www.retrovirology.com/content/7/1/23/suppl/S4">4</a>) <a name="d31808e1621"></a>[<a onclick="LoadInParent('#B39'); return false;" href="#B39">39</a>]. Only one sequence per patient was subjected for analyses. Multiple alignments of
         CRF02_AG V3 amino acid sequences were performed as described, and positively [K and
         R] and negatively [D and E] charged amino acids were counted. In codons with amino
         acid mixtures, all possible permutations were assessed (Additional files <a name="d31808e1625"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/suppl/S3','',630,470); return false;" href="http://www.retrovirology.com/content/7/1/23/suppl/S3">3</a> and <a name="d31808e1628"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/suppl/S4','',630,470); return false;" href="http://www.retrovirology.com/content/7/1/23/suppl/S4">4</a>). The combination resulting in the highest net charge was used for phenotype prediction.
         The performance of different sequence motif-based rules was measured in terms of sensitivity,
         specificity, positive predictive value (PPV) and negative predictive value (NPV).
         The sensitivity was determined as the fraction of predicted X4 sequences among the
         sequences from viruses phenotyped as CXCR4-using, the specificity as the fraction
         of predicted R5 sequences among the sequences from viruses phenotyped as CCR5-using
         only, the PPV as the fraction of correctly predicted X4 sequences among all predicted
         X4 sequences, and the NPV as the fraction of correctly predicted R5 sequences among
         all predicted R5 sequences.
      </p><div class="figs">
      <div class="addfile"><p><a onclick="popup('http://www.retrovirology.com/content/7/1/23/suppl/S2','S2',630,470); return false;" href="http://www.retrovirology.com/content/7/1/23/suppl/S2"></a><strong>Additional file 2.</strong> <strong>Table S2 - Reference dataset of CRF02_AG sequences used in the molecular analysis</strong>. Accession numbers of the HIV-1 reference dataset of HIV-1 CRF02_AG sequences with
         phenotypically determined coreceptor tropism used in the molecular analysis.</p><p>

         	Format: DOC 
         	Size: 42KB <a href="http://www.retrovirology.com/content/supplementary/1742-4690-7-23-s2.doc">Download file</a></p><p>This file can be viewed with: <a onclick="largepopup('http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=3657ce88-7cfa-457a-9aec-f4f827f20cac,'Help');return false;" href="http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=3657ce88-7cfa-457a-9aec-f4f827f20cac">Microsoft Word Viewer</a><span class="open-data-icon"><a href="http://www.retrovirology.com/about/access/#opendata"><img title="Open Data" alt="Open Data" src="opendata.gif"></a></span></p></div>
      <div class="addfile"><p><a onclick="popup('http://www.retrovirology.com/content/7/1/23/suppl/S3','S3',630,470); return false;" href="http://www.retrovirology.com/content/7/1/23/suppl/S3"></a><strong>Additional file 3.</strong> <strong>Table S3 - Alignment and molecular characteristics of HIV-1 CRF02_AG V3 amino acid
            sequences from study samples and references with determined CCR5 tropism</strong>. Summary of the molecular characteristics of the CCR5 tropic sequences used in the
         genotypic analysis.</p><p>

         	Format: DOC 
         	Size: 258KB <a href="http://www.retrovirology.com/content/supplementary/1742-4690-7-23-s3.doc">Download file</a></p><p>This file can be viewed with: <a onclick="largepopup('http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=3657ce88-7cfa-457a-9aec-f4f827f20cac,'Help');return false;" href="http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=3657ce88-7cfa-457a-9aec-f4f827f20cac">Microsoft Word Viewer</a><span class="open-data-icon"><a href="http://www.retrovirology.com/about/access/#opendata"><img title="Open Data" alt="Open Data" src="opendata.gif"></a></span></p></div>
      <div class="addfile"><p><a onclick="popup('http://www.retrovirology.com/content/7/1/23/suppl/S4','S4',630,470); return false;" href="http://www.retrovirology.com/content/7/1/23/suppl/S4"></a><strong>Additional file 4.</strong> <strong>Table S4 - Alignment and molecular characteristics of HIV-1 CRF02_AG V3 amino acid
            sequences from study samples and references with determined CXCR4 tropism</strong>. Summary of the molecular characteristics of the CXCR4 tropic sequences used in the
         genotypic analysis.</p><p>

         	Format: DOC 
         	Size: 170KB <a href="http://www.retrovirology.com/content/supplementary/1742-4690-7-23-s4.doc">Download file</a></p><p>This file can be viewed with: <a onclick="largepopup('http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=3657ce88-7cfa-457a-9aec-f4f827f20cac,'Help');return false;" href="http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=3657ce88-7cfa-457a-9aec-f4f827f20cac">Microsoft Word Viewer</a><span class="open-data-icon"><a href="http://www.retrovirology.com/about/access/#opendata"><img title="Open Data" alt="Open Data" src="opendata.gif"></a></span></p></div></div>
      <h4>Statistics</h4>
      <p style="line-height: 160%;">All statistical analysis was performed using SPSS 16.0.</p>
      <h4>Ethics</h4>
      <p style="line-height: 160%;">The study was approved by the Research Ethics Committee at the Karolinska Institute,
         Stockholm, and the Ministries of Health and the Interior in Guinea-Bissau.
      </p>
      <h4>Nucleotide sequence accession numbers</h4>
      <p style="line-height: 160%;">Nucleotide sequences were deposited in GenBank under the following accession numbers:
         <a target="_top" href="http://www.ncbi.nih.gov/entrez/query.fcgi?db=Nucleotide&amp;cmd=search&amp;term=GQ401717">GQ401717</a>-<a target="_top" href="http://www.ncbi.nih.gov/entrez/query.fcgi?db=Nucleotide&amp;cmd=search&amp;term=GQ401744">GQ401744</a>, and <a target="_top" href="http://www.ncbi.nih.gov/entrez/query.fcgi?db=Nucleotide&amp;cmd=search&amp;term=FJ831886">FJ831886</a>-<a target="_top" href="http://www.ncbi.nih.gov/entrez/query.fcgi?db=Nucleotide&amp;cmd=search&amp;term=FJ831893">FJ831893</a>.
      </p>
   </div>
</section>
<section><a name="sec3"></a><h3><span class="arrow"></span>Results</h3>
   <div class="collapsible-content">
      <h4>Subtype determination</h4>
      <p style="line-height: 160%;">The HIV-1 V3 region from 29 plasma samples, collected during late-stage disease from
         29 treatment-naïve individuals, was amplified and sequenced. A phylogenetic tree with
         these sequences and reference sequences of different subtypes was reconstructed. Sequences
         were well separated with long branches in the phylogeny, indicating patient-specific
         origin of sequences (Fig. <a name="d31808e1707"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/figure/F1','',800,470); return false;" href="http://www.retrovirology.com/content/7/1/23/figure/F1">1</a>). Twenty-eight sequences formed a subtype A cluster together with reference sequences
         of A1, A2 and CRF02_AG. The non-subtype A sequence clustered with reference strains
         of subtype C. The V3 region of CRF02_AG is subtype A-derived, and to further characterize
         these sequences we made a BLAST-search using each sample sequence as query sequence.
         The three most similar full genome (&gt; 8000 bp) hits were used as reference sequences
         to distinguish subtype A from CRF02_AG. Among the 28 patients with subtype A-like
         sequences, 22 clustered with CRF02_AG reference sequences, and six with the previously
         described subsubtype A3 (Fig. <a name="d31808e1710"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/figure/F1','',800,470); return false;" href="http://www.retrovirology.com/content/7/1/23/figure/F1">1</a>, Table <a name="d31808e1713"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/table/T2','',800,470); return false;" href="http://www.retrovirology.com/content/7/1/23/table/T2">2</a>) <a name="d31808e1717"></a>[<a onclick="LoadInParent('#B48'); return false;" href="#B48">48</a>].
      </p>
      <h4>Accuracy of the phenotypic method for subtype A and CRF02_AG</h4>
      <p style="line-height: 160%;">Construction of chimeric viruses is a commonly used method for coreceptor tropism
         analysis, and both commercial and in-house variants can be found in the literature
         <a name="d31808e1727"></a><a name="d31808e1729"></a><a name="d31808e1731"></a>[<a onclick="LoadInParent('#B29'); return false;" href="#B29">29</a>,<a onclick="LoadInParent('#B30'); return false;" href="#B30">30</a>,<a onclick="LoadInParent('#B49'); return false;" href="#B49">49</a>]. Here, we used for subtype B the well-established Tropism Recombinant Test (TRT)
         <a name="d31808e1735"></a><a name="d31808e1737"></a>[<a onclick="LoadInParent('#B29'); return false;" href="#B29">29</a>,<a onclick="LoadInParent('#B50'); return false;" href="#B50">50</a>]. To confirm that the TRT assay performs equally well for HIV-1 subtype A and CRF02_AG
         as for HIV-1 subtype B, we used a control panel of HIV-1 subtype A and CRF02_AG isolates
         with known coreceptor tropism. We reanalyzed and confirmed previous results of the
         isolates by infecting the cell lines U87.CD4-CCR5 and U87.CD4-CXCR4, and by direct
         sequencing of the V3 region with flanking regions (Table <a name="d31808e1740"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/table/T1','',800,470); return false;" href="http://www.retrovirology.com/content/7/1/23/table/T1">1</a>). The V1-V3 region of each isolate was amplified and used for production of chimeric
         viruses. Tropism results of infections with chimeric viruses were in concordance with
         results from isolate infections, showing that the TRT assay can be used for determination
         of coreceptor tropism of HIV-1 subtype A and CRF02_AG (Table <a name="d31808e1743"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/table/T1','',800,470); return false;" href="http://www.retrovirology.com/content/7/1/23/table/T1">1</a>). The high concordance also suggests that the major determinants for coreceptor use
         are located within the V1-V3 region.
      </p>
      <h4>Prevalence of HIV-1 CXCR4-using populations in subtype A and CRF02_AG infected individuals
         in late-stage disease
      </h4>
      <p style="line-height: 160%;">To investigate the prevalence of CXCR4-using populations in the 28 individuals from
         Guinea-Bissau infected with HIV-1 subtype A or CRF02_AG, we constructed infectious
         chimeric viruses with patient-specific gp120 V1-V3 regions. All chimeric viruses that
         were tested established productive infections with a significant increase in p24 antigen
         production over time (1-9 days) in U87.CD4-CCR5 and/or U87.CD4-CXCR4 cells. Twenty-one
         (75%) of the individuals studied had viruses that used both CCR5 and CXCR4 for cellular
         entry, whereas one (4%) and six (21%) individuals had pure X4 or R5 populations, respectively
         (Table <a name="d31808e1752"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/table/T2','',800,470); return false;" href="http://www.retrovirology.com/content/7/1/23/table/T2">2</a>). In subtype A infected individuals, three of six had CXCR4-using populations, whereas
         the corresponding number in CRF02_AG infected individuals were 19 of 22 (86%).
      </p>
      <p style="line-height: 160%;">Recently, an evolving epidemic with increasing frequency of CXCR4 tropism among subtype
         C-infected individuals was suggested <a name="d31808e1758"></a>[<a onclick="LoadInParent('#B11'); return false;" href="#B11">11</a>]. To determine if a similar pattern could be seen in our HIV-1 CRF02_AG material from
         Guinea-Bissau, we divided the data set in one early group (samples collected from
         1997 to 2001), and one late group (samples collected from 2003 to 2007). In the early
         group, five out of eight samples had viruses that were CXCR4-tropic, whereas all samples
         in the late group were of this phenotype (14 out of 14) (p = 0.036, two-tailed Fisher's
         exact test). The two groups were well balanced with no significant differences in
         CD4+ T cell counts (p = 0.646 and p = 0.220 for CD4tot and CD4%, respectively, Mann-Whitney
         U test). To investigate if this difference could be found in a larger material of
         HIV-1 CRF02_AG we added all available data from the literature and in The Los Alamos
         Sequence Data Base where we could couple patient-specific disease status with coreceptor
         tropism and sampling year (Additional file <a name="d31808e1761"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/suppl/S5','',630,470); return false;" href="http://www.retrovirology.com/content/7/1/23/suppl/S5">5</a>). We found data of 43 different individuals sampled during 1997-2001 or 2003-2007
         with defined late-stage disease and with known coreceptor tropism (42 from Cameroon
         and 1 from Ghana). When analyzed together with our samples from Guinea-Bissau, 28
         out of 50 had viruses that were CXCR4-tropic in the early group, whereas 14 out of
         15 were of this phenotype in the late group (p = 0.012, two-tailed Fisher's exact
         test) (Additional file <a name="d31808e1764"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/suppl/S5','',630,470); return false;" href="http://www.retrovirology.com/content/7/1/23/suppl/S5">5</a>).
      </p><div class="figs">
      <div class="addfile"><p><a onclick="popup('http://www.retrovirology.com/content/7/1/23/suppl/S5','S5',630,470); return false;" href="http://www.retrovirology.com/content/7/1/23/suppl/S5"></a><strong>Additional file 5.</strong> <strong>Table S5. Data used to investigate an evolving epidemic available data from HIV-1
            CRF02_AG</strong>. Summary of the data obtained from the literature and Los Alamos Sequence Data Base
         to investigate if the HIV-1 CRF02_AG epidemic represents an evolving epidemic.</p><p>

         	Format: DOC 
         	Size: 35KB <a href="http://www.retrovirology.com/content/supplementary/1742-4690-7-23-s5.doc">Download file</a></p><p>This file can be viewed with: <a onclick="largepopup('http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=3657ce88-7cfa-457a-9aec-f4f827f20cac,'Help');return false;" href="http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=3657ce88-7cfa-457a-9aec-f4f827f20cac">Microsoft Word Viewer</a><span class="open-data-icon"><a href="http://www.retrovirology.com/about/access/#opendata"><img title="Open Data" alt="Open Data" src="opendata.gif"></a></span></p></div></div>
      <h4>Molecular characterization of the V3 region coupled to coreceptor tropism</h4>
      <p style="line-height: 160%;">To create a data set that would allow for comparison between the R5 and R5X4/X4 groups,
         we combined the 22 CRF02_AG plasma-derived sequences with the three CRF02_AG sequences
         from the control panel and the 86 CRF02_AG sequences available in Los Alamos sequence
         data base with known phenotype resulting in a final dataset of 111 sequences (75 R5
         and 36 R5X4 or X4 sequences) (Additional files <a name="d31808e1785"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/suppl/S3','',630,470); return false;" href="http://www.retrovirology.com/content/7/1/23/suppl/S3">3</a> and <a name="d31808e1788"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/suppl/S4','',630,470); return false;" href="http://www.retrovirology.com/content/7/1/23/suppl/S4">4</a>). Sequences of subsubtype A3 were too few, and therefore not subjected for further
         analysis. Several sequence motif-based rules and bioinformatic tools have been developed
         to predict coreceptor tropism based on V3 sequences of HIV-1. These have mainly been
         based on subtype B, and the most common are the 11/25 rule (positively charged amino
         acids in V3 position 11 and/or 25 predicts CXCR4 tropism) and the net charge rule
         (a net charge of ≥+5 predicts CXCR4 tropism) <a name="d31808e1792"></a>[<a onclick="LoadInParent('#B51'); return false;" href="#B51">51</a>]. Raymond <em>et al. </em>used the 11/25 rule in combination with the net charge rule to develop a genotypic
         rule specific for CRF02_AG (Table <a name="d31808e1798"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/table/T3','',800,470); return false;" href="http://www.retrovirology.com/content/7/1/23/table/T3">3</a>) <a name="d31808e1803"></a>[<a onclick="LoadInParent('#B52'); return false;" href="#B52">52</a>]. Here, we used these rules and two widely used bioinformatic tools, WebPSSM and Geno2Pheno
         to predict the HIV-1 tropism based on the V3 sequence <a name="d31808e1807"></a><a name="d31808e1809"></a>[<a onclick="LoadInParent('#B32'); return false;" href="#B32">32</a>,<a onclick="LoadInParent('#B53'); return false;" href="#B53">53</a>] (Table <a name="d31808e1812"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/table/T3','',800,470); return false;" href="http://www.retrovirology.com/content/7/1/23/table/T3">3</a>). The sensitivity reflects how many sequences of the true CXCR4-using viruses that
         the genotypic rule or bioinformatic tool identifies as CXCR4-using, whereas the specificity
         reflects how many sequences of the true CCR5-using viruses that the genotypic rule
         or bioinformatic tool identifies as CCR5-using. The PPV (R5X4/X4) and the NPV (R5)
         reflect how many of the predictions that are accurate. Of existing rules and tools,
         Geno2Pheno had the highest sensitivity, whereas the combined rule by Raymond <em>et al. </em>had the highest specificity (Table <a name="d31808e1818"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/table/T3','',800,470); return false;" href="http://www.retrovirology.com/content/7/1/23/table/T3">3</a>).
      </p><div class="figs">
      <div class="table"><p><a onclick="popup('http://www.retrovirology.com/content/7/1/23/table/T3','T3',800,470); return false;" href="http://www.retrovirology.com/content/7/1/23/table/T3"><strong>Table 3.</strong></a> Comparison of different genotypic rules and bioinformatic tools for prediction of
         HIV-1 coreceptor tropism based on the HIV-1 V3 amino acid sequence.
      </p></div></div>
      <p style="line-height: 160%;">The sensitivity is of particular interest in clinical settings since it reflects the
         likelihood of detecting true CXCR4-tropism, and patients having HIV-1 of this tropism
         may not be suitable for treatment with coreceptor antagonists. Due to the low sensitivity
         of previous rules for HIV-1 of CRF02_AG, we used our data set to develop new rules.
         The mean net charges of V3 in the R5 and R5X4/X4 groups were 3.45 (95% CI: 3.27-3.64),
         and 4.47 (95% CI: 4.01-4.94), respectively. Modifying the net charge rule by setting
         the cutoff for prediction of CXCR4 tropism to ≥+4, we improved the sensitivity to
         72%, although to a cost of both specificity (47%) and PPV (46%) (Table <a name="d31808e2339"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/table/T3','',800,470); return false;" href="http://www.retrovirology.com/content/7/1/23/table/T3">3</a>). We then examined another approach by counting the total number of charged amino
         acids. The mean numbers of charged amino acids were 6.60 (95% CI: 6.36-6.84) and 7.58
         (95% CI: 7.16-8.01) for the R5 and R5X4/X4 group, respectively. Setting the cutoff
         to ≥ 8 charged amino acids for prediction of CXCR4 tropism resulted in a sensitivity
         equal to the one obtained by Geno2Pheno (50%) (Table <a name="d31808e2342"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/table/T3','',800,470); return false;" href="http://www.retrovirology.com/content/7/1/23/table/T3">3</a>).
      </p>
      <p style="line-height: 160%;">Finally, we tested all possible combinations of previous genotypic rules analyzed
         in this study, and the net charge rule of ≥+4 and/or the total charge rule of ≥ 8
         for prediction of CXCR4 use (data not shown). Of all combinations tested, the combination
         of the net charge rule of ≥+5 and total charge rule of ≥ 8 resulted in the best improvement
         in sensitivity (64%), without losing too much in specificity (73%) (Table <a name="d31808e2347"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/table/T3','',800,470); return false;" href="http://www.retrovirology.com/content/7/1/23/table/T3">3</a>).
      </p>
   </div>
</section>
<section><a name="sec4"></a><h3><span class="arrow"></span>Discussion</h3>
   <div class="collapsible-content">
      <p style="line-height: 160%;">In this study, we show that the V1-V3 region of the HIV-1 envelope is the major determinant
         for coreceptor tropism also for subtype A and CRF02_AG, and that the TRT assay accurately
         determines coreceptor tropism for these subtypes <a name="d31808e2357"></a>[<a onclick="LoadInParent('#B29'); return false;" href="#B29">29</a>]. In our samples, we found a high prevalence of X4 populations (79%) during late-stage
         disease and an increasing frequency of HIV-1 with CXCR4 tropism in CRF02_AG-infected
         patients over time. We also demonstrate that the total number of charged amino acids
         may contribute to the development of genotypic rules and bioinformatic tools for prediction
         of coreceptor tropism of HIV-1 CRF02_AG.
      </p>
      <p style="line-height: 160%;">HIV-1 subtype determination was done by amplifying and sequencing the envelope gp120
         V3 region of 29 infected individuals in late-stage disease from Guinea-Bissau, West
         Africa. Twenty-eight of the individuals had HIV-1 of subtype A or CRF02_AG, confirming
         previous results that these are the dominating HIV-1 forms in Guinea-Bissau <a name="d31808e2363"></a>[<a onclick="LoadInParent('#B54'); return false;" href="#B54">54</a>]. The remaining study subject was infected with HIV-1 subtype C, a subtype that has
         never been described in Guinea-Bissau before, even though it has been shown to circulate
         to some extent in West Africa <a name="d31808e2367"></a>[<a onclick="LoadInParent('#B55'); return false;" href="#B55">55</a>]. A detailed analysis of the six patients infected with subtype A showed close genetic
         relationship to the previously described sub-subtype A3 (Fig. <a name="d31808e2370"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/figure/F1','',800,470); return false;" href="http://www.retrovirology.com/content/7/1/23/figure/F1">1</a>). This sub-subtype was first described in Senegal, but has been shown to be prevalent
         in several West African countries, including Guinea-Bissau <a name="d31808e2374"></a><a name="d31808e2376"></a>[<a onclick="LoadInParent('#B48'); return false;" href="#B48">48</a>,<a onclick="LoadInParent('#B56'); return false;" href="#B56">56</a>].
      </p>
      <p style="line-height: 160%;">A control panel of 11 HIV-1 subtype A and CRF02_AG isolates was used to show that
         the, for subtype B, well-characterized TRT assay is accurate also for subtype A and
         CRF02_AG. This finding suggests that the gp120 V1-V3 region is the major determinant
         of coreceptor phenotype also for subtype A and CRF02_AG. This conclusion is in line
         with the general concept previously established for subtype B that coreceptor phenotype
         is determined by the V1-V3 region of gp120, in particular the V3 region <a name="d31808e2382"></a>[<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a>]. Our results show that the TRT can be used as a reliable alternative to the commercially
         available Trofile assay (Monogram Biosciences, San Francisco, USA), at least for the
         tested subtypes. The Trofile assay require amplification of the entire gp160 (2,500
         bp), and it has been proposed that use of V1-V3 (940 bp) can be a more sensitive approach
         as it could reduce the risk of losing minority populations seen when amplifying a
         larger fragment <a name="d31808e2386"></a>[<a onclick="LoadInParent('#B50'); return false;" href="#B50">50</a>].
      </p>
      <p style="line-height: 160%;">Since we had a reliable phenotypic assay, we were able to analyze genetic traits that
         would best predict coreceptor phenotype for viruses from our HIV-1 CRF02_AG samples.
         Applying the 11/25 rule (positively charged amino acids in V3 position 11 and/or 25
         predicts CXCR4 tropism) we found a sensitivity of 31%, and a specificity 95%. This
         is similar to a previous study of 113 CRF02_AG isolates, where the 11/25 rule had
         a sensitivity of 33%, and a specificity of 96% <a name="d31808e2392"></a>[<a onclick="LoadInParent('#B13'); return false;" href="#B13">13</a>]. In another study the authors used the combined criteria of the 11/25 and the net
         charge rule of V3 sequences of 52 CRF02_AG isolates and found both high sensitivity
         (70%) and high specificity (98%) <a name="d31808e2396"></a>[<a onclick="LoadInParent('#B52'); return false;" href="#B52">52</a>]. The results of the combined criteria could, however, not be verified by our data.
         A reason for the poor performance of this rule on our dataset may be that it relies
         on the 11/25 rule. In sequences from viruses with CXCR4 tropism from Guinea-Bissau
         we found in general negatively charged or non-charged amino acids in positions 11
         and 25, whereas at least one of these positions in most cases were positively charged
         in the sequences used in the study by Raymond <em>et al</em>. (Additional files <a name="d31808e2402"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/suppl/S3','',630,470); return false;" href="http://www.retrovirology.com/content/7/1/23/suppl/S3">3</a> and <a name="d31808e2405"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/suppl/S4','',630,470); return false;" href="http://www.retrovirology.com/content/7/1/23/suppl/S4">4</a>). We also observed many charged (both positive and negative) amino acids as a profound
         characteristic of sequences derived from CXCR4-using viruses (in the complete dataset).
         Therefore, we counted the total number of charged amino acids in the V3 region and
         combined it with the net charge rule. Due to sensitivity, this rule performed better
         than all of the analyzed rules and bioinformatic tools, without losing too much in
         specificity. This rule is also different to those rules that are used for subtype
         B, suggesting that molecular differences at the level of virus-cell receptor interaction
         may exist among HIV-1 subtypes. Further studies are needed to investigate subtype-specific
         genotypic differences involved in HIV-1 coreceptor tropism, and the sensitivity of
         existing genotypic rules and bioinformatic tools have to be increased before they
         can be used in a clinical setting.
      </p>
      <p style="line-height: 160%;">Next, we examined the coreceptor phenotype of HIV-1 in 28 subtype A or CRF02_AG infected
         individuals in late-stage disease. CXCR4-using viral populations were found in as
         much as 79% of the analyzed samples, demonstrating the importance of analyzing samples
         from patients in late-stage disease when investigating HIV-1 subtype-specific predisposal
         for CXCR4 tropism. For HIV-1 subtype B, it is well known that CCR5 is the dominant
         coreceptor early in infection. Switch or broadening of coreceptor use from CCR5 to
         CXCR4 occurs late in disease <a name="d31808e2411"></a>[<a onclick="LoadInParent('#B6'); return false;" href="#B6">6</a>]. To our knowledge, only one previous study has reported on CRF02_AG coreceptor tropism
         in late-stage disease <a name="d31808e2415"></a>[<a onclick="LoadInParent('#B13'); return false;" href="#B13">13</a>]. Vergne <em>et al. </em>found that 56% of the studied isolates were positive for MT-2 cell tropism, to be
         compared to our result of CXCR4 use of 86%. Although, it is important to note that
         the viruses in the study by Vergne <em>et al. </em>were isolated from samples collected between 1996 and 2001, and the corresponding
         number of CXCR4-tropism among our samples during this time-frame was 63%.
      </p>
      <p style="line-height: 160%;">Connell <em>et al. </em>(2008) analyzed the results of 19 subtype C isolates, isolated in 2005, and found
         a higher prevalence (30%) of CXCR4 tropism than has been shown in earlier studies
         <a name="d31808e2430"></a>[<a onclick="LoadInParent('#B11'); return false;" href="#B11">11</a>]. They suggested that HIV-1 subtype C might be an evolving epidemic, showing an increasing
         prevalence of CXCR4 phenotype over time in South Africa. In the present study, we
         performed a direct comparison by dividing our 22 CRF02_AG samples into two groups:
         Samples from 1997 to 2001, and samples from 2003 to 2007. The CD4+ T cell counts and
         clinical data were similar between the two groups, excluding any bias due to differences
         in disease status. In addition, all of the investigated individuals were treatment
         naïve. We found a significant difference between the groups, suggesting a similar
         kind of evolving epidemic for CRF02_AG in Guinea-Bissau that has been suggested for
         subtype C in South Africa. We also analyzed our data together with available data
         of HIV-1 CRF02_AG infected patients with known clinical parameters and coreceptor
         tropism and found that the trend of an evolving epidemic was consistent. Competition
         assays between HIV-1 R5 and X4 viruses have shown that X4 viruses in general out-compete
         R5 viruses due to both higher replication kinetics and higher CXCR4 than CCR5 expression
         in PBMC <a name="d31808e2434"></a>[<a onclick="LoadInParent('#B57'); return false;" href="#B57">57</a>]. Moreover, it has been shown that sequence change occur at a rate of 1% per year
         in HIV-1 <em>env</em>, illustrating the constant evolution of HIV-1 on the genetic level <a name="d31808e2441"></a>[<a onclick="LoadInParent('#B58'); return false;" href="#B58">58</a>]. Further studies (based on larger sample sizes than studied here and by Connell <em>et al.</em>) are needed to investigate if HIV-1 is evolving towards a more predisposed state
         of changing into CXCR4 phenotype on a population level, and a confirmation of this
         finding in larger cohorts may have implications for viral transmission, pathogenesis
         and disease progression.
      </p>
      <p style="line-height: 160%;">To further the view of our results, we performed a literature review of published
         results regarding subtype-specific coreceptor tropism in late-stage disease, which,
         to our knowledge, has not been presented before. The analysis included 498 patient-specific
         HIV-1 samples of six different subtypes, sampled over more than 20 years (1988-2008).
         Data of CD4+ T cell counts and/or clinical status for all patients were examined,
         and only samples from individuals diagnosed with AIDS or having CD4+ T cell counts
         ≤ 200 cells/μl were included in the analysis. In cases were both CD4+ T cell counts
         and clinical status could be found, the criterion of CD4+ T cell count was used. The
         vast majority of patients were included based on CD4+ T cell counts. Moreover, only
         one sample per patient was allowed, and in cases where the same patients appeared
         in several studies the patient data were only used once (for details about the analysis,
         see Additional file <a name="d31808e2450"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/suppl/S6','',630,470); return false;" href="http://www.retrovirology.com/content/7/1/23/suppl/S6">6</a>). HIV-1 coreceptor tropism determined as MT-2 non-syncytium or syncytium inducing
         was regarded as CCR5 or CXCR4 tropism, respectively <a name="d31808e2454"></a>[<a onclick="LoadInParent('#B22'); return false;" href="#B22">22</a>]. If HIV-1 subtype and coreceptor tropism were not specified, this data were confirmed
         by personal communication with the authors (Additional file <a name="d31808e2457"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/suppl/S6','',630,470); return false;" href="http://www.retrovirology.com/content/7/1/23/suppl/S6">6</a>, and Acknowledgement). Our analysis revealed a high frequency of HIV-1 R5X4 or X4
         populations in late stage disease among all analyzed subtypes, except for subtype
         C (Fig. <a name="d31808e2460"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/figure/F2','',800,470); return false;" href="http://www.retrovirology.com/content/7/1/23/figure/F2">2</a>). Only 15% of the individuals infected with HIV-1 subtype C had CXCR4-using populations,
         compared to 66% (60%-77%) in individuals infected with HIV-1 of non-subtype C (p &lt;
         0.001, two-tailed Fisher's exact test).
      </p><div class="figs">
      <div class="addfile"><p><a onclick="popup('http://www.retrovirology.com/content/7/1/23/suppl/S6','S6',630,470); return false;" href="http://www.retrovirology.com/content/7/1/23/suppl/S6"></a><strong>Additional file 6.</strong> <strong>Table S6 - Overview of the literature analysis</strong>. Summary of the data obtained from the literature and used to determine the amount
         of CXCR4 tropism in late stage disease in the studied HIV-1 subtypes.</p><p>

         	Format: DOC 
         	Size: 156KB <a href="http://www.retrovirology.com/content/supplementary/1742-4690-7-23-s6.doc">Download file</a></p><p>This file can be viewed with: <a onclick="largepopup('http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=3657ce88-7cfa-457a-9aec-f4f827f20cac,'Help');return false;" href="http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=3657ce88-7cfa-457a-9aec-f4f827f20cac">Microsoft Word Viewer</a><span class="open-data-icon"><a href="http://www.retrovirology.com/about/access/#opendata"><img title="Open Data" alt="Open Data" src="opendata.gif"></a></span></p></div>
      <div class="fig"><p><a onclick="popup('http://www.retrovirology.com/content/7/1/23/figure/F2','F2',800,470); return false;" href="http://www.retrovirology.com/content/7/1/23/figure/F2"><img src="1742-4690-7-23-2.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 2.</strong></a> <strong>Prevalence of HIV-1 CXCR4-tropism in late-stage disease in different subtypes</strong>. Combined results of new data presented in this study and data from the literature
         analysis, showing the prevalence of HIV-1 CXCR4-tropic viruses in late stage disease.
         The number of individuals used in each diagram is specified within brackets. A detailed
         description of the analysis can be found in Additional file <a name="d31808e2487"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/suppl/S6','',630,470); return false;" href="http://www.retrovirology.com/content/7/1/23/suppl/S6">6</a>.
      </p></div></div>
      <p style="line-height: 160%;">The data from the literature review also allowed us to investigate if we could confirm
         the results of Connell <em>et al. </em>of an evolving epidemic in South Africa, and if this could be seen for subtype C in
         general <a name="d31808e2497"></a>[<a onclick="LoadInParent('#B11'); return false;" href="#B11">11</a>]. We divided the subtype C data set (208 patients, samples collected in Cameroon,
         Ethiopia, India, Malawi, South Africa, Sweden and Zimbabwe) into an early group (samples
         before the year of 2000) and a late group (samples after 2000) (Additional file <a name="d31808e2500"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/suppl/S7','',630,470); return false;" href="http://www.retrovirology.com/content/7/1/23/suppl/S7">7</a>). In the early group from South Africa eight of 46 patients (17%) had CXCR4-using
         viruses, whereas the corresponding number in the late group was 11 of 36 (31%) (p
         = 0.19, two-tailed Fisher's exact test). Connell <em>et al. </em>reported that 30% of their isolates were able to use CXCR4 and compared this to previous
         studies from South Africa showing no syncytium-inducing (SI) capacity of HIV-1 isolates
         collected during the 1980s, whereas 10-17% of the studied isolates had SI capacity
         during the 1990s. No statistical evaluation was performed in their study. In the early
         group of our complete subtype C data set, 11 of 145 patients (8%) had CXCR4-using
         viruses, whereas the corresponding number in the late group was 21 of 63 (33%) (p
         &lt; 0.001, two-tailed Fisher's exact test). Our results confirms the numbers presented
         by Connell <em>et al</em>. in South Africa (even though the difference between the early and the late group
         was not statistically significant), and further strengthens the concern of an evolving
         HIV-1 subtype C-epidemic on the population level.
      </p><div class="figs">
      <div class="addfile"><p><a onclick="popup('http://www.retrovirology.com/content/7/1/23/suppl/S7','S7',630,470); return false;" href="http://www.retrovirology.com/content/7/1/23/suppl/S7"></a><strong>Additional file 7.</strong> <strong>Table S7 - Overview of the subtype C material</strong>. Summary of the subtype C data obtained from the literature review used in the analysis
         of an evolving epidemic for subtype C.</p><p>

         	Format: DOC 
         	Size: 64KB <a href="http://www.retrovirology.com/content/supplementary/1742-4690-7-23-s7.doc">Download file</a></p><p>This file can be viewed with: <a onclick="largepopup('http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=3657ce88-7cfa-457a-9aec-f4f827f20cac,'Help');return false;" href="http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=3657ce88-7cfa-457a-9aec-f4f827f20cac">Microsoft Word Viewer</a><span class="open-data-icon"><a href="http://www.retrovirology.com/about/access/#opendata"><img title="Open Data" alt="Open Data" src="opendata.gif"></a></span></p></div></div>
      <p style="line-height: 160%;">Understanding subtype-specific differences regarding coreceptor tropism is important
         for several reasons. First, these studies may help us understanding differences in
         HIV-1 pathogenesis. Several studies have indicated differences in relative pathogenicity
         between different subtypes, where subtype D appears to be more pathogenic compared
         to other subtypes <a name="d31808e2525"></a>[<a onclick="LoadInParent('#B59'); return false;" href="#B59">59</a>]. Moreover, it has been shown that HIV-1 subtype D has a preference for CXCR4 tropism
         early in infection, and a connection with the faster disease progression seen in this
         subtype has been suggested <a name="d31808e2529"></a>[<a onclick="LoadInParent('#B3'); return false;" href="#B3">3</a>]. Second, understanding HIV-1 subtype-specific differences regarding the ability to
         develop CXCR4-using populations may be of great importance for future treatment guidelines
         for coreceptor antagonists. Fätkenheuer <em>et al. </em>(2008) reported a striking difference in the appearance of X4 populations among patients
         with experienced treatment failure, after receiving either the CCR5-antagonist Maraviroc
         or a placebo treatment <a name="d31808e2536"></a>[<a onclick="LoadInParent('#B60'); return false;" href="#B60">60</a>]. Despite a short follow-up period of only 48-weeks, as many as 57% of patients receiving
         Maraviroc developed X4 viruses (76 of 133 patients), compared to only 6% (6 of 95
         patients) in the placebo group. This finding strengthens the concerns that CCR5 antagonists
         could increase the risk of HIV-1 populations to shift away from CCR5 to CXCR4 use,
         potentially leading to treatment failure and faster disease progression <a name="d31808e2540"></a><a name="d31808e2542"></a>[<a onclick="LoadInParent('#B61'); return false;" href="#B61">61</a>,<a onclick="LoadInParent('#B62'); return false;" href="#B62">62</a>]. Most of the participants in the above study were infected with HIV-1 subtype B,
         and more studies are needed to determine if the outgrowth of R5X4 or X4 populations
         will be as distinct also for other subtypes, such as subtype C.
      </p>
      <p style="line-height: 160%;">Based on the accumulated knowledge on subtype C infections, it is tempting to speculate
         that subtype C-infected patients in general may be more suitable for treatment with
         CCR5 antagonists than patients infected with other subtypes, at least in late-stage
         disease. Taken these thoughts further, different subtypes and CRFs may be more or
         less predisposed for the emergence of R5X4 or X4 populations, although it seems that
         CXCR4-using HIV-1 populations likely will arise in most HIV-1 non-subtype C infections
         (Fig. <a name="d31808e2547"></a><a onclick="popup('http://www.retrovirology.com/content/7/1/23/figure/F2','',800,470); return false;" href="http://www.retrovirology.com/content/7/1/23/figure/F2">2</a>). The comparative picture of subtype-dependent HIV-1 coreceptor tropism in late-stage
         disease underlines the importance of further studies on HIV-1 subtype-dependent coreceptor
         tropism, an issue of direct clinical importance for the development of treatment guidelines
         of the recently introduced coreceptor antagonists in HIV treatment regimens.
      </p>
   </div>
</section>
<section><a name="sec5"></a><h3><span class="arrow"></span>Conclusion</h3>
   <div class="collapsible-content">
      <p style="line-height: 160%;">In summary, we show that the TRT assay accurately determines the HIV-1 coreceptor
         tropism of subtype A and CRF02_AG. Using this assay we found a high amount of HIV-1
         CXCR4-using populations (79%) in our plasma samples and an increasing frequency of
         CXCR4 tropism in CRF02_AG-infected patients over time. The emergence of CXCR4 use
         may have implications for viral transmission, pathogenesis and disease progression.
         We also demonstrate that the combined criteria of the total number of charged amino
         acids and net charge of the V3 region is a more sensitive predictor of CXCR4 tropism
         for HIV-1 CRF02_AG compared to the analyzed genotypic rules and bioinformatic tools.
         Finally, we conducted an extensive literature analysis of 498 individuals which, to
         our knowledge, is the most extensive comparison of subtype-specific coreceptor tropism
         in late-stage disease. We found a generally high frequency of CXCR4 tropism among
         all major subtypes and CRFs, except for subtype C. These results demonstrate the need
         for further studies investigating subtype-specific emergence for CXCR4-tropism, this
         may be particularly important due to the introduction of CCR5-antagonists in HIV treatment
         regimens.
      </p>
   </div>
</section>
<section><a name="sec6"></a><h3><span class="arrow"></span>Competing interests</h3>
   <div class="collapsible-content">
      <p style="line-height: 160%;">The authors declare that they have no competing interests.</p>
   </div>
</section>
<section><a name="sec7"></a><h3><span class="arrow"></span>Authors' contributions</h3>
   <div class="collapsible-content">
      <p style="line-height: 160%;">JE designed the study, optimized the experimental protocols, performed RNA extraction
         and sequencing, analyzed and interpreted the data, performed the literature analysis,
         and wrote the manuscript. FM contributed in study design, clinically evaluated the
         patient data and participated in patient selection. WMA performed most of the cloning
         and colony-PCR, and participated in optimization of the cloning strategy. EV carried
         out cell assays, ELISA experiments, and participated in optimization of the TRT assay.
         AJB was medically and organizationally responsible for the clinical sites with biological
         samples of the study participants in the cohort. ZJdS was responsible for analyses
         of HIV serology at the laboratory in Guinea-Bissau. EMF participated in interpretation
         of the results, and contributed to the literature analysis. HN contributed in study
         design, and participated in patient selection. PM designed the study, participated
         in analyzing and in interpretation of the data, and helped to draft the manuscript.
         All authors read and approved the manuscript.
      </p>
   </div>
</section>
<section><a name="ack"></a><h3><span class="arrow"></span>Acknowledgements</h3>
   <div class="collapsible-content">
      <p style="line-height: 160%;">We thank Professor Jan Albert (Swedish Institute for Infectious Disease Control) for
         providing the isolates used in the evaluation of the TRT assay and SCIBLU Genomics
         (Swegene Centre for Integrative Biology at Lund University) for technical assistance.
         We also thank Professor Jan Albert, Professor Hanneke Schuitemaker, Dr. Gabriella
         Scarlatti and Dr. Åsa Björndal for providing information about clinical data and coreceptor
         tropism from previous studies. We would also like to acknowledge the anonymous reviewers
         for valuable comments on the manuscript.
      </p>
      <p style="line-height: 160%;">This work was supported by grants from the Swedish Research Council, the Swedish International
         Development Cooperation Agency/Department for Research Cooperation (SIDA/SAREC), the
         Crafoord Foundation, Lund, Sweden, the Royal Physiographic Society, Lund, Sweden,
         The Lars Hierta Memorial Foundation, Stockholm, Sweden, and Konsul Thure Carlsson
         Fund for Medical Research, Lund, Sweden. DNA sequencing was performed at the SWEGENE
         Center of Genomic Ecology, supported by the Crafoord foundation, through the SWEGENE
         consortium.
      </p>
   </div>
</section>
<section><a name="refs"></a><h3><span class="arrow"></span>References</h3>
   <div id="article-references" class="collapsible-content">
      <ol id="references">
         <li id="B1">
            <p><a name="B1"></a> Taylor BS,  Sobieszczyk ME,  McCutchan FE,  Hammer SM: <strong> The challenge of HIV-1 subtype diversity. </strong></p><p><em>N Engl J Med</em> 2008,  <strong>358</strong><strong>:</strong>1590-1602.  <a target="_top" href="http://www.retrovirology.com/pubmed/18403767">PubMed&nbsp;Abstract</a> |  <a target="_top" href="http://dx.doi.org/10.1056/NEJMra0706737">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=18403767">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B1"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B2">
            <p><a name="B2"></a> Berger EA,  Doms RW,  Fenyo EM,  Korber BT,  Littman DR,  Moore JP,  Sattentau QJ,  Schuitemaker H,  Sodroski J,  Weiss RA: <strong> A newclassification for HIV-1. </strong></p><p><em>Nature</em> 1998,  <strong>391</strong><strong>:</strong>240.  <a target="_top" href="http://www.retrovirology.com/pubmed/9440686">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9440686">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B2"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B3">
            <p><a name="B3"></a> Kaleebu P,  Nankya IL,  Yirrell DL,  Shafer LA,  Kyosiimire-Lugemwa J,  Lule DB,  Morgan D,  Beddows S,  Weber J,  Whitworth JA: <strong> Relation between chemokine receptor use, disease stage, and HIV-1 subtypes A and D:
                  results from a rural Ugandan cohort. </strong></p><p><em>J Acquir Immune Defic Syndr</em> 2007,  <strong>45</strong><strong>:</strong>28-33.  <a target="_top" href="http://www.retrovirology.com/pubmed/17310935">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17310935">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B3"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B4">
            <p><a name="B4"></a> Asjo B,  Morfeldt-Manson L,  Albert J,  Biberfeld G,  Karlsson A,  Lidman K,  Fenyo EM: <strong> Replicative capacity of human immunodeficiency virus from patients with varying severity
                  of HIV infection. </strong></p><p><em>Lancet</em> 1986,  <strong>2</strong><strong>:</strong>660-662.  <a target="_top" href="http://www.retrovirology.com/pubmed/2429124">PubMed&nbsp;Abstract</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B4"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B5">
            <p><a name="B5"></a> Karlsson A,  Parsmyr K,  Sandstrom E,  Fenyo EM,  Albert J: <strong> MT-2 cell tropism as prognostic marker for disease progression in human immunodeficiency
                  virus type 1 infection. </strong></p><p><em>J Clin Microbiol</em> 1994,  <strong>32</strong><strong>:</strong>364-370.  <a target="_top" href="http://www.retrovirology.com/pubmed/7908672">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=7908672">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B5"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B6">
            <p><a name="B6"></a> Koot M,  Keet IP,  Vos AH,  de Goede RE,  Roos MT,  Coutinho RA,  Miedema F,  Schellekens PT,  Tersmette M: <strong> Prognostic value of HIV-1 syncytium-inducing phenotype for rate of CD4+ cell depletion
                  and progression to AIDS. </strong></p><p><em>Ann Intern Med</em> 1993,  <strong>118</strong><strong>:</strong>681-688.  <a target="_top" href="http://www.retrovirology.com/pubmed/8096374">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=8096374">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B6"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B7">
            <p><a name="B7"></a> Bjorndal A,  Deng H,  Jansson M,  Fiore JR,  Colognesi C,  Karlsson A,  Albert J,  Scarlatti G,  Littman DR,  Fenyo EM: <strong> Coreceptor usage of primary human immunodeficiency virus type 1 isolates varies according
                  to biological phenotype. </strong></p><p><em>J Virol</em> 1997,  <strong>71</strong><strong>:</strong>7478-7487.  <a target="_top" href="http://www.retrovirology.com/pubmed/9311827">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=9311827">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B7"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B8">
            <p><a name="B8"></a> Abebe A,  Demissie D,  Goudsmit J,  Brouwer M,  Kuiken CL,  Pollakis G,  Schuitemaker H,  Fontanet AL,  Rinke de Wit TF: <strong> HIV-1 subtype C syncytium- and non-syncytium-inducing phenotypes and coreceptor usage
                  among Ethiopian patients with AIDS. </strong></p><p><em>Aids</em> 1999,  <strong>13</strong><strong>:</strong>1305-1311.  <a target="_top" href="http://www.retrovirology.com/pubmed/10449282">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10449282">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B8"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B9">
            <p><a name="B9"></a> Bjorndal A,  Sonnerborg A,  Tscherning C,  Albert J,  Fenyo EM: <strong> Phenotypic characteristics of human immunodeficiency virus type 1 subtype C isolates
                  of Ethiopian AIDS patients. </strong></p><p><em>AIDS Res Hum Retroviruses</em> 1999,  <strong>15</strong><strong>:</strong>647-653.  <a target="_top" href="http://www.retrovirology.com/pubmed/10331443">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10331443">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B9"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B10">
            <p><a name="B10"></a> Ping LH,  Nelson JA,  Hoffman IF,  Schock J,  Lamers SL,  Goodman M,  Vernazza P,  Kazembe P,  Maida M,  Zimba D,  Goodenow MM,  Eron JJ Jr,  Fiscus SA,  Cohen MS,  Swanstrom R: <strong> Characterization of V3 sequence heterogeneity in subtype C human immunodeficiency
                  virus type 1 isolates from Malawi: underrepresentation of X4 variants. </strong></p><p><em>J Virol</em> 1999,  <strong>73</strong><strong>:</strong>6271-6281.  <a target="_top" href="http://www.retrovirology.com/pubmed/10400718">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=10400718">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B10"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B11">
            <p><a name="B11"></a> Connell BJ,  Michler K,  Capovilla A,  Venter WD,  Stevens WS,  Papathanasopoulos MA: <strong> Emergence of X4 usage among HIV-1 subtype C: evidence for an evolving epidemic in
                  South Africa. </strong></p><p><em>Aids</em> 2008,  <strong>22</strong><strong>:</strong>896-899.  <a target="_top" href="http://www.retrovirology.com/pubmed/18427209">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18427209">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B11"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B12">
            <p><a name="B12"></a> Yu XF,  Wang Z,  Beyrer C,  Celentano DD,  Khamboonruang C,  Allen E,  Nelson K: <strong> Phenotypic and genotypic characteristics of human immunodeficiency virus type 1 from
                  patients with AIDS in northern Thailand. </strong></p><p><em>J Virol</em> 1995,  <strong>69</strong><strong>:</strong>4649-4655.  <a target="_top" href="http://www.retrovirology.com/pubmed/7609029">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=7609029">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B12"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B13">
            <p><a name="B13"></a> Vergne L,  Bourgeois A,  Mpoudi-Ngole E,  Mougnutou R,  Mbuagbaw J,  Liegeois F,  Laurent C,  Butel C,  Zekeng L,  Delaporte E,  Peeters M: <strong> Biological and genetic characteristics of HIV infections in Cameroon reveals dual
                  group M and O infections and a correlation between SI-inducing phenotype of the predominant
                  CRF02_AG variant and disease stage. </strong></p><p><em>Virology</em> 2003,  <strong>310</strong><strong>:</strong>254-266.  <a target="_top" href="http://www.retrovirology.com/pubmed/12781713">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12781713">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B13"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B14">
            <p><a name="B14"></a> Richman DD,  Bozzette SA: <strong> The impact of the syncytium-inducing phenotype of human immunodeficiency virus on
                  disease progression. </strong></p><p><em>J Infect Dis</em> 1994,  <strong>169</strong><strong>:</strong>968-974.  <a target="_top" href="http://www.retrovirology.com/pubmed/7909549">PubMed&nbsp;Abstract</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B14"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B15">
            <p><a name="B15"></a> Mild M,  Esbjornsson J,  Fenyo EM,  Medstrand P: <strong> Frequent intrapatient recombination between human immunodeficiency virus type 1 R5
                  and X4 envelopes: implications for coreceptor switch. </strong></p><p><em>J Virol</em> 2007,  <strong>81</strong><strong>:</strong>3369-3376.  <a target="_top" href="http://www.retrovirology.com/pubmed/17251288">PubMed&nbsp;Abstract</a> |  <a target="_top" href="http://dx.doi.org/10.1128/JVI.01295-06">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17251288">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B15"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B16">
            <p><a name="B16"></a> Bratt G,  Leandersson AC,  Albert J,  Sandstrom E,  Wahren B: <strong> MT-2 tropism and CCR-5 genotype strongly influence disease progression in HIV-1-infected
                  individuals. </strong></p><p><em>Aids</em> 1998,  <strong>12</strong><strong>:</strong>729-736.  <a target="_top" href="http://www.retrovirology.com/pubmed/9619804">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9619804">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B16"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B17">
            <p><a name="B17"></a> Peeters M,  Vincent R,  Perret JL,  Lasky M,  Patrel D,  Liegeois F,  Courgnaud V,  Seng R,  Matton T,  Molinier S,  Delaporte E: <strong> Evidence for differences in MT2 cell tropism according to genetic subtypes of HIV-1:
                  syncytium-inducing variants seem rare among subtype C HIV-1 viruses. </strong></p><p><em>J Acquir Immune Defic Syndr Hum Retrovirol</em> 1999,  <strong>20</strong><strong>:</strong>115-121.  <a target="_top" href="http://www.retrovirology.com/pubmed/10048897">PubMed&nbsp;Abstract</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B17"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B18">
            <p><a name="B18"></a> Spijkerman IJ,  Koot M,  Prins M,  Keet IP,  Hoek AJ,  Miedema F,  Coutinho RA: <strong> Lower prevalence and incidence of HIV-1 syncytium-inducing phenotype among injecting
                  drug users compared with homosexual men. </strong></p><p><em>Aids</em> 1995,  <strong>9</strong><strong>:</strong>1085-1092.  <a target="_top" href="http://www.retrovirology.com/pubmed/8527083">PubMed&nbsp;Abstract</a> |  <a target="_top" href="http://dx.doi.org/10.1097/00002030-199509000-00016">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B18"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B19">
            <p><a name="B19"></a> Cilliers T,  Nhlapo J,  Coetzer M,  Orlovic D,  Ketas T,  Olson WC,  Moore JP,  Trkola A,  Morris L: <strong> The CCR5 and CXCR4 coreceptors are both used by human immunodeficiency virus type
                  1 primary isolates from subtype C. </strong></p><p><em>J Virol</em> 2003,  <strong>77</strong><strong>:</strong>4449-4456.  <a target="_top" href="http://www.retrovirology.com/pubmed/12634405">PubMed&nbsp;Abstract</a> |  <a target="_top" href="http://dx.doi.org/10.1128/JVI.77.7.4449-4456.2003">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=12634405">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B19"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B20">
            <p><a name="B20"></a> Huang W,  Eshleman SH,  Toma J,  Fransen S,  Stawiski E,  Paxinos EE,  Whitcomb JM,  Young AM,  Donnell D,  Mmiro F,  Musoke P,  Guay LA,  Jackson JB,  Parkin NT,  Petropoulos CJ: <strong> Coreceptor tropism in human immunodeficiency virus type 1 subtype D: high prevalence
                  of CXCR4 tropism and heterogeneous composition of viral populations. </strong></p><p><em>J Virol</em> 2007,  <strong>81</strong><strong>:</strong>7885-7893.  <a target="_top" href="http://www.retrovirology.com/pubmed/17507467">PubMed&nbsp;Abstract</a> |  <a target="_top" href="http://dx.doi.org/10.1128/JVI.00218-07">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17507467">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B20"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B21">
            <p><a name="B21"></a> Tscherning C,  Alaeus A,  Fredriksson R,  Bjorndal A,  Deng H,  Littman DR,  Fenyo EM,  Albert J: <strong> Differences in chemokine coreceptor usage between genetic subtypes of HIV-1. </strong></p><p><em>Virology</em> 1998,  <strong>241</strong><strong>:</strong>181-188.  <a target="_top" href="http://www.retrovirology.com/pubmed/9499793">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9499793">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B21"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B22">
            <p><a name="B22"></a> Zhang L,  Huang Y,  He T,  Cao Y,  Ho DD: <strong> HIV-1 subtype and second-receptor use. </strong></p><p><em>Nature</em> 1996,  <strong>383</strong><strong>:</strong>768.  <a target="_top" href="http://www.retrovirology.com/pubmed/8892998">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=8892998">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B22"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B23">
            <p><a name="B23"></a> Hartley O,  Klasse PJ,  Sattentau QJ,  Moore JP: <strong> V3: HIV'sswitch-hitter. </strong></p><p><em>AIDS Res Hum Retroviruses</em> 2005,  <strong>21</strong><strong>:</strong>171-189.  <a target="_top" href="http://www.retrovirology.com/pubmed/15725757">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15725757">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B23"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B24">
            <p><a name="B24"></a> Pastore C,  Nedellec R,  Ramos A,  Pontow S,  Ratner L,  Mosier DE: <strong> Human immunodeficiency virus type 1 coreceptor switching: V1/V2 gain-of-fitness mutations
                  compensate for V3 loss-of-fitness mutations. </strong></p><p><em>J Virol</em> 2006,  <strong>80</strong><strong>:</strong>750-758.  <a target="_top" href="http://www.retrovirology.com/pubmed/16378977">PubMed&nbsp;Abstract</a> |  <a target="_top" href="http://dx.doi.org/10.1128/JVI.80.2.750-758.2006">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=16378977">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B24"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B25">
            <p><a name="B25"></a> Sax PE: <strong> FDA approval: maraviroc. </strong></p><p><em>AIDS Clin Care</em> 2007,  <strong>19</strong><strong>:</strong>75.  <a target="_top" href="http://www.retrovirology.com/pubmed/18398993">PubMed&nbsp;Abstract</a> |  <a target="_top" href="http://dx.doi.org/10.1080/09540120600746899">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B25"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B26">
            <p><a name="B26"></a> Weber J,  Piontkivska H,  Quinones-Mateu ME: <strong> HIV type 1 tropism and inhibitors of viral entry: clinical implications. </strong></p><p><em>AIDS Rev</em> 2006,  <strong>8</strong><strong>:</strong>60-77.  <a target="_top" href="http://www.retrovirology.com/pubmed/16848274">PubMed&nbsp;Abstract</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B26"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B27">
            <p><a name="B27"></a> Westby M,  Smith-Burchnell C,  Mori J,  Lewis M,  Mosley M,  Stockdale M,  Dorr P,  Ciaramella G,  Perros M: <strong> Reduced maximal inhibition in phenotypic susceptibility assays indicates that viral
                  strains resistant to the CCR5 antagonist maraviroc utilize inhibitor-bound receptor
                  for entry. </strong></p><p><em>J Virol</em> 2007,  <strong>81</strong><strong>:</strong>2359-2371.  <a target="_top" href="http://www.retrovirology.com/pubmed/17182681">PubMed&nbsp;Abstract</a> |  <a target="_top" href="http://dx.doi.org/10.1128/JVI.02006-06">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17182681">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B27"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B28">
            <p><a name="B28"></a> Wolinsky SM,  Veazey RS,  Kunstman KJ,  Klasse PJ,  Dufour J,  Marozsan AJ,  Springer MS,  Moore JP: <strong> Effect of a CCR5 inhibitor on viral loads in macaques dual-infected with R5 and X4
                  primate immunodeficiency viruses. </strong></p><p><em>Virology</em> 2004,  <strong>328</strong><strong>:</strong>19-29.  <a target="_top" href="http://www.retrovirology.com/pubmed/15380354">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15380354">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B28"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B29">
            <p><a name="B29"></a> Trouplin V,  Salvatori F,  Cappello F,  Obry V,  Brelot A,  Heveker N,  Alizon M,  Scarlatti G,  Clavel F,  Mammano F: <strong> Determination of coreceptor usage of human immunodeficiency virus type 1 from patient
                  plasma samples by using a recombinant phenotypic assay. </strong></p><p><em>J Virol</em> 2001,  <strong>75</strong><strong>:</strong>251-259.  <a target="_top" href="http://www.retrovirology.com/pubmed/11119595">PubMed&nbsp;Abstract</a> |  <a target="_top" href="http://dx.doi.org/10.1128/JVI.75.1.251-259.2001">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=11119595">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B29"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B30">
            <p><a name="B30"></a> Whitcomb JM,  Huang W,  Fransen S,  Limoli K,  Toma J,  Wrin T,  Chappey C,  Kiss LD,  Paxinos EE,  Petropoulos CJ: <strong> Development and characterization of a novel single-cycle recombinant-virus assay to
                  determine human immunodeficiency virus type 1 coreceptor tropism. </strong></p><p><em>Antimicrob Agents Chemother</em> 2007,  <strong>51</strong><strong>:</strong>566-575.  <a target="_top" href="http://www.retrovirology.com/pubmed/17116663">PubMed&nbsp;Abstract</a> |  <a target="_top" href="http://dx.doi.org/10.1128/AAC.00853-06">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17116663">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B30"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B31">
            <p><a name="B31"></a> Garrido C,  Roulet V,  Chueca N,  Poveda E,  Aguilera A,  Skrabal K,  Zahonero N,  Carlos S,  Garcia F,  Faudon JL,  Soriano V,  de Mendoza C: <strong> Evaluation of eight different bioinformatics tools to predict viral tropism in different
                  human immunodeficiency virus type 1 subtypes. </strong></p><p><em>J Clin Microbiol</em> 2008,  <strong>46</strong><strong>:</strong>887-891.  <a target="_top" href="http://www.retrovirology.com/pubmed/18199789">PubMed&nbsp;Abstract</a> |  <a target="_top" href="http://dx.doi.org/10.1128/JCM.01611-07">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=18199789">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B31"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B32">
            <p><a name="B32"></a> Jensen MA,  Li FS,  van 't Wout AB,  Nickle DC,  Shriner D,  He HX,  McLaughlin S,  Shankarappa R,  Margolick JB,  Mullins JI: <strong> Improved coreceptor usage prediction and genotypic monitoring of R5-to-X4 transition
                  by motif analysis of human immunodeficiency virus type 1 env V3 loopsequences. </strong></p><p><em>J Virol</em> 2003,  <strong>77</strong><strong>:</strong>13376-13388.  <a target="_top" href="http://www.retrovirology.com/pubmed/14645592">PubMed&nbsp;Abstract</a> |  <a target="_top" href="http://dx.doi.org/10.1128/JVI.77.24.13376-13388.2003">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=14645592">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B32"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B33">
            <p><a name="B33"></a> Andersson S,  Norrgren H,  da Silva Z,  Biague A,  Bamba S,  Kwok S,  Christopherson C,  Biberfeld G,  Albert J: <strong> Plasma viral load in HIV-1 and HIV-2 singly and dually infected individuals in Guinea-Bissau,
                  West Africa: significantly lower plasma virus set point in HIV-2 infection than in
                  HIV-1 infection. </strong></p><p><em>Arch Intern Med</em> 2000,  <strong>160</strong><strong>:</strong>3286-3293.  <a target="_top" href="http://www.retrovirology.com/pubmed/11088091">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11088091">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B33"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B34">
            <p><a name="B34"></a> Norrgren H,  Andersson S,  Naucler A,  Dias F,  Johansson I,  Biberfeld G: <strong> HIV-1, HIV-2, HTLV-I/II and Treponema pallidum infections: incidence, prevalence,
                  and HIV-2-associated mortality in an occupational cohort in Guinea-Bissau. </strong></p><p><em>J Acquir Immune Defic Syndr Hum Retrovirol</em> 1995,  <strong>9</strong><strong>:</strong>422-428.  <a target="_top" href="http://www.retrovirology.com/pubmed/7600111">PubMed&nbsp;Abstract</a> |  <a target="_top" href="http://dx.doi.org/10.1097/00042560-199508000-00014">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B34"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B35">
            <p><a name="B35"></a> Mansson F,  Biague A,  da Silva ZJ,  Dias F,  Nilsson LA,  Andersson S,  Fenyo EM,  Norrgren H: <strong> Prevalence and incidence of HIV-1 and HIV-2 before, during and after a civil war in
                  an occupational cohort in Guinea-Bissau, West Africa. </strong></p><p><em>Aids</em> 2009,  <strong>23</strong><strong>:</strong>1575-1582.  <a target="_top" href="http://www.retrovirology.com/pubmed/19521234">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19521234">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B35"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B36">
            <p><a name="B36"></a><strong> Guidelines for national human immunodeficiency virus case surveillance, including
                  monitoring for human immunodeficiency virus infection and acquired immunodeficiency
                  syndrome. Centers for Disease Control andPrevention </strong></p><p><em>MMWR Recomm Rep</em> 1999,  <strong>48</strong><strong>:</strong>1-27. 
               </p><p>29-31</p> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B36"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a><p></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B37">
            <p><a name="B37"></a> CB H: <strong> WHO Staging System for HIV Infection and Disease in Adolescents and Adults. </strong>In <em>Book WHO Staging System for HIV Infection and Disease in Adolescents and Adults (Editor
                  ed.^eds.)</em>.  City: UCSF Center for HIV Information;  2006. <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B37"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B38">
            <p><a name="B38"></a> Leitner T,  Korovina G,  Marquina S,  Smolskaya T,  Albert J: <strong> Molecular epidemiology and MT-2 cell tropism of Russian HIV type 1 variant. </strong></p><p><em>AIDS Res Hum Retroviruses</em> 1996,  <strong>12</strong><strong>:</strong>1595-1603.  <a target="_top" href="http://www.retrovirology.com/pubmed/8947294">PubMed&nbsp;Abstract</a> |  <a target="_top" href="http://dx.doi.org/10.1089/aid.1996.12.1595">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B38"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B39">
            <p><a name="B39"></a><strong> Los Alamos Sequence Database </strong>[<a target="_top" href="http://www.hiv.lanl.gov/">http://www.hiv.lanl.gov/</a>] <a href="http://www.webcitation.org/query.php?url=http://www.hiv.lanl.gov/&amp;refdoi=10.1186/1742-4690-7-23" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p><p></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B40">
            <p><a name="B40"></a> Tamura K,  Dudley J,  Nei M,  Kumar S: <strong> MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0. </strong></p><p><em>Mol Biol Evol</em> 2007,  <strong>24</strong><strong>:</strong>1596-1599.  <a target="_top" href="http://www.retrovirology.com/pubmed/17488738">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17488738">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B40"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B41">
            <p><a name="B41"></a> Thompson JD,  Higgins DG,  Gibson TJ: <strong> CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through
                  sequence weighting, position-specific gap penalties and weight matrix choice. </strong></p><p><em>Nucleic Acids Res</em> 1994,  <strong>22</strong><strong>:</strong>4673-4680.  <a target="_top" href="http://www.retrovirology.com/pubmed/7984417">PubMed&nbsp;Abstract</a> |  <a target="_top" href="http://dx.doi.org/10.1093/nar/22.22.4673">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=7984417">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B41"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B42">
            <p><a name="B42"></a> Posada D,  Crandall KA: <strong> MODELTEST: testing the model ofDNA substitution. </strong></p><p><em>Bioinformatics</em> 1998,  <strong>14</strong><strong>:</strong>817-818.  <a target="_top" href="http://www.retrovirology.com/pubmed/9918953">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9918953">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B42"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B43">
            <p><a name="B43"></a> Felsenstein J: <strong> Confidence limits on phylogenies: An approach using the bootstrap. </strong></p><p><em>Evolution</em> 1985,  <strong>39</strong><strong>:</strong>783-791.   <a target="_top" href="http://dx.doi.org/10.2307/2408678">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B43"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B44">
            <p><a name="B44"></a> Nei MKS: <em>Molecular evolution and phylogenetics</em>.  Oxford, Oxford University Press;  2000. <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B44"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B45">
            <p><a name="B45"></a> Dopazo J: <strong> Estimating errors and confidence intervals for branch lengths in phylogenetic trees
                  by a bootstrap approach. </strong></p><p><em>J Mol Evol</em> 1994,  <strong>38</strong><strong>:</strong>300-304.  <a target="_top" href="http://www.retrovirology.com/pubmed/8006997">PubMed&nbsp;Abstract</a> |  <a target="_top" href="http://dx.doi.org/10.1007/BF00176092">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B45"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B46">
            <p><a name="B46"></a> Deng H,  Liu R,  Ellmeier W,  Choe S,  Unutmaz D,  Burkhart M,  Di Marzio P,  Marmon S,  Sutton RE,  Hill CM,  Davis CB,  Peiper SC,  Schall TJ,  Littman DR,  Landau NR: <strong> Identification of a major co-receptor for primary isolates of HIV-1. </strong></p><p><em>Nature</em> 1996,  <strong>381</strong><strong>:</strong>661-666.  <a target="_top" href="http://www.retrovirology.com/pubmed/8649511">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=8649511">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B46"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B47">
            <p><a name="B47"></a> Mild M,  Bjorndal A,  Medstrand P,  Fenyo EM: <strong> Isolation of human immunodeficiency virus-type 1 (HIV-1) clones with biological and
                  molecular properties of the primary isolate. </strong></p><p><em>Virology</em> 2006,  <strong>350</strong><strong>:</strong>58-66.  <a target="_top" href="http://www.retrovirology.com/pubmed/16563458">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16563458">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B47"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B48">
            <p><a name="B48"></a> Meloni ST,  Kim B,  Sankale JL,  Hamel DJ,  Tovanabutra S,  Mboup S,  McCutchan FE,  Kanki PJ: <strong> Distinct human immunodeficiency virus type 1 subtype A virus circulating in West Africa:
                  sub-subtype A3. </strong></p><p><em>J Virol</em> 2004,  <strong>78</strong><strong>:</strong>12438-12445.  <a target="_top" href="http://www.retrovirology.com/pubmed/15507630">PubMed&nbsp;Abstract</a> |  <a target="_top" href="http://dx.doi.org/10.1128/JVI.78.22.12438-12445.2004">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=15507630">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B48"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B49">
            <p><a name="B49"></a> Raymond S,  Delobel P,  Mavigner M,  Cazabat M,  Souyris C,  Sandres-Saune K,  Cuzin L,  Marchou B,  Massip P,  Izopet J: <strong> Correlation between genotypic predictions based on V3 sequences and phenotypic determination
                  of HIV-1 tropism. </strong></p><p><em>Aids</em> 2008,  <strong>22</strong><strong>:</strong>F11-16.  <a target="_top" href="http://www.retrovirology.com/pubmed/18753930">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18753930">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B49"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B50">
            <p><a name="B50"></a> Skrabal K,  Low AJ,  Dong W,  Sing T,  Cheung PK,  Mammano F,  Harrigan PR: <strong> Determining human immunodeficiency virus coreceptor use in a clinical setting: degree
                  of correlation between two phenotypic assays and a bioinformatic model. </strong></p><p><em>J Clin Microbiol</em> 2007,  <strong>45</strong><strong>:</strong>279-284.  <a target="_top" href="http://www.retrovirology.com/pubmed/17122004">PubMed&nbsp;Abstract</a> |  <a target="_top" href="http://dx.doi.org/10.1128/JCM.01118-06">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17122004">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B50"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B51">
            <p><a name="B51"></a> Resch W,  Hoffman N,  Swanstrom R: <strong> Improved success of phenotype prediction of the human immunodeficiency virus type
                  1 from envelope variable loop 3 sequence using neural networks. </strong></p><p><em>Virology</em> 2001,  <strong>288</strong><strong>:</strong>51-62.  <a target="_top" href="http://www.retrovirology.com/pubmed/11543657">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11543657">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B51"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B52">
            <p><a name="B52"></a> Raymond S,  Delobel P,  Mavigner M,  Cazabat M,  Souyris C,  Encinas S,  Sandres-Saune K,  Pasquier C,  Marchou B,  Massip P,  Izopet J: <strong> Genotypic Prediction of HIV-1 CRF02-AG Tropism. </strong></p><p><em>J Clin Microbiol</em> 2009,  <strong>47</strong>(7)<strong>:</strong>2292-4.  <a target="_top" href="http://www.retrovirology.com/pubmed/19439544">PubMed&nbsp;Abstract</a> |  <a target="_top" href="http://dx.doi.org/10.1128/JCM.02439-08">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19439544">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B52"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B53">
            <p><a name="B53"></a> Sing T,  Low AJ,  Beerenwinkel N,  Sander O,  Cheung PK,  Domingues FS,  Buch J,  Daumer M,  Kaiser R,  Lengauer T,  Harrigan PR: <strong> Predicting HIV coreceptor usage on the basis of genetic and clinical covariates. </strong></p><p><em>Antivir Ther</em> 2007,  <strong>12</strong><strong>:</strong>1097-1106.  <a target="_top" href="http://www.retrovirology.com/pubmed/18018768">PubMed&nbsp;Abstract</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B53"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B54">
            <p><a name="B54"></a> Andersson S,  Norrgren H,  Dias F,  Biberfeld G,  Albert J: <strong> Molecular characterization of human immunodeficiency virus (HIV)-1 and -2 in individuals
                  from guinea-bissau with single or dual infections: predominance of a distinct HIV-1
                  subtype A/G recombinant in West Africa. </strong></p><p><em>Virology</em> 1999,  <strong>262</strong><strong>:</strong>312-320.  <a target="_top" href="http://www.retrovirology.com/pubmed/10502510">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10502510">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B54"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B55">
            <p><a name="B55"></a> Hamel DJ,  Sankale JL,  Eisen G,  Meloni ST,  Mullins C,  Gueye-Ndiaye A,  Mboup S,  Kanki PJ: <strong> Twenty years of prospective molecular epidemiology in Senegal: changes in HIV diversity. </strong></p><p><em>AIDS Res Hum Retroviruses</em> 2007,  <strong>23</strong><strong>:</strong>1189-1196.  <a target="_top" href="http://www.retrovirology.com/pubmed/17961103">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17961103">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B55"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B56">
            <p><a name="B56"></a> Meloni ST,  Sankale JL,  Hamel DJ,  Eisen G,  Gueye-Ndiaye A,  Mboup S,  Kanki PJ: <strong> Molecular epidemiology of human immunodeficiency virus type 1 sub-subtype A3 in Senegal
                  from 1988 to 2001. </strong></p><p><em>J Virol</em> 2004,  <strong>78</strong><strong>:</strong>12455-12461.  <a target="_top" href="http://www.retrovirology.com/pubmed/15507632">PubMed&nbsp;Abstract</a> |  <a target="_top" href="http://dx.doi.org/10.1128/JVI.78.22.12455-12461.2004">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=15507632">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B56"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B57">
            <p><a name="B57"></a> Tebit DM,  Nankya I,  Arts EJ,  Gao Y: <strong> HIV diversity, recombination and disease progression: how does fitness "fit" into
                  the puzzle? </strong></p><p><em>AIDS Rev</em> 2007,  <strong>9</strong><strong>:</strong>75-87.  <a target="_top" href="http://www.retrovirology.com/pubmed/17694675">PubMed&nbsp;Abstract</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B57"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B58">
            <p><a name="B58"></a> Shankarappa R,  Margolick JB,  Gange SJ,  Rodrigo AG,  Upchurch D,  Farzadegan H,  Gupta P,  Rinaldo CR,  Learn GH,  He X,  Huang XL,  Mullins JI: <strong> Consistent viral evolutionary changes associated with the progression of human immunodeficiency
                  virus type 1 infection. </strong></p><p><em>J Virol</em> 1999,  <strong>73</strong><strong>:</strong>10489-10502.  <a target="_top" href="http://www.retrovirology.com/pubmed/10559367">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=10559367">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B58"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B59">
            <p><a name="B59"></a> Kaleebu P,  French N,  Mahe C,  Yirrell D,  Watera C,  Lyagoba F,  Nakiyingi J,  Rutebemberwa A,  Morgan D,  Weber J,  Gilks C,  Whitworth J: <strong> Effect of human immunodeficiency virus (HIV) type 1 envelope subtypes A and D on disease
                  progression in a large cohort of HIV-1-positive persons in Uganda. </strong></p><p><em>J Infect Dis</em> 2002,  <strong>185</strong><strong>:</strong>1244-1250.  <a target="_top" href="http://www.retrovirology.com/pubmed/12001041">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12001041">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B59"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B60">
            <p><a name="B60"></a> Fatkenheuer G,  Nelson M,  Lazzarin A,  Konourina I,  Hoepelman AI,  Lampiris H,  Hirschel B,  Tebas P,  Raffi F,  Trottier B,  Bellos N,  Saag M,  Cooper DA,  Westby M,  Tawadrous M,  Sullivan JF,  Ridgway C,  Dunne MW,  Felstead S,  Mayer H,  Ryst E, MOTIVATE 1, MOTIVATE 2 Study Teams: <strong> Subgroup analyses of maraviroc in previously treated R5 HIV-1 infection. </strong></p><p><em>N Engl J Med</em> 2008,  <strong>359</strong><strong>:</strong>1442-1455.  <a target="_top" href="http://www.retrovirology.com/pubmed/18832245">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18832245">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B60"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B61">
            <p><a name="B61"></a> Dolin R: <strong> A new class of anti-HIV therapy and new challenges. </strong></p><p><em>N Engl J Med</em> 2008,  <strong>359</strong><strong>:</strong>1509-1511.  <a target="_top" href="http://www.retrovirology.com/pubmed/18832250">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18832250">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B61"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B62">
            <p><a name="B62"></a> Kilby JM,  Eron JJ: <strong> Novel therapies based on mechanisms of HIV-1 cell entry. </strong></p><p><em>N Engl J Med</em> 2003,  <strong>348</strong><strong>:</strong>2228-2238.  <a target="_top" href="http://www.retrovirology.com/pubmed/12773651">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12773651">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.retrovirology.com/sfx_links?ui=1742-4690-7-23&amp;bibl=B62"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
      </ol>
   </div>
</section>
</div>
    </div></div>
</div>


		</div>
</div>
    			
		<dl class="google-ad wide  advert-text">
	<dt style="display: block;" class="hide">
					<a class="banner-ad" href="http://www.biomedcentral.com/advertisers/digital_advertising"></a>
				
			</dt>
		<dd>

		<!-- OAS AD 'Bottom' begin -->
		<script type="text/javascript">

		OAS_AD('Bottom');

		</script>
		<!-- OAS AD 'Bottom' end -->
		</dd>

		</dl>

		<noscript>

			<dl class="google-ad  noscript">  
    <dt class="hide"><a href="http://www.biomedcentral.com/advertisers/digital_advertising">Advertisement</dt>
    
    <dd>
        <a href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/retrovirology.com/article/10.1186/1742/4690/7/23/15876034934@Bottom?">
            <img src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/retrovirology.com/article/10.1186/1742/4690/7/23/14976004755@Bottom?" alt="advert"/>
        </a>
    </dd>
			</dl>
		</noscript>
			    <hr class="hide">
	
    



 
    
    
                                                                                                                                                                                                                                
                                                        
    
            

            
        <script>window.bmcIsMobile = "classic";</script>
        
        
    
<div id="footer">

									<span class="views"><a href="http://www.retrovirology.com/content/7/1/23?fmt_view=mobile">Mobile view</a> | <strong>Desktop view</strong></span>
			
	<div class="content">
    <div class="rounded ">
<div class="wrap-inner ">
        	
    <ul class="desktop">
                                                    
				        <li class="noborder"><a href="http://www.biomedcentral.com/about/tandc">Terms and Conditions</a></li>
<li><a href="http://www.biomedcentral.com/about/privacy">Privacy statement</a></li>
<li><a href="http://www.biomedcentral.com/presscenter">Press</a></li>
	<li><a href="http://www.biomedcentral.com/advertisers">Information for advertisers</a></li>
<li><a href="http://www.biomedcentral.com/about/bmcjobs">Jobs at BMC</a></li>
<li><a href="http://www.retrovirology.com/support">Support</a></li>
<li><a href="http://www.retrovirology.com/about/contact">Contact us</a></li>


    
    	    </ul>
    <p id="copyright">© 2013 
	BioMed Central Ltd unless otherwise stated. Part of Springer Science+Business Media.</p>
        </div>
</div>

	</div>

</div>
<div class="springer">
    <a href="http://www.springer.com/" target="blank"><img style="float: right; margin: 0px;" src="Springer_Branding_footer_logo.png"></a>
</div>
					    

<script>

// getElementsByClassName Polyfill
var getElementsByClassName=function(e,t,n){if(document.getElementsByClassName){getElementsByClassName=function(e,t,n){n=n||document;var r=n.getElementsByClassName(e),i=t?new RegExp("\\b"+t+"\\b","i"):null,s=[],o;for(var u=0,a=r.length;u<a;u+=1){o=r[u];if(!i||i.test(o.nodeName)){s.push(o)}}return s}}else if(document.evaluate){getElementsByClassName=function(e,t,n){t=t||"*";n=n||document;var r=e.split(" "),i="",s="http://www.w3.org/1999/xhtml",o=document.documentElement.namespaceURI===s?s:null,u=[],a,f;for(var l=0,c=r.length;l<c;l+=1){i+="[contains(concat(' ', @class, ' '), ' "+r[l]+" ')]"}try{a=document.evaluate(".//"+t+i,n,o,0,null)}catch(h){a=document.evaluate(".//"+t+i,n,null,0,null)}while(f=a.iterateNext()){u.push(f)}return u}}else{getElementsByClassName=function(e,t,n){t=t||"*";n=n||document;var r=e.split(" "),i=[],s=t==="*"&&n.all?n.all:n.getElementsByTagName(t),o,u=[],a;for(var f=0,l=r.length;f<l;f+=1){i.push(new RegExp("(^|\\s)"+r[f]+"(\\s|$)"))}for(var c=0,h=s.length;c<h;c+=1){o=s[c];a=false;for(var p=0,d=i.length;p<d;p+=1){a=i[p].test(o.className);if(!a){break}}if(a){u.push(o)}}return u}}return getElementsByClassName(e,t,n)}
</script>



<script src="jquery.min.js"></script>

<script type="text/javascript">

if (typeof jQuery == 'undefined') { document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/jquery.min.js' type='text/javascript'%3E%3C/script%3E")); }


document.documentElement.id = "js";

var site = {
			abbreviation: "",
	        type: "BMC_NICHE",
        portalId: "9001",
                section: "browse",
                page: "fulltext",
    		
									homePortal: "biomedcentral.com",
		  

			siteShowAds: "true",
				showAds: "true",
				url: "retrovirology.com",
			portal: false,
	        adpage_url: "retrovirology.com/article/10.1186/1742/4690/7/23",
        id: "10056",
	name: "Retrovirology",
	plugins: {
		scrollable: (getElementsByClassName("scrollable").length > 0),
		flowplayer: ((document.getElementById("flowplayer") != null) || (getElementsByClassName("myPlayer").length > 0) || (document.getElementById("videoarticle") != null) || (getElementsByClassName("flowplayer").length > 0) || (document.getElementById("audio") != null) || (document.getElementById("elifeplayer") != null)),
		lightbox: ((document.getElementById("image-highlight") != null) || (getElementsByClassName("lightbox").length > 0)),
		thickbox: (getElementsByClassName("thickbox").length > 0),
		altmetric: ((document.getElementById("altmetric") != null) || (getElementsByClassName("altmetric").length > 0))
	}
	
};



if(typeof quoteSearchUri == "undefined") { quoteSearchUri=null; }
</script>



        <script type="text/javascript" src="behaviours-0.js"></script>







		<script>

	if (site.plugins.scrollable == true) { 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/jquery.scrollable-1.0.1.min.js' %3E%3C/script%3E")); 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/jquery.tools.min.js'%3E%3C/script%3E"));
	}

	if (site.plugins.flowplayer == true) { 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/flowplayer-3.2.6.min.js'%3E%3C/script%3E"));
	}

	if (site.plugins.lightbox == true) { 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/browse_imagehighlight.js'%3E%3C/script%3E"));
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/jquery.colorbox.js' %3E%3C/script%3E"));
	}

	if (site.plugins.thickbox == true) { 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/thickbox-compressed.js' %3E%3C/script%3E"));
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/thickbox-flashhidden-patch.js' %3E%3C/script%3E"));
	}
	</script>




				        <script type="text/javascript" src="mobile-0.js"></script>
	        <script type="text/javascript" src="jquery-ui.min.js"></script>
	        	        <script src="hammer.js"></script>
		



            <script type="text/javascript" src="jquery.fancybox-1.3.1.pack-0.js"></script>
            <script type="text/javascript" src="articles-0.js"></script>
		<div class=" fb_reset" id="fb-root"><div style="position: absolute; top: -10000px; height: 0px; width: 0px;"><div><iframe src="23_6.html" style="border: medium none;" tab-index="-1" title="Facebook Cross Domain Communication Frame" aria-hidden="true" id="fb_xdm_frame_http" allowtransparency="true" name="fb_xdm_frame_http" frameborder="0" scrolling="no"></iframe><iframe src="23_7.html" style="border: medium none;" tab-index="-1" title="Facebook Cross Domain Communication Frame" aria-hidden="true" id="fb_xdm_frame_https" allowtransparency="true" name="fb_xdm_frame_https" frameborder="0" scrolling="no"></iframe></div></div><div style="position: absolute; top: -10000px; height: 0px; width: 0px;"><div></div></div></div>










<!--[if lte IE 6]>
            <script type="text/javascript" src="/javascript/plugins/DD_belatedPNG_0.0.8a-0.js"></script>
    <script type="text/javascript">
        DD_belatedPNG.fix('.png_bg, ul.primary-nav li a, ul.primary-nav li a span, ul.secondary-nav li a, ul.secondary-nav li a span, ul li i, .plus-button, .facebook, .twitter, .fancy-bg, #fancybox-close');
    </script>
	                <link rel="stylesheet" type="text/css" href="/css/hacks/ie6-0.css"/>
    <![endif]-->

<!-- Claudia Whitcombe wanted this comment here -->

       <div class="hide">
      <dl class="google-ad" id="x96banner">
        <dt style="display: block;" class="hide">
          <a class="skyscraper-ad" href="http://www.biomedcentral.com/advertisers/digital_advertising">Advertisement</a>
       </dt>
        <dd>
          <!-- OAS AD 'x96' begin -->
          <script type="text/javascript">

            OAS_AD('x96');
          </script>
        </dd>
      </dl>
    </div>  
    



<div id="fancybox-tmp"></div><div id="fancybox-loading"><div></div></div><div id="fancybox-overlay"></div><div id="fancybox-wrap"><div id="fancybox-outer"><div class="fancy-bg" id="fancy-bg-n"></div><div class="fancy-bg" id="fancy-bg-ne"></div><div class="fancy-bg" id="fancy-bg-e"></div><div class="fancy-bg" id="fancy-bg-se"></div><div class="fancy-bg" id="fancy-bg-s"></div><div class="fancy-bg" id="fancy-bg-sw"></div><div class="fancy-bg" id="fancy-bg-w"></div><div class="fancy-bg" id="fancy-bg-nw"></div><div id="fancybox-inner"></div><a id="fancybox-close"></a><a href="javascript:;" id="fancybox-left"><span class="fancy-ico" id="fancybox-left-ico"></span></a><a href="javascript:;" id="fancybox-right"><span class="fancy-ico" id="fancybox-right-ico"></span></a></div></div></body>
</html>
