<!DOCTYPE html>
<html style="overflow: hidden;">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">


<meta name="wpd_version" content="0.2">
<meta name="wpd_baseurl" content="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science/article/pii/S0022283608013442?np=y">
<meta name="wpd_url" content="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science/article/pii/S0022283608013442?np=y">
<meta name="wpd_date" content="2013-04-24T10:16Z">
<script async="async" type="text/javascript">s_c_il[1].cdw=s_c_il[1].d.write;s_c_il[1].cdwb="";s_c_il[1].d.write=function(m){s_c_il[1].cdwb+=m};
s.channel="publications";
s.pageName="article";
if (typeof (SDM) != 'undefined') {
  if (typeof(SDM.pageType) != 'undefined' &&
      typeof(SDM.issn_or_isbn) != 'undefined' &&
      typeof(SDM.pm) != 'undefined' &&
      typeof(SDM.pm.pii) != 'undefined' &&
      typeof(SDM.pm.contentType) != 'undefined') {
      var ct=SDM.pm.contentType.toLowerCase();
      if (ct=="jl") {
        s.prop1="journal";
      } else if (ct=="bk") {
         s.prop1="book";
      } else if (ct=="rw") {
          s.prop1="refwork";
      } else if (ct=="bs") {
          s.prop1="bookseries";
      } else if (ct=="hb") {
           s.prop1="handbook";
      }
      s.eVar17=SDM.pageType.toLowerCase();
      s.eVar14=SDM.issn_or_isbn; 
      s.eVar28=SDM.issn_or_isbn;
      s.products=";"+SDM.pm.pii;
    }
}
s.events="prodView,event5";
s.prop2=s.prop1;
s.prop4="article";
s_c_il[1].d.write=s_c_il[1].cdw;s_c_il[1].crl(0,1)</script><script src="sciencedirect.js" async="async" type="text/javascript"></script>
<script src="204774041.js"></script>
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<script> var ts1 = new Date().getTime(); </script>
<!-- static const char articletmpl_sccsId[] = "%W%  %G% %U%  Retrieved: %H% %T%"; -->
<meta name="robots" content="NOARCHIVE">
<meta name="robots" content="noodp">
<meta name="robots" content="noydir">
<link rel="canonical" href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science/article/pii/S0022283608013442">


<link rel="shortcut icon" href="faviconSD.ico" type="image/x-icon">
<title>

ScienceDirect.com - 

    
        
    
    Journal of Molecular Biology
 - Structure and Function of HIV-1 Reverse Transcriptase: Molecular Mechanisms of Polymerization and Inhibition</title>






<script>

if( typeof(ts1) == "undefined" ){ ts1= new Date().getTime(); }
Array.prototype.rt = function(label,ts){this.push( "{lbl:"+label+",ts:"+(ts||ts==0?ts:new Date().getTime())+"}" );}
var prs = [];
prs.rt( "head", ts1 );
function loadPDF(){}
</script>


<!--[if lte IE 7]>
<script src="http://cdn.els-cdn.com/sd/js/json2_131R4.js.min"></script>
<![endif]-->

<link rel="stylesheet" type="text/css" href="S0022283608013442.css" media="all">
</head>
<body style="overflow: hidden;">


<div id="dic_bubble" class="selection_bubble fontSize13 noSelect" style="z-index: 9999; border: 1px solid rgb(74, 174, 222);"></div><script>prs.rt('body_start');</script>
<script>

var refLinkURL = "http://www.sciencedirect.com/science/relLink/ajaxReq?_eid=1-s2.0-S0022283608013442";
var updatedCitedBy = "http://www.sciencedirect.com/science/referenceResolution/ajaxRefResol?_updateCitedBy=";
var refResolvePath = "http://www.sciencedirect.com/science/referenceResolution/ajaxRefResol?_eid=1-s2.0-S0022283608013442&_refCnt=164&_docType=article";

// these variables handle the reference resolution. Please ignore the naming.
// the first variable myMap will hold an associative array of links by bibID
// the second variable myXabsCounts holds the scopus counts.
var myMap = []; // html of links for references.
var myXabsCounts = []; // scopus counts
</script>
<div style="width: 1606px; left: 155px; display: none;" class="js" id="header-area">
  <div id="header" class="outerWrapper">
<div style="background-color: rgb(255, 255, 255); position: relative;" class="outerWrapper">
<div id="masthead">

<!-- INSERTS THE TEMPLATE VARIABLES POPULATED FROM DB for SCIVERSE and PRODUCT labels -->

<div class="logoSciVerseBlk"> <a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/" <span="" class="logoSciVerseImg">&nbsp;</a> </div> <div class="logoScienceDirectBlk"><a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/" title="ScienceDirect -The world's leading full-text scientific database"><span class="logoScienceDirectImg">&nbsp;</span></a></div>

</div>
<div id="suites" class="clearfix" aria-label="SciVerse" role="navigation">
<ul class="suites"><li><a href="http://www.hub.sciverse.com/action/home" title="Hub - SciVerse's integrated search">Hub</a></li><li><a title="ScienceDirect - the world's leading full-text scientific database" class="selected">ScienceDirect</a></li><li><a href="http://www.scopus.com.libgate.library.nuigalway.ie/" title="Scopus - the largest abstract and citation database of research literature and quality web sources">Scopus</a></li><li><a href="http://www.applications.sciverse.com/action/userhome" title=" Applications - SciVerse Applications" class="last">Applications</a></li></ul>
</div>
</div>		

     <!-- if not logged in -->
	   

			

    <div id="login" class="clearfix">
    	<ul>
		<li>
    
      <a href="https://www-sciencedirect-com.libgate.library.nuigalway.ie/science?_ob=RegistrationURL&amp;_method=display&amp;_type=ip&amp;_returnURL=http%3A%2F%2Fwww.sciencedirect.com%2Fsciencenp%3Dy&amp;_acct=C000007922&amp;_version=1&amp;_userid=103680&amp;md5=a003226ccf19a695734972d3bd54db42" target="_top" title="Register a ScienceDirect username and password for using alerts and other personal features">Register</a>
    
  </li>
  <!--top right widget-->
  <li class="login " id="loginPlus"><a href="javascript:LoginBox.flipLogin('loginBox',null);" title="Login to ScienceDirect" class="novisit plus nojs" id="loginPlusScript">Login</a><noscript><a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=HomePageURL&_method=userHomePage&_fl=y&_acct=C000007922&_version=1&_urlVersion=0&_userid=103680&md5=0773bdb54cea8075a9f1176d3dba54dc" title="Open the Login Box" class="plus" id="loginPlusNoScript" >Login</a></noscript>
  </li>
           
           <li class="last"><a href="http://www.scival.com/" title="Go to SciVal Suite">Go to SciVal Suite</a></li> 
        </ul>
    </div>
    
	<div id="loginBox" style="width: 600px; display: none;">
		
        <div style="width: 285px; float: right; border-left: 1px solid rgb(155, 155, 155); padding-left: 10px; min-height: 180px;">
        
        <div class="sampletext">
        <div style="font-weight: bold; padding-bottom: 5px;">Login via your institution</div>
		
		
		<div><a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=FederationURL&amp;_method=display&amp;_type=f&amp;prevURL=http%3A%2F%2Fwww.sciencedirect.com%2Fscience%2Farticle%2Fpii%2FS0022283608013442%3Fnp%3Dy&amp;_acct=C000007922&amp;_version=1&amp;_userid=103680&amp;md5=f3508a614debf1e80eb4d8a0d8308a63" title="Go to Athens/ Institution login">Athens / Institution login</a></div>
		
        </div>
	
				
		
                  
                
            
            
        </div>
		
        <div class="padding" style="width: 285px;">
            <form name="sdLogin" method="POST" action="https://www-sciencedirect-com.libgate.library.nuigalway.ie/science/login" id="login-box-form" style="margin: 0px;">
            <b><span style="text-align: right; font-weight: bold;">Login using your SciVerse credentials</span></b>
			<input name="login" value="login" type="hidden">
            <div style="margin-top: 20px;">
            <label for="username"><span style="text-align: right;">Username:</span>
            <input id="username" name="userid" title="Please enter your username here" type="text"></label>
            </div>
            
            <div style="margin-top: 7px;">
            <label for="password"><span style="text-align: right;">Password:</span>
            <input id="password" name="password" maxlength="20" title="Please enter your password here" onkeypress="if(13==event.keyCode)this.form.submit();if (13==event.keyCode)return false;" type="password"></label>
            </div>
                   
                       
			
			<div class="remember">
			<input checked="" name="rememberid" id="rememberMe" value="Y" class="allCheck top2fix" title="Checking this option will log you in automatically next time you visit ScienceDirect" type="checkbox">
			<label for="rememberMe" title="Checking this option will log you in automatically next time you visit ScienceDirect">Remember me</label>
			</div>
            
			<input name="_ob" value="MiamiLogonURL" type="hidden">
			<input name="targetURL" value="http://www.sciencedirect.com/science/article/pii/S0022283608013442?np=y" type="hidden">
			<input name="returnURL" value="http://www.sciencedirect.com/science/article/pii/S0022283608013442?np=y" type="hidden">
            <div class="submit">
			
			<div style="float: left; padding-left: 0px;"><input class="button" name="arrow" alt="Login to ScienceDirect" title="Login to ScienceDirect" value="Login" type="Submit"></div>
        
			
			<div class="notRegistered"><span class="astPad">| </span><span><a href="https://www-sciencedirect-com.libgate.library.nuigalway.ie/science?_ob=RegistrationURL&amp;_method=display&amp;_type=ip&amp;_returnURL=http%3A%2F%2Fwww.sciencedirect.com%2Fsciencenp%3Dy&amp;_acct=C000007922&amp;_version=1&amp;_userid=103680&amp;md5=a003226ccf19a695734972d3bd54db42" title="" target="_top">Not Registered?</a></span></div>
			
			
			<div style="clear: left;"></div>
            </div>
			
            <div class="forgotpass">
			<a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=ReminderURL&amp;_method=display&amp;_acct=C000007922&amp;_version=1&amp;_userid=103680&amp;md5=d42879e2c3ee578ec4edb1badc8cf513" target="passwd" onclick="var remindWin;remindWin=window.open('','passwd','scrollbars=yes,resizable=yes,directories=no,toolbar=n,menubar=no,status=no,width=650,height=300');" title="Click here for help in recovering your ScienceDirect access">Forgotten username or password?</a>
            </div>
            </form>
        </div>
	</div>

	<!-- logged in -->
	     



<div id="navigationTop" class="clearfix" aria-label="ScienceDirect" role="navigation">
    <ul class="nav main">
        <li class="last">
            <a class=" canonHomeLink" href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/" method="userHomePage" btn="Y" zone="TopNavBar" origin="home" target="_top" name="Home" title="Home">Home</a>
        </li>
        <li>
            <!--<h1>-->
            <a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science/browse" type="all" auth="y" zone="TopNavBar" origin="article" target="_top" name="Publications" title="Publications">Publications</a>
            <!--</h1>-->
        </li>
        
        <li>
            <a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=MiamiSearchURL&amp;_method=requestForm&amp;_btn=Y&amp;_zone=TopNavBar&amp;_origin=article&amp;_acct=C000007922&amp;_version=1&amp;_urlVersion=1&amp;_userid=103680&amp;md5=6f00b17e70582bccfd79c0043e2d7e9d" target="_top" name="Search" title="Search">Search</a>
        </li>
        
        
        <li>
            <a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=UserSubscriptionURL&amp;_method=begin&amp;_btn=Y&amp;_zone=TopNavBar&amp;_origin=article&amp;_acct=C000007922&amp;_version=1&amp;_urlVersion=1&amp;_userid=103680&amp;md5=8246689b76c81456079cab859fc0613c" target="_top" name="account" title="My settings">My settings</a>
        </li>
        
        
        
        <li>
            <a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=MiamiSDIURL&amp;_method=listAlerts&amp;_btn=Y&amp;_zone=TopNavBar&amp;_origin=article&amp;_acct=C000007922&amp;_version=1&amp;_urlVersion=0&amp;_userid=103680&amp;md5=7614e1367f59f6ad24497b7cb1295fc7" target="_top" name="alert" title="Alerts">My alerts</a>
        </li>
        
        
    </ul>
    <ul class="nav misc">
        <li><span id="ChatHeader"></span></li>          
        <li class="last" style="margin-right: 247px;"><a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=HelpURL&amp;_file=browse_open_doc.htm&amp;_btn=Y&amp;_zone=TopNavBar&amp;_origin=article&amp;_acct=C000007922&amp;_version=1&amp;_urlVersion=0&amp;_userid=103680&amp;md5=3078a5cf677d428f5de42e4c8b948657" target="sdhelp" title="Help (Opens new window)" onclick="var helpWin; helpWin=window.open('http://www.sciencedirect.com/science?_ob=HelpURL&amp;_file=browse_open_doc.htm&amp;_btn=Y&amp;_zone=TopNavBar&amp;_origin=article&amp;_acct=C000007922&amp;_version=1&amp;_urlVersion=0&amp;_userid=103680&amp;md5=3078a5cf677d428f5de42e4c8b948657','sdhelp','scrollbars=yes,resizable=yes,directories=no,toolbar=no,menubar=no,status=no,width=760,height=570'); helpWin.focus(); return false">Help</a></li>      
    </ul> 
    
                
        <div id="bannerTop">                
            <div>Brought to you by:<br>
            
                <a href="http://www.library.nuigalway.ie.libgate.library.nuigalway.ie/" onclick="openNS('http://www.library.nuigalway.ie'); return false;">James Hardiman Library</a>
            
            </div>
        </div>
        
               
</div>

</div>


</div>













<div style="top: 0px; width: 1606px; left: 155px; bottom: 0px; border-top: 3px solid rgb(108, 157, 48);" class="js" id="page-area">
  <div style="width: 1606px;" class="js" id="articleToolbar">
    <div id="articleNav">
  <ul aria-label="Article options" role="navigation">
    <li>
      <!-- return to gateway -->
       
      
    </li>





<li>
      <div class="icon_exportarticlesci_dir" title="Export citation">
        <div class="icon"></div>
        <a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=DownloadURL&amp;_method=confirm&amp;_eidkey=1-s2.0-S0022283608013442&amp;count=1&amp;_docType=FLA&amp;zone=toolbar&amp;_acct=C000007922&amp;_version=1&amp;_userid=103680&amp;md5=525bd47fce3179f851ea560ba64df095">Export citation</a>
      </div>
</li>
  




<li>
  <div class="icon_pdf">
    <div class="icon"></div>
    <a id="pdfLink" href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=MiamiImageURL&amp;_cid=272582&amp;_user=103680&amp;_pii=S0022283608013442&amp;_check=y&amp;_origin=article&amp;_zone=toolbar&amp;_coverDate=2009-Jan-23&amp;view=c&amp;originContentFamily=serial&amp;wchp=dGLbVlS-zSkWz&amp;md5=8886164ebb7ac0b6955dd8f28193a6d8&amp;pid=1-s2.0-S0022283608013442-main.pdf" target="newPdfWin" pdfurl="http://www.sciencedirect.com/science?_ob=MiamiImageURL&amp;_cid=272582&amp;_user=103680&amp;_pii=S0022283608013442&amp;_check=y&amp;_origin=article&amp;_zone=toolbar&amp;_coverDate=2009-Jan-23&amp;view=c&amp;originContentFamily=serial&amp;wchp=dGLbVlS-zSkWz&amp;md5=8886164ebb7ac0b6955dd8f28193a6d8&amp;pid=1-s2.0-S0022283608013442-main.pdf" class="big pdf ext_sdlink" style="cursor: pointer;" title="Download PDF">PDF  (918 K)</a>
  </div>
</li>
 
 



<li class="optionsPos">
      <div id="moreOptionsButton" title="Show article options">
        <a href="#">More options...</a>
        <div class="down_sci_dir"></div>
      </div>

<!-- More Options Starts-->
<div id="moreOptionsMenu" class="articleOptions articleOptionssci_dir">
<ul class="menuContent">


    
    <li>
    <ul class="ereader">
        <li class="title">

            eReader format &nbsp;<a title="Help" onclick="var help; help=window.open('http://www.sciencedirect.com/science/help');">What's this?</a>
        </li>
        <li>
            <a id="epubLink" class="erLink" href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science/article/pii/S0022283608013442?np=y" title="ePub">
                <span class="icon epub">&nbsp;</span><span class="ereadertext">ePub</span>
            </a>
        </li>
        <li>
            <a id="mobiLink" class="erLink" href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science/article/pii/S0022283608013442?np=y" title="Mobipocket">
                <span class="icon mobi">&nbsp;</span><span class="ereadertext">Mobipocket</span>
            </a>
        </li>
    </ul>
    </li>
    





<li>
    <div class="emailsci_dir">
        <div></div>
        <a title="Email this article" href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=EmailFriendURL&amp;_method=gatherInfo&amp;refSource=html&amp;count=1&amp;_eidkey=1-s2.0-S0022283608013442&amp;_acct=C000007922&amp;_version=1&amp;_userid=103680&amp;md5=4949cdbd3e8133014a6f9d1afa3865f8">Email article</a>
    </div>
</li>

  




   
       <li>
          <div id="alertOn" style="display: none;" class="alertsci_dir">
            <div></div>
            <span>

            Signed up for journal alerts&nbsp;[<a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=PublicationURL&amp;newIssueAlert=2&amp;_cid=272582&amp;_pubType=J&amp;_auth=y&amp;originContentFamily=serial&amp;_pii=S0022283608013442&amp;_acct=C000007922&amp;_version=1&amp;_urlVersion=0&amp;_userid=103680&amp;md5=bdeafdf7e18f34c6b4caee0e9f39d4ff">remove</a>]</span>
          </div>
          <div id="alertOff" style="display: inline;" class="alertsci_dir">
            <div></div>
            <a title="Set new Volumes / Issues alert" href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=PublicationURL&amp;newIssueAlert=2&amp;_cid=272582&amp;_pubType=J&amp;_auth=y&amp;originContentFamily=serial&amp;_pii=S0022283608013442&amp;_acct=C000007922&amp;_version=1&amp;_urlVersion=0&amp;_userid=103680&amp;md5=bdeafdf7e18f34c6b4caee0e9f39d4ff">

            Alert me about new articles in this journal
            </a>
          </div>
          <div id="rsltInd" style="display: none;">
            <div class="imgIcon"><h3><img src="errorIcon.gif" alt="Failure" title="Failure"></h3></div>
            <div class="confirmMsgText">Your selection(s) could not be saved due to an internal error. Please try again.</div>
          </div>          
      </li>
    


<li>
    <div class="thumbnailsci_dir">
        <div></div>
        <a href="javascript:void(0);" id="optImgToggle" title="">Show thumbnail images</a>
    </div>
</li>

</ul>
<div id="articleOptionsAd" role="banner" class="ad"></div>
</div>
<!-- More options Ends -->
</li>
<li id="erToolbar"></li>




</ul>
</div>

<div class="sdSearch" aria-label="Quick search" role="search">
    <fieldset>
        <div class="left">
            <input style="color: rgb(155, 155, 155);" id="quickSearch" name="quickSearch" value="Search ScienceDirect" size="30" maxlength="450" title="Perform a quick search on ScienceDirect" type="text"><button class="submit" title="Submit quick search">Search</button>
        </div>
        <div id="quickSearchButton" class="right" title="Show more quick search options">
            <div class="down_sci_dir"></div>
        </div>
    </fieldset>
</div>
<div class="clear"></div>
<div style="width: 1579px;" class="quickSearch" id="sdQuickSearch" aria-label="Extended search" role="search">
<div class="extSearch" role="search" aria-label="Quick Search">


<form name="qkSrch" method="get" target="_top" action="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science" style="margin: 0px;">
<input name="_ob" value="QuickSearchURL" type="hidden">
<input name="_method" value="submitForm" type="hidden">
<input name="_acct" value="C000007922" type="hidden">
<input name="_origin" value="article" type="hidden">
<input name="_zone" value="qSearch" type="hidden">
<input name="md5" value="150d94845014a5f578fff24e6f5ff0a9" type="hidden">

<table style="margin: 0px;" border="0" cellpadding="0" cellspacing="0" width="100%">
<tbody><tr valign="middle">

<!-- JS enabled. Code for Articles -->

<td align="center" nowrap="nowrap">
<label for="Articles" style="cursor: pointer; padding-right: 0px; margin-left: 2px;" class="nojs">Articles</label>
<input class="qsRadio nojs" name="searchtype" id="Articles" value="a" checked="checked" style="" type="radio">
<noscript>

<!-- JS disabled. Code for Articles -->
<span style="margin-left:13px;">Articles</span>
</noscript>
</td>


<!-- Code related for toggling labels -->


<td align="right"><label for="qs_all" id="fieldLabel">&nbsp;&nbsp;&nbsp;&nbsp;All fields</label></td>
<td align="left"><input class="textbox qsinput xpstyle qsImgBlurFocus" name="qs_all" id="qs_all" value="" size="30" maxlength="450" title="For example: heart attack AND behavior?" type="text"></td>




<td align="right"><label for="qs_author">&nbsp;&nbsp;&nbsp;&nbsp;Author</label></td>
<td colspan="5" align="left"><input class="textbox qsinput xpstyle" name="qs_author" id="qs_author" value="" size="33" maxlength="450" title="e.g. J S Smith or John Smith or Smith JS" style="_width:100%" type="text"></td>
<td nowrap="nowrap">

</td><td></td><td></td>
<td align="right" nowrap="nowrap" valign="middle" width="90%">
<a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=MiamiSearchURL&amp;_method=requestForm&amp;_btn=Y&amp;_acct=C000007922&amp;_version=1&amp;_urlVersion=1&amp;_userid=103680&amp;md5=36be44d02ea7cffa13df23aca528620a" style="vertical-align: bottom; font-size: 0.92em;">Advanced search</a>
</td>
</tr>
<tr>

<!-- JS enabled. Code for images -->

<td align="center" nowrap="nowrap">
<label for="Images" style="cursor: pointer; padding-right: 0px;" class="nojs">Images</label>
<input class="qsRadio nojs" name="searchtype" id="Images" value="i" type="radio">
<noscript>

<!-- JS disabled. Code for Images -->
<span style="margin-left:13px;"><a href="/science?np=y&imgSel=Y" >Images</a></span>
</noscript>
</td>


<td align="right"><label for="qs_title">&nbsp;&nbsp;&nbsp;&nbsp;Journal/Book&nbsp;title</label></td>
<td align="left"><input autocomplete="off" class="textbox qsinput xpstyle ui-autocomplete-input" id="qs_title" name="qs_title" value="" size="30" maxlength="450" title="For example: journal of molecular biology" type="text"><span class="ui-helper-hidden-accessible" aria-live="polite" role="status"></span></td>


<td class="toggleQukSrch2" align="right"><label for="qs_vol" id="volField">&nbsp;&nbsp;&nbsp;&nbsp;Volume</label></td>
<td class="toggleQukSrch" align="left"><input class="textbox qsinput xpstyle" name="qs_vol" id="qs_vol" value="" size="3" maxlength="10" style="width: 30px;" type="text"></td>
<td class="toggleQukSrch2" align="right"><label for="qs_issue" id="issueField">&nbsp;&nbsp;Issue</label></td>
<td class="toggleQukSrch" align="left"><input class="textbox qsinput xpstyle" name="qs_issue" id="qs_issue" value="" size="3" maxlength="10" style="width: 30px;" type="text"></td>
<td class="toggleQukSrch2" align="right"><label for="qs_pages" id="pageField">&nbsp;&nbsp;Page</label></td>
<td class="toggleQukSrch" align="right"><input class="textbox qsinput xpstyle" name="qs_pages" id="qs_pages" value="" size="3" maxlength="10" title="For example: 14-27" style="width: 30px;" type="text"></td>


<td align="right" nowrap="nowrap">
<input class="button" id="submit_search" alt="Submit Quick Search" title="Submit Quick Search" value="Search" name="sdSearch" type="Submit">
</td> 
<td colspan="8" align="right" nowrap="nowrap" valign="bottom">
<a class="icon_qmarkHelpsci_dir" href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=HelpURL&amp;_origin=srch_form&amp;_zone=TopNavBar&amp;_file=qs_tips.htm&amp;_acct=C000007922&amp;_version=1&amp;_urlVersion=0&amp;_userid=103680&amp;md5=ae1e488939a34c669c24f09774e72b25" target="sdhelp" onmouseover="window.status='Help is Available';return true" onmouseout="window.status='';return true" onclick="var helpWin;helpWin=window.open('http://www.sciencedirect.com/science?_ob=HelpURL&amp;_file=qs_tips.htm&amp;_acct=C000007922&amp;_version=1&amp;_urlVersion=0&amp;_userid=103680&amp;md5=426a1ddabd0771203259aa91ecaa495c','sdhelp','scrollbars=yes,resizable=yes,directories=no,toolbar=no,menubar=no,status=no,width=760,height=570');helpWin.focus();return false" style="font-size: 0.92em; padding-right: 0px;">Search tips</a>
</td>
</tr>
</tbody></table>
</form>
</div>



 
</div>

<div style="color: rgb(255, 0, 0); font-weight: bold; border: 0px solid red;">
  
  
  
  
</div>





  </div>
  <div style="clear: both;"></div>
  <div style="margin-top: 26px; left: 3px;" class="js" id="leftCloseBar">
      <div class="closed" title="Open to view article outline"></div>
  </div>
  <div style="top: 26px; left: 3px; width: 258px;" id="leftPane" class="column js" aria-label="table of contents" role="navigation">
  <div class="js" id="leftPaneInner"><div id="olGraphCbBox"><label><input id="outlineGraphicsCheckBox" checked="true" type="checkBox">Show thumbnails in outline</label></div>
    <div data-st="0" style="top: 32px; overflow: hidden; background: none repeat ; 0% rgb(236, 242, 246);" id="outline" class="outline js">
      <script type="text/javascript">

      document.write("<div class='outlineMsg'>Article outline is loading...</div>");
      </script>
      <noscript>

      <div class="infoMsg">
      <div style="float: left; width: 20px; "><img border="0" src="http://cdn.els-cdn.com/sd/alertIcon.gif" valign="top" alt="Alert message"></div>
      <div style="padding-left:25px;">JavaScript is disabled on your browser. 
<div style="padding-top:10px;">
      Please enable JavaScript to use all the features on this page.</div></div>
      </div>
      </noscript>
      <ul style="display: block; background: none repeat ; 0% rgb(236, 242, 246);" id="itemOutline"><li class="" style="margin-right: 15px;"><div id="ol_section_abstract" class="io item h2sec"><a class="tocLink" href="#section_abstract" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">Abstract<br></a></div></li><li class="" style="margin-right: 15px;"><div id="ol_kwd_1" class="io item h2sec"><a class="tocLink" href="#kwd_1" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">Abbreviations<br></a></div></li><li class="" style="margin-right: 15px;"><div id="ol_kwd_2" class="io item h2sec"><a class="tocLink" href="#kwd_2" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">Keywords<br></a></div></li><li class="" style="margin-right: 15px;"><div id="ol_section_lbl1" class="io item h2sec"><a class="tocLink" href="#section_lbl1" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">Role of HIV-1 reverse transcriptase in viral replication<br></a></div></li><li class="outlineFocus outlineFocusFirst" style="margin-right: 15px;"><div id="ol_fig1" class="io item h2sec fig"><a class="tocLink" href="#fig1" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');"><img data-loaded="true" src="1-s2.0-S0022283608013442-gr1.sml" class="smlImgLazyJSB greyImg displayImg" data-smlsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr1.sml" data-thumbheight="164"></a></div></li><li class="outlineFocus" style="margin-right: 15px;"><div id="ol_section_lbl2" class="io item h2sec"><a class="tocLink" href="#section_lbl2" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">Structure of HIV-1 reverse transcriptase<br></a></div></li><li class="outlineFocus outlineFocusLast" style="margin-right: 15px;"><div id="ol_fig2" class="io item h2sec fig"><a class="tocLink" href="#fig2" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');"><img data-loaded="true" src="1-s2.0-S0022283608013442-gr2.sml" class="smlImgLazyJSB greyImg displayImg" data-smlsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr2.sml" data-thumbheight="162"></a></div></li><li style="margin-right: 15px;"><div id="ol_fig3" class="io item h2sec fig"><a class="tocLink" href="#fig3" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');"><img data-loaded="true" src="1-s2.0-S0022283608013442-gr3.sml" class="smlImgLazyJSB greyImg displayImg" data-smlsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr3.sml" data-thumbheight="148"></a></div></li><li style="margin-right: 15px;"><div id="ol_section_lbl3" class="io item h2sec"><a class="tocLink" href="#section_lbl3" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">Molecular mechanisms<br></a></div></li><li style="margin-right: 15px;"><div id="ol_section_lbl3.1" class="io item h3sec"><a class="tocLink" href="#section_lbl3.1" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">Molecular mechanism of polymerization<br></a></div></li><li style="margin-right: 15px;"><div id="ol_fig4" class="io item h3sec fig"><a class="tocLink" href="#fig4" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');"><img data-loaded="true" src="1-s2.0-S0022283608013442-gr4.sml" class="smlImgLazyJSB greyImg displayImg" data-smlsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr4.sml" data-thumbheight="28"></a></div></li><li style="margin-right: 15px;"><div id="ol_section_lbl3.2" class="io item h3sec"><a class="tocLink" href="#section_lbl3.2" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">Molecular mechanisms of HIV-1 reverse transcriptase inhibition<br></a></div></li><li style="margin-right: 15px;"><div id="ol_section_lbl3.2.1" class="io item h4sec"><a class="tocLink" href="#section_lbl3.2.1" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">HIV-1 RT inhibitors<br></a></div></li><li style="margin-right: 15px;"><div id="ol_fig5" class="io item h4sec fig"><a class="tocLink" href="#fig5" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');"><img data-loaded="true" src="1-s2.0-S0022283608013442-gr5.sml" class="smlImgLazyJSB greyImg displayImg" data-smlsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr5.sml" data-thumbheight="163"></a></div></li><li style="margin-right: 15px;"><div id="ol_fig6" class="io item h4sec fig"><a class="tocLink" href="#fig6" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');"><img data-loaded="true" src="1-s2.0-S0022283608013442-gr6.sml" class="smlImgLazyJSB greyImg displayImg" data-smlsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr6.sml" data-thumbheight="164"></a></div></li><li style="margin-right: 15px;"><div id="ol_section_lbl3.2.2" class="io item h4sec"><a class="tocLink" href="#section_lbl3.2.2" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">Inhibition by nucleoside reverse transcriptase inhibitors (NRTIs)<br></a></div></li><li style="margin-right: 15px;"><div id="ol_section_lbl3.2.3" class="io item h4sec"><a class="tocLink" href="#section_lbl3.2.3" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">Inhibition by non-nucleoside reverse transcriptase inhibitors (NNRTIs)<br></a></div></li><li style="margin-right: 15px;"><div id="ol_fig7" class="io item h4sec fig"><a class="tocLink" href="#fig7" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');"><img data-loaded="true" src="1-s2.0-S0022283608013442-gr7.sml" class="smlImgLazyJSB greyImg displayImg" data-smlsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr7.sml" data-thumbheight="163"></a></div></li><li style="margin-right: 15px;"><div id="ol_section_lbl3.3" class="io item h3sec"><a class="tocLink" href="#section_lbl3.3" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">Molecular mechanisms of resistance<br></a></div></li><li style="margin-right: 15px;"><div id="ol_section_lbl3.3.1" class="io item h4sec"><a class="tocLink" href="#section_lbl3.3.1" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">NRTI resistance<br></a></div></li><li style="margin-right: 15px;"><div id="ol_section_lbl3.3.2" class="io item h4sec"><a class="tocLink" href="#section_lbl3.3.2" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">NRTI exclusion<br></a></div></li><li style="margin-right: 15px;"><div id="ol_section_lbl3.3.3" class="io item h4sec"><a class="tocLink" href="#section_lbl3.3.3" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">NRTI excision<br></a></div></li><li style="margin-right: 15px;"><div id="ol_section_lbl3.3.4" class="io item h4sec"><a class="tocLink" href="#section_lbl3.3.4" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">NNRTIs and NNRTI resistance<br></a></div></li><li style="margin-right: 15px;"><div id="ol_section_lbl3.3.5" class="io item h4sec"><a class="tocLink" href="#section_lbl3.3.5" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">Loss/change of key hydrophobic interactions<br></a></div></li><li style="margin-right: 15px;"><div id="ol_section_lbl3.3.6" class="io item h4sec"><a class="tocLink" href="#section_lbl3.3.6" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">Steric hindrance<br></a></div></li><li style="margin-right: 15px;"><div id="ol_section_lbl3.3.7" class="io item h4sec"><a class="tocLink" href="#section_lbl3.3.7" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">Pocket entrance mutations<br></a></div></li><li style="margin-right: 15px;"><div id="ol_section_lbl4" class="io item h2sec"><a class="tocLink" href="#section_lbl4" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">Molecular mechanism of RNase H activity<br></a></div></li><li style="margin-right: 15px;"><div id="ol_section_lbl4.1" class="io item h3sec"><a class="tocLink" href="#section_lbl4.1" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">Inhibitors of RNase H<br></a></div></li><li style="margin-right: 15px;"><div id="ol_fig8" class="io item h3sec fig"><a class="tocLink" href="#fig8" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');"><img src="grey_pxl.gif" class="smlImgLazyJSB greyImg" data-smlsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr8.sml" data-thumbheight="164"></a></div></li><li style="margin-right: 15px;"><div id="ol_fig9" class="io item h3sec fig"><a class="tocLink" href="#fig9" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');"><img src="grey_pxl.gif" class="smlImgLazyJSB greyImg" data-smlsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr9.sml" data-thumbheight="164"></a></div></li><li style="margin-right: 15px;"><div id="ol_section_lbl4.1.1" class="io item h4sec"><a class="tocLink" href="#section_lbl4.1.1" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">Future perspectives<br></a></div></li><li style="margin-right: 15px;"><div id="ol_section_lbl4.1.2" class="io item h4sec"><a class="tocLink" href="#section_lbl4.1.2" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">Inhibitors of NRTI excision<br></a></div></li><li style="margin-right: 15px;"><div id="ol_fig10" class="io item h4sec fig"><a class="tocLink" href="#fig10" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');"><img src="grey_pxl.gif" class="smlImgLazyJSB greyImg" data-smlsrc="http://ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr10.sml" data-thumbheight="156"></a></div></li><li style="margin-right: 15px;"><div id="ol_ack001" class="io item h2sec"><a class="tocLink" href="#ack001" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">Acknowledgements<br></a></div></li><li style="margin-right: 15px;"><div id="ol_bibl001" class="io item h2sec"><a class="tocLink" href="#bibl001" onclick="SD_UTIL.sendUserKeyEvent('displayTOCSection', 'article', 'leftPane', 'serial');">References<br></a></div></li></ul><div style="width: 241px; background: none repeat ; 0% rgb(236, 242, 246);" id="articleLeftAd"><iframe class="usmonndchupfwwcnugwg" style="background: none repeat ; 0% rgb(236, 242, 246);" border="0" cellspacing="0" src="S0022283608013442_1.html" frameborder="0" scrolling="no"></iframe><iframe class="usmonndchupfwwcnugwg" border="0" cellspacing="0" src="S0022283608013442_2.html" frameborder="0" scrolling="no"></iframe></div></div>
      <div id="leftOpenBar">
        <div class="open" title="Open to view article outline"></div>
      </div>
  </div>
  </div>
  <div tabindex="-1" data-st="4137" data-dir="u" style="top: 26px; left: 261px;" id="centerPane" class="column js">
    <div id="centerInner" class="centerInner svBigBox js" aria-label="Article" role="main">
      
<script>prs.rt('data_start');</script>
      <div class="authClaim"></div><div class="head"><div class="publisherLogo"><img data-loaded="true" style="display: inline;" alt="" id="pubLogo" class="imgLazyJSB" src="elsevier.gif" data-defaultlogo="/elsevier.gif"><noscript><img alt="" src="http://origin-cdn.els-cdn.com/sd/elsevier.gif"></noscript></div><div class="publicationHead"><div class="title"><a title="Go to Journal of Molecular Biology on SciVerse ScienceDirect" href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science/journal/00222836" querystr="?zone=centerPane" class="cLink"><span>Journal of Molecular Biology</span></a></div><p class="volIssue"><a title="Go to table of contents for this volume/issue" href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science/journal/00222836/385/3">Volume 385, Issue 3</a>, 23 January 2009, Pages 693–713</p></div><div class="publicationCover"><a querystr="?zone=publicationLogo" class="cLink" href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science/journal/00222836/385/3"><img data-loaded="true" style="display: inline;" alt="Cover image" src="1-s2.0-S0022283608X0049X-cov150h.gif" class="toprightlogo imgLazyJSB" data-publogo="1-s2.0-S0022283608X0049X-cov150h.gif"><noscript><img alt="Cover image" src="http://origin-ars.els-cdn.com/content/image/1-s2.0-S0022283608X0049X-cov150h.gif" class="toprightlogo"></noscript></a></div></div><div id="" class="publicationType"><span id="">Review</span></div><div class="module_topic_paths"></div><h1 class="svTitle" id="">Structure and Function of HIV-1 Reverse Transcriptase: Molecular Mechanisms of Polymerization and Inhibition</h1><ul class="authorGroup noCollab" id=""><li><a href="#" class="authorName" id="authname_N4555b8b8N2d886db0" data-t="a" data-fn="Stefan G." data-ln="Sarafianos" data-pos="1">Stefan G. Sarafianos</a><a title="Affiliation: 1" href="#aff1" class="intra_ref auth_aff" id="baff1"><sup>1</sup></a>, </li><li><a href="#" class="authorName" id="authname_N4555b8b8N2d886e40" data-t="a" data-fn="Bruno" data-ln="Marchand" data-pos="2">Bruno Marchand</a><a title="Affiliation: 1" href="#aff1" class="intra_ref auth_aff" id="baff1"><sup>1</sup></a>, </li><li><a href="#" class="authorName" id="authname_N4555b8b8N2d886ed0" data-t="a" data-fn="Kalyan" data-ln="Das" data-pos="3">Kalyan Das</a><a title="Affiliation: 2" href="#aff2" class="intra_ref auth_aff" id="baff2"><sup>2</sup></a>, </li><li><a href="#" class="authorName" id="authname_N4555b8b8N2d886f60" data-t="a" data-fn="Daniel M." data-ln="Himmel" data-pos="4">Daniel M. Himmel</a><a title="Affiliation: 2" href="#aff2" class="intra_ref auth_aff" id="baff2"><sup>2</sup></a>, </li><li><a href="#" class="authorName" id="authname_N4555b8b8N2d886ff0" data-t="a" data-fn="Michael A." data-ln="Parniak" data-pos="5">Michael A. Parniak</a><a title="Affiliation: 3" href="#aff3" class="intra_ref auth_aff" id="baff3"><sup>3</sup></a>, </li><li><a href="#" class="authorName" id="authname_N4555b8b8N2d887080" data-t="a" data-fn="Stephen H." data-ln="Hughes" data-pos="6">Stephen H. Hughes</a><a title="Affiliation: 4" href="#aff4" class="intra_ref auth_aff" id="baff4"><sup>4</sup></a>, </li><li><a href="#" class="authorName" id="authname_N4555b8b8N2d887110" data-t="a" data-fn="Eddy" data-ln="Arnold" data-pos="7">Eddy Arnold</a><a title="Affiliation: 2" href="#aff2" class="intra_ref auth_aff" id="baff2"><sup>2</sup></a><sup>, </sup><a title="Corresponding author contact information" href="#cor1" id="bcor1" class="intra_ref auth_corr"><img data-loaded="true" style="display: inline;" class="imgLazyJSB" alt="Corresponding author contact information" src="REcor.gif" data-inlimg="/entities/REcor.gif"><noscript><img alt="Corresponding author contact information" src="http://origin-cdn.els-cdn.com/sd/entities/REcor.gif"></noscript></a><sup>, </sup><a href="mailto:arnold@cabm.rutgers.edu"><img data-loaded="true" style="display: inline;" class="imgLazyJSB" src="REemail.gif" alt="E-mail the corresponding author" data-inlimg="/entities/REemail.gif"><noscript><img src="http://origin-cdn.els-cdn.com/sd/entities/REemail.gif" alt="E-mail the corresponding author"></noscript></a></li></ul><div class="endLine"></div><ul class="affiliation"><li id="aff1"><sup>1</sup> <span id="">Christopher Bond Life Sciences Center, Department of Molecular Microbiology &amp; Immunology, University of Missouri School of Medicine, Columbia, MO 65211, USA</span></li><li id="aff2"><sup>2</sup> <span id="">Center for Advanced Biotechnology and Medicine and Department of Chemistry and Chemical Biology, Rutgers University, Piscataway, NJ 08854-5627, USA</span></li><li id="aff3"><sup>3</sup> <span id="">Department of Molecular Genetics and Biochemistry, University of Pittsburgh School of Medicine, Pittsburgh, PA 15261, USA</span></li><li id="aff4"><sup>4</sup> <span id="">HIV Drug Resistance Program, National Cancer Institute, Frederick, MD 21702-1201, USA</span></li></ul><!--VALIDHTML--><dl class="articleDates"><dd>Received 7 July 2008</dd><dd>Revised 15 October 2008</dd><dd>Accepted 15 October 2008</dd><dd>Available online 3 November 2008</dd><dd id="">Edited by I. Wilson</dd></dl><!--VALIDHTML--><ul class="articleDoi"><li><a id="ddDoi" href="http://dx.doi.org.libgate.library.nuigalway.ie/10.1016/j.jmb.2008.10.071" target="doilink" onclick="var doiWin; doiWin=window.open('http://dx.doi.org/10.1016/j.jmb.2008.10.071','doilink','scrollbars=yes,resizable=yes,directories=yes,toolbar=yes,menubar=yes,status=yes'); doiWin.focus()">http://dx.doi.org/10.1016/j.jmb.2008.10.071</a>, <a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science/help/doi.htm" target="sdhelp" onmouseover="window.status='Help is Available';return true" onclick="var helpWin; helpWin=window.open('http://www.sciencedirect.com/science/help/doi.htm','sdhelp','scrollbars=yes,resizable=yes,directories=no,toolbar=no,menubar=no,status=no,width=760,height=570'); helpWin.focus()">How to Cite or Link Using DOI</a></li></ul><!--VALIDHTML--><ul class="articleInfo"><li class="permissions" id="permissions"><div class="boxLink"><a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=RedirectURL&amp;_method=outwardLink&amp;_partnerName=936&amp;_eid=1-s2.0-S0022283608013442&amp;_pii=S0022283608013442&amp;_origin=article&amp;_zone=art_page&amp;_targetURL=https%3A%2F%2Fs100.copyright.com%2FAppDispatchServlet%3FpublisherName%3DELS%26contentID%3DS0022283608013442%26orderBeanReset%3Dtrue&amp;_acct=C000007922&amp;_version=1&amp;_userid=103680&amp;md5=9784454e353e9a57c56e8a9839bb17f1" rel="nofollow" class="olSpawnWindow" target="outwardLink">Permissions &amp; Reprints</a></div></li></ul><!--VALIDHTML--><div id="ppvPlaceHolder" class="hidden"></div><!--VALIDHTML--><div id="refersToAndreferredToBy"><dl id="referredToBy" class="documentThread"><!--Referred To By--></dl></div><!--VALIDHTML--><hr id="author1"><div class="abstract svAbstract "><h2 class="secHeading" id="section_abstract">Abstract</h2><p id="">The rapid replication of HIV-1 and the errors made during viral replication cause the virus to evolve rapidly in patients, making the problems of vaccine development and drug therapy particularly challenging. In the absence of an effective vaccine, drugs are the only useful treatment. Anti-HIV drugs work; so far drug therapy has saved more than three million years of life. Unfortunately, HIV-1 develops resistance to all of the available drugs. Although a number of useful anti-HIV drugs have been approved for use in patients, the problems associated with drug toxicity and the development of resistance means that the search for new drugs is an ongoing process. The three viral enzymes, reverse transcriptase (RT), integrase (IN), and protease (PR) are all good drug targets. Two distinct types of RT inhibitors, both of which block the polymerase activity of RT, have been approved to treat HIV-1 infections, nucleoside analogs (NRTIs) and nonnucleosides (NNRTIs), and there are promising leads for compounds that either block the RNase H activity or block the polymerase in other ways. A better understanding of the structure and function(s) of RT and of the mechanism(s) of inhibition can be used to generate better drugs; in particular, drugs that are effective against the current drug-resistant strains of HIV-1.</p></div><!--VALIDHTML--><script>prs.rt("abs_end");</script><div id="SD_BA1P" class="sgfNoTitleBar sgfNoGadgetBorder svDoNotLink"></div><!--VALIDHTML--><h2 id="kwd_1">Abbreviations</h2><ul class="keyword" id=""><li id=""><span id="">HIV-1</span>, <span id="">human immunodeficiency virus type 1</span>; </li><li id=""><span id="">RT</span>, <span id="">reverse transcriptase</span>; </li><li id=""><span id="">PR</span>, <span id="">protease</span>; </li><li id=""><span id="">IN</span>, <span id="">integrase</span>; </li><li id=""><span id="">PBS</span>, <span id="">primer binding site</span>; </li><li id=""><span id="">R</span>, <span id="">repeat</span>; </li><li id=""><span id="">PPT</span>, <span id="">polypurine tract</span>; </li><li id=""><span id="">LTR</span>, <span id="">long terminal repeat</span>; </li><li id=""><span id="">DEC</span>, <span id="">dead-end complex</span>; </li><li id=""><span id="">NRTI</span>, <span id="">nucleoside RT inhibitor</span>; </li><li id=""><span id="">NNRTI</span>, <span id="">nonnucleoside RT inhibitor</span>; </li><li id=""><span id="">NNIBP</span>, <span id="">NNRTI-binding pocket</span>; </li><li id=""><span id="">BBNH</span>, <span id="">acylhydrazone <em>N</em>-(4-tert-butylbenzoyl)-2-hydroxy-1-naphthaldehyde hydrazone</span>; </li><li id=""><span id="">CPHM</span>, <span id="">aryl hydrazone 4-chlorophenyl hydrazone mesoxalate</span></li></ul><!--VALIDHTML--><h2 id="kwd_2">Keywords</h2><ul class="keyword" id=""><li id=""><span id="">polymerase structure and mechanism</span>; </li><li id=""><span id="">AIDS</span>; </li><li id=""><span id="">drug resistance</span>; </li><li id=""><span id="">drug design</span>; </li><li id=""><span id="">ribonuclease H</span></li></ul><!--VALIDHTML--><p class="svArticle section" id="">In the absence of an effective vaccine, drugs are the only therapeutic tools that can be used to treat human immunodeficiency virus type 1 (HIV-1) infections. Unfortunately, HIV-1 infections cannot be cured, so that drug therapy, once initiated, must be continued for the life of the patient. This places a special burden on the design of anti-HIV drugs: They need to be relatively nontoxic so that they can be used in long-term therapy. HIV-1 replication is error prone<span id="bbib1"><a href="#bib1" id="ancbbib1" class="intra_ref"><sup>1</sup></a></span> and the errors that arise during the viral life-cycle, together with the rapid replication of the virus in patients, allow the virus to escape the host immune system and develop resistance to all of the available drugs.<span id="bbib2"><a href="#bib2" id="ancbbib2" class="intra_ref"><sup>2</sup></a></span> The virus evolves sufficiently rapidly that, unless the therapy is well-designed, resistance will develop in all treated patients. The only way to stop the development of resistance is to completely block viral replication; this, in turn, stops the evolution of resistance. It takes a combination of drugs (usually three) to completely block viral replication; this is the reason that three-drug regimens are used in standard HIV-1 therapies. Of the approved drugs, most target two of the three virally encoded enzymes that carry out viral replication, reverse transcriptase (RT) and protease (PR). A new drug, raltegravir, which targets the third enzyme, integrase (IN), has recently been approved<span id="bfn1"><a href="#fn1" id="ancbfn1" class="intra_ref">†</a></span>. In addition to the drugs that target the viral enzymes, there are two approved drugs, enfurvitide and maraviroc, that target different aspects of viral entry.</p><!--VALIDHTML--><p class="svArticle section" id="">Because standard anti-HIV therapies include anti-RT drugs, it is important to understand the drug target (RT) itself, the role(s) RT plays in the viral life-cycle, the ways the drugs act to inhibit the normal functions of RT, and the mechanisms that the virus, and more importantly RT, uses to evade the available drugs. Armed with a better understanding of RT, and the mechanisms of inhibition and drug resistance that involve RT directly, it should be possible to develop drugs that will be more effective; in particular to develop drugs that can block the replication of viruses that are resistant to the currently available drugs.</p><!--VALIDHTML--><h2 class="svArticle" id="section_lbl1">Role of HIV-1 reverse transcriptase in viral replication</h2><p class="svArticle section" id="">Viral infections are initiated by the fusion of the viral and cellular membranes; this fusion reaction is caused by the interactions of the viral envelope glycoprotein with its receptor (CD4) and a co-receptor, usually either CCR5 or CXCR4 (for a review of the retroviral life-cycle, and an overview of reverse transcription, see Ref. <span id="bbib1"><a href="#bib1" id="ancbbib1" class="intra_ref">1</a></span>). Binding the receptor and co-receptor causes changes in the structure of the envelope glycoprotein, which leads to membrane fusion. Membrane fusion places the viral core, which contains RT, into the cytoplasm of the cell. A poorly understood process called uncoating modifies the core in ways that promote reverse transcription.</p><!--VALIDHTML--><p class="svArticle section" id="">The HIV-1 virion contains, in addition to the viral proteins, two copies of a single-stranded RNA genome. RT has two enzymatic activities; a DNA polymerase that can copy either a DNA or an RNA template, and an RNase H that cleaves RNA only if the RNA is part of an RNA/DNA duplex. The two enzymatic functions of RT, polymerase and RNase H, cooperate to convert the RNA into a double-stranded linear DNA. This conversion takes place in the cytoplasm of the infected cell. After DNA synthesis has been completed, the resulting linear double-stranded viral DNA is translocated to the nucleus where the viral DNA is inserted into the host genome by IN. This inserted DNA copy, called a provirus, is the source of both viral genomic and viral messenger RNAs, which are generated by the host DNA-dependent RNA polymerase. Although other viral proteins (notably the nucleic acid chaperone nucleocapsid, and perhaps IN), and probably some cellular factors, help RT carry out the reactions that convert the viral RNA into DNA, RT contains all the necessary enzymatic activities for the conversion.</p><!--VALIDHTML--><p class="svArticle section" id="">Like many other DNA polymerases, RT requires both a primer and a template. DNA synthesis is initiated from a host tRNA primer (in HIV-1 the primer is tRNA<sub>lys3</sub>). Near the 5′ end of the viral genome there is a segment 18 nucleotides long, called the primer binding site (PBS) that is complementary to the 18 nucleotides at the 3′ end of tRNA<sub>lys3</sub>. The tRNA primer is hybridized to the PBS (<span id="bfig1"><a href="#fig1" id="ancbfig1" class="intra_ref">Fig. 1</a></span>). The viral RNA genome, which serves as the template, is plus-strand. First (minus) strand DNA synthesis is initiated from the tRNA primer, allowing RT to copy the 5′ end of the viral RNA genome. Synthesis of the minus-strand DNA generates an RNA/DNA hybrid that is a substrate for RNase H, which degrades the RNA strand, leaving the nascent minus-strand DNA single stranded. The sequences at the 5′ and 3′ ends of the viral RNA genome are identical (R, or repeat, see <span id="bfig1"><a href="#fig1" id="ancbfig1" class="intra_ref">Fig. 1</a></span>). This allows the minus-strand DNA to hybridize with the R sequence at the 3′ end of one of the two viral RNAs in the virion, a step that is called the first jump, or the first or minus-strand transfer. After the nascent DNA hybridizes to R, minus-strand synthesis can continue along the viral RNA. As DNA synthesis proceeds, RNase H degrades the RNA strand. Although most of the RNase H cleavages do not appear to be sequence specific, there is a purine-rich sequence (called the polypurine tract or PPT) near the 3′ end of the viral RNA (<span id="bfig1"><a href="#fig1" id="ancbfig1" class="intra_ref">Fig. 1</a></span>). The PPT is relatively resistant to cleavage by RNase H, and it serves as the primer for second (plus) strand DNA synthesis. Plus-strand DNA synthesis proceeds until RT starts to copy the tRNA primer. The first 18 nucleotides can be copied; however, the next nucleotide in the tRNA is a modified A that cannot be copied by RT. Once the 3′ end of the tRNA has been reverse transcribed, an RNA/DNA hybrid that is a substrate for RNase H is created. In the reverse transcription of most retroviral genomes, the RNase H of RT removes the entire tRNA primer. HIV-1 is the exception; its RNase H cleaves precisely one nucleotide from the tRNA/DNA junction, leaving a ribo-A on the 3′ end of the viral minus-strand DNA. This sets the stage for the second jump, also called the second or plus-strand transfer. The removal of the tRNA primer exposes a single-stranded portion of the plus-strand DNA that has the same sequence as the PBS. Exposure of the 3′ end of the plus-strand DNA allows the 5′ end of the minus-strand (once the PBS has been copied) to be transferred to the plus-strand (<span id="bfig1"><a href="#fig1" id="ancbfig1" class="intra_ref">Fig. 1</a></span>). Once this second transfer happens, both the minus and plus strands are extended until the entire DNA is double stranded, creating a DNA that has the same sequences at both ends (these repeats are called long terminal repeats or LTRs). <span id="bfig1"><a href="#fig1" id="ancbfig1" class="intra_ref">Figure 1</a></span> shows that the DNA is longer than the RNA genome(s) from which it derives, allowing the viral DNA, once integrated, to serve as the template from which new copies of the viral genome (and the viral messenger RNAs) can be copied by the host enzyme DNA-dependent RNA polymerase.</p><!--VALIDHTML--><div class="figTblUpiOuter svArticle" id="figure_fig1"><div class="
                    textboxdefault
                "><dl class="figure" id="fig1" data-t="f"><dt class="autoScroll" data-style="height:756px;width:580px;"><a class="figureLink" href="#gr1" title="Full-size image (78 K)"><img data-loaded="true" style="display: inline;" class="imgLazyJSB figure large smallImg" alt="Full-size image (78 K)" src="1-s2.0-S0022283608013442-gr1.jpg" data-thumbeid="1-s2.0-S0022283608013442-gr1.sml" data-fulleid="1-s2.0-S0022283608013442-gr1.jpg" data-thumbheight="164" data-thumbwidth="126" data-fullheight="756" data-fullwidth="580" border="0"><noscript class="figure large" border="0" alt="Full-size image" data-thumbeid="1-s2.0-S0022283608013442-gr1.sml" data-fulleid="1-s2.0-S0022283608013442-gr1.jpg"><img class="figure large" border="0" alt="Full-size image (78 K)" src="http://origin-ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr1.jpg" data-thumbEID="1-s2.0-S0022283608013442-gr1.sml" data-fullEID="1-s2.0-S0022283608013442-gr1.jpg"></noscript></a></dt><dd id="labelCaptionfig1"><p class="caption">Fig. 1.&nbsp;<span id="">Reverse transcription of the HIV-1 genome (with permission from Katz and Skalka<span id="fig1bbib3"><a href="#bib3" id="ancfig1bbib3" class="intra_ref"><sup>3</sup></a></span>). Each retroviral particle contains two copies of the RNA genome. Minus-strand DNA synthesis starts near the 5′ end of the plus-stand RNA genome using a host tRNA<sub>lys3</sub> that anneals at the primer-binding site (PBS) as a primer. <em>Step 1:</em> Synthesis proceeds to the 5′ end of the RNA genome through the U5 region, ending at the R region at the 5′end, forming the minus-strand strong stop DNA. <em>Step 2:</em> DNA synthesis is accompanied by RNase H digestion of the RNA portion of the RNA/DNA hybrid product, thus exposing the single-stranded DNA product. <em>Step 3:</em> This exposure facilitates hybridization with the R region at the 3′ end of the same, or the second, RNA genome, a strand-transfer reaction known as the first jump. <em>Step 4:</em> When minus-strand elongation passes a polypurine-rich region called the polypurine tract (PPT) region, a unique plus-strand RNA primer is formed by RNase H cleavage at its borders. Plus-strand synthesis then continues back to the U5 region using the minus-strand DNA as a template. <em>Step</em><em>5:</em> Meanwhile, minus-strand synthesis continues through the genome using the plus-strand RNA as a template, and removing the RNA template in its wake <em>via</em> RNase H activity. <em>Step 6:</em> The RNase H digestion products formed are presumed to provide additional primers for plus-strand synthesis at a number of internal locations along the minus-strand DNA. <em>Step 7:</em> PPT-initiated plus-strand DNA synthesis stops after copying the annealed portion of the tRNA to generate the plus-strand DNA form of the PBS, forming the plus-strand strong stop product. The tRNA is then removed by the RNase H activity of RT. <em>Step 8:</em> This may facilitate annealing to the PBS complement on the minus-strand DNA, providing the complementarity for the second jump. DNA synthesis then continues. <em>Step 9:</em> Strand displacement synthesis by RT to the PBS and PPT ends, and/or repair and ligation of a circular intermediate, produces a linear duplex with long terminal repeats (LTRs) at both ends.<!--Comment--></span><!--caption--></p></dd><dd class="menuButtonLinks"><div class="btnHolder"><a class="menuTitle" href="#">Figure options</a><div class="down_Btn"></div></div><ul class="menuButton"><li><a href="#fig1" class="viewWS" title="View in workspace">View in workspace</a></li><li><a href="http://origin-ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr1.jpg" title="View full-size image - Opens in new window">Download full-size image</a></li><li><a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0022283608013442&amp;_image=1-s2.0-S0022283608013442-gr1_lrg.jpg&amp;_ba=&amp;_fmt=full&amp;_orig=na&amp;_issn=00222836&amp;_pii=S0022283608013442&amp;_isHiQual=Y&amp;_acct=C000049402&amp;_version=1&amp;_urlVersion=0&amp;_userid=963121&amp;md5=1ce7a790cc6aec1bb0753c40fbe67fb6" title="View high quality image - Opens in new window">Download high-quality image (652 K)</a></li><li><a class="ppt" title="Download as PowerPoint slide" href="http://services.elsevier.com.libgate.library.nuigalway.ie/SDWebExport/export/figure/S0022283608013442/1-s2.0-S0022283608013442-gr1.jpg/ppt">Download as PowerPoint slide</a></li></ul></dd></dl></div></div><!--VALIDHTML--><p class="svArticle section" id="">Although there are complexities to the reverse transcription process that are beyond the scope of this review, there are a few additional points that should be made here. As already mentioned, the end product of the reverse transcription process is the substrate for IN. As such, the ends of the linear viral DNA need to be relatively precise. As is shown in <span id="bfig1"><a href="#fig1" id="ancbfig1" class="intra_ref">Fig. 1</a></span>, it is the RNase H cleavages that generate and remove the PPT primer that define the U3 end of the linear viral DNA, and it is the removal of the tRNA primer that defines the U5 end. Although RNase H has no mechanism that allows it to recognize specific sequences, it carries out these particular cleavage reactions with absolute specificity; and the ends of the linear viral DNA genome are defined to the exact nucleotide.</p><!--VALIDHTML--><p class="svArticle section" id="">There is also the question of why virions contain two copies of the viral RNA genome instead of one. In theory, the reactions outlined in <span id="bfig1"><a href="#fig1" id="ancbfig1" class="intra_ref">Fig. 1</a></span> could be carried out with only one copy of the viral RNA genome. The obvious explanation is that, if there was only one copy of the viral RNA in a virion, a single break in the RNA would be fatal, preventing the synthesis of a complete DNA copy. However, if there is a second copy of the RNA, and minus-strand DNA synthesis is blocked by a break in the RNA template, synthesis can be continued if minus-strand DNA synthesis is transferred to the second RNA genome. In fact, if two copies of the RNA genome are present, a complete DNA copy can be made even if both RNA genomes are extensively nicked, so long as there is no site at which both RNA copies are nicked. If a virion is produced by a cell that contains only one integrated DNA genome, the two RNA copies will be identical (unless RNA polymerase makes an error), and the fact that, during minus-strand DNA synthesis, RT often shifts back and forth between the two RNA templates will have no significant consequence. However, if a host cell contains two different integrated viral DNA genomes, then virions can be produced that contain two related RNA genomes that have different sequences. This sets the stage for the generation of viral recombinants. If RT makes a double-stranded viral DNA by copying from two different RNA genomes, the resulting DNA will contain sequences that are derived from both of the parental genomes. In HIV-1, recombination during reverse transcription is the rule, not the exception. This can have important consequences. For example, recombination can lead to the generation of viruses that are resistant to multiple drugs from parental viruses that are each resistant to only a single drug.</p><!--VALIDHTML--><h2 class="svArticle" id="section_lbl2">Structure of HIV-1 reverse transcriptase</h2><p class="svArticle section" id="">The RT of HIV-1 is an asymmetric heterodimer composed of two related subunits, p66 and p51. Both subunits derive, by cleavage by the viral protease (PR), from a Gag-Pol polyprotein that is synthesized from unspliced viral RNA.<sup><a href="#bib4" id="bbib4" class="intra_ref">4</a>&nbsp;and&nbsp;<a href="#bib5" id="bbib5" class="intra_ref">5</a></sup> p66 and p51 share a common amino terminus; p66 is 560 amino acids in length, p51 is 440 amino acids long. As has been discussed briefly, both of the enzymatic functions of RT, the DNA polymerase and RNase H, are essential for copying the single-stranded RNA genome found in virions into the double-stranded DNA that is inserted into the host genome by IN (for a review of the retroviral life-cycle, and an overview of reverse transcription, see Ref. <span id="bbib1"><a href="#bib1" id="ancbbib1" class="intra_ref">1</a></span>). The larger subunit of the RT heterodimer, p66, contains the active sites for both of the enzymatic activities of RT (polymerase and RNase H); the smaller subunit has a structural role (<span id="bfig2"><a href="#fig2" id="ancbfig2" class="intra_ref">Fig. 2</a></span>). Important structural features of RT were elucidated by the first crystallographic studies.<sup><a href="#bib6" id="bbib6" class="intra_ref">6</a>&nbsp;and&nbsp;<a href="#bib7" id="bbib7" class="intra_ref">7</a></sup> p66 is composed of two spatially distinct domains, polymerase and RNase H. The polymerase domain is composed of four subdomains: fingers (residues 1–85 and 118–155), palm (residues 86–117 and 156–236), thumb (237–318), and connection (319–426).<sup><a href="#bib6" id="bbib6" class="intra_ref">6</a>&nbsp;and&nbsp;<a href="#bib7" id="bbib7" class="intra_ref">7</a></sup> p51 folds into the same four subdomains as the polymerase domain of p66 (fingers, palm, thumb, and connection); however, the positions of the subdomains relative to each other are different in p66 and p51 (see <span id="bfig2"><a href="#fig2" id="ancbfig2" class="intra_ref">Fig. 2</a></span>).</p><!--VALIDHTML--><div class="figTblUpiOuter svArticle" id="figure_fig2"><div class="
                    textboxdefault
                "><dl class="figure" id="fig2" data-t="f"><dt class="autoScroll" data-style="height:330px;width:446px;"><a class="figureLink" href="#gr2" title="Full-size image (54 K)"><img data-loaded="true" style="display: inline; height: 330px; width: 446px;" class="imgLazyJSB figure large" alt="Full-size image (54 K)" src="1-s2.0-S0022283608013442-gr2.jpg" data-thumbeid="1-s2.0-S0022283608013442-gr2.sml" data-fulleid="1-s2.0-S0022283608013442-gr2.jpg" data-thumbheight="162" data-thumbwidth="219" data-fullheight="330" data-fullwidth="446" border="0" height="330" width="446"><noscript class="figure large" border="0" alt="Full-size image" data-thumbeid="1-s2.0-S0022283608013442-gr2.sml" data-fulleid="1-s2.0-S0022283608013442-gr2.jpg"><img class="figure large" border="0" alt="Full-size image (54 K)" src="http://origin-ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr2.jpg" data-thumbEID="1-s2.0-S0022283608013442-gr2.sml" data-fullEID="1-s2.0-S0022283608013442-gr2.jpg"></noscript></a></dt><dd id="labelCaptionfig2"><p class="caption">Fig. 2.&nbsp;<span id="">Ribbon representation of HIV-1 RT in a complex with nucleic acid. The fingers, palm, thumb, connection, and RNase H subdomains of the p66 subunit are shown in blue, red, green, yellow, and orange, respectively. The p51 subunit is shown in dark brown. The template and primer DNA strands are shown in light gray and dark gray, respectively.<!--Comment--></span><!--caption--></p></dd><dd class="menuButtonLinks"><div class="btnHolder"><a class="menuTitle" href="#">Figure options</a><div class="down_Btn"></div></div><ul class="menuButton"><li><a href="#fig2" class="viewWS" title="View in workspace">View in workspace</a></li><li><a href="http://origin-ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr2.jpg" title="View full-size image - Opens in new window">Download full-size image</a></li><li><a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0022283608013442&amp;_image=1-s2.0-S0022283608013442-gr2_lrg.jpg&amp;_ba=&amp;_fmt=full&amp;_orig=na&amp;_issn=00222836&amp;_pii=S0022283608013442&amp;_isHiQual=Y&amp;_acct=C000049402&amp;_version=1&amp;_urlVersion=0&amp;_userid=963121&amp;md5=bab49e909ccb4d7e75f0f05ca24f6109" title="View high quality image - Opens in new window">Download high-quality image (555 K)</a></li><li><a class="ppt" title="Download as PowerPoint slide" href="http://services.elsevier.com.libgate.library.nuigalway.ie/SDWebExport/export/figure/S0022283608013442/1-s2.0-S0022283608013442-gr2.jpg/ppt">Download as PowerPoint slide</a></li></ul></dd></dl></div></div><!--VALIDHTML--><p class="svArticle section" id="">The nucleic-acid binding cleft is formed primarily by the p66 fingers, palm, thumb, connection, and RNase H subdomains of p66. The connection and thumb subdomains of p51 form the floor of the binding cleft (<span id="bfig2"><a href="#fig2" id="ancbfig2" class="intra_ref">Fig. 2</a></span>). The binding cleft is configured so that the nucleic acid contacts both the polymerase and the RNase H active sites; these are located about 17 or 18 basepairs apart on the nucleic acid substrate. The αH and αI helices of the p66 thumb help to properly position the nucleic acid through interactions that involve both the primer and template strands. The DNA primer grip, a highly conserved structural motif<span id="bbib8"><a href="#bib8" id="ancbbib8" class="intra_ref"><sup>8</sup></a></span> that consists of the p66 β12-β13 hairpin in HIV-1 RT<span id="bbib7"><a href="#bib7" id="ancbbib7" class="intra_ref"><sup>7</sup></a></span> helps position the 3′-OH end of the primer strand at the polymerase active site. Mutational studies have shown that changes in the DNA primer grip alter nucleic acid binding and may affect both polymerase and RNase H activities of RT.<sup><a href="#bib9" id="bbib9" class="intra_ref">9</a>, <a href="#bib10" id="bbib10" class="intra_ref">10</a>, <a href="#bib11" id="bbib11" class="intra_ref">11</a>, <a href="#bib12" id="bbib12" class="intra_ref">12</a>, <a href="#bib13" id="bbib13" class="intra_ref">13</a>, <a href="#bib14" id="bbib14" class="intra_ref">14</a>, <a href="#bib15" id="bbib15" class="intra_ref">15</a>, <a href="#bib16" id="bbib16" class="intra_ref">16</a>&nbsp;and&nbsp;<a href="#bib17" id="bbib17" class="intra_ref">17</a></sup></p><!--VALIDHTML--><p class="svArticle section" id="">The polymerase active site is composed of three catalytic carboxylates in the palm subdomain of p66 (D110, D185, and D186) that bind two divalent ions (<span id="bfig3"><a href="#fig3" id="ancbfig3" class="intra_ref">Fig. 3</a></span>) (Mg<sup>2+</sup> appears to be the divalent cation used <em>in vivo</em>; Mn<sup>2+</sup> can support polymerization <em>in vitro</em>) that are required for catalysis.<span id="bbib18"><a href="#bib18" id="ancbbib18" class="intra_ref"><sup>18</sup></a></span> D185 and D186 are part of the YXDD motif that is highly conserved in retroviral RTs (X is Met in HIV RT and Val, Leu, or Ala in other retroviruses). <sup><a href="#bib19" id="bbib19" class="intra_ref">19</a>, <a href="#bib20" id="bbib20" class="intra_ref">20</a>&nbsp;and&nbsp;<a href="#bib21" id="bbib21" class="intra_ref">21</a></sup> Other conserved residues that help form the dNTP-binding site of RT include: (a) R72 and K65 that are involved in the binding the β- and γ-phosphates, respectively, in the incoming dNTP.<span id="bbib22"><a href="#bib22" id="ancbbib22" class="intra_ref"><sup>22</sup></a></span> and (b) residue Y115, which contributes to the binding of the deoxyribose ring of the incoming dNTP and has been termed a steric gate that discriminates between deoxy and ribonucleoside triphosphates; <sup><a href="#bib23" id="bbib23" class="intra_ref">23</a>, <a href="#bib24" id="bbib24" class="intra_ref">24</a>&nbsp;and&nbsp;<a href="#bib25" id="bbib25" class="intra_ref">25</a></sup> and (c) Q151, a residue that interacts directly with the 3′-OH of the incoming dNTP.<span id="bbib22"><a href="#bib22" id="ancbbib22" class="intra_ref"><sup>22</sup></a></span></p><!--VALIDHTML--><div class="figTblUpiOuter svArticle" id="figure_fig3"><div class="
                    textboxdefault
                "><dl class="figure" id="fig3" data-t="f"><dt class="autoScroll" data-style="height:308px;width:454px;"><a class="figureLink" href="#gr3" title="Full-size image (35 K)"><img data-loaded="true" style="display: inline; height: 308px; width: 454px;" class="imgLazyJSB figure large" alt="Full-size image (35 K)" src="1-s2.0-S0022283608013442-gr3.jpg" data-thumbeid="1-s2.0-S0022283608013442-gr3.sml" data-fulleid="1-s2.0-S0022283608013442-gr3.jpg" data-thumbheight="148" data-thumbwidth="219" data-fullheight="308" data-fullwidth="454" border="0" height="308" width="454"><noscript class="figure large" border="0" alt="Full-size image" data-thumbeid="1-s2.0-S0022283608013442-gr3.sml" data-fulleid="1-s2.0-S0022283608013442-gr3.jpg"><img class="figure large" border="0" alt="Full-size image (35 K)" src="http://origin-ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr3.jpg" data-thumbEID="1-s2.0-S0022283608013442-gr3.sml" data-fullEID="1-s2.0-S0022283608013442-gr3.jpg"></noscript></a></dt><dd id="labelCaptionfig3"><p class="caption">Fig. 3.&nbsp;<span id="">Metal chelation in the polymerase active site of an enzyme/DNA/dNTP ternary complex. The nucleotide binding site (N site or pre-translocation site) and the priming site (P site or post-translocation site) are shown.<!--Comment--></span><!--caption--></p></dd><dd class="menuButtonLinks"><div class="btnHolder"><a class="menuTitle" href="#">Figure options</a><div class="down_Btn"></div></div><ul class="menuButton"><li><a href="#fig3" class="viewWS" title="View in workspace">View in workspace</a></li><li><a href="http://origin-ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr3.jpg" title="View full-size image - Opens in new window">Download full-size image</a></li><li><a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0022283608013442&amp;_image=1-s2.0-S0022283608013442-gr3_lrg.jpg&amp;_ba=&amp;_fmt=full&amp;_orig=na&amp;_issn=00222836&amp;_pii=S0022283608013442&amp;_isHiQual=Y&amp;_acct=C000049402&amp;_version=1&amp;_urlVersion=0&amp;_userid=963121&amp;md5=5eafaa518d5807a4c2bd89f199924c67" title="View high quality image - Opens in new window">Download high-quality image (300 K)</a></li><li><a class="ppt" title="Download as PowerPoint slide" href="http://services.elsevier.com.libgate.library.nuigalway.ie/SDWebExport/export/figure/S0022283608013442/1-s2.0-S0022283608013442-gr3.jpg/ppt">Download as PowerPoint slide</a></li></ul></dd></dl></div></div><!--VALIDHTML--><h2 class="svArticle" id="section_lbl3">Molecular mechanisms</h2><h3 class="svArticle" id="section_lbl3.1">Molecular mechanism of polymerization</h3><p class="svArticle section" id="">The mechanism of DNA polymerization by HIV RT is reasonably well understood; extensive biochemical and crystallographic data have helped define the individual steps of the process (<span id="bfig4"><a href="#fig4" id="ancbfig4" class="intra_ref">Fig. 4</a></span>). The reaction begins with the binding of RT to the nucleic acid substrate, which results in a conformational change in the position of the p66 thumb, from a closed to an open conformation. Like many other DNA polymerases, RT requires both a primer and a template. In most sequence contexts, RT preferentially binds a double-stranded nucleic acid so that the 3′ end of the primer strand is bound at the priming site (P site), adjacent to the polymerase active site.<sup><a href="#bib7" id="bbib7" class="intra_ref">7</a>, <a href="#bib23" id="bbib23" class="intra_ref">23</a>, <a href="#bib24" id="bbib24" class="intra_ref">24</a>, <a href="#bib25" id="bbib25" class="intra_ref">25</a>, <a href="#bib26" id="bbib26" class="intra_ref">26</a>, <a href="#bib27" id="bbib27" class="intra_ref">27</a>&nbsp;and&nbsp;<a href="#bib28" id="bbib28" class="intra_ref">28</a></sup> The initial step in nucleotide incorporation is the binding of the incoming dNTP at the nucleotide binding site (N site) to form a ternary complex.<span id="bbib22"><a href="#bib22" id="ancbbib22" class="intra_ref"><sup>22</sup></a></span> The rate-limiting step in the polymerization reaction is a conformational change in which a portion of the p66 fingers subdomain closes down on the incoming dNTP, which helps to precisely align the 3′-OH of the primer, the α-phosphate of the dNTP, and the polymerase active site.<sup><a href="#bib22" id="bbib22" class="intra_ref">22</a>, <a href="#bib29" id="bbib29" class="intra_ref">29</a>&nbsp;and&nbsp;<a href="#bib30" id="bbib30" class="intra_ref">30</a></sup> The chemical step that follows leads to the formation of a phosphodiester bond between the newly incorporated nucleoside and the primer with the concomitant generation of pyrophosphate. The fingers open to allow the pyrophosphate to leave the active site. In processive DNA synthesis, the nucleic acid substrate must translocate relative to RT to free the nucleotide-binding site so that RT can bind the next incoming dNTP.</p><!--VALIDHTML--><div class="figTblUpiOuter svArticle" id="figure_fig4"><div class="
                    textboxdefault
                "><dl class="figure" id="fig4" data-t="f"><dt class="autoScroll" data-style="height:58px;width:444px;"><a class="figureLink" href="#gr4" title="Full-size image (6 K)"><img style="display: inline;" class="imgLazyJSB figure large" alt="Full-size image (6 K)" src="grey_pxl.gif" data-thumbeid="1-s2.0-S0022283608013442-gr4.sml" data-fulleid="1-s2.0-S0022283608013442-gr4.jpg" data-thumbheight="28" data-thumbwidth="219" data-fullheight="58" data-fullwidth="444" border="0" height="58" width="444"><noscript class="figure large" border="0" alt="Full-size image" data-thumbeid="1-s2.0-S0022283608013442-gr4.sml" data-fulleid="1-s2.0-S0022283608013442-gr4.jpg"><img class="figure large" border="0" alt="Full-size image (6 K)" src="http://origin-ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr4.jpg" data-thumbEID="1-s2.0-S0022283608013442-gr4.sml" data-fullEID="1-s2.0-S0022283608013442-gr4.jpg"></noscript></a></dt><dd id="labelCaptionfig4"><p class="caption">Fig. 4.&nbsp;<span id="">Reaction steps in the mechanism of DNA polymerization by HIV-1 RT.<!--Comment--></span><!--caption--></p></dd><dd class="menuButtonLinks"><div class="btnHolder"><a class="menuTitle" href="#">Figure options</a><div class="down_Btn"></div></div><ul class="menuButton"><li><a href="#fig4" class="viewWS" title="View in workspace">View in workspace</a></li><li><a href="http://origin-ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr4.jpg" title="View full-size image - Opens in new window">Download full-size image</a></li><li><a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0022283608013442&amp;_image=1-s2.0-S0022283608013442-gr4_lrg.jpg&amp;_ba=&amp;_fmt=full&amp;_orig=na&amp;_issn=00222836&amp;_pii=S0022283608013442&amp;_isHiQual=Y&amp;_acct=C000049402&amp;_version=1&amp;_urlVersion=0&amp;_userid=963121&amp;md5=8df4432d78e5d1c6ca1f048c071cab97" title="View high quality image - Opens in new window">Download high-quality image (49 K)</a></li><li><a class="ppt" title="Download as PowerPoint slide" href="http://services.elsevier.com.libgate.library.nuigalway.ie/SDWebExport/export/figure/S0022283608013442/1-s2.0-S0022283608013442-gr4.jpg/ppt">Download as PowerPoint slide</a></li></ul></dd></dl></div></div><!--VALIDHTML--><p class="svArticle section" id="">Both steady state and pre-steady state kinetic experiments have been used to characterize dNTP binding and nucleotide incorporation.<sup><a href="#bib29" id="bbib29" class="intra_ref">29</a>&nbsp;and&nbsp;<a href="#bib30" id="bbib30" class="intra_ref">30</a></sup> Generally, dNTP binding has <em>K</em><sub>d</sub> in the low micromolar range. It was shown that dNTP binding is slightly more efficient on a DNA/DNA template-primer (<em>K</em><sub>d</sub> 4&nbsp;μM) than to an RNA/DNA substrate (<em>K</em><sub>d</sub> 14&nbsp;μM). The rate of incorporation of the bound nucleotide (<em>k</em><sub>pol</sub>) is slightly increased when RNA/DNA is the substrate (∼2-fold increase compared to DNA/DNA). Transient kinetics studies have shown that the rate-limiting step for single nucleotide incorporation is a conformational change (fingers closing down on the active site). However, steady state experiments have shown that the overall reaction is limited by the dissociation rate of RT from the nucleic acid substrate. <sup><a href="#bib29" id="bbib29" class="intra_ref">29</a>&nbsp;and&nbsp;<a href="#bib31" id="bbib31" class="intra_ref">31</a></sup></p><!--VALIDHTML--><p class="svArticle section" id="">The approved NRTIs all lack a 3′-OH and act as chain terminators when they are incorporated into viral DNA by RT. If sufficient dNTPs are present, incorporation of a chain-terminator into DNA may result in the formation of a stable dead-end complex (DEC), where RT has incorporated the NRTI, the end of the primer (with the NRTI at the 3′ end) has been translocated, and the incoming dNTP is bound in a stable closed complex. This tighter binding of the nucleic acid substrate in the closed conformation can be demonstrated by electrophoresis in a native polyacrylamide gel.<span id="bbib32"><a href="#bib32" id="ancbbib32" class="intra_ref"><sup>32</sup></a></span></p><!--VALIDHTML--><p class="svArticle section" id="">The chemical step requires two divalent metal ions and there is good reason to believe that the normal <em>in vivo</em> metals are both Mg<sup>2+</sup>. The metals coordinate the oxygens of all three phosphates of the incoming dNTP and the side chains of the three catalytic Asp residues. (110,185,186) Metal coordination at the polymerase site facilitates the attack of the 3′-OH on the α-phosphate of the incoming nucleotide by activating the hydroxyl group (<span id="bfig3"><a href="#fig3" id="ancbfig3" class="intra_ref">Fig. 3</a></span>). The two metal ions also stabilize the charge of the reaction intermediates.<span id="bbib33"><a href="#bib33" id="ancbbib33" class="intra_ref"><sup>33</sup></a></span></p><!--VALIDHTML--><p class="svArticle section" id="">Insights into the molecular details of the translocation step that follows incorporation of the nucleotide during processive DNA synthesis have been provided by crystal structures of RT covalently cross-linked to nucleic acid in pre- and post-translocation states (N and P complexes, respectively).<span id="bbib34"><a href="#bib34" id="ancbbib34" class="intra_ref"><sup>34</sup></a></span> A comparison of these structures indicates unfavorable interactions between the conserved YMDD loop of the polymerase active site and the terminal phosphate of the extended primer terminus that may contribute to translocation after dNMP incorporation. It has not been possible to determine whether translocation precedes the release of pyrophosphate product. Recently, using a site-specific footprinting assay, it was demonstrated that phosphonoformic acid, an analog of pyrophosphate, stabilizes the pre-translocation complex (N complex), suggesting that the pyrophosphate product is released before translocation of RT.<span id="bbib27"><a href="#bib27" id="ancbbib27" class="intra_ref"><sup>27</sup></a></span></p><!--VALIDHTML--><h3 class="svArticle" id="section_lbl3.2">Molecular mechanisms of HIV-1 reverse transcriptase inhibition</h3><h4 class="svArticle" id="section_lbl3.2.1">HIV-1 RT inhibitors</h4><p class="svArticle section" id="">Because of its central role in the life-cycle of HIV, RT has been a key target for the development of antiviral therapies. Nearly half of the anti-AIDS drugs target the polymerase activity of RT<span id="bfn2"><a href="#fn2" id="ancbfn2" class="intra_ref">†</a></span>. The approved anti-RT drugs belong to one of two broad classes: nucleoside RT inhibitors (NRTIs) (<span id="bfig5"><a href="#fig5" id="ancbfig5" class="intra_ref">Fig. 5</a></span>) and nonnucleoside RT inhibitors (NNRTIs) (<span id="bfig6"><a href="#fig6" id="ancbfig6" class="intra_ref">Fig. 6</a></span>). Most standard three-drug regimens involve two NRTIs combined with either a PR inhibitor or an NNRTI; other combination therapies, some of which involve the entry/fusion inhibitors, are used to treat some of the patients who have failed the standard therapies. The following NRTIs have been approved by the US Food and Drug Administration: AZT-zidovudine, 3TC-lamivudine, FTC-emtricitabine, ddI-didanosine, and ABC-abacavir. d4T-Stavudine and ddC-zalcitabine are no longer used in therapies. TFV-tenofovir is a nucleo<em>t</em>ide RT inhibitor and, for simplicity, it will be discussed with the NRTIs. Some of these inhibitors are available as combinations, for example TFV+FTC (truvada), AZT&nbsp;+&nbsp;3TC (combivir), and ABC&nbsp;+&nbsp;3TC&nbsp;+&nbsp;AZT (trizivir)<span id="bfn3"><a href="#fn3" id="ancbfn3" class="intra_ref">‡</a></span>. Four NNRTIs have been approved for the treatment of AIDS (the first generation NNRTIs are nevirapine and delavirdine, the second generation NNRTI is efavirenz, and the third generation is etravirine (TMC125)). Additional NNRTIs are currently in late stage clinical trials. These include TMC278 (rilpivirine), UK-453,061, and RDEA-806; these third generation NNRTIs have increased effectiveness against drug-resistant HIV strains. Both tenofovir and TMC120 (dapivirine) are in large-scale trials as a chemopreventives; specific antiretroviral chemoprevention strategies have great potential to reduce the sexual transmission of HIV. While some specific inhibitors of the RNase H activity of RT have been described, none has yet been approved for the treatment of HIV infections.</p><!--VALIDHTML--><div class="figTblUpiOuter svArticle" id="figure_fig5"><div class="
                    textboxdefault
                "><dl class="figure" id="fig5" data-t="f"><dt class="autoScroll" data-style="height:432px;width:580px;"><a class="figureLink" href="#gr5" title="Full-size image (35 K)"><img style="display: inline;" class="imgLazyJSB figure large smallImg" alt="Full-size image (35 K)" src="grey_pxl.gif" data-thumbeid="1-s2.0-S0022283608013442-gr5.sml" data-fulleid="1-s2.0-S0022283608013442-gr5.jpg" data-thumbheight="163" data-thumbwidth="219" data-fullheight="432" data-fullwidth="580" border="0"><noscript class="figure large" border="0" alt="Full-size image" data-thumbeid="1-s2.0-S0022283608013442-gr5.sml" data-fulleid="1-s2.0-S0022283608013442-gr5.jpg"><img class="figure large" border="0" alt="Full-size image (35 K)" src="http://origin-ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr5.jpg" data-thumbEID="1-s2.0-S0022283608013442-gr5.sml" data-fullEID="1-s2.0-S0022283608013442-gr5.jpg"></noscript></a></dt><dd id="labelCaptionfig5"><p class="caption">Fig. 5.&nbsp;<span id="">Chemical structures of NRTIs.<!--Comment--></span><!--caption--></p></dd><dd class="menuButtonLinks"><div class="btnHolder"><a class="menuTitle" href="#">Figure options</a><div class="down_Btn"></div></div><ul class="menuButton"><li><a href="#fig5" class="viewWS" title="View in workspace">View in workspace</a></li><li><a href="http://origin-ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr5.jpg" title="View full-size image - Opens in new window">Download full-size image</a></li><li><a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0022283608013442&amp;_image=1-s2.0-S0022283608013442-gr5_lrg.jpg&amp;_ba=&amp;_fmt=full&amp;_orig=na&amp;_issn=00222836&amp;_pii=S0022283608013442&amp;_isHiQual=Y&amp;_acct=C000049402&amp;_version=1&amp;_urlVersion=0&amp;_userid=963121&amp;md5=c2ac795054d44221d99e2761d623404c" title="View high quality image - Opens in new window">Download high-quality image (288 K)</a></li><li><a class="ppt" title="Download as PowerPoint slide" href="http://services.elsevier.com.libgate.library.nuigalway.ie/SDWebExport/export/figure/S0022283608013442/1-s2.0-S0022283608013442-gr5.jpg/ppt">Download as PowerPoint slide</a></li></ul></dd></dl></div></div><!--VALIDHTML--><div class="figTblUpiOuter svArticle" id="figure_fig6"><div class="
                    textboxdefault
                "><dl class="figure" id="fig6" data-t="f"><dt class="autoScroll" data-style="height:521px;width:647px;"><a class="figureLink" href="#gr6" title="Full-size image (43 K)"><img style="display: inline;" class="imgLazyJSB figure large smallImg" alt="Full-size image (43 K)" src="grey_pxl.gif" data-thumbeid="1-s2.0-S0022283608013442-gr6.sml" data-fulleid="1-s2.0-S0022283608013442-gr6.jpg" data-thumbheight="164" data-thumbwidth="204" data-fullheight="521" data-fullwidth="647" border="0"><noscript class="figure large" border="0" alt="Full-size image" data-thumbeid="1-s2.0-S0022283608013442-gr6.sml" data-fulleid="1-s2.0-S0022283608013442-gr6.jpg"><img class="figure large" border="0" alt="Full-size image (43 K)" src="http://origin-ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr6.jpg" data-thumbEID="1-s2.0-S0022283608013442-gr6.sml" data-fullEID="1-s2.0-S0022283608013442-gr6.jpg"></noscript></a></dt><dd id="labelCaptionfig6"><p class="caption">Fig. 6.&nbsp;<span id="">Chemical structures of NNRTIs.<!--Comment--></span><!--caption--></p></dd><dd class="menuButtonLinks"><div class="btnHolder"><a class="menuTitle" href="#">Figure options</a><div class="down_Btn"></div></div><ul class="menuButton"><li><a href="#fig6" class="viewWS" title="View in workspace">View in workspace</a></li><li><a href="http://origin-ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr6.jpg" title="View full-size image - Opens in new window">Download full-size image</a></li><li><a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0022283608013442&amp;_image=1-s2.0-S0022283608013442-gr6_lrg.jpg&amp;_ba=&amp;_fmt=full&amp;_orig=na&amp;_issn=00222836&amp;_pii=S0022283608013442&amp;_isHiQual=Y&amp;_acct=C000049402&amp;_version=1&amp;_urlVersion=0&amp;_userid=963121&amp;md5=2f4360a279b46273ab2ac6db0456630b" title="View high quality image - Opens in new window">Download high-quality image (345 K)</a></li><li><a class="ppt" title="Download as PowerPoint slide" href="http://services.elsevier.com.libgate.library.nuigalway.ie/SDWebExport/export/figure/S0022283608013442/1-s2.0-S0022283608013442-gr6.jpg/ppt">Download as PowerPoint slide</a></li></ul></dd></dl></div></div><!--VALIDHTML--><h4 class="svArticle" id="section_lbl3.2.2">Inhibition by nucleoside reverse transcriptase inhibitors (NRTIs)</h4><p class="svArticle section" id="">NRTIs are structurally diverse analogs of the natural substrates of DNA synthesis (<span id="bfig5"><a href="#fig5" id="ancbfig5" class="intra_ref">Fig. 5</a></span>). As already discussed, the approved NRTIs all lack the 3′-OH and act as chain terminators when incorporated in viral DNA by RT. For NRTIs to be effective against HIV, they must be taken up by the host cell and then phosphorylated by cellular enzymes to convert them to their active form, the NRTI triphosphates. The efficiency of this conversion to the active metabolite and the stability of NRTIs (and their triphosphates) in the presence of catabolic enzymes are important considerations in antiviral therapies, because these factors help determine the concentration of the inhibitor in the bloodstream that is required for the NRTI to be effective. Although all NRTIs are analogs of normal nucleosides, not all NRTIs are good substrates for the cellular kinases.<span id="bbib35"><a href="#bib35" id="ancbbib35" class="intra_ref"><sup>35</sup></a></span> For example, AZTTP levels are low because the addition of the second and third phosphates is limiting.<span id="bbib36"><a href="#bib36" id="ancbbib36" class="intra_ref"><sup>36</sup></a></span> However, for other NRTIs, the addition of the first phosphate is the limiting step. For this reason it can be helpful to develop prodrug versions of NRTIs that contain the first phosphate. Prodrugs are modified by the addition of hydrophobic moieties that allow them to enter cells. The hydrophobic groups are designed to be removed once the compound enters the cell. For example, TFV is given to patients as a prodrug phosphonate nucleotide. As such, TFV needs the addition of two, instead of three, phosphates (<span id="bfig5"><a href="#fig5" id="ancbfig5" class="intra_ref">Fig. 5</a></span>). The NRTIs ABC, and ddI (<span id="bfig5"><a href="#fig5" id="ancbfig5" class="intra_ref">Fig. 5</a></span>) are modified by cellular enzymes to carbovir-triphosphate and dideoxyadenosine-triphosphate (ddATP), the respective active metabolites. RT incorporates NRTI triphosphates with variable efficiencies. For example, RT incorporates AZTTP, a thymidine analog, almost as efficiently as dTTP.<span id="bbib37"><a href="#bib37" id="ancbbib37" class="intra_ref"><sup>37</sup></a></span> In contrast, ddCTP is incorporated less efficiently than dCTP.<span id="bbib38"><a href="#bib38" id="ancbbib38" class="intra_ref"><sup>38</sup></a></span> 3TC-TP, another cytidine analog, is given to patients as the <span class="smallcaps">l</span> rather than the <span class="smallcaps">d</span> enantiomer, and is incorporated less efficiently than either dCTP or ddCTP.<span id="bbib13"><a href="#bib13" id="ancbbib13" class="intra_ref"><sup>13</sup></a></span></p><!--VALIDHTML--><h4 class="svArticle" id="section_lbl3.2.3">Inhibition by non-nucleoside reverse transcriptase inhibitors (NNRTIs)</h4><p class="svArticle section" id="">NNRTIs (<span id="bfig6"><a href="#fig6" id="ancbfig6" class="intra_ref">Fig. 6</a></span>) bind at the NNRTI-binding pocket (NNIBP), a hydrophobic pocket adjacent to the polymerase active site (∼&nbsp;10&nbsp;Å) (<span id="bfig7"><a href="#fig7" id="ancbfig7" class="intra_ref">Fig. 7</a></span>). The NNIBP consists of residues L100, K101, K103, V106, T107, V108, V179, Y181, Y188, V189, G190, F227, W229, L234, and Y318 of p66 and E138 of p51.<sup><a href="#bib6" id="bbib6" class="intra_ref">6</a>&nbsp;and&nbsp;<a href="#bib39" id="bbib39" class="intra_ref">39</a></sup></p><!--VALIDHTML--><div class="figTblUpiOuter svArticle" id="figure_fig7"><div class="
                    textboxdefault
                "><dl class="figure" id="fig7" data-t="f"><dt class="autoScroll" data-style="height:347px;width:329px;"><a class="figureLink" href="#gr7" title="Full-size image (38 K)"><img style="display: inline;" class="imgLazyJSB figure large" alt="Full-size image (38 K)" src="grey_pxl.gif" data-thumbeid="1-s2.0-S0022283608013442-gr7.sml" data-fulleid="1-s2.0-S0022283608013442-gr7.jpg" data-thumbheight="163" data-thumbwidth="155" data-fullheight="347" data-fullwidth="329" border="0" height="347" width="329"><noscript class="figure large" border="0" alt="Full-size image" data-thumbeid="1-s2.0-S0022283608013442-gr7.sml" data-fulleid="1-s2.0-S0022283608013442-gr7.jpg"><img class="figure large" border="0" alt="Full-size image (38 K)" src="http://origin-ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr7.jpg" data-thumbEID="1-s2.0-S0022283608013442-gr7.sml" data-fullEID="1-s2.0-S0022283608013442-gr7.jpg"></noscript></a></dt><dd id="labelCaptionfig7"><p class="caption">Fig. 7.&nbsp;<span id="">Ribbon representation of the NNRTI-binding pocket, showing the residues where NNRTI-resistance mutations occur.<!--Comment--></span><!--caption--></p></dd><dd class="menuButtonLinks"><div class="btnHolder"><a class="menuTitle" href="#">Figure options</a><div class="down_Btn"></div></div><ul class="menuButton"><li><a href="#fig7" class="viewWS" title="View in workspace">View in workspace</a></li><li><a href="http://origin-ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr7.jpg" title="View full-size image - Opens in new window">Download full-size image</a></li><li><a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0022283608013442&amp;_image=1-s2.0-S0022283608013442-gr7_lrg.jpg&amp;_ba=&amp;_fmt=full&amp;_orig=na&amp;_issn=00222836&amp;_pii=S0022283608013442&amp;_isHiQual=Y&amp;_acct=C000049402&amp;_version=1&amp;_urlVersion=0&amp;_userid=963121&amp;md5=73847d0e4c4ce3d53d647994a4255eaf" title="View high quality image - Opens in new window">Download high-quality image (358 K)</a></li><li><a class="ppt" title="Download as PowerPoint slide" href="http://services.elsevier.com.libgate.library.nuigalway.ie/SDWebExport/export/figure/S0022283608013442/1-s2.0-S0022283608013442-gr7.jpg/ppt">Download as PowerPoint slide</a></li></ul></dd></dl></div></div><!--VALIDHTML--><p class="svArticle section" id="">Biochemical data have shown that NNRTIs are noncompetitive inhibitors and do not interfere directly with the binding of either the dNTP or the nucleic acid substrates of RT. Pre-steady state kinetic analysis of single nucleotide addition in the presence of NNRTIs has shown that binding of NNRTI interferes with the chemical step of DNA synthesis.<sup><a href="#bib29" id="bbib29" class="intra_ref">29</a>&nbsp;and&nbsp;<a href="#bib30" id="bbib30" class="intra_ref">30</a></sup> However, the molecular details of NNRTI inhibition are not clearly understood.</p><!--VALIDHTML--><p class="svArticle section" id="">Structural studies of HIV-1 RT have shown that: (1) the NNIBP does not exist in structures of HIV-1 RT that have no bound NNRTI; (2) the NNIBP is created by large structural rearrangements, particularly an extended conformation of the primer grip and rearrangements of the aromatic-ring containing residues Y181 and Y188 (<span id="bfig7"><a href="#fig7" id="ancbfig7" class="intra_ref">Fig. 7</a></span>) and this structural rearrangement locks the p66 thumb and fingers in their hyper-extended conformations;<sup><a href="#bib6" id="bbib6" class="intra_ref">6</a>, <a href="#bib39" id="bbib39" class="intra_ref">39</a>, <a href="#bib40" id="bbib40" class="intra_ref">40</a>, <a href="#bib41" id="bbib41" class="intra_ref">41</a>&nbsp;and&nbsp;<a href="#bib42" id="bbib42" class="intra_ref">42</a></sup> (3) the NNIBP is created primarily from the p66 subunit (near the polymerase active site); (4) p51, which contains amino acids identical with the polymerase domain of p66, does not have a second NNIBP; (5) the NNRTI resistance mutations are located in and around the NNIBP;<sup><a href="#bib6" id="bbib6" class="intra_ref">6</a>, <a href="#bib39" id="bbib39" class="intra_ref">39</a>&nbsp;and&nbsp;<a href="#bib43" id="bbib43" class="intra_ref">43</a></sup> and (6) there is no metal co-factor bound at the active site with persuasive coordination geometry when an NNRTI is bound to RT.<span id="bbib44"><a href="#bib44" id="ancbbib44" class="intra_ref"><sup>44</sup></a></span></p><!--VALIDHTML--><p class="svArticle section" id="">The structural data suggest that NNRTI-binding distorts the position of the primer grip, thus affecting the alignment of the primer terminus with the polymerase active site, which could affect the chemical step of viral DNA synthesis. There is also structural evidence to suggest that NNRTI binding affects the conformation of the catalytic carboxylates that bind the metal cofactors.<span id="bbib44"><a href="#bib44" id="ancbbib44" class="intra_ref"><sup>44</sup></a></span> Specifically, the β9–β10 loop that contains the conserved YMDD motif (<span id="bfig3"><a href="#fig3" id="ancbfig3" class="intra_ref">Fig. 3</a></span>) assumes different conformations during the course of DNA polymerization.<span id="bbib34"><a href="#bib34" id="ancbbib34" class="intra_ref"><sup>34</sup></a></span> In the RT/DNA/dNTP complex the YMDD loop flexes and moves “down” by 2&nbsp;Å to bind dNTP and the metal ions. However, in the RT/NNRTI complexes, the YMDD loop always assumes the opposite conformation (“up”). This suggests that NNRTI binding restricts a conformational change of the YMDD loop that is needed to attain the metal-binding conformation and that this restriction could affect the translocation of the nucleic acid that normally occurs after the incorporation of each nucleotide.<span id="bbib44"><a href="#bib44" id="ancbbib44" class="intra_ref"><sup>44</sup></a></span></p><!--VALIDHTML--><h3 class="svArticle" id="section_lbl3.3">Molecular mechanisms of resistance</h3><h4 class="svArticle" id="section_lbl3.3.1">NRTI resistance</h4><p class="svArticle section" id="">Drug resistance remains a central challenge in HIV therapies. Because resistant viruses can be transmitted, the prevalence of resistant viruses is increasing in untreated HIV-1 patients. For the virus to replicate and be transmitted, HIV-1 RT must be able to complete viral DNA synthesis, and NRTI-resistant RTs must retain the ability to incorporate normal dNTPs with reasonable efficiency. This means that NRTI resistance involves enhanced discrimination between normal nucleosides and NRTIs. Two basic types of NRTI-resistance mechanisms are known for HIV-1 RT: (1) one resistance mechanism (exclusion) involves enhanced discrimination at the time the NRTI-TP is incorporated. The M184V/I mutations provide a clear example of the exclusion mechanism, M184V/I selectively reduce the incorporation of 3TC and FTC by steric hindrance.<sup><a href="#bib22" id="bbib22" class="intra_ref">22</a>, <a href="#bib45" id="bbib45" class="intra_ref">45</a>&nbsp;and&nbsp;<a href="#bib46" id="bbib46" class="intra_ref">46</a></sup> (2) The second mechanism involves the selective removal of the NRTI from the end of the viral DNA after it has been incorporated by RT.<sup><a href="#bib47" id="bbib47" class="intra_ref">47</a>, <a href="#bib48" id="bbib48" class="intra_ref">48</a>, <a href="#bib49" id="bbib49" class="intra_ref">49</a>&nbsp;and&nbsp;<a href="#bib50" id="bbib50" class="intra_ref">50</a></sup> This is the excision mechanism; a well-studied example involves AZT resistance caused by a set of mutations including M41L, D67N, K70R, L210W, T215F/Y, and K219E/Q (these mutations will be referred to collectively as AZT resistant or AZTr; they are referred to also as thymidine-analog mutations, TAMs, or excision-enhancing mutations, EEMs). Although the HIV-1 RTs isolated from AZT-resistant viruses in patients do not ordinarily have all of the AZTr mutations, combinations of these mutations give rise to high levels of resistance to AZT and to much lower levels of resistance to some other NRTIs.<sup><a href="#bib51" id="bbib51" class="intra_ref">51</a>, <a href="#bib52" id="bbib52" class="intra_ref">52</a>, <a href="#bib53" id="bbib53" class="intra_ref">53</a>, <a href="#bib54" id="bbib54" class="intra_ref">54</a>&nbsp;and&nbsp;<a href="#bib55" id="bbib55" class="intra_ref">55</a></sup> HIV-1 RTs carrying the AZTr mutations can acquire additional mutations that enhance their ability to excise AZT and allow these RTs to excise most other NRTIs more efficiently.</p><!--VALIDHTML--><h4 class="svArticle" id="section_lbl3.3.2">NRTI exclusion</h4><p class="svArticle section" id="">In addition to the M184V/I mutations that cause resistance to 3TC and FTC by the exclusion mechanism (which has already been discussed), the mutations L74V (ddI resistance),<span id="bbib56"><a href="#bib56" id="ancbbib56" class="intra_ref"><sup>56</sup></a></span> K65R (TFV and ddI resistance)<sup><a href="#bib57" id="bbib57" class="intra_ref">57</a>, <a href="#bib58" id="bbib58" class="intra_ref">58</a>&nbsp;and&nbsp;<a href="#bib59" id="bbib59" class="intra_ref">59</a></sup> and a complex set of several mutations that involves Q151M (resistance to most NRTIs) cause resistance by the exclusion mechanism.<span id="bbib60"><a href="#bib60" id="ancbbib60" class="intra_ref"><sup>60</sup></a></span> All of the amino acids involved in resistance by the exclusion mechanism are in the fingers or the palm of RT; all are in positions that could affect the binding of an incoming dNTP. We do not yet have as detailed a picture for the exact mechanisms of resistance for these mutations as we do for the M184V/I mutations. For K65R, biochemical studies have shown that it has a decreased rate of nucleotide incorporation and markedly decreased rate of TFV incorporation.<sup><a href="#bib61" id="bbib61" class="intra_ref">61</a>&nbsp;and&nbsp;<a href="#bib62" id="bbib62" class="intra_ref">62</a></sup> We know also that the K65R mutation causes hypersusceptibility to AZT by reducing the efficiency of ATP-mediated excision, which counteracts a decreased rate of AZT incorporation.<sup><a href="#bib63" id="bbib63" class="intra_ref">63</a>&nbsp;and&nbsp;<a href="#bib64" id="bbib64" class="intra_ref">64</a></sup> Finally, it has been shown that in the background of K65R, M184V causes partial resensitization to TFV.<span id="bbib66"><a href="#bib66" id="ancbbib66" class="intra_ref"><sup>66</sup></a></span> Q151 interacts with the 3′-OH of a normal triphosphate.<span id="bbib22"><a href="#bib22" id="ancbbib22" class="intra_ref"><sup>22</sup></a></span> It appears that the Q151M mutation, and the additional mutations that usually accompany the Q151M mutation, cause differences in the hydrogen bonding network between the deoxyribose of an incoming dNTP and the enzyme, and that these changes enhance the importance of the interactions of the enzyme with the 3′-OH.<span id="bbib65"><a href="#bib65" id="ancbbib65" class="intra_ref"><sup>65</sup></a></span> This allows the mutant enzyme to better discriminate between normal dNTPs, which have the 3′-OH, and NRTIs, which do not have a 3′-OH. Presumably L74V<span id="bbib67"><a href="#bib67" id="ancbbib67" class="intra_ref"><sup>67</sup></a></span> (and the less common mutation V75T<span id="bbib68"><a href="#bib68" id="ancbbib68" class="intra_ref"><sup>68</sup></a></span>) both disrupt the hydrogen bonding network that involves the 3′-OH, although the details of how this occurs are not well defined,<span id="bbib69"><a href="#bib69" id="ancbbib69" class="intra_ref"><sup>69</sup></a></span> and the discrimination is selective for ddI (which is converted to ddA in cells) and, to a lesser degree, for ddC. What is striking is that, despite the fact that the mechanism apparently involves enhanced recognition of the 3′-OH, the L74V enzyme is not resistant to other NRTIs.<span id="bbib70"><a href="#bib70" id="ancbbib70" class="intra_ref"><sup>70</sup></a></span></p><!--VALIDHTML--><h4 class="svArticle" id="section_lbl3.3.3">NRTI excision</h4><p class="svArticle section" id="">The first example of NRTI resistance to be described was resistance to AZT. However, it took more than 10 years for the underlying molecular resistance mechanism to be understood. This was, at least in part, because the mechanism was unexpected. An AZT-resistant RT incorporates AZT-TP as efficiently as wild-type HIV-1 RT. However, AZT-resistant RTs have an enhanced ability to remove the incorporated AZT-MP from the 5′ end of the template strand. The underlying mechanism involves ATP-dependent excision; the excision mechanism is related to the normal mechanism of polymerization run in reverse.<sup><a href="#bib47" id="bbib47" class="intra_ref">47</a>, <a href="#bib48" id="bbib48" class="intra_ref">48</a>&nbsp;and&nbsp;<a href="#bib49" id="bbib49" class="intra_ref">49</a></sup> Although there was some initial controversy about whether ATP or PPi is the pyrophosphate donor in the excision reaction that gives rise to AZT resistance, it is now clear that the excision reaction that gives rise to AZT resistance <em>in vivo</em> uses ATP as the pyrophosphate donor, rather than PPi, which is the normal product of the polymerization reaction. These proposals answered some important questions but left other questions unanswered. (1) If the mechanism is ATP-dependent excision, why is AZT excised so much more efficiently than other NRTIs? (2) How do the AZT-resistance mutations cause enhanced excision? In 2001, Boyer <em>et al</em>.<span id="bbib50"><a href="#bib50" id="ancbbib50" class="intra_ref"><sup>50</sup></a></span> proposed a specific model in which ATP is the pyrophosphate donor, and at least some of the mutations that cause AZT resistance enhance the appropriate binding of ATP. In particular, a key mutation, T215F/Y, is selected because an F or a Y at position 215 can stack with the adenine ring of ATP. Although previous studies have shown little or no difference in ATP binding in the presence of AZT resistance mutations, <sup><a href="#bib71" id="bbib71" class="intra_ref">71</a>&nbsp;and&nbsp;<a href="#bib72" id="bbib72" class="intra_ref">72</a></sup> Dharmasena <em>et al</em>.<span id="bbib73"><a href="#bib73" id="ancbbib73" class="intra_ref"><sup>73</sup></a></span> show, in experiments using the excision product AZTMP-PPPA, that the primary mutations that give rise to AZT resistance enhance the ability of RT to bind ATP. For the excision reaction to occur, the end of the primer strand must be at the polymerase active site (also called the N, or nucleotide binding site) (<span id="bfig3"><a href="#fig3" id="ancbfig3" class="intra_ref">Fig. 3</a></span>). Normally, translocation moves an incorporated dNTP or NRTI-TP from the N site to the P or priming site so that the next incoming dNTP can bind. In most cases the concentration of dNTPs in cells is high enough that the incoming dNTP can be bound. NRTIs lack the 3′-OH; so that, if a NRTI-TP has been incorporated and the end of the primer is translocated to the P site, the next incoming dNTP can be bound, but cannot be incorporated. At reasonable concentrations of dNTPs, this is the inhibited state of an RT that has just incorporated an NRTI. The recently incorporated NRTI-MP is in the P site and the enzyme forms a closed complex with the incoming dNTP bound at the N site. The closed complex is relatively stable and, as was just discussed, excision requires that the NRTI be at the N, and not the P site. This is why an AZTr RT cannot efficiently excise most NRTI-MPs from the end of the primer strand: The NRTI-MP at the end of the primer cannot be translocated from the P to the N site because the bound incoming dNTP blocks the N site. In the Boyer <em>et al.</em> model, AZT-MP is excised more efficiently than are other NRTI-MPs because, when an AZT-MP is the nucleotide at the 3′ end of the primer, the long azido group of the AZT-MP interferes with the appropriate binding of the incoming dNTP, which destabilizes the closed complex. Because the closed complex is less stable, an AZT-MP terminated primer has better access to the N site (active site), where it can be excised. There are structural data that support this part of the model.<span id="bbib34"><a href="#bib34" id="ancbbib34" class="intra_ref"><sup>34</sup></a></span> In addition, the model is supported both by the observations that the aromaticity of the nucleobase on the pyrophosphate donor determines how well it functions in an excision reaction and by the fact that diphosphate nucleotides are poor excision substrates (presumably because when the nucleobase binds to T215F/Y the phosphates are not in position to react with the end of the primer).<span id="bbib74"><a href="#bib74" id="ancbbib74" class="intra_ref"><sup>74</sup></a></span></p><!--VALIDHTML--><p class="svArticle section" id="">There are complexities to the excision reaction that are still not completely understood. For example, many other nucleoside analogs are excised less efficiently than AZTMP even in the absence of an incoming dNTP.<span id="bbib75"><a href="#bib75" id="ancbbib75" class="intra_ref"><sup>75</sup></a></span> One possibility is that an AZT-terminated primer is a good substrate for excision because the incoming dNTP binds poorly<sup><a href="#bib32" id="bbib32" class="intra_ref">32</a>, <a href="#bib49" id="bbib49" class="intra_ref">49</a>&nbsp;and&nbsp;<a href="#bib50" id="bbib50" class="intra_ref">50</a></sup> and because, in the absence of an incoming dNTP, an AZT-terminated primer preferentially binds to the N site.<sup><a href="#bib26" id="bbib26" class="intra_ref">26</a>&nbsp;and&nbsp;<a href="#bib28" id="bbib28" class="intra_ref">28</a></sup> However, this may not be the only explanation, and it is entirely possible that there are other factors that affect the efficiency of excision. For example, it was shown that not all 3′-azido analogs are good substrates for excision, which suggests that other components of the primer terminus, such as the base, also contribute to the efficiency of excision.<span id="bbib76"><a href="#bib76" id="ancbbib76" class="intra_ref"><sup>76</sup></a></span></p><!--VALIDHTML--><p class="svArticle section" id="">As already mentioned, AZTr RTs can acquire additional mutations that allow the RT to excise a much broader set of NRTIs than does AZTr RT. One set of mutations that enhance the breadth and efficiency of NRTI excision by AZTr RTs involves the insertion of amino acids near position 69 of the fingers subdomain of HIV-1 RT, these insertions are accompanied by mutations of the adjacent amino acids.<sup><a href="#bib77" id="bbib77" class="intra_ref">77</a>, <a href="#bib78" id="bbib78" class="intra_ref">78</a>, <a href="#bib79" id="bbib79" class="intra_ref">79</a>&nbsp;and&nbsp;<a href="#bib80" id="bbib80" class="intra_ref">80</a></sup> There are a number of related “fingers insertion” mutations.<sup><a href="#bib81" id="bbib81" class="intra_ref">81</a>, <a href="#bib82" id="bbib82" class="intra_ref">82</a>, <a href="#bib83" id="bbib83" class="intra_ref">83</a>, <a href="#bib84" id="bbib84" class="intra_ref">84</a>, <a href="#bib85" id="bbib85" class="intra_ref">85</a>, <a href="#bib86" id="bbib86" class="intra_ref">86</a>&nbsp;and&nbsp;<a href="#bib87" id="bbib87" class="intra_ref">87</a></sup> The insertions are in the β3–β4 loop of the fingers, which is the loop that closes down on the incoming dNTP to form the closed complex. The insertion mutations apparently broadly destabilize the formation of dNTP-induced stable closed complexes for primers terminated with a variety of NRTI-MPs so that, in the presence of physiological dNTP concentrations, excision of these NRTIs would be inhibited to a lesser extent than with WT RT.<sup><a href="#bib77" id="bbib77" class="intra_ref">77</a>&nbsp;and&nbsp;<a href="#bib78" id="bbib78" class="intra_ref">78</a></sup> However, the finger mutations significantly enhance excision in the absence of dNTPs; there are data to show that insertion of SG or AG can increase the rate of excision directly.<span id="bbib78"><a href="#bib78" id="ancbbib78" class="intra_ref"><sup>78</sup></a></span> RTs acquire the fingers insertion mutations only if they already have AZTr mutations, and are able to bind ATP efficiently and appropriately for the excision. The addition of the fingers insertion mutations allows a much broader array of incorporated NRTI-MPs access to the N site where they can be excised.<span id="bbib26"><a href="#bib26" id="ancbbib26" class="intra_ref"><sup>26</sup></a></span> Conversely, there are NRTI and NNRTI resistance mutations that reduce the ability of HIV-1 RT to excise AZT (and other NRTIs). For example, the M184V/I mutations that give rise to 3TC/FTC resistance also reduce the excision efficiency of WT and drug-resistant HIV-1 RTs,<sup><a href="#bib88" id="bbib88" class="intra_ref">88</a>&nbsp;and&nbsp;<a href="#bib89" id="bbib89" class="intra_ref">89</a></sup> although another study that used a different template-primer sequence did not show a significant reduction in excision.<span id="bbib90"><a href="#bib90" id="ancbbib90" class="intra_ref"><sup>90</sup></a></span></p><!--VALIDHTML--><p class="svArticle section" id="">One of the connection subdomain mutations, N348I, was shown to confer moderate cross-resistance to both NRTIs and NNRTIs.<sup><a href="#bib91" id="bbib91" class="intra_ref">91</a>&nbsp;and&nbsp;<a href="#bib92" id="bbib92" class="intra_ref">92</a></sup> This mutation is present in a significant number of clinical isolates, usually in the presence of other NRTI-resistance mutations.</p><!--VALIDHTML--><h4 class="svArticle" id="section_lbl3.3.4">NNRTIs and NNRTI resistance</h4><p class="svArticle section" id="">There are four NNRTI drugs (nevirapine, delavirdine (first generation), efavirenz (second generation), and etravirine (third generation)) that are currently approved for treating HIV-1 infections and several other potent NNRTIs that inhibit HIV-1 at nanomolar concentrations (EC<sub>50</sub>) are in clinical trials. However, there are mutations in RT that can cause resistance to all of the approved NNRTIs. Most of the NNRTI resistance mutations are found in and around the NNIBP. K103N and Y181C are the most frequently observed resistance mutations in patients treated with the approved NNRTIs. Other NNRTI resistance mutations observed in patients include L100I, K101E, V106A, V179D, Y188L, G190A, and P236L; the NNRTI resistance mutations can occur singly, or in combinations. Unfortunately, resistance to first and second generation NNRTIs can evolve relatively quickly. The most promising of the third generation NNRTIs are effective against HIV strains that carry most common single and double mutations; however, viral strains carrying multiple NNRTI-resistance mutations can exhibit significant levels of drug resistance.</p><!--VALIDHTML--><p class="svArticle section" id="">Extensive crystallographic, molecular modeling, and biochemical studies have contributed towards understanding NNRTI drug resistance and the development of better NNRTIs, which is an ongoing effort. Our current understanding suggests that there are at least three broad classes of NNRTI-resistance mechanisms.</p><!--VALIDHTML--><h4 class="svArticle" id="section_lbl3.3.5">Loss/change of key hydrophobic interactions</h4><p class="svArticle section" id="">Amino acid residues Y181, Y188, and F227 are located in the hydrophobic core of the NNIBP (<span id="bfig7"><a href="#fig7" id="ancbfig7" class="intra_ref">Fig. 7</a></span>).<sup><a href="#bib6" id="bbib6" class="intra_ref">6</a>, <a href="#bib40" id="bbib40" class="intra_ref">40</a>, <a href="#bib93" id="bbib93" class="intra_ref">93</a>&nbsp;and&nbsp;<a href="#bib94" id="bbib94" class="intra_ref">94</a></sup> Specific residues in this core have extensive interactions with NNRTIs. Mutations in some of the key residues (Y181C, Y188L, and F227L) cause significant resistance through the loss of the aromatic ring interactions with NNRTIs, which are generally hydrophobic.<sup><a href="#bib5" id="bbib5" class="intra_ref">5</a>, <a href="#bib93" id="bbib93" class="intra_ref">93</a>, <a href="#bib94" id="bbib94" class="intra_ref">94</a>, <a href="#bib95" id="bbib95" class="intra_ref">95</a>, <a href="#bib96" id="bbib96" class="intra_ref">96</a>, <a href="#bib97" id="bbib97" class="intra_ref">97</a>&nbsp;and&nbsp;<a href="#bib98" id="bbib98" class="intra_ref">98</a></sup> This causes high levels of resistance to the first generation NNRTIs, which are relatively rigid. More advanced NNRTIs, however, are designed with so-called strategic flexibility. This intrinsic flexibility makes it possible for the newer drugs to have compensatory interactions with RTs that have mutations causing resistance to the first-generation NNRTIs.<span id="bbib99"><a href="#bib99" id="ancbbib99" class="intra_ref"><sup>99</sup></a></span> This flexibility in the binding has been called wiggling and jiggling, and its structural basis has been described in recent structural studies of wild-type, K103N/Y181C, and L100I/K103N HIV-1 RT complexes with TMC278/rilpivirine.<sup><a href="#bib100" id="bbib100" class="intra_ref">100</a>&nbsp;and&nbsp;<a href="#bib101" id="bbib101" class="intra_ref">101</a></sup> Wiggling and jiggling allow NNRTIs to adapt to changes in the NNIBP caused by resistance mutations; the side chains of the pocket residues adjust to accommodate inhibitor binding in a “shrink-wrap” mode.</p><!--VALIDHTML--><h4 class="svArticle" id="section_lbl3.3.6">Steric hindrance</h4><p class="svArticle section" id="">Amino acid residues L100 and G190 are in the central region of the NNIBP. Mutations in either of these residues cause high levels of resistance to many NNRTIs. The L100I mutation confers resistance by changing the shape of the pocket (the amino acid is β-branched instead of γ-branched),<span id="bbib98"><a href="#bib98" id="ancbbib98" class="intra_ref"><sup>98</sup></a></span> whereas G190A introduces a bulge.<span id="bbib69"><a href="#bib69" id="ancbbib69" class="intra_ref"><sup>69</sup></a></span> For example, HBY 097, which effectively inhibits viruses carrying the Y181C mutation, cannot inhibit viruses that carry the G190A mutation.<span id="bbib40"><a href="#bib40" id="ancbbib40" class="intra_ref"><sup>40</sup></a></span> In the crystal structures of the wild-type RT/HBY 097 complex,<span id="bbib94"><a href="#bib94" id="ancbbib94" class="intra_ref"><sup>94</sup></a></span> the bulky and rigid quinoxaline ring of HBY 097 is near the β6–β10–β9 sheet (that contains G190). The G190A mutation would cause the Cβ atom of A190 to have a steric clash with the quinoxaline moiety and reduce the binding of HBY 097.</p><!--VALIDHTML--><h4 class="svArticle" id="section_lbl3.3.7">Pocket entrance mutations</h4><p class="svArticle section" id="">The K103N and K101E mutations are NNRTI-resistance mutations that frequently cause resistance to first generation NNRTIs. Amino acid residues K101 and K103 are located at the rim of the entrance to the NNIBP with their side chains pointing out. These mutations apparently cause resistance by interfering with the entry of NNRTIs into the pocket.<sup><a href="#bib101" id="bbib101" class="intra_ref">101</a>&nbsp;and&nbsp;<a href="#bib102" id="bbib102" class="intra_ref">102</a></sup> Second generation NNRTIs were designed to overcome this problem. For example, DAPY NNRTIs are able to inhibit the K103N mutant because they interact with the side chain of the mutated N103 residue.<sup><a href="#bib99" id="bbib99" class="intra_ref">99</a>&nbsp;and&nbsp;<a href="#bib103" id="bbib103" class="intra_ref">103</a></sup> However, new drug-resistance mutations, or combinations of mutations, will be selected when the more advanced NNRTIs are used to treat HIV-1-infected patients.</p><!--VALIDHTML--><p class="svArticle section" id="">Recently, it has been reported that a number of mutations in the connection, or RNase H subdomains of RT, can enhance resistance to both NRTI and NNRTI inhibitors of RT.<sup><a href="#bib91" id="bbib91" class="intra_ref">91</a>, <a href="#bib92" id="bbib92" class="intra_ref">92</a>, <a href="#bib104" id="bbib104" class="intra_ref">104</a>&nbsp;and&nbsp;<a href="#bib105" id="bbib105" class="intra_ref">105</a></sup> The biochemical mechanism of this effect is still the subject of investigation; however, it does appear that the reduction in RNase H activity reduces the degradation of the RNA template, which allows more time for the excision mechanism to occur, increasing resistance to AZT.<span id="bbib106"><a href="#bib106" id="ancbbib106" class="intra_ref"><sup>106</sup></a></span> In addition, it was shown that connection domain mutations cause accumulation of short AZT-terminated RNase H cleavage products that are better substrates for the excision reaction, which results in AZT resistance.<span id="bbib107"><a href="#bib107" id="ancbbib107" class="intra_ref"><sup>107</sup></a></span></p><!--VALIDHTML--><h2 class="svArticle" id="section_lbl4">Molecular mechanism of RNase H activity</h2><p class="svArticle section" id="">As mentioned earlier, RNase H is responsible for the degradation of the RNA portion of the RNA/DNA substrate that is formed during minus strand synthesis. It is responsible also for the removal of the priming tRNA<sup><a href="#bib108" id="bbib108" class="intra_ref">108</a>&nbsp;and&nbsp;<a href="#bib109" id="bbib109" class="intra_ref">109</a></sup> and the PPT.<span id="bbib110"><a href="#bib110" id="ancbbib110" class="intra_ref"><sup>110</sup></a></span> Viruses deficient in RNase H activity are non-infectious,<sup><a href="#bib111" id="bbib111" class="intra_ref">111</a>&nbsp;and&nbsp;<a href="#bib112" id="bbib112" class="intra_ref">112</a></sup> making RNase H an interesting target for anti-HIV inhibitors. HIV-1 RNase H was the first segment of RT to be crystallized.<span id="bbib113"><a href="#bib113" id="ancbbib113" class="intra_ref"><sup>113</sup></a></span> It was shown to be structurally similar to other forms of RNase H, such as RNase HI from <em>Escherichia coli</em> and <em>Thermus thermophilus</em>. Crystal structures of full-length HIV-1 RT with a DNA/DNA <sup><a href="#bib7" id="bbib7" class="intra_ref">7</a>&nbsp;and&nbsp;<a href="#bib39" id="bbib39" class="intra_ref">39</a></sup> or RNA/DNA<span id="bbib11"><a href="#bib11" id="ancbbib11" class="intra_ref"><sup>11</sup></a></span> substrate show a distance of ∼&nbsp;60&nbsp;Å between the polymerase active site and the RNase H active site. In terms of nucleotides, the distance is 17 basepairs for a DNA/DNA substrate and 18 basepairs for a RNA/DNA substrate, which is in agreement with biochemical analysis. <sup><a href="#bib114" id="bbib114" class="intra_ref">114</a>, <a href="#bib115" id="bbib115" class="intra_ref">115</a>, <a href="#bib116" id="bbib116" class="intra_ref">116</a>, <a href="#bib117" id="bbib117" class="intra_ref">117</a>, <a href="#bib118" id="bbib118" class="intra_ref">118</a>, <a href="#bib119" id="bbib119" class="intra_ref">119</a>, <a href="#bib120" id="bbib120" class="intra_ref">120</a>&nbsp;and&nbsp;<a href="#bib121" id="bbib121" class="intra_ref">121</a></sup></p><!--VALIDHTML--><p class="svArticle section" id="">HIV-1 RNase H was shown to function in a two metal ion mechanism similar to other polymerase-associated nucleases. The crystal structure of the RNase H fragment shows two Mn<sup>2+</sup> bound the active site at a distance of ∼&nbsp;4&nbsp;Å from each other and interacting with conserved residues D442, E478, D498, and D549.<span id="bbib113"><a href="#bib113" id="ancbbib113" class="intra_ref"><sup>113</sup></a></span> Calorimetry and solution NMR experiments also demonstrated the presence of two Mn<sup>2+</sup> or two Mg<sup>2+</sup> bound to RNase H.<sup><a href="#bib122" id="bbib122" class="intra_ref">122</a>&nbsp;and&nbsp;<a href="#bib123" id="bbib123" class="intra_ref">123</a></sup> Biochemical assays suggested that the optimal RNase H activity is obtained in the presence of one Mg<sup>2+</sup> and one Mn<sup>2+</sup>;<span id="bbib124"><a href="#bib124" id="ancbbib124" class="intra_ref"><sup>124</sup></a></span> however, it is likely that, <em>in vivo</em>, both metals are Mg<sup>2+</sup>. Recently, the crystal structure of RNase H complexed with an RNA/DNA hybrid confirmed a mechanism for substrate recognition and two metal ion-dependent catalysis that has been proposed to be common between the functionally similar transposases, retroviral integrases, Holliday junction resolvases, and RISC nuclease Argonaute.<span id="bbib125"><a href="#bib125" id="ancbbib125" class="intra_ref"><sup>125</sup></a></span></p><!--VALIDHTML--><p class="svArticle section" id="">It has been suggested that the RNase H specificity for RNA/DNA substrate relies on the width of the minor groove.<sup><a href="#bib126" id="bbib126" class="intra_ref">126</a>, <a href="#bib127" id="bbib127" class="intra_ref">127</a>, <a href="#bib128" id="bbib128" class="intra_ref">128</a>, <a href="#bib129" id="bbib129" class="intra_ref">129</a>, <a href="#bib130" id="bbib130" class="intra_ref">130</a>, <a href="#bib131" id="bbib131" class="intra_ref">131</a>, <a href="#bib132" id="bbib132" class="intra_ref">132</a>&nbsp;and&nbsp;<a href="#bib133" id="bbib133" class="intra_ref">133</a></sup> This proposal comes from the observation that RNA/DNA has an intermediate conformation between <em>A</em>- and <em>B</em>-form double-stranded nucleic acids named <em>H</em>-form, which has a minor groove width of ∼&nbsp;9–10&nbsp;Å. The HIV-1 RNase H cleaves substrates with different minor groove widths much less efficiently. These include the junction of an RNA–DNA hybrid annealed to a DNA template where the minor groove can be as narrow as 4.5–5.5&nbsp;Å.<span id="bbib133"><a href="#bib133" id="ancbbib133" class="intra_ref"><sup>133</sup></a></span> The crystal structure of RT bound to an RNA/DNA substrate supports the idea that it is the width of the minor groove, in combination with the RNase H primer grip, that binds and positions the primer strand on the enzyme, which is responsible for cleavage specificity.<span id="bbib11"><a href="#bib11" id="ancbbib11" class="intra_ref"><sup>11</sup></a></span> Mutagenesis suggests that the residues in the RNase H that are in contact with the primer strand are more important than the residues contacting the template strand. <sup><a href="#bib10" id="bbib10" class="intra_ref">10</a>&nbsp;and&nbsp;<a href="#bib134" id="bbib134" class="intra_ref">134</a></sup></p><!--VALIDHTML--><p class="svArticle section" id="">As already described, the PPT is used as primer for the initiation of the plus-strand synthesis.<span id="bbib1"><a href="#bib1" id="ancbbib1" class="intra_ref"><sup>1</sup></a></span> A crystal structure of HIV-1 RT in complex with the PPT as RNA template annealed to a DNA primer provides insights regarding the resistance of the PPT to RNase H degradation.<span id="bbib11"><a href="#bib11" id="ancbbib11" class="intra_ref"><sup>11</sup></a></span> The structure shows a narrower than expected minor groove in the RNA/DNA substrate near the RNase H active site (∼&nbsp;7&nbsp;Å), which positions the RNA template approximately 3&nbsp;Å away from the RNase H active site. This distance of 3&nbsp;Å is approximately the same as the difference between the minor groove width in the PPT RNA/DNA structure and the predicted minor groove width in <em>H</em>-form nucleic acids. The difference in width of the minor groove is thought to be due to the presence of A-tracts in the PPT, which are known to have a narrower minor groove and to provide more stiffness to the nucleic acid.</p><!--VALIDHTML--><p class="svArticle section" id="">As already discussed, RNase H is responsible also for the removal of the PPT from the 5′ end of the genome, and the tRNA used as primer from the 3′ end of the genome, these cleavages define the ends of the linear DNA that is the substrate for integration. The RNase H cleaves at the junction between the PPT RNA and the DNA transcript made by RT.<span id="bbib1"><a href="#bib1" id="ancbbib1" class="intra_ref"><sup>1</sup></a></span> Using a single molecule fluorescence resonance energy transfer assay, Abbondanzieri <em>et al</em>. recently showed that the preferred orientation of RT on a nucleic acid substrate depends on both the composition of the substrate (RNA/DNA <em>versus</em> DNA/DNA) and on the sequence of the nucleic acid. They showed also that RT can switch rapidly between orientations when it binds to polypurine RNA primers for plus-strand DNA synthesis.<span id="bbib135"><a href="#bib135" id="ancbbib135" class="intra_ref"><sup>135</sup></a></span></p><!--VALIDHTML--><p class="svArticle section" id="">In most retroviruses, RNase H removes the priming tRNA by cleaving at the junction of the RNA and the DNA genome. In most cases, IN removes two nucleotides from each of the 3′ ends of viral DNA, exposing a conserved CA dinucleotide that is then joined to the host DNA. In the case of HIV-1, there is only a single nucleotide between the CA and the PBS, where viral DNA synthesis is initiated. Sequence analysis of the 2-LTR circle junctions that are derived from the ligation of the ends of the viral linear DNA suggested that RNase H cleaves one nucleotide from the 3′ end of the tRNA, giving rise to viral DNA substrate that would allow IN to remove two nucleotides;<sup><a href="#bib136" id="bbib136" class="intra_ref">136</a>&nbsp;and&nbsp;<a href="#bib137" id="bbib137" class="intra_ref">137</a></sup> and <em>in vitro</em> analysis of the RNase H cleavage on model substrates supports this interpretation. <sup><a href="#bib108" id="bbib108" class="intra_ref">108</a>&nbsp;and&nbsp;<a href="#bib109" id="bbib109" class="intra_ref">109</a></sup></p><!--VALIDHTML--><h3 class="svArticle" id="section_lbl4.1">Inhibitors of RNase H</h3><p class="svArticle section" id="">As mentioned earlier, RT RNase H activity is essential for HIV-1 replication. For that reason, RNase H is a promising target for the development of antiretroviral drugs.<sup><a href="#bib138" id="bbib138" class="intra_ref">138</a>&nbsp;and&nbsp;<a href="#bib139" id="bbib139" class="intra_ref">139</a></sup> RNase H inhibitors (RNHIs) are likely to be active against all of the current drug-resistant HIV-1 variants. Only six RNHIs with reasonable <em>in vitro</em> potency have been described in any detail (<span id="bfig8"><a href="#fig8" id="ancbfig8" class="intra_ref">Fig. 8</a></span>a, structures <em>1</em> – <em>6</em>), and most of these have minimal therapeutic value because of limited cell uptake or cytotoxicity. Although all of the available inhibitors of the polymerase activity of RT are bound either at or near the polymerase active site, not all the RNHIs are bound at or near the RNase H active site. The acylhydrazone <em>N</em>-(4-tert-butylbenzoyl)-2-hydroxy-1-naphthaldehyde hydrazone (BBNH; <span id="bfig8"><a href="#fig8" id="ancbfig8" class="intra_ref">Fig. 8</a></span>a, structure <em>1</em>) is an RNHI with reasonable potency.<span id="bbib140"><a href="#bib140" id="ancbbib140" class="intra_ref"><sup>140</sup></a></span> BBNH is a bifunctional inhibitor of HIV-1 RT; it inhibits both the RT DNA polymerase and RNase H activities of the enzyme with similar potency (IC<sub>50</sub>&nbsp;≈&nbsp;3&nbsp;μM). A crystal structure of HIV-1 RT complexed with an analog of BBNH, dihydroxy benzoyl naphthyl hydrazone (DHBNH) has been published.<span id="bbib141"><a href="#bib141" id="ancbbib141" class="intra_ref"><sup>141</sup></a></span>] Unlike BBNH, DHBNH (<span id="bfig8"><a href="#fig8" id="ancbfig8" class="intra_ref">Fig. 8</a></span>a, structure <em>2</em>) is a specific inhibitor of RT RNase H activity (IC<sub>50</sub>&nbsp;≈&nbsp;0.5&nbsp;μM) and has relatively limited activity against the DNA polymerase activity of RT. DHBNH is non-cytotoxic and inhibits the replication of a variety of drug-resistant HIV-1 RT mutants.<span id="bbib142"><a href="#bib142" id="ancbbib142" class="intra_ref"><sup>142</sup></a></span> Surprisingly, in the crystal structure, DHBNH binds more than 50&nbsp;Å away from the RNase H active site, at a site that is, in some sense, between the polymerase active site and the NNIBP (<span id="bfig9"><a href="#fig9" id="ancbfig9" class="intra_ref">Fig. 9</a></span>). The mechanism by which DHBNH binding inhibits RT RNase H is not clear, but may involve repositioning of the polymerase primer grip and the thumb of RT in a manner that alters the trajectory of the nucleic acid near the RNase H active site, thereby preventing RNase H-catalyzed cleavage.<span id="bbib141"><a href="#bib141" id="ancbbib141" class="intra_ref"><sup>141</sup></a></span> Alternatively, DHBNH (and other acylhydrazone analogs) may also bind to a second site in or near the RNase H domain that was not seen in the crystal.<span id="bbib142"><a href="#bib142" id="ancbbib142" class="intra_ref"><sup>142</sup></a></span> Recent NMR studies of the interaction of DHBNH with an active RNase H fragment that does not contain the polymerase domain are consistent with this possibility (M. A. Parniak &amp; R. Ishima, unpublished results).</p><!--VALIDHTML--><div class="figTblUpiOuter svArticle" id="figure_fig8"><div class="
                    textboxdefault
                "><dl class="figure" id="fig8" data-t="f"><dt class="autoScroll" data-style="height:646px;width:669px;"><a class="figureLink" href="#gr8" title="Full-size image (53 K)"><img style="display: inline;" class="imgLazyJSB figure large smallImg" alt="Full-size image (53 K)" src="grey_pxl.gif" data-thumbeid="1-s2.0-S0022283608013442-gr8.sml" data-fulleid="1-s2.0-S0022283608013442-gr8.jpg" data-thumbheight="164" data-thumbwidth="170" data-fullheight="646" data-fullwidth="669" border="0"><noscript class="figure large" border="0" alt="Full-size image" data-thumbeid="1-s2.0-S0022283608013442-gr8.sml" data-fulleid="1-s2.0-S0022283608013442-gr8.jpg"><img class="figure large" border="0" alt="Full-size image (53 K)" src="http://origin-ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr8.jpg" data-thumbEID="1-s2.0-S0022283608013442-gr8.sml" data-fullEID="1-s2.0-S0022283608013442-gr8.jpg"></noscript></a></dt><dd id="labelCaptionfig8"><p class="caption">Fig. 8.&nbsp;<span id="">Chemical structure of RNase H inhibitors (a): 1, <em>N</em>-(4-t-butylbenzoyl)-2-hydroxy-1-naphthaldehyde hydrazone (BBNH); 2, dihydroxy benzoyl naphthyl hydrazone (DHBNH); 3, 4-chlorophenylhydrazone of mesoxalic acid (CPHM); 4, diketo acid; 5, <em>N</em>-hydroxyimide; 6, hydroxyl tropolone (2,7-dihydroxy-4-1(methylethyl)-2,4,6-cycloheptatrien-1-one, or β-thujaplicinol); excision inhibitors (b), and 4′-substituted nucleoside analogs (c): 1, 4′-ethynyl-2-fluoro deoxyadenosine (EFdA); 2, 4′methyl-deoxythymidine (MdT), 4′ethynyl-deoxythymidine (EdT).<!--Comment--></span><!--caption--></p></dd><dd class="menuButtonLinks"><div class="btnHolder"><a class="menuTitle" href="#">Figure options</a><div class="down_Btn"></div></div><ul class="menuButton"><li><a href="#fig8" class="viewWS" title="View in workspace">View in workspace</a></li><li><a href="http://origin-ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr8.jpg" title="View full-size image - Opens in new window">Download full-size image</a></li><li><a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0022283608013442&amp;_image=1-s2.0-S0022283608013442-gr8_lrg.jpg&amp;_ba=&amp;_fmt=full&amp;_orig=na&amp;_issn=00222836&amp;_pii=S0022283608013442&amp;_isHiQual=Y&amp;_acct=C000049402&amp;_version=1&amp;_urlVersion=0&amp;_userid=963121&amp;md5=91580a76709c00d097049e09a0c676d3" title="View high quality image - Opens in new window">Download high-quality image (434 K)</a></li><li><a class="ppt" title="Download as PowerPoint slide" href="http://services.elsevier.com.libgate.library.nuigalway.ie/SDWebExport/export/figure/S0022283608013442/1-s2.0-S0022283608013442-gr8.jpg/ppt">Download as PowerPoint slide</a></li></ul></dd></dl></div></div><!--VALIDHTML--><div class="figTblUpiOuter svArticle" id="figure_fig9"><div class="
                    textboxdefault
                "><dl class="figure" id="fig9" data-t="f"><dt class="autoScroll" data-style="height:402px;width:444px;"><a class="figureLink" href="#gr9" title="Full-size image (72 K)"><img style="display: inline;" class="imgLazyJSB figure large" alt="Full-size image (72 K)" src="grey_pxl.gif" data-thumbeid="1-s2.0-S0022283608013442-gr9.sml" data-fulleid="1-s2.0-S0022283608013442-gr9.jpg" data-thumbheight="164" data-thumbwidth="181" data-fullheight="402" data-fullwidth="444" border="0" height="402" width="444"><noscript class="figure large" border="0" alt="Full-size image" data-thumbeid="1-s2.0-S0022283608013442-gr9.sml" data-fulleid="1-s2.0-S0022283608013442-gr9.jpg"><img class="figure large" border="0" alt="Full-size image (72 K)" src="http://origin-ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr9.jpg" data-thumbEID="1-s2.0-S0022283608013442-gr9.sml" data-fullEID="1-s2.0-S0022283608013442-gr9.jpg"></noscript></a></dt><dd id="labelCaptionfig9"><p class="caption">Fig. 9.&nbsp;<span id="">Ribbon representation of HIV-1 RT with DHBNH bound.<span id="fig9bbib142"><a href="#bib142" id="ancfig9bbib142" class="intra_ref"><sup>142</sup></a></span> In the crystal structure of DHBNH, an RNase H inhibitor binds &gt;&nbsp;50&nbsp;Å away from the RNase H subdomain, at a site that partially overlaps the NNIBP. The subdomains of the p66 subunit are color-coded (fingers in blue, palm in red, thumb in green, connection in yellow, and RNase H in gold). Upper left inset: a close-up of the DHBNH binding site. The inhibitor is shown in magenta. The pocket that is occupied by NNRTIs is shown in gray (the NNRTI pocket is not occupied in this structure).<!--Comment--></span><!--caption--></p></dd><dd class="menuButtonLinks"><div class="btnHolder"><a class="menuTitle" href="#">Figure options</a><div class="down_Btn"></div></div><ul class="menuButton"><li><a href="#fig9" class="viewWS" title="View in workspace">View in workspace</a></li><li><a href="http://origin-ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr9.jpg" title="View full-size image - Opens in new window">Download full-size image</a></li><li><a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0022283608013442&amp;_image=1-s2.0-S0022283608013442-gr9_lrg.jpg&amp;_ba=&amp;_fmt=full&amp;_orig=na&amp;_issn=00222836&amp;_pii=S0022283608013442&amp;_isHiQual=Y&amp;_acct=C000049402&amp;_version=1&amp;_urlVersion=0&amp;_userid=963121&amp;md5=7769f5e814504bba9cd3fb13d5813a50" title="View high quality image - Opens in new window">Download high-quality image (512 K)</a></li><li><a class="ppt" title="Download as PowerPoint slide" href="http://services.elsevier.com.libgate.library.nuigalway.ie/SDWebExport/export/figure/S0022283608013442/1-s2.0-S0022283608013442-gr9.jpg/ppt">Download as PowerPoint slide</a></li></ul></dd></dl></div></div><!--VALIDHTML--><p class="svArticle section" id="">The aryl hydrazone 4-chlorophenyl hydrazone mesoxalate (CPHM; <span id="bfig8"><a href="#fig8" id="ancbfig8" class="intra_ref">Fig. 8</a></span>a, structure <em>3</em>) inhibits the RNase H of HIV-1 RT with potency similar to that of BBNH.<span id="bbib143"><a href="#bib143" id="ancbbib143" class="intra_ref"><sup>143</sup></a></span> Unlike BBNH, CPHM is specific for RNase H and does not inhibit the DNA polymerase activity of RT. CPHM has no antiviral activity, and it is possible that this charged molecule cannot enter cells. The Merck group has described a diketo acid inhibitor of RNase H, 4-[5-benzoylamino) thien-2-yl]-2,4-dioxobutanoic acid (<span id="bfig8"><a href="#fig8" id="ancbfig8" class="intra_ref">Fig. 8</a></span>a, structure <em>4</em>), that was discovered in their integrase inhibitor discovery program.<span id="bbib144"><a href="#bib144" id="ancbbib144" class="intra_ref"><sup>144</sup></a></span> Like CPHM, this molecule has no antiviral activity. Structural information for the interaction of CPHM and RT is not available. On the basis of the proposed mechanism for the diketo acid inhibitors of integrase, it seems likely that CPHM and diketo acids inhibit RNase H by binding to the active site and chelating the essential Mg<sup>2+</sup>. Other RNHIs include the <em>N</em>-hydroxyimides () (<span id="bfig8"><a href="#fig8" id="ancbfig8" class="intra_ref">Fig. 8</a></span>a, structure <em>5</em>)<span id="bbib124"><a href="#bib124" id="ancbbib124" class="intra_ref"><sup>124</sup></a></span><sup>,</sup><span id="bbib145"><a href="#bib145" id="ancbbib145" class="intra_ref"><sup>145</sup></a></span> and hydroxy tropolones (<span id="bfig8"><a href="#fig8" id="ancbfig8" class="intra_ref">Fig. 8</a></span>a, structure <em>6</em>).<span id="bbib146"><a href="#bib146" id="ancbbib146" class="intra_ref"><sup>146</sup></a></span> These RNHIs also possess functional groups spaced to allow the coordination of the two metal cations in the RNase H active site. These compounds are not candidate therapeutics because they are cytotoxic.</p><!--VALIDHTML--><h4 class="svArticle" id="section_lbl4.1.1">Future perspectives</h4><h5 class="svArticle" id="section_lbl4.1.1.1">Delayed chain terminators</h5><p class="svArticle section" id="">Because excision is a major pathway for HIV-1 NRTI resistance, it is important to develop NRTIs that are relatively resistant to excision. Conventional NRTIs are susceptible to excision because they remain at the 3′ end of the primer strand, where they could be excised, if resistance mutations in HIV-1 RT allow the 3′ end of the primer good access to the N site. Nucleoside analogs that cause DNA chain termination only after additional normal nucleotides have been incorporated (delayed chain terminators) should be less susceptible to excision. The conformationally locked nucleosides North methanocarbathymidine (N-MCT) and North methanocarba-2′-deoxyadenosine (N-MCdA) act as delayed chain terminators in an <em>in vitro</em> HIV-1 RT polymerization reaction. The N-MC compounds are poorly excised by excision-proficient RTs. Unfortunately N-MCT is poorly phosphorylated in cultured cells; <sup><a href="#bib147" id="bbib147" class="intra_ref">147</a>&nbsp;and&nbsp;<a href="#bib148" id="bbib148" class="intra_ref">148</a></sup> however, the compound can be tested against HIV-1 (or HIV-1 based vectors) in cell lines that express herpes simplex virus thymidine kinase (HSV-TK), which allows N-MCT to be phosphorylated reasonably well. N-MCT potently inhibits HIV-1 replication in cells that express HSV-TK and efficiently blocks the replication of HIV-1 variants that replicate using excision-proficient RTs.<span id="bbib149"><a href="#bib149" id="ancbbib149" class="intra_ref"><sup>149</sup></a></span> Although the N-MC compounds are not drug candidates, these results suggested that this is a promising approach to the problem posed by excision-proficient NRTI-resistant mutants.</p><!--VALIDHTML--><p class="svArticle section" id="">The results obtained with the N-MC compounds illustrate a key point: To be active in cells, nucleoside analogs must be phosphorylated by host cell kinases. The deoxyribose rings of nucleosides are not flat, and can exist in various conformations. Two preferred conformations have been called North (found in <em>A</em>-form nucleic acids) and South (found in <em>B</em>-form nucleic acids). Most DNA polymerases (including HIV-1 RT) prefer to incorporate dNTPs that are in the North conformation; <sup><a href="#bib22" id="bbib22" class="intra_ref">22</a>, <a href="#bib149" id="bbib149" class="intra_ref">149</a>, <a href="#bib150" id="bbib150" class="intra_ref">150</a>, <a href="#bib151" id="bbib151" class="intra_ref">151</a>&nbsp;and&nbsp;<a href="#bib152" id="bbib152" class="intra_ref">152</a></sup> however, cellular kinases prefer the South conformation. <sup><a href="#bib153" id="bbib153" class="intra_ref">153</a>, <a href="#bib154" id="bbib154" class="intra_ref">154</a>, <a href="#bib155" id="bbib155" class="intra_ref">155</a>&nbsp;and&nbsp;<a href="#bib156" id="bbib156" class="intra_ref">156</a></sup> The N-MC compounds are poorly phosphorylated because they are locked in the North conformation.</p><!--VALIDHTML--><h5 class="svArticle" id="section_lbl4.1.1.2">4′ substituted NRTIs</h5><p class="svArticle section" id="">Several groups have concentrated on developing compounds that have 4′ modifications on the pseudosugar ring and interfere with viral DNA synthesis. Mitsuya and colleagues have done extensive work on 4′-modified nucleosides and identified promising compounds.<span id="bbib157"><a href="#bib157" id="ancbbib157" class="intra_ref"><sup>157</sup></a></span> The most potent of the compounds described thus far have a 4′-ethynyl (4′E) substitution and have antiviral activity against wild-type and drug-resistant HIV strains.<span id="bbib158"><a href="#bib158" id="ancbbib158" class="intra_ref"><sup>158</sup></a></span> Second generation 4′-ethynyl-substituted compounds have an additional 2-fluoro (2F) substitution on the adenine ring, and inhibit HIV at picomolar concentrations.<sup><a href="#bib159" id="bbib159" class="intra_ref">159</a>&nbsp;and&nbsp;<a href="#bib160" id="bbib160" class="intra_ref">160</a></sup> 4′E-2FdA (<span id="bfig8"><a href="#fig8" id="ancbfig8" class="intra_ref">Fig. 8</a></span>c) retains significant activity against multi-drug-resistant viruses isolated from patients. The compound appears to have a low level of cytotoxicity and it does not appear to inhibit human DNA polymerases α or β, or mitochondrial DNA polymerase γ.<span id="bbib161"><a href="#bib161" id="ancbbib161" class="intra_ref"><sup>161</sup></a></span></p><!--VALIDHTML--><p class="svArticle section" id="">Marquez and colleagues have shown that the triphosphates of 4′ methyl T and 4′-ethyl thymidine (<span id="bfig8"><a href="#fig8" id="ancbfig8" class="intra_ref">Fig. 8</a></span>c) are potent inhibitors of HIV RT <em>in</em><em>vitro</em>.<span id="bbib162"><a href="#bib162" id="ancbbib162" class="intra_ref"><sup>162</sup></a></span> While these nucleosides were not activated efficiently by host cell kinases, they were highly effective against both wild type HIV-1 and a number of NRTI-resistant mutants in the presence of HSV-TK.</p><!--VALIDHTML--><p class="svArticle section" id="">Interestingly, the 4′-substituted compounds shown in <span id="bfig8"><a href="#fig8" id="ancbfig8" class="intra_ref">Fig. 8</a></span>c have a 3′-OH. How then do they inhibit viral replication? None of these compounds appear to be a delayed chain terminator. 4′-Methyl TTP was incorporated into DNA reasonably efficiently by HIV-1 RT; however, in many cases, the next (normal) nucleotide, which would be joined to the 4′-methyl T at the 3′ end of the primer, was incorporated slowly. The degree of pausing depended on both the sequence and the nature of the template (DNA or RNA). The underlying mechanisms of RT inhibition by N-MC or 4′E compounds appear to be distinct. The N-MC compounds presumably cause delayed chain termination because of a steric clash between the cyclopropane ring on the pseudosugar ring of the N-MC compounds and HIV-1 RT at a position near the base of the thumb of the p66 subunit.<span id="bbib149"><a href="#bib149" id="ancbbib149" class="intra_ref"><sup>149</sup></a></span> The molecular details of RT inhibition by 4′-substituted dNTPs that retain the 3′OH are currently under investigation. Notably, addition of a 4′-ethynyl group to the classical chain-terminator d4T improves its potency and reduces its cytotoxicity.<sup><a href="#bib163" id="bbib163" class="intra_ref">163</a>&nbsp;and&nbsp;<a href="#bib164" id="bbib164" class="intra_ref">164</a></sup></p><!--VALIDHTML--><h4 class="svArticle" id="section_lbl4.1.2">Inhibitors of NRTI excision</h4><p class="svArticle section" id="">As discussed earlier,<sup><a href="#bib47" id="bbib47" class="intra_ref">47</a>, <a href="#bib48" id="bbib48" class="intra_ref">48</a>&nbsp;and&nbsp;<a href="#bib49" id="bbib49" class="intra_ref">49</a></sup> excision is an important mechanism of NRTI resistance. Specific inhibitors of NRTI excision could be therapeutically useful if they could be used to prevent the excision of an incorporated NRTI, which should restore the potency of an NRTI that would otherwise be excised. Optimally, such inhibitors should not reduce the incorporation of the NRTI because this would antagonize the antiretroviral activity of the NRTI. Parniak and colleagues reported that certain bisphosphonate compounds such as BPH218 (<span id="bfig8"><a href="#fig8" id="ancbfig8" class="intra_ref">Fig. 8</a></span>b) are potent inhibitors of ATP-mediated phosphorolytic excision of 3′-terminal AZT-MP (<em>in vitro</em> IC<sub>50</sub> ∼&nbsp;2&nbsp;μM). Bisphosphonates are highly charged at neutral pH and are unable to traverse the plasma membrane of cells, thus BPH218 has no antiviral efficacy either in the presence or in the absence of an NRTI. However, copolymer carriers greatly facilitate cell uptake of bisphosphonates (M.A. Parniak <em>et al</em>., unpublished results). BPH218 administered to cells in this manner has no effect on HIV-1 replication; however, the compound provides up to 30-fold potentiation of the antiviral activity of AZT against AZTr HIV-1 (<span id="bfig10"><a href="#fig10" id="ancbfig10" class="intra_ref">Fig. 10</a></span>). This has potential clinical significance, in that substantial numbers of treatment-experienced patients are infected with AZTr-HIV. Although bisphosphonates such as BPH218 are not therapeutics, these findings show that RT-catalyzed NRTI excision is a target for the development of adjuvant drugs that could restore the efficacy of NRTIs in patients infected with excision-proficient NRTI-resistant HIV.</p><!--VALIDHTML--><div class="figTblUpiOuter svArticle" id="figure_fig10"><div class="
                    textboxdefault
                "><dl class="figure" id="fig10" data-t="f"><dt class="autoScroll" data-style="height:235px;width:329px;"><a class="figureLink" href="#gr10" title="Full-size image (20 K)"><img style="display: inline;" class="imgLazyJSB figure large" alt="Full-size image (20 K)" src="grey_pxl.gif" data-thumbeid="1-s2.0-S0022283608013442-gr10.sml" data-fulleid="1-s2.0-S0022283608013442-gr10.jpg" data-thumbheight="156" data-thumbwidth="219" data-fullheight="235" data-fullwidth="329" border="0" height="235" width="329"><noscript class="figure large" border="0" alt="Full-size image" data-thumbeid="1-s2.0-S0022283608013442-gr10.sml" data-fulleid="1-s2.0-S0022283608013442-gr10.jpg"><img class="figure large" border="0" alt="Full-size image (20 K)" src="http://origin-ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr10.jpg" data-thumbEID="1-s2.0-S0022283608013442-gr10.sml" data-fullEID="1-s2.0-S0022283608013442-gr10.jpg"></noscript></a></dt><dd id="labelCaptionfig10"><p class="caption">Fig. 10.&nbsp;<span id="">Potentiation of AZT antiviral activity against AZT-resistant HIV-1 by BPH218.<!--Comment--></span><!--caption--></p></dd><dd class="menuButtonLinks"><div class="btnHolder"><a class="menuTitle" href="#">Figure options</a><div class="down_Btn"></div></div><ul class="menuButton"><li><a href="#fig10" class="viewWS" title="View in workspace">View in workspace</a></li><li><a href="http://origin-ars.els-cdn.com/content/image/1-s2.0-S0022283608013442-gr10.jpg" title="View full-size image - Opens in new window">Download full-size image</a></li><li><a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S0022283608013442&amp;_image=1-s2.0-S0022283608013442-gr10_lrg.jpg&amp;_ba=&amp;_fmt=full&amp;_orig=na&amp;_issn=00222836&amp;_pii=S0022283608013442&amp;_isHiQual=Y&amp;_acct=C000049402&amp;_version=1&amp;_urlVersion=0&amp;_userid=963121&amp;md5=dbc9d5780ecc15c692e71a94f8427b20" title="View high quality image - Opens in new window">Download high-quality image (78 K)</a></li><li><a class="ppt" title="Download as PowerPoint slide" href="http://services.elsevier.com.libgate.library.nuigalway.ie/SDWebExport/export/figure/S0022283608013442/1-s2.0-S0022283608013442-gr10.jpg/ppt">Download as PowerPoint slide</a></li></ul></dd></dl></div></div><!--VALIDHTML--><p class="svArticle section" id="">In conclusion, structural studies of wild-type and drug-resistant mutant HIV RTs in complex with inhibitors or substrates, together with biochemical and virological experiments have provided valuable insights into the mechanisms of DNA polymerization, inhibition and drug resistance. Importantly, these studies continue to guide the design of better inhibitors that offer promise for improved treatments of HIV infections.</p><!--VALIDHTML--><div class="articleText svArticle"><div class="articleText_indent"><h2 class="h2 secHeading" id="ack001">Acknowledgements</h2><p class="svArticle section" id="">S.G.S. acknowledges support by NIH (grants AI076119, AI079801, and AI074389). Support for B.M. comes from an amfAR Mathilde Kim Fellowship grant. S.H.H. was supported by the Intramural Research Program of NIH, NCI, Center for Cancer Research, and NIGMS. M.A.P. was supported by NIH grants AI060452, AI73975, AI076119, AI07980. E.A. is grateful for support from NIH (grants AI27690 MERIT Award and P01 GM 066671).</p></div></div><!--VALIDHTML--><script type="text/javascript">

              prs.rt("ref_start");
            </script><h2 id="bibl001">References</h2><ol class="references"><li id="bib1" data-t="r"><ul class="reference" id="bibsbref1"><li class="referenceLabel"><a href="#bib1" class="label intra_ref">1</a></li><li class="author">J.M. Coffin, S.H. Hughes, H.E. Varmus</li><li class="title"><p>Retroviruses</p></li><li class="source"><p>Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY (1997)</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref1" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref1'] = new Array();
        </script></li><li id="bib2" data-t="r"><ul class="reference" id="bibsbref2"><li class="referenceLabel"><a href="#bib2" class="label intra_ref">2</a></li><li class="author">J.M. Coffin</li><li class="title"><p>HIV population dynamics in vivo: implications for genetic variation, pathogenesis, and therapy</p></li><li class="source"><p>Science, 267 (1995), pp. 483–489</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref2" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref2'] = new Array();
        </script></li><li id="bib3" data-t="r"><ul class="reference" id="bibsbref3"><li class="referenceLabel"><a href="#bib3" class="label intra_ref">3</a></li><li class="author">R.A. Katz, A.M. Skalka</li><li class="title"><p>The retroviral enzymes</p></li><li class="source"><p>Ann. Rev. Biochem., 63 (1994), pp. 133–173</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref3" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref3'] = new Array();
        </script></li><li id="bib4" data-t="r"><ul class="reference" id="bibsbref4"><li class="referenceLabel"><a href="#bib4" class="label intra_ref">4</a></li><li class="author">F. di Marzo Veronese, T.D. Copeland, A.L. DeVico, R. Rahman, S. Oroszlan, R.C. Gallo, M.G. Sarngadharan</li><li class="title"><p>Characterization of highly immunogenic p66/p51 as the reverse transcriptase of HTLV-III/LAV</p></li><li class="source"><p>Science, 231 (1986), pp. 1289–1291</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref4" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref4'] = new Array();
        </script></li><li id="bib5" data-t="r"><ul class="reference" id="bibsbref5"><li class="referenceLabel"><a href="#bib5" class="label intra_ref">5</a></li><li class="author">D.M. Lowe, A. Aitken, C. Bradley, G.K. Darby, B.A. Larder, K.L. Powell <em>et al.</em></li><li class="title"><p>HIV-1 reverse transcriptase: crystallization and analysis of domain structure by limited proteolysis</p></li><li class="source"><p>Biochemistry, 27 (1988), pp. 8884–8889</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref5" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref5'] = new Array();
        </script></li><li id="bib6" data-t="r"><ul class="reference" id="bibsbref6"><li class="referenceLabel"><a href="#bib6" class="label intra_ref">6</a></li><li class="author">L.A. Kohlstaedt, J. Wang, J.M. Friedman, P.A. Rice, T.A. Steitz</li><li class="title"><p>Crystal structure at 3.5&nbsp;Å resolution of HIV-1 reverse transcriptase complexed with an inhibitor</p></li><li class="source"><p>Science, 256 (1992), pp. 1783–1790</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref6" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref6'] = new Array();
        </script></li><li id="bib7" data-t="r"><ul class="reference" id="bibsbref7"><li class="referenceLabel"><a href="#bib7" class="label intra_ref">7</a></li><li class="author">A. Jacobo-Molina, J. Ding, R.G. Nanni, A.D. Clark Jr, X. Lu, C. Tantillo <em>et al.</em></li><li class="title"><p>Crystal structure of human immunodeficiency virus type 1 reverse transcriptase complexed with double-stranded DNA at 3.0&nbsp;Å resolution shows bent DNA</p></li><li class="source"><p>Proc. Natl Acad. Sci. USA, 90 (1993), pp. 6320–6324</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref7" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref7'] = new Array();
        </script></li><li id="bib8" data-t="r"><ul class="reference" id="bibsbref8"><li class="referenceLabel"><a href="#bib8" class="label intra_ref">8</a></li><li class="author">Y. Xiong, T.H. Eickbush</li><li class="title"><p>Origin and evolution of retroelements based upon their reverse transcriptase sequences</p></li><li class="source"><p>EMBO J., 9 (1990), pp. 3353–3362</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref8" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref8'] = new Array();
        </script></li><li id="bib9" data-t="r"><ul class="reference" id="bibsbref9"><li class="referenceLabel"><a href="#bib9" class="label intra_ref">9</a></li><li class="author">M. Ghosh, P.S. Jacques, D.W. Rodgers, M. Ottman, J.L. Darlix, S.F. Le Grice</li><li class="title"><p>Alterations to the primer grip of p66 HIV-1 reverse transcriptase and their consequences for template-primer utilization</p></li><li class="source"><p>Biochemistry, 35 (1996), pp. 8553–8562</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref9" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref9'] = new Array();
        </script></li><li id="bib10" data-t="r"><ul class="reference" id="bibsbref10"><li class="referenceLabel"><a href="#bib10" class="label intra_ref">10</a></li><li class="author">J.G. Julias, M.J. McWilliams, S.G. Sarafianos, E. Arnold, S.H. Hughes</li><li class="title"><p>Mutations in the RNase H domain of HIV-1 reverse transcriptase affect the initiation of DNA synthesis and the specificity of RNase H cleavage in vivo</p></li><li class="source"><p>Proc. Natl Acad. Sci. USA, 99 (2002), pp. 9515–9520</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref10" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref10'] = new Array();
        </script></li><li id="bib11" data-t="r"><ul class="reference" id="bibsbref11"><li class="referenceLabel"><a href="#bib11" class="label intra_ref">11</a></li><li class="author">S.G. Sarafianos, K. Das, C. Tantillo, A.D. Clark Jr, J. Ding, J.M. Whitcomb <em>et al.</em></li><li class="title"><p>Crystal structure of HIV-1 reverse transcriptase in complex with a polypurine tract RNA:DNA</p></li><li class="source"><p>EMBO J., 20 (2001), pp. 1449–1461</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref11" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref11'] = new Array();
        </script></li><li id="bib12" data-t="r"><ul class="reference" id="bibsbref12"><li class="referenceLabel"><a href="#bib12" class="label intra_ref">12</a></li><li class="author">M. Wisniewski, C. Palaniappan, Z. Fu, S.F. Le Grice, P. Fay, R.A. Bambara</li><li class="title"><p>Mutations in the primer grip region of HIV reverse transcriptase can increase replication fidelity</p></li><li class="source"><p>J. Biol. Chem., 274 (1999), pp. 28175–28184</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref12" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref12'] = new Array();
        </script></li><li id="bib13" data-t="r"><ul class="reference" id="bibsbref13"><li class="referenceLabel"><a href="#bib13" class="label intra_ref">13</a></li><li class="author">R. Krebs, U. Immendorfer, S.H. Thrall, B.M. Wohrl, R.S. Goody</li><li class="title"><p>Single-step kinetics of HIV-1 reverse transcriptase mutants responsible for virus resistance to nucleoside inhibitors zidovudine and 3-TC</p></li><li class="source"><p>Biochemistry, 36 (1997), pp. 10292–10300</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref13" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref13'] = new Array();
        </script></li><li id="bib14" data-t="r"><ul class="reference" id="bibsbref14"><li class="referenceLabel"><a href="#bib14" class="label intra_ref">14</a></li><li class="author">M.D. Powell, M. Ghosh, P.S. Jacques, K.J. Howard, S.F. Le Grice, J.G. Levin</li><li class="title"><p>Alanine-scanning mutations in the “primer grip” of p66 HIV-1 reverse transcriptase result in selective loss of RNA priming activity</p></li><li class="source"><p>J. Biol. Chem., 272 (1997), pp. 13262–13269</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref14" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref14'] = new Array();
        </script></li><li id="bib15" data-t="r"><ul class="reference" id="bibsbref15"><li class="referenceLabel"><a href="#bib15" class="label intra_ref">15</a></li><li class="author">C. Palaniappan, M. Wisniewski, P.S. Jacques, S.F. Le Grice, P.J. Fay, R.A. Bambara</li><li class="title"><p>Mutations within the primer grip region of HIV-1 reverse transcriptase result in loss of RNase H function</p></li><li class="source"><p>J. Biol. Chem., 272 (1997), pp. 11157–11164</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref15" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref15'] = new Array();
        </script></li><li id="bib16" data-t="r"><ul class="reference" id="bibsbref16"><li class="referenceLabel"><a href="#bib16" class="label intra_ref">16</a></li><li class="author">M. Ghosh, J. Williams, M.D. Powell, J.G. Levin, S.F. Le Grice</li><li class="title"><p>Mutating a conserved motif of the HIV-1 reverse transcriptase palm subdomain alters primer utilization</p></li><li class="source"><p>Biochemistry, 36 (1997), pp. 5758–5768</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref16" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref16'] = new Array();
        </script></li><li id="bib17" data-t="r"><ul class="reference" id="bibsbref17"><li class="referenceLabel"><a href="#bib17" class="label intra_ref">17</a></li><li class="author">P.S. Jacques, B.M. Wohrl, M. Ottmann, J.L. Darlix, S.F. Le Grice</li><li class="title"><p>Mutating the “primer grip” of p66 HIV-1 reverse transcriptase implicates tryptophan-229 in template-primer utilization</p></li><li class="source"><p>J. Biol. Chem., 269 (1994), pp. 26472–26478</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref17" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref17'] = new Array();
        </script></li><li id="bib18" data-t="r"><ul class="reference" id="bibsbref18"><li class="referenceLabel"><a href="#bib18" class="label intra_ref">18</a></li><li class="author">B.A. Larder, D.J. Purifoy, K.L. Powell, G. Darby</li><li class="title"><p>Site-specific mutagenesis of AIDS virus reverse transcriptase</p></li><li class="source"><p>Nature, 327 (1987), pp. 716–717</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref18" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref18'] = new Array();
        </script></li><li id="bib19" data-t="r"><ul class="reference" id="bibsbref19"><li class="referenceLabel"><a href="#bib19" class="label intra_ref">19</a></li><li class="author">M. Delarue, O. Poch, N. Tordo, D. Moras, P. Argos</li><li class="title"><p>An attempt to unify the structure of polymerases</p></li><li class="source"><p>Protein Eng., 3 (1990), pp. 461–467</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref19" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref19'] = new Array();
        </script></li><li id="bib20" data-t="r"><ul class="reference" id="bibsbref20"><li class="referenceLabel"><a href="#bib20" class="label intra_ref">20</a></li><li class="author">O. Poch, I. Sauvaget, M. Delarue, N. Tordo</li><li class="title"><p>Identification of four conserved motifs among the RNA-dependent polymerase encoding elements</p></li><li class="source"><p>EMBO J., 8 (1989), pp. 3867–3874</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref20" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref20'] = new Array();
        </script></li><li id="bib21" data-t="r"><ul class="reference" id="bibsbref21"><li class="referenceLabel"><a href="#bib21" class="label intra_ref">21</a></li><li class="author">M.S. Johnson, M.A. McClure, D.F. Feng, J. Gray, R.F. Doolittle</li><li class="title"><p>Computer analysis of retroviral pol genes: assignment of enzymatic functions to specific sequences and homologies with nonviral enzymes</p></li><li class="source"><p>Proc. Natl Acad. Sci. USA, 83 (1986), pp. 7648–7652</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref21" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref21'] = new Array();
        </script></li><li id="bib22" data-t="r"><ul class="reference" id="bibsbref22"><li class="referenceLabel"><a href="#bib22" class="label intra_ref">22</a></li><li class="author">H. Huang, R. Chopra, G.L. Verdine, S.C. Harrison</li><li class="title"><p>Structure of a covalently trapped catalytic complex of HIV-1 reverse transcriptase: implications for drug resistance</p></li><li class="source"><p>Science, 282 (1998), pp. 1669–1675</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref22" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref22'] = new Array();
        </script></li><li id="bib23" data-t="r"><ul class="reference" id="bibsbref23"><li class="referenceLabel"><a href="#bib23" class="label intra_ref">23</a></li><li class="author">A.M. Martin-Hernandez, E. Domingo, L. Menendez-Arias</li><li class="title"><p>Human immunodeficiency virus type 1 reverse transcriptase: role of Tyr115 in deoxynucleotide binding and misinsertion fidelity of DNA synthesis</p></li><li class="source"><p>EMBO J., 15 (1996), pp. 4434–4442</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref23" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref23'] = new Array();
        </script></li><li id="bib24" data-t="r"><ul class="reference" id="bibsbref24"><li class="referenceLabel"><a href="#bib24" class="label intra_ref">24</a></li><li class="author">G. Gao, M. Orlova, M.M. Georgiadis, W.A. Hendrickson, S.P. Goff</li><li class="title"><p>Conferring RNA polymerase activity to a DNA polymerase: a single residue in reverse transcriptase controls substrate selection</p></li><li class="source"><p>Proc. Natl Acad. Sci. USA, 94 (1997), pp. 407–411</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref24" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref24'] = new Array();
        </script></li><li id="bib25" data-t="r"><ul class="reference" id="bibsbref25"><li class="referenceLabel"><a href="#bib25" class="label intra_ref">25</a></li><li class="author">P.L. Boyer, S.G. Sarafianos, E. Arnold, S.H. Hughes</li><li class="title"><p>Analysis of mutations at positions 115 and 116 in the dNTP binding site of HIV-1 reverse transcriptase</p></li><li class="source"><p>Proc. Natl Acad. Sci. USA, 97 (2000), pp. 3056–3061</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref25" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref25'] = new Array();
        </script></li><li id="bib26" data-t="r"><ul class="reference" id="bibsbref26"><li class="referenceLabel"><a href="#bib26" class="label intra_ref">26</a></li><li class="author">B. Marchand, M. Gotte</li><li class="title"><p>Site-specific footprinting reveals differences in the translocation status of HIV-1 reverse transcriptase. Implications for polymerase translocation and drug resistance</p></li><li class="source"><p>J. Biol. Chem., 278 (2003), pp. 35362–35372</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref26" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref26'] = new Array();
        </script></li><li id="bib27" data-t="r"><ul class="reference" id="bibsbref27"><li class="referenceLabel"><a href="#bib27" class="label intra_ref">27</a></li><li class="author">B. Marchand, E.P. Tchesnokov, M. Gotte</li><li class="title"><p>The pyrophosphate analogue foscarnet traps the pre-translocational state of HIV-1 reverse transcriptase in a Brownian ratchet model of polymerase translocation</p></li><li class="source"><p>J. Biol. Chem., 282 (2007), pp. 3337–3346</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref27" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref27'] = new Array();
        </script></li><li id="bib28" data-t="r"><ul class="reference" id="bibsbref28"><li class="referenceLabel"><a href="#bib28" class="label intra_ref">28</a></li><li class="author">P.R. Meyer, W. Rutvisuttinunt, S.E. Matsuura, A.G. So, W.A. Scott</li><li class="title"><p>Stable complexes formed by HIV-1 reverse transcriptase at distinct positions on the primer-template controlled by binding deoxynucleoside triphosphates or foscarnet</p></li><li class="source"><p>J. Mol. Biol., 369 (2007), pp. 41–54</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref28" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref28'] = new Array();
        </script></li><li id="bib29" data-t="r"><ul class="reference" id="bibsbref29"><li class="referenceLabel"><a href="#bib29" class="label intra_ref">29</a></li><li class="author">W.M. Kati, K.A. Johnson, L.F. Jerva, K.S. Anderson</li><li class="title"><p>Mechanism and fidelity of HIV reverse transcriptase</p></li><li class="source"><p>J. Biol. Chem., 267 (1992), pp. 25988–25997</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref29" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref29'] = new Array();
        </script></li><li id="bib30" data-t="r"><ul class="reference" id="bibsbref30"><li class="referenceLabel"><a href="#bib30" class="label intra_ref">30</a></li><li class="author">J.E. Reardon</li><li class="title"><p>Human immunodeficiency virus reverse transcriptase: steady-state and pre-steady-state kinetics of nucleotide incorporation</p></li><li class="source"><p>Biochemistry, 31 (1992), pp. 4473–4479</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref30" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref30'] = new Array();
        </script></li><li id="bib31" data-t="r"><ul class="reference" id="bibsbref31"><li class="referenceLabel"><a href="#bib31" class="label intra_ref">31</a></li><li class="author">K. Rittinger, G. Divita, R.S. Goody</li><li class="title"><p>Human immunodeficiency virus reverse transcriptase substrate-induced conformational changes and the mechanism of inhibition by nonnucleoside inhibitors</p></li><li class="source"><p>Proc. Natl Acad. Sci. USA, 92 (1995), pp. 8046–8049</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref31" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref31'] = new Array();
        </script></li><li id="bib32" data-t="r"><ul class="reference" id="bibsbref32"><li class="referenceLabel"><a href="#bib32" class="label intra_ref">32</a></li><li class="author">W. Tong, C.D. Lu, S.K. Sharma, S. Matsuura, A.G. So, W.A. Scott</li><li class="title"><p>Nucleotide-induced stable complex formation by HIV-1 reverse transcriptase</p></li><li class="source"><p>Biochemistry, 36 (1997), pp. 5749–5757</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref32" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref32'] = new Array();
        </script></li><li id="bib33" data-t="r"><ul class="reference" id="bibsbref33"><li class="referenceLabel"><a href="#bib33" class="label intra_ref">33</a></li><li class="author">T.A. Steitz</li><li class="title"><p>A mechanism for all polymerases</p></li><li class="source"><p>Nature, 391 (1998), pp. 231–232</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref33" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref33'] = new Array();
        </script></li><li id="bib34" data-t="r"><ul class="reference" id="bibsbref34"><li class="referenceLabel"><a href="#bib34" class="label intra_ref">34</a></li><li class="author">S.G. Sarafianos, A.D. Clark Jr, K. Das, S. Tuske, J.J. Birktoft, P. Ilankumaran <em>et al.</em></li><li class="title"><p>Structures of HIV-1 reverse transcriptase with pre- and post-translocation AZTMP-terminated DNA</p></li><li class="source"><p>EMBO J., 21 (2002), pp. 6614–6624</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref34" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref34'] = new Array();
        </script></li><li id="bib35" data-t="r"><ul class="reference" id="bibsbref35"><li class="referenceLabel"><a href="#bib35" class="label intra_ref">35</a></li><li class="author">C.F. Perno, R. Yarchoan, D.A. Cooney, N.R. Hartman, S. Gartner, M. Popovic <em>et al.</em></li><li class="title"><p>Inhibition of human immunodeficiency virus (HIV-1/HTLV-IIIBa-L) replication in fresh and cultured human peripheral blood monocytes/macrophages by azidothymidine and related 2′,3′-dideoxynucleosides</p></li><li class="source"><p>J. Exp. Med., 168 (1988), pp. 1111–1125</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref35" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref35'] = new Array();
        </script></li><li id="bib36" data-t="r"><ul class="reference" id="bibsbref36"><li class="referenceLabel"><a href="#bib36" class="label intra_ref">36</a></li><li class="author">P.A. Furman, J.A. Fyfe, M.H. St Clair, K. Weinhold, J.L. Rideout, G.A. Freeman <em>et al.</em></li><li class="title"><p>Phosphorylation of 3′-azido-3′-deoxythymidine and selective interaction of the 5′-triphosphate with human immunodeficiency virus reverse transcriptase</p></li><li class="source"><p>Proc. Natl Acad. Sci. USA, 83 (1986), pp. 8333–8337</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref36" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref36'] = new Array();
        </script></li><li id="bib37" data-t="r"><ul class="reference" id="bibsbref37"><li class="referenceLabel"><a href="#bib37" class="label intra_ref">37</a></li><li class="author">S.G. Kerr, K.S. Anderson</li><li class="title"><p>Pre-steady-state kinetic characterization of wild type and 3′-azido-3′-deoxythymidine (AZT) resistant human immunodeficiency virus type 1 reverse transcriptase: implication of RNA directed DNA polymerization in the mechanism of AZT resistance</p></li><li class="source"><p>Biochemistry, 36 (1997), pp. 14064–14070</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref37" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref37'] = new Array();
        </script></li><li id="bib38" data-t="r"><ul class="reference" id="bibsbref38"><li class="referenceLabel"><a href="#bib38" class="label intra_ref">38</a></li><li class="author">J.Y. Feng, K.S. Anderson</li><li class="title"><p>Mechanistic studies comparing the incorporation of (+) and (–) isomers of 3TCTP by HIV-1 reverse transcriptase</p></li><li class="source"><p>Biochemistry, 38 (1999), pp. 55–63</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref38" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref38'] = new Array();
        </script></li><li id="bib39" data-t="r"><ul class="reference" id="bibsbref39"><li class="referenceLabel"><a href="#bib39" class="label intra_ref">39</a></li><li class="author">J. Ding, K. Das, Y. Hsiou, S.G. Sarafianos, A.D. Clark Jr, A. Jacobo-Molina <em>et al.</em></li><li class="title"><p>Structure and functional implications of the polymerase active site region in a complex of HIV-1 RT with a double-stranded DNA template-primer and an antibody Fab fragment at 2.8 A resolution</p></li><li class="source"><p>J. Mol. Biol., 284 (1998), pp. 1095–1111</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref39" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref39'] = new Array();
        </script></li><li id="bib40" data-t="r"><ul class="reference" id="bibsbref40"><li class="referenceLabel"><a href="#bib40" class="label intra_ref">40</a></li><li class="author">J. Ren, R. Esnouf, E. Garman, D. Somers, C. Ross, I. Kirby <em>et al.</em></li><li class="title"><p>High resolution structures of HIV-1 RT from four RT-inhibitor complexes</p></li><li class="source"><p>Nature Struct. Biol., 2 (1995), pp. 293–302</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref40" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref40'] = new Array();
        </script></li><li id="bib41" data-t="r"><ul class="reference" id="bibsbref41"><li class="referenceLabel"><a href="#bib41" class="label intra_ref">41</a></li><li class="author">D.W. Rodgers, S.J. Gamblin, B.A. Harris, S. Ray, J.S. Culp, B. Hellmig <em>et al.</em></li><li class="title"><p>The structure of unliganded reverse transcriptase from the human immunodeficiency virus type 1</p></li><li class="source"><p>Proc. Natl Acad. Sci. USA, 92 (1995), pp. 1222–1226</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref41" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref41'] = new Array();
        </script></li><li id="bib42" data-t="r"><ul class="reference" id="bibsbref42"><li class="referenceLabel"><a href="#bib42" class="label intra_ref">42</a></li><li class="author">Y. Hsiou, J. Ding, K. Das, A.D. Clark Jr, S.H. Hughes, E. Arnold</li><li class="title"><p>Structure of unliganded HIV-1 reverse transcriptase at 2.7&nbsp;Å resolution: implications of conformational changes for polymerization and inhibition mechanisms</p></li><li class="source"><p>Structure, 4 (1996), pp. 853–860</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref42" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref42'] = new Array();
        </script></li><li id="bib43" data-t="r"><ul class="reference" id="bibsbref43"><li class="referenceLabel"><a href="#bib43" class="label intra_ref">43</a></li><li class="author">C. Tantillo, J. Ding, A. Jacobo-Molina, R.G. Nanni, P.L. Boyer, S.H. Hughes <em>et al.</em></li><li class="title"><p>Locations of anti-AIDS drug binding sites and resistance mutations in the three-dimensional structure of HIV-1 reverse transcriptase. Implications for mechanisms of drug inhibition and resistance</p></li><li class="source"><p>J. Mol. Biol., 243 (1994), pp. 369–387</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref43" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref43'] = new Array();
        </script></li><li id="bib44" data-t="r"><ul class="reference" id="bibsbref44"><li class="referenceLabel"><a href="#bib44" class="label intra_ref">44</a></li><li class="author">K. Das, S.G. Sarafianos, A.D. Clark Jr, P.L. Boyer, S.H. Hughes, E. Arnold</li><li class="title"><p>Crystal structures of clinically relevant Lys103Asn/Tyr181Cys double mutant HIV-1 reverse transcriptase in complexes with ATP and non-nucleoside inhibitor HBY 097</p></li><li class="source"><p>J. Mol. Biol., 365 (2007), pp. 77–89</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref44" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref44'] = new Array();
        </script></li><li id="bib45" data-t="r"><ul class="reference" id="bibsbref45"><li class="referenceLabel"><a href="#bib45" class="label intra_ref">45</a></li><li class="author">S.G. Sarafianos, K. Das, A.D. Clark Jr, J. Ding, P.L. Boyer <em>et al.</em></li><li class="title"><p>Lamivudine (3TC) resistance in HIV-1 reverse transcriptase involves steric hindrance with beta-branched amino acids</p></li><li class="source"><p>Proc. Natl Acad. Sci. USA, 96 (1999), pp. 10027–10032</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref45" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref45'] = new Array();
        </script></li><li id="bib46" data-t="r"><ul class="reference" id="bibsbref46"><li class="referenceLabel"><a href="#bib46" class="label intra_ref">46</a></li><li class="author">H.Q. Gao, P.L. Boyer, S.G. Sarafianos, E. Arnold, S.H. Hughes</li><li class="title"><p>The role of steric hindrance in 3TC resistance of human immunodeficiency virus type-1 reverse transcriptase</p></li><li class="source"><p>J. Mol. Biol., 300 (2000), pp. 403–418</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref46" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref46'] = new Array();
        </script></li><li id="bib47" data-t="r"><ul class="reference" id="bibsbref47"><li class="referenceLabel"><a href="#bib47" class="label intra_ref">47</a></li><li class="author">D. Arion, N. Kaushik, S. McCormick, G. Borkow, M.A. Parniak</li><li class="title"><p>Phenotypic mechanism of HIV-1 resistance to 3′-azido-3′-deoxythymidine (AZT): increased polymerization processivity and enhanced sensitivity to pyrophosphate of the mutant viral reverse transcriptase</p></li><li class="source"><p>Biochemistry, 37 (1998), pp. 15908–15917</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref47" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref47'] = new Array();
        </script></li><li id="bib48" data-t="r"><ul class="reference" id="bibsbref48"><li class="referenceLabel"><a href="#bib48" class="label intra_ref">48</a></li><li class="author">P.R. Meyer, S.E. Matsuura, A.G. So, W.A. Scott</li><li class="title"><p>Unblocking of chain-terminated primer by HIV-1 reverse transcriptase through a nucleotide-dependent mechanism</p></li><li class="source"><p>Proc. Natl Acad. Sci. USA, 95 (1998), pp. 13471–13476</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref48" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref48'] = new Array();
        </script></li><li id="bib49" data-t="r"><ul class="reference" id="bibsbref49"><li class="referenceLabel"><a href="#bib49" class="label intra_ref">49</a></li><li class="author">P.R. Meyer, S.E. Matsuura, A.M. Mian, A.G. So, W.A. Scott</li><li class="title"><p>A mechanism of AZT resistance: an increase in nucleotide-dependent primer unblocking by mutant HIV-1 reverse transcriptase</p></li><li class="source"><p>Mol. Cell, 4 (1999), pp. 35–43</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref49" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref49'] = new Array();
        </script></li><li id="bib50" data-t="r"><ul class="reference" id="bibsbref50"><li class="referenceLabel"><a href="#bib50" class="label intra_ref">50</a></li><li class="author">P.L. Boyer, S.G. Sarafianos, E. Arnold, S.H. Hughes</li><li class="title"><p>Selective excision of AZTMP by drug-resistant human immunodeficiency virus reverse transcriptase</p></li><li class="source"><p>J. Virol., 75 (2001), pp. 4832–4842</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref50" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref50'] = new Array();
        </script></li><li id="bib51" data-t="r"><ul class="reference" id="bibsbref51"><li class="referenceLabel"><a href="#bib51" class="label intra_ref">51</a></li><li class="author">B.A. Larder, S.D. Kemp</li><li class="title"><p>Multiple mutations in HIV-1 reverse transcriptase confer high-level resistance to zidovudine (AZT)</p></li><li class="source"><p>Science, 246 (1989), pp. 1155–1158</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref51" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref51'] = new Array();
        </script></li><li id="bib52" data-t="r"><ul class="reference" id="bibsbref52"><li class="referenceLabel"><a href="#bib52" class="label intra_ref">52</a></li><li class="author">R. Rooke, M.A. Parniak, M. Tremblay, H. Soudeyns, X.G. Li, Q. Gao <em>et al.</em></li><li class="title"><p>Biological comparison of wild-type and zidovudine-resistant isolates of human immunodeficiency virus type 1 from the same subjects: susceptibility and resistance to other drugs</p></li><li class="source"><p>Antimicrob. Agents Chemother., 35 (1991), pp. 988–991</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref52" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref52'] = new Array();
        </script></li><li id="bib53" data-t="r"><ul class="reference" id="bibsbref53"><li class="referenceLabel"><a href="#bib53" class="label intra_ref">53</a></li><li class="author">P. Kellam, C.A. Boucher, B.A. Larder</li><li class="title"><p>Fifth mutation in human immunodeficiency virus type 1 reverse transcriptase contributes to the development of high-level resistance to zidovudine</p></li><li class="source"><p>Proc. Natl Acad. Sci. USA, 89 (1992), pp. 1934–1938</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref53" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref53'] = new Array();
        </script></li><li id="bib54" data-t="r"><ul class="reference" id="bibsbref54"><li class="referenceLabel"><a href="#bib54" class="label intra_ref">54</a></li><li class="author">P.R. Harrigan, I. Kinghorn, S. Bloor, S.D. Kemp, I. Najera, A. Kohli, B.A. Larder</li><li class="title"><p>Significance of amino acid variation at human immunodeficiency virus type 1 reverse transcriptase residue 210 for zidovudine susceptibility</p></li><li class="source"><p>J. Virol., 70 (1996), pp. 5930–5934</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref54" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref54'] = new Array();
        </script></li><li id="bib55" data-t="r"><ul class="reference" id="bibsbref55"><li class="referenceLabel"><a href="#bib55" class="label intra_ref">55</a></li><li class="author">D.J. Hooker, G. Tachedjian, A.E. Solomon, A.D. Gurusinghe, S. Land, C. Birch, J.L. Anderson, B.M. Roy, E. Arnold, N.J. Deacon</li><li class="title"><p>An in vivo mutation from leucine to tryptophan at position 210 in human immunodeficiency virus type-1 reverse transcriptase contributes to high-level resistance to 3'-azido-3' deoxythymidine</p></li><li class="source"><p>J. Virol., 70 (1996), pp. 8010–8018</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref55" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref55'] = new Array();
        </script></li><li id="bib56" data-t="r"><ul class="reference" id="bibsbref56"><li class="referenceLabel"><a href="#bib56" class="label intra_ref">56</a></li><li class="author">J.L. Martin, J.E. Wilson, R.L. Haynes, P.A. Furman</li><li class="title"><p>Mechanism of resistance of human immunodeficiency virus type 1 to 2′,3′-dideoxyinosine</p></li><li class="source"><p>Proc. Natl Acad. Sci. USA, 90 (1993), pp. 6135–6139</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref56" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref56'] = new Array();
        </script></li><li id="bib57" data-t="r"><ul class="reference" id="bibsbref57"><li class="referenceLabel"><a href="#bib57" class="label intra_ref">57</a></li><li class="author">M.A. Winters, R.W. Shafer, R.A. Jellinger, G. Mamtora, T. Gingeras, T.C. Merigan</li><li class="title"><p>Human immunodeficiency virus type 1 reverse transcriptase genotype and drug susceptibility changes in infected individuals receiving dideoxyinosine monotherapy for 1 to 2 years</p></li><li class="source"><p>Antimicrob. Agents Chemother., 41 (1997), pp. 757–762</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref57" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref57'] = new Array();
        </script></li><li id="bib58" data-t="r"><ul class="reference" id="bibsbref58"><li class="referenceLabel"><a href="#bib58" class="label intra_ref">58</a></li><li class="author">P.R. Harrigan, C. Stone, P. Griffin, I. Najera, S. Bloor, S. Kemp <em>et al.</em></li><li class="title"><p>Resistance profile of the human immunodeficiency virus type 1 reverse transcriptase inhibitor abacavir (1592U89) after monotherapy and combination therapy. CNA2001 Investigative Group</p></li><li class="source"><p>J. Infect. Dis., 181 (2000), pp. 912–920</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref58" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref58'] = new Array();
        </script></li><li id="bib59" data-t="r"><ul class="reference" id="bibsbref59"><li class="referenceLabel"><a href="#bib59" class="label intra_ref">59</a></li><li class="author">N.A. Margot, B. Lu, A. Cheng, M.D. Miller</li><li class="title"><p>Resistance development over 144 weeks in treatment-naive patients receiving tenofovir disoproxil fumarate or stavudine with lamivudine and efavirenz in Study 903</p></li><li class="source"><p>HIV Med., 7 (2006), pp. 442–450</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref59" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref59'] = new Array();
        </script></li><li id="bib60" data-t="r"><ul class="reference" id="bibsbref60"><li class="referenceLabel"><a href="#bib60" class="label intra_ref">60</a></li><li class="author">T. Ueno, T. Shirasaka, H. Mitsuya</li><li class="title"><p>Enzymatic characterization of human immunodeficiency virus type 1 reverse transcriptase resistant to multiple 2′,3′-dideoxynucleoside 5′-triphosphates</p></li><li class="source"><p>J. Biol. Chem., 270 (1995), pp. 23605–23611</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref60" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref60'] = new Array();
        </script></li><li id="bib61" data-t="r"><ul class="reference" id="bibsbref61"><li class="referenceLabel"><a href="#bib61" class="label intra_ref">61</a></li><li class="author">K.L. White, J.M. Chen, J.Y. Feng, N.A. Margot, J.K. Ly, A.S. Ray <em>et al.</em></li><li class="title"><p>The K65R reverse transcriptase mutation in HIV-1 reverses the excision phenotype of zidovudine resistance mutations</p></li><li class="source"><p>Antivir. Ther., 11 (2006), pp. 155–163</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref61" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref61'] = new Array();
        </script></li><li id="bib62" data-t="r"><ul class="reference" id="bibsbref62"><li class="referenceLabel"><a href="#bib62" class="label intra_ref">62</a></li><li class="author">Z. Gu, E.J. Arts, M.A. Parniak, M.A. Wainberg</li><li class="title"><p>Mutated K65R recombinant reverse transcriptase of human immunodeficiency virus type 1 shows diminished chain termination in the presence of 2′,3′-dideoxycytidine 5′-triphosphate and other drugs</p></li><li class="source"><p>Proc. Natl Acad. Sci. USA, 92 (1995), pp. 2760–2764</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref62" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref62'] = new Array();
        </script></li><li id="bib63" data-t="r"><ul class="reference" id="bibsbref63"><li class="referenceLabel"><a href="#bib63" class="label intra_ref">63</a></li><li class="author">U.M. Parikh, S. Zelina, N. Sluis-Cremer, J.W. Mellors</li><li class="title"><p>Molecular mechanisms of bidirectional antagonism between K65R and thymidine analog mutations in HIV-1 reverse transcriptase</p></li><li class="source"><p>AIDS, 21 (2007), pp. 1405–1414</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref63" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref63'] = new Array();
        </script></li><li id="bib64" data-t="r"><ul class="reference" id="bibsbref64"><li class="referenceLabel"><a href="#bib64" class="label intra_ref">64</a></li><li class="author">K.L. White, N.A. Margot, J.K. Ly, J.M. Chen, A.S. Ray, M. Pavelko <em>et al.</em></li><li class="title"><p>A combination of decreased NRTI incorporation and decreased excision determines the resistance profile of HIV-1 K65R RT</p></li><li class="source"><p>AIDS, 19 (2005), pp. 1751–1760</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref64" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref64'] = new Array();
        </script></li><li id="bib65" data-t="r"><ul class="reference" id="bibsbref65"><li class="referenceLabel"><a href="#bib65" class="label intra_ref">65</a></li><li class="author">J. Deval, K.L. White, M.D. Miller, N.T. Parkin, J. Courcambeck, P. Halfon <em>et al.</em></li><li class="title"><p>Mechanistic basis for reduced viral and enzymatic fitness of HIV-1 reverse transcriptase containing both K65R and M184V mutations</p></li><li class="source"><p>J. Biol. Chem., 279 (2004), pp. 509–516</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref65" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref65'] = new Array();
        </script></li><li id="bib66" data-t="r"><ul class="reference" id="bibsbref66"><li class="referenceLabel"><a href="#bib66" class="label intra_ref">66</a></li><li class="author">J.Y. Feng, F.T. Myrick, N.A. Margot, G.B. Mulamba, L. Rimsky, K. Borroto-Esoda <em>et al.</em></li><li class="title"><p>Virologic and enzymatic studies revealing the mechanism of K65R- and Q151M-associated HIV-1 drug resistance towards emtricitabine and lamivudine</p></li><li class="source"><p>Nucleosides Nucleotides Nucleic Acids, 25 (2006), pp. 89–107</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref66" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref66'] = new Array();
        </script></li><li id="bib67" data-t="r"><ul class="reference" id="bibsbref67"><li class="referenceLabel"><a href="#bib67" class="label intra_ref">67</a></li><li class="author">J. Deval, J.M. Navarro, B. Selmi, J. Courcambeck, J. Boretto, P. Halfon <em>et al.</em></li><li class="title"><p>A loss of viral replicative capacity correlates with altered DNA polymerization kinetics by the human immunodeficiency virus reverse transcriptase bearing the K65R and L74V dideoxynucleoside resistance substitutions</p></li><li class="source"><p>J. Biol. Chem., 279 (2004), pp. 25489–25496</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref67" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref67'] = new Array();
        </script></li><li id="bib68" data-t="r"><ul class="reference" id="bibsbref68"><li class="referenceLabel"><a href="#bib68" class="label intra_ref">68</a></li><li class="author">B. Selmi, J. Boretto, J.M. Navarro, J. Sire, S. Longhi, C. Guerreiro <em>et al.</em></li><li class="title"><p>The valine-to-threonine 75 substitution in human immunodeficiency virus type 1 reverse transcriptase and its relation with stavudine resistance</p></li><li class="source"><p>J. Biol. Chem., 276 (2001), pp. 13965–13974</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref68" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref68'] = new Array();
        </script></li><li id="bib69" data-t="r"><ul class="reference" id="bibsbref69"><li class="referenceLabel"><a href="#bib69" class="label intra_ref">69</a></li><li class="author">S.G. Sarafianos, K. Das, S.H. Hughes, E. Arnold</li><li class="title"><p>Taking aim at a moving target: designing drugs to inhibit drug-resistant HIV-1 reverse transcriptases</p></li><li class="source"><p>Curr. Opin. Struct. Biol., 14 (2004), pp. 716–730</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref69" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref69'] = new Array();
        </script></li><li id="bib70" data-t="r"><ul class="reference" id="bibsbref70"><li class="referenceLabel"><a href="#bib70" class="label intra_ref">70</a></li><li class="author">L.R. Miranda, M. Gotte, F. Liang, D.R. Kuritzkes</li><li class="title"><p>The L74V mutation in human immunodeficiency virus type 1 reverse transcriptase counteracts enhanced excision of zidovudine monophosphate associated with thymidine analog resistance mutations</p></li><li class="source"><p>Antimicrob. Agents Chemother., 49 (2005), pp. 2648–2656</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref70" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref70'] = new Array();
        </script></li><li id="bib71" data-t="r"><ul class="reference" id="bibsbref71"><li class="referenceLabel"><a href="#bib71" class="label intra_ref">71</a></li><li class="author">A.S. Ray, E. Murakami, A. Basavapathruni, J.A. Vaccaro, D. Ulrich, C.K. Chu <em>et al.</em></li><li class="title"><p>Probing the molecular mechanisms of AZT drug resistance mediated by HIV-1 reverse transcriptase using a transient kinetic analysis</p></li><li class="source"><p>Biochemistry, 42 (2003), pp. 8831–8841</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref71" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref71'] = new Array();
        </script></li><li id="bib72" data-t="r"><ul class="reference" id="bibsbref72"><li class="referenceLabel"><a href="#bib72" class="label intra_ref">72</a></li><li class="author">B. Marchand, K.L. White, J.K. Ly, N.A. Margot, R. Wang, M. McDermott <em>et al.</em></li><li class="title"><p>Effects of the translocation status of human immunodeficiency virus type 1 reverse transcriptase on the efficiency of excision of tenofovir</p></li><li class="source"><p>Antimicrob. Agents Chemother., 51 (2007), pp. 2911–2919</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref72" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref72'] = new Array();
        </script></li><li id="bib73" data-t="r"><ul class="reference" id="bibsbref73"><li class="referenceLabel"><a href="#bib73" class="label intra_ref">73</a></li><li class="author">S. Dharmasena, Z. Pongracz, E. Arnold, S.G. Sarafianos, M.A. Parniak</li><li class="title"><p>3′-Azido-3′-deoxythymidine-(5′)-tetraphospho-(5′)-adenosine, the product of ATP-mediated excision of chain-terminating AZTMP, is a potent chain-terminating substrate for HIV-1 reverse transcriptase</p></li><li class="source"><p>Biochemistry, 46 (2007), pp. 828–836</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref73" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref73'] = new Array();
        </script></li><li id="bib74" data-t="r"><ul class="reference" id="bibsbref74"><li class="referenceLabel"><a href="#bib74" class="label intra_ref">74</a></li><li class="author">P.R. Meyer, S.E. Matsuura, A.A. Tolun, I. Pfeifer, A.G. So, J.W. Mellors, W.A. Scott</li><li class="title"><p>Effects of specific zidovudine resistance mutations and substrate structure on nucleotide-dependent primer unblocking by human immunodeficiency virus type 1 reverse transcriptase</p></li><li class="source"><p>Antimicrob. Agents Chemother., 46 (2002), pp. 1540–1545</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref74" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref74'] = new Array();
        </script></li><li id="bib75" data-t="r"><ul class="reference" id="bibsbref75"><li class="referenceLabel"><a href="#bib75" class="label intra_ref">75</a></li><li class="author">P.R. Meyer, A.J. Smith, S.E. Matsuura, W.A. Scott</li><li class="title"><p>Effects of primer-template sequence on ATP-dependent removal of chain-terminating nucleotide analogues by HIV-1 reverse transcriptase</p></li><li class="source"><p>J. Biol. Chem., 279 (2004), pp. 45389–45398</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref75" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref75'] = new Array();
        </script></li><li id="bib76" data-t="r"><ul class="reference" id="bibsbref76"><li class="referenceLabel"><a href="#bib76" class="label intra_ref">76</a></li><li class="author">N. Sluis-Cremer, D. Arion, U. Parikh, D. Koontz, R.F. Schinazi, J.W. Mellors, M.A. Parniak</li><li class="title"><p>The 3′-azido group is not the primary determinant of 3′-azido-3′-deoxythymidine (AZT) responsible for the excision phenotype of AZT-resistant HIV-1</p></li><li class="source"><p>J. Biol. Chem., 280 (2005), pp. 29047–29052</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref76" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref76'] = new Array();
        </script></li><li id="bib77" data-t="r"><ul class="reference" id="bibsbref77"><li class="referenceLabel"><a href="#bib77" class="label intra_ref">77</a></li><li class="author">P.L. Boyer, S.G. Sarafianos, E. Arnold, S.H. Hughes</li><li class="title"><p>Nucleoside analog resistance caused by insertions in the fingers of human immunodeficiency virus type 1 reverse transcriptase involves ATP-mediated excision</p></li><li class="source"><p>J. Virol., 76 (2002), pp. 9143–9151</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref77" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref77'] = new Array();
        </script></li><li id="bib78" data-t="r"><ul class="reference" id="bibsbref78"><li class="referenceLabel"><a href="#bib78" class="label intra_ref">78</a></li><li class="author">P.R. Meyer, J. Lennerstrand, S.E. Matsuura, B.A. Larder, W.A. Scott</li><li class="title"><p>Effects of dipeptide insertions between codons 69 and 70 of human immunodeficiency virus type 1 reverse transcriptase on primer unblocking, deoxynucleoside triphosphate inhibition, and DNA chain elongation</p></li><li class="source"><p>J. Virol., 77 (2003), pp. 3871–3877</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref78" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref78'] = new Array();
        </script></li><li id="bib79" data-t="r"><ul class="reference" id="bibsbref79"><li class="referenceLabel"><a href="#bib79" class="label intra_ref">79</a></li><li class="author">K.L. White, J.M. Chen, N.A. Margot, T. Wrin, C.J. Petropoulos, L.K. Naeger <em>et al.</em></li><li class="title"><p>Molecular mechanisms of tenofovir resistance conferred by human immunodeficiency virus type 1 reverse transcriptase containing a diserine insertion after residue 69 and multiple thymidine analog-associated mutations</p></li><li class="source"><p>Antimicrob. Agents Chemother., 48 (2004), pp. 992–1003</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref79" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref79'] = new Array();
        </script></li><li id="bib80" data-t="r"><ul class="reference" id="bibsbref80"><li class="referenceLabel"><a href="#bib80" class="label intra_ref">80</a></li><li class="author">A. Mas, M. Parera, C. Briones, V. Soriano, M.A. Martinez, E. Domingo, L. Menendez-Arias</li><li class="title"><p>Role of a dipeptide insertion between codons 69 and 70 of HIV-1 reverse transcriptase in the mechanism of AZT resistance</p></li><li class="source"><p>EMBO J., 19 (2000), pp. 5752–5761</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref80" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref80'] = new Array();
        </script></li><li id="bib81" data-t="r"><ul class="reference" id="bibsbref81"><li class="referenceLabel"><a href="#bib81" class="label intra_ref">81</a></li><li class="author">L. Andreoletti, L. Weiss, A. Si-Mohamed, C. Piketty, T. Prazuck, G. Calamy <em>et al.</em></li><li class="title"><p>Multidrug-resistant HIV-1 RNA and proviral DNA variants harboring new dipeptide insertions in the reverse transcriptase pol gene</p></li><li class="source"><p>J. Acquir. Immune Defic. Syndr., 29 (2002), pp. 102–104</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref81" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref81'] = new Array();
        </script></li><li id="bib82" data-t="r"><ul class="reference" id="bibsbref82"><li class="referenceLabel"><a href="#bib82" class="label intra_ref">82</a></li><li class="author">P. Bonfanti, I. Faggion, S. La Seta Catamancio, M. Violin, C. Balotta, S. Rusconi</li><li class="title"><p>Response to antiretroviral therapy in a patient with an uncommon codon 69 insertion in the human immunodeficiency virus type 1 reverse transcriptase</p></li><li class="source"><p>Antimicrob. Agents Chemother., 44 (2000), p. 1767</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref82" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref82'] = new Array();
        </script></li><li id="bib83" data-t="r"><ul class="reference" id="bibsbref83"><li class="referenceLabel"><a href="#bib83" class="label intra_ref">83</a></li><li class="author">A. De Antoni, A. Foli, J. Lisziewicz, F. Lori</li><li class="title"><p>Mutations in the pol gene of human immunodeficiency virus type 1 in infected patients receiving didanosine and hydroxyurea combination therapy</p></li><li class="source"><p>J. Infect. Dis., 176 (1997), pp. 899–903</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref83" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref83'] = new Array();
        </script></li><li id="bib84" data-t="r"><ul class="reference" id="bibsbref84"><li class="referenceLabel"><a href="#bib84" class="label intra_ref">84</a></li><li class="author">J.J. de Jong, J. Goudsmit, V.V. Lukashov, M.E. Hillebrand, E. Baan, R. Huismans <em>et al.</em></li><li class="title"><p>Insertion of two amino acids combined with changes in reverse transcriptase containing tyrosine-215 of HIV-1 resistant to multiple nucleoside analogs</p></li><li class="source"><p>AIDS, 13 (1999), pp. 75–80</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref84" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref84'] = new Array();
        </script></li><li id="bib85" data-t="r"><ul class="reference" id="bibsbref85"><li class="referenceLabel"><a href="#bib85" class="label intra_ref">85</a></li><li class="author">B.A. Larder, S. Bloor, S.D. Kemp, K. Hertogs, R.L. Desmet, V. Miller <em>et al.</em></li><li class="title"><p>A family of insertion mutations between codons 67 and 70 of human immunodeficiency virus type 1 reverse transcriptase confer multinucleoside analog resistance</p></li><li class="source"><p>Antimicrob. Agents Chemother., 43 (1999), pp. 1961–1967</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref85" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref85'] = new Array();
        </script></li><li id="bib86" data-t="r"><ul class="reference" id="bibsbref86"><li class="referenceLabel"><a href="#bib86" class="label intra_ref">86</a></li><li class="author">A. Rakik, M. Ait-Khaled, P. Griffin, T.A. Thomas, M. Tisdale, J.P. Kleim</li><li class="title"><p>A novel genotype encoding a single amino acid insertion and five other substitutions between residues 64 and 74 of the HIV-1 reverse transcriptase confers high-level cross-resistance to nucleoside reverse transcriptase inhibitors. Abacavir CNA2007 International Study Group</p></li><li class="source"><p>J. Acquir. Immune Defic. Syndr., 22 (1999), pp. 139–145</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref86" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref86'] = new Array();
        </script></li><li id="bib87" data-t="r"><ul class="reference" id="bibsbref87"><li class="referenceLabel"><a href="#bib87" class="label intra_ref">87</a></li><li class="author">M.A. Winters, K.L. Coolley, Y.A. Girard, D.J. Levee, H. Hamdan, R.W. Shafer <em>et al.</em></li><li class="title"><p>A 6-basepair insert in the reverse transcriptase gene of human immunodeficiency virus type 1 confers resistance to multiple nucleoside inhibitors</p></li><li class="source"><p>J. Clin. Invest., 102 (1998), pp. 1769–1775</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref87" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref87'] = new Array();
        </script></li><li id="bib88" data-t="r"><ul class="reference" id="bibsbref88"><li class="referenceLabel"><a href="#bib88" class="label intra_ref">88</a></li><li class="author">M. Gotte, D. Arion, M.A. Parniak, M.A. Wainberg</li><li class="title"><p>The M184V mutation in the reverse transcriptase of human immunodeficiency virus type 1 impairs rescue of chain-terminated DNA synthesis</p></li><li class="source"><p>J. Virol., 74 (2000), pp. 3579–3585</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref88" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref88'] = new Array();
        </script></li><li id="bib89" data-t="r"><ul class="reference" id="bibsbref89"><li class="referenceLabel"><a href="#bib89" class="label intra_ref">89</a></li><li class="author">P.L. Boyer, S.G. Sarafianos, E. Arnold, S.H. Hughes</li><li class="title"><p>The M184V mutation reduces the selective excision of zidovudine 5′-monophosphate (AZTMP) by the reverse transcriptase of human immunodeficiency virus type 1</p></li><li class="source"><p>J. Virol., 76 (2002), pp. 3248–3256</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref89" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref89'] = new Array();
        </script></li><li id="bib90" data-t="r"><ul class="reference" id="bibsbref90"><li class="referenceLabel"><a href="#bib90" class="label intra_ref">90</a></li><li class="author">L.K. Naeger, N.A. Margot, M.D. Miller</li><li class="title"><p>Increased drug susceptibility of HIV-1 reverse transcriptase mutants containing M184V and zidovudine-associated mutations: analysis of enzyme processivity, chain-terminator removal and viral replication</p></li><li class="source"><p>Antivir. Ther., 6 (2001), pp. 115–126</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref90" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref90'] = new Array();
        </script></li><li id="bib91" data-t="r"><ul class="reference" id="bibsbref91"><li class="referenceLabel"><a href="#bib91" class="label intra_ref">91</a></li><li class="author">S.H. Yap, C.W. Sheen, J. Fahey, M. Zanin, D. Tyssen, V.D. Lima <em>et al.</em></li><li class="title"><p>N348I in the connection domain of HIV-1 reverse transcriptase confers zidovudine and nevirapine resistance</p></li><li class="source"><p>PLoS Med., 4 (2007), p. e335</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref91" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref91'] = new Array();
        </script></li><li id="bib92" data-t="r"><ul class="reference" id="bibsbref92"><li class="referenceLabel"><a href="#bib92" class="label intra_ref">92</a></li><li class="author">A. Hachiya, E.N. Kodama, S.G. Sarafianos, M.M. Schuckmann, Y. Sakagami, M. Matsuoka <em>et al.</em></li><li class="title"><p>Amino acid mutation N348I in the connection subdomain of human immunodeficiency virus type 1 reverse transcriptase confers multiclass resistance to nucleoside and nonnucleoside reverse transcriptase inhibitors</p></li><li class="source"><p>J. Virol., 82 (2008), pp. 3261–3270</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref92" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref92'] = new Array();
        </script></li><li id="bib93" data-t="r"><ul class="reference" id="bibsbref93"><li class="referenceLabel"><a href="#bib93" class="label intra_ref">93</a></li><li class="author">J. Ding, K. Das, H. Moereels, L. Koymans, K. Andries, P.A. Janssen <em>et al.</em></li><li class="title"><p>Structure of HIV-1 RT/TIBO R 86183 complex reveals similarity in the binding of diverse nonnucleoside inhibitors</p></li><li class="source"><p>Nature Struct. Biol., 2 (1995), pp. 407–415</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref93" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref93'] = new Array();
        </script></li><li id="bib94" data-t="r"><ul class="reference" id="bibsbref94"><li class="referenceLabel"><a href="#bib94" class="label intra_ref">94</a></li><li class="author">Y. Hsiou, K. Das, J. Ding, A.D. Clark Jr, J.P. Kleim, M. Rosner <em>et al.</em></li><li class="title"><p>Structures of Tyr188Leu mutant and wild-type HIV-1 reverse transcriptase complexed with the non-nucleoside inhibitor HBY 097: Inhibitor flexibility is a useful design feature for reducing drug resistance</p></li><li class="source"><p>J. Mol. Biol., 284 (1998), pp. 313–323</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref94" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref94'] = new Array();
        </script></li><li id="bib95" data-t="r"><ul class="reference" id="bibsbref95"><li class="referenceLabel"><a href="#bib95" class="label intra_ref">95</a></li><li class="author">K. Das, J. Ding, Y. Hsiou, A.D. Clark Jr, H. Moereels, L. Koymans <em>et al.</em></li><li class="title"><p>Crystal structures of 8-Cl and 9-Cl TIBO complexed with wild-type HIV-1 RT and 8-Cl TIBO complexed with the Tyr181Cys HIV-1 RT drug-resistant mutant</p></li><li class="source"><p>J. Mol. Biol., 264 (1996), pp. 1085–1100</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref95" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref95'] = new Array();
        </script></li><li id="bib96" data-t="r"><ul class="reference" id="bibsbref96"><li class="referenceLabel"><a href="#bib96" class="label intra_ref">96</a></li><li class="author">J. Ren, R. Esnouf, A. Hopkins, C. Ross, Y. Jones, D. Stammers, D. Stuart</li><li class="title"><p>The structure of HIV-1 reverse transcriptase complexed with 9-chloro-TIBO: lessons for inhibitor design</p></li><li class="source"><p>Structure, 3 (1995), pp. 915–926</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref96" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref96'] = new Array();
        </script></li><li id="bib97" data-t="r"><ul class="reference" id="bibsbref97"><li class="referenceLabel"><a href="#bib97" class="label intra_ref">97</a></li><li class="author">J. Ren, C. Nichols, L. Bird, P. Chamberlain, K. Weaver, S. Short <em>et al.</em></li><li class="title"><p>Structural mechanisms of drug resistance for mutations at codons 181 and 188 in HIV-1 reverse transcriptase and the improved resilience of second generation non-nucleoside inhibitors</p></li><li class="source"><p>J. Mol. Biol., 312 (2001), pp. 795–805</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref97" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref97'] = new Array();
        </script></li><li id="bib98" data-t="r"><ul class="reference" id="bibsbref98"><li class="referenceLabel"><a href="#bib98" class="label intra_ref">98</a></li><li class="author">J. Ren, C.E. Nichols, P.P. Chamberlain, K.L. Weaver, S.A. Short, D.K. Stammers</li><li class="title"><p>Crystal structures of HIV-1 reverse transcriptases mutated at codons 100, 106 and 108 and mechanisms of resistance to non-nucleoside inhibitors</p></li><li class="source"><p>J. Mol. Biol., 336 (2004), pp. 569–578</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref98" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref98'] = new Array();
        </script></li><li id="bib99" data-t="r"><ul class="reference" id="bibsbref99"><li class="referenceLabel"><a href="#bib99" class="label intra_ref">99</a></li><li class="author">K. Das, A.D. Clark Jr, P.J. Lewi, J. Heeres, M.R. De Jonge, L.M. Koymans <em>et al.</em></li><li class="title"><p>Roles of conformational and positional adaptability in structure-based design of TMC125-R165335 (etravirine) and related non-nucleoside reverse transcriptase inhibitors that are highly potent and effective against wild-type and drug-resistant HIV-1 variants</p></li><li class="source"><p>J. Med. Chem., 47 (2004), pp. 2550–2560</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref99" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref99'] = new Array();
        </script></li><li id="bib100" data-t="r"><ul class="reference" id="bibsbref100"><li class="referenceLabel"><a href="#bib100" class="label intra_ref">100</a></li><li class="author">K. Das, J.D. Bauman, A.D. Clark Jr., Y.V. Frenkel, P.J. Lewi, A.J. Shatkin, S.H. Hughes, E. Arnold</li><li class="title"><p>High-resolution structures of HIV-1 reverse transcriptase/TMC278 complexes: strategic flexibility explains potency against resistance mutations</p></li><li class="source"><p>Proc. Natl. Acad. Sci. U.S.A., 105 (2008), pp. 1466–1471</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref100" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref100'] = new Array();
        </script></li><li id="bib101" data-t="r"><ul class="reference" id="bibsbref101"><li class="referenceLabel"><a href="#bib101" class="label intra_ref">101</a></li><li class="author">Y. Hsiou, J. Ding, K. Das, A.D. Clark Jr, P.L. Boyer, P. Lewi <em>et al.</em></li><li class="title"><p>The Lys103Asn mutation of HIV-1 RT: a novel mechanism of drug resistance</p></li><li class="source"><p>J. Mol. Biol., 309 (2001), pp. 437–445</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref101" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref101'] = new Array();
        </script></li><li id="bib102" data-t="r"><ul class="reference" id="bibsbref102"><li class="referenceLabel"><a href="#bib102" class="label intra_ref">102</a></li><li class="author">J. Ren, C.E. Nichols, P.P. Chamberlain, K.L. Weaver, S.A. Short, J.H. Chan <em>et al.</em></li><li class="title"><p>Relationship of potency and resilience to drug resistant mutations for GW420867X revealed by crystal structures of inhibitor complexes for wild-type, Leu100Ile, Lys101Glu, and Tyr188Cys mutant HIV-1 reverse transcriptases</p></li><li class="source"><p>J. Med. Chem., 50 (2007), pp. 2301–2309</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref102" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref102'] = new Array();
        </script></li><li id="bib103" data-t="r"><ul class="reference" id="bibsbref103"><li class="referenceLabel"><a href="#bib103" class="label intra_ref">103</a></li><li class="author">P.A. Janssen, P.J. Lewi, E. Arnold, F. Daeyaert, M. de Jonge, J. Heeres <em>et al.</em></li><li class="title"><p>In search of a novel anti-HIV drug: multidisciplinary coordination in the discovery of 4-[[4-[[4-[(1E)-2-cyanoethenyl]-2,6-dimethylphenyl]amino]-2- pyrimidinyl]amino]benzonitrile (R278474, rilpivirine)</p></li><li class="source"><p>J. Med. Chem., 48 (2005), pp. 1901–1909</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref103" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref103'] = new Array();
        </script></li><li id="bib104" data-t="r"><ul class="reference" id="bibsbref104"><li class="referenceLabel"><a href="#bib104" class="label intra_ref">104</a></li><li class="author">G.N. Nikolenko, K.A. Delviks-Frankenberry, S. Palmer, F. Maldarelli, M.J. Fivash Jr, J.M. Coffin, V.K. Pathak</li><li class="title"><p>Mutations in the connection domain of HIV-1 reverse transcriptase increase 3′-azido-3′-deoxythymidine resistance</p></li><li class="source"><p>Proc. Natl Acad. Sci. USA, 104 (2007), pp. 317–322</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref104" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref104'] = new Array();
        </script></li><li id="bib105" data-t="r"><ul class="reference" id="bibsbref105"><li class="referenceLabel"><a href="#bib105" class="label intra_ref">105</a></li><li class="author">K.A. Delviks-Frankenberry, G.N. Nikolenko, R. Barr, V.K. Pathak</li><li class="title"><p>Mutations in human immunodeficiency virus type 1 RNase H primer grip enhance 3′-azido-3′-deoxythymidine resistance</p></li><li class="source"><p>J. Virol., 81 (2007), pp. 6837–6845</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref105" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref105'] = new Array();
        </script></li><li id="bib106" data-t="r"><ul class="reference" id="bibsbref106"><li class="referenceLabel"><a href="#bib106" class="label intra_ref">106</a></li><li class="author">K.A. Delviks-Frankenberry, G.N. Nikolenko, P.L. Boyer, S.H. Hughes, J.M. Coffin, A. Jere, V.K. Pathak</li><li class="title"><p>HIV-1 reverse transcriptase connection subdomain mutations reduce template RNA degradation and enhance AZT excision</p></li><li class="source"><p>Proc. Natl Acad. Sci. USA, 105 (2008), pp. 10943–10948</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref106" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref106'] = new Array();
        </script></li><li id="bib107" data-t="r"><ul class="reference" id="bibsbref107"><li class="referenceLabel"><a href="#bib107" class="label intra_ref">107</a></li><li class="author">M. Ehteshami, G.L. Beilhartz, B.J. Scarth, E.P. Tchesnokov, S. McCormick, B. Wynhoven <em>et al.</em></li><li class="title"><p>Connection domain mutations N348I and A360V in HIV-1 reverse transcriptase enhance resistance to 3′-azido-3′-deoxythymidine through both RNase H-dependent and -independent mechanisms</p></li><li class="source"><p>J. Biol. Chem., 283 (2008), pp. 22222–22232</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref107" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref107'] = new Array();
        </script></li><li id="bib108" data-t="r"><ul class="reference" id="bibsbref108"><li class="referenceLabel"><a href="#bib108" class="label intra_ref">108</a></li><li class="author">E.S. Furfine, J.E. Reardon</li><li class="title"><p>Human immunodeficiency virus reverse transcriptase ribonuclease H: specificity of tRNA(Lys3)-primer excision</p></li><li class="source"><p>Biochemistry, 30 (1991), pp. 7041–7046</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref108" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref108'] = new Array();
        </script></li><li id="bib109" data-t="r"><ul class="reference" id="bibsbref109"><li class="referenceLabel"><a href="#bib109" class="label intra_ref">109</a></li><li class="author">K.A. Pullen, L.K. Ishimoto, J.J. Champoux</li><li class="title"><p>Incomplete removal of the RNA primer for minus-strand DNA synthesis by human immunodeficiency virus type 1 reverse transcriptase</p></li><li class="source"><p>J. Virol., 66 (1992), pp. 367–373</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref109" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref109'] = new Array();
        </script></li><li id="bib110" data-t="r"><ul class="reference" id="bibsbref110"><li class="referenceLabel"><a href="#bib110" class="label intra_ref">110</a></li><li class="author">H.E. Huber, C.C. Richardson</li><li class="title"><p>Processing of the primer for plus strand DNA synthesis by human immunodeficiency virus 1 reverse transcriptase</p></li><li class="source"><p>J. Biol. Chem., 265 (1990), pp. 10565–10573</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref110" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref110'] = new Array();
        </script></li><li id="bib111" data-t="r"><ul class="reference" id="bibsbref111"><li class="referenceLabel"><a href="#bib111" class="label intra_ref">111</a></li><li class="author">N. Tanese, S.P. Goff</li><li class="title"><p>Domain structure of the Moloney murine leukemia virus reverse transcriptase: mutational analysis and separate expression of the DNA polymerase and RNase H activities</p></li><li class="source"><p>Proc. Natl Acad. Sci. USA, 85 (1988), pp. 1777–1781</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref111" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref111'] = new Array();
        </script></li><li id="bib112" data-t="r"><ul class="reference" id="bibsbref112"><li class="referenceLabel"><a href="#bib112" class="label intra_ref">112</a></li><li class="author">O. Schatz, F.V. Cromme, F. Gruninger-Leitch, S.F. Le Grice</li><li class="title"><p>Point mutations in conserved amino acid residues within the C-terminal domain of HIV-1 reverse transcriptase specifically repress RNase H function</p></li><li class="source"><p>FEBS Lett., 257 (1989), pp. 311–314</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref112" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref112'] = new Array();
        </script></li><li id="bib113" data-t="r"><ul class="reference" id="bibsbref113"><li class="referenceLabel"><a href="#bib113" class="label intra_ref">113</a></li><li class="author">J.F. Davies II, Z. Hostomska, Z. Hostomsky, S.R. Jordan, D.A. Matthews</li><li class="title"><p>Crystal structure of the ribonuclease H domain of HIV-1 reverse transcriptase</p></li><li class="source"><p>Science, 252 (1991), pp. 88–95</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref113" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref113'] = new Array();
        </script></li><li id="bib114" data-t="r"><ul class="reference" id="bibsbref114"><li class="referenceLabel"><a href="#bib114" class="label intra_ref">114</a></li><li class="author">O. Schatz, J. Mous, S.F. Le Grice</li><li class="title"><p>HIV-1 RT-associated ribonuclease H displays both endonuclease and 3′–5′ exonuclease activity</p></li><li class="source"><p>EMBO J., 9 (1990), pp. 1171–1176</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref114" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref114'] = new Array();
        </script></li><li id="bib115" data-t="r"><ul class="reference" id="bibsbref115"><li class="referenceLabel"><a href="#bib115" class="label intra_ref">115</a></li><li class="author">B.M. Wohrl, K. Moelling</li><li class="title"><p>Interaction of HIV-1 ribonuclease H with polypurine tract containing RNA-DNA hybrids</p></li><li class="source"><p>Biochemistry, 29 (1990), pp. 10141–10147</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref115" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref115'] = new Array();
        </script></li><li id="bib116" data-t="r"><ul class="reference" id="bibsbref116"><li class="referenceLabel"><a href="#bib116" class="label intra_ref">116</a></li><li class="author">V. Gopalakrishnan, J.A. Peliska, S.J. Benkovic</li><li class="title"><p>Human immunodeficiency virus type 1 reverse transcriptase: spatial and temporal relationship between the polymerase and RNase H activities</p></li><li class="source"><p>Proc. Natl Acad. Sci. USA, 89 (1992), pp. 10763–10767</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref116" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref116'] = new Array();
        </script></li><li id="bib117" data-t="r"><ul class="reference" id="bibsbref117"><li class="referenceLabel"><a href="#bib117" class="label intra_ref">117</a></li><li class="author">M. Ghosh, K.J. Howard, C.E. Cameron, S.J. Benkovic, S.H. Hughes, S.F. Le Grice</li><li class="title"><p>Truncating alpha-helix E' of p66 human immunodeficiency virus reverse transcriptase modulates RNase H function and impairs DNA strand transfer</p></li><li class="source"><p>J. Biol. Chem., 270 (1995), pp. 7068–7076</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref117" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref117'] = new Array();
        </script></li><li id="bib118" data-t="r"><ul class="reference" id="bibsbref118"><li class="referenceLabel"><a href="#bib118" class="label intra_ref">118</a></li><li class="author">M. Gotte, S. Fackler, T. Hermann, E. Perola, L. Cellai, H.J. Gross <em>et al.</em></li><li class="title"><p>HIV-1 reverse transcriptase-associated RNase H cleaves RNA/RNA in arrested complexes: implications for the mechanism by which RNase H discriminates between RNA/RNA and RNA/DNA</p></li><li class="source"><p>EMBO J., 14 (1995), pp. 833–841</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref118" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref118'] = new Array();
        </script></li><li id="bib119" data-t="r"><ul class="reference" id="bibsbref119"><li class="referenceLabel"><a href="#bib119" class="label intra_ref">119</a></li><li class="author">H.Q. Gao, P.L. Boyer, E. Arnold, S.H. Hughes</li><li class="title"><p>Effects of mutations in the polymerase domain on the polymerase, RNase H and strand transfer activities of human immunodeficiency virus type 1 reverse transcriptase</p></li><li class="source"><p>J. Mol. Biol., 277 (1998), pp. 559–572</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref119" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref119'] = new Array();
        </script></li><li id="bib120" data-t="r"><ul class="reference" id="bibsbref120"><li class="referenceLabel"><a href="#bib120" class="label intra_ref">120</a></li><li class="author">H.Q. Gao, S.G. Sarafianos, E. Arnold, S.H. Hughes</li><li class="title"><p>Similarities and differences in the RNase H activities of human immunodeficiency virus type 1 reverse transcriptase and Moloney murine leukemia virus reverse transcriptase</p></li><li class="source"><p>J. Mol. Biol., 294 (1999), pp. 1097–1113</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref120" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref120'] = new Array();
        </script></li><li id="bib121" data-t="r"><ul class="reference" id="bibsbref121"><li class="referenceLabel"><a href="#bib121" class="label intra_ref">121</a></li><li class="author">M. Gotte, G. Maier, H.J. Gross, H. Heumann</li><li class="title"><p>Localization of the active site of HIV-1 reverse transcriptase-associated RNase H domain on a DNA template using site-specific generated hydroxyl radicals</p></li><li class="source"><p>J. Biol. Chem., 273 (1998), pp. 10139–10146</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref121" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref121'] = new Array();
        </script></li><li id="bib122" data-t="r"><ul class="reference" id="bibsbref122"><li class="referenceLabel"><a href="#bib122" class="label intra_ref">122</a></li><li class="author">J.A. Cowan, T. Ohyama, K. Howard, J.W. Rausch, S.M. Cowan, S.F. Le Grice</li><li class="title"><p>Metal-ion stoichiometry of the HIV-1 RT ribonuclease H domain: evidence for two mutually exclusive sites leads to new mechanistic insights on metal-mediated hydrolysis in nucleic acid biochemistry</p></li><li class="source"><p>J. Biol. Inorg. Chem., 5 (2000), pp. 67–74</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref122" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref122'] = new Array();
        </script></li><li id="bib123" data-t="r"><ul class="reference" id="bibsbref123"><li class="referenceLabel"><a href="#bib123" class="label intra_ref">123</a></li><li class="author">K. Pari, G.A. Mueller, E.F. DeRose, T.W. Kirby, R.E. London</li><li class="title"><p>Solution structure of the RNase H domain of the HIV-1 reverse transcriptase in the presence of magnesium</p></li><li class="source"><p>Biochemistry, 42 (2003), pp. 639–650</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref123" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref123'] = new Array();
        </script></li><li id="bib124" data-t="r"><ul class="reference" id="bibsbref124"><li class="referenceLabel"><a href="#bib124" class="label intra_ref">124</a></li><li class="author">K. Klumpp, J.Q. Hang, S. Rajendran, Y. Yang, A. Derosier, P. Wong Kai In <em>et al.</em></li><li class="title"><p>Two-metal ion mechanism of RNA cleavage by HIV RNase H and mechanism-based design of selective HIV RNase H inhibitors</p></li><li class="source"><p>Nucleic Acids Res., 31 (2003), pp. 6852–6859</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref124" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref124'] = new Array();
        </script></li><li id="bib125" data-t="r"><ul class="reference" id="bibsbref125"><li class="referenceLabel"><a href="#bib125" class="label intra_ref">125</a></li><li class="author">M. Nowotny, S.A. Gaidamakov, R.J. Crouch, W. Yang</li><li class="title"><p>Crystal structures of RNase H bound to an RNA/DNA hybrid: substrate specificity and metal-dependent catalysis</p></li><li class="source"><p>Cell, 121 (2005), pp. 1005–1016</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref125" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref125'] = new Array();
        </script></li><li id="bib126" data-t="r"><ul class="reference" id="bibsbref126"><li class="referenceLabel"><a href="#bib126" class="label intra_ref">126</a></li><li class="author">O. Fedoroff, M. Salazar, B.R. Reid</li><li class="title"><p>Structure of a DNA:RNA hybrid duplex. Why RNase H does not cleave pure RNA</p></li><li class="source"><p>J Mol Biol, 233 (1993), pp. 509–523</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref126" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref126'] = new Array();
        </script></li><li id="bib127" data-t="r"><ul class="reference" id="bibsbref127"><li class="referenceLabel"><a href="#bib127" class="label intra_ref">127</a></li><li class="author">M. Salazar, O. Fedoroff, L. Zhu, B.R. Reid</li><li class="title"><p>The solution structure of the r(gcg)d(TATACCC):d(GGGTATACGC) Okazaki fragment contains two distinct duplex morphologies connected by a junction</p></li><li class="source"><p>J. Mol. Biol., 241 (1994), pp. 440–455</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref127" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref127'] = new Array();
        </script></li><li id="bib128" data-t="r"><ul class="reference" id="bibsbref128"><li class="referenceLabel"><a href="#bib128" class="label intra_ref">128</a></li><li class="author">L. Zhu, M. Salazar, B.R. Reid</li><li class="title"><p>DNA duplexes flanked by hybrid duplexes: the solution structure of chimeric junctions in [r(cgcg)d(TATACGCG)]2</p></li><li class="source"><p>Biochemistry, 34 (1995), pp. 2372–2380</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref128" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref128'] = new Array();
        </script></li><li id="bib129" data-t="r"><ul class="reference" id="bibsbref129"><li class="referenceLabel"><a href="#bib129" class="label intra_ref">129</a></li><li class="author">N.C. Horton, B.C. Finzel</li><li class="title"><p>The structure of an RNA/DNA hybrid: a substrate of the ribonuclease activity of HIV-1 reverse transcriptase</p></li><li class="source"><p>J. Mol. Biol., 264 (1996), pp. 521–533</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref129" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref129'] = new Array();
        </script></li><li id="bib130" data-t="r"><ul class="reference" id="bibsbref130"><li class="referenceLabel"><a href="#bib130" class="label intra_ref">130</a></li><li class="author">O.Y. Fedoroff, Y. Ge, B.R. Reid</li><li class="title"><p>Solution structure of r(gaggacug):d(CAGTCCTC) hybrid: implications for the initiation of HIV-1 (+)-strand synthesis</p></li><li class="source"><p>J. Mol. Biol., 269 (1997), pp. 225–239</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref130" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref130'] = new Array();
        </script></li><li id="bib131" data-t="r"><ul class="reference" id="bibsbref131"><li class="referenceLabel"><a href="#bib131" class="label intra_ref">131</a></li><li class="author">G.W. Han, M.L. Kopka, D. Cascio, K. Grzeskowiak, R.E. Dickerson</li><li class="title"><p>Structure of a DNA analog of the primer for HIV-1 RT second strand synthesis</p></li><li class="source"><p>J. Mol. Biol., 269 (1997), pp. 811–826</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref131" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref131'] = new Array();
        </script></li><li id="bib132" data-t="r"><ul class="reference" id="bibsbref132"><li class="referenceLabel"><a href="#bib132" class="label intra_ref">132</a></li><li class="author">M. Bachelin, G. Hessler, G. Kurz, J.G. Hacia, P.B. Dervan, H. Kessler</li><li class="title"><p>Structure of a stereoregular phosphorothioate DNA/RNA duplex</p></li><li class="source"><p>Nature Struct. Biol., 5 (1998), pp. 271–276</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref132" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref132'] = new Array();
        </script></li><li id="bib133" data-t="r"><ul class="reference" id="bibsbref133"><li class="referenceLabel"><a href="#bib133" class="label intra_ref">133</a></li><li class="author">T. Szyperski, M. Gotte, M. Billeter, E. Perola, L. Cellai, H. Heumann, K. Wuthrich</li><li class="title"><p>NMR structure of the chimeric hybrid duplex r(gcaguggc).r(gcca)d(CTGC) comprising the tRNA-DNA junction formed during initiation of HIV-1 reverse transcription</p></li><li class="source"><p>J. Biomol. NMR, 13 (1999), pp. 343–355</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref133" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref133'] = new Array();
        </script></li><li id="bib134" data-t="r"><ul class="reference" id="bibsbref134"><li class="referenceLabel"><a href="#bib134" class="label intra_ref">134</a></li><li class="author">J.W. Rausch, D. Lener, J.T. Miller, J.G. Julias, S.H. Hughes, S.F. Le Grice</li><li class="title"><p>Altering the RNase H primer grip of human immunodeficiency virus reverse transcriptase modifies cleavage specificity</p></li><li class="source"><p>Biochemistry, 41 (2002), pp. 4856–4865</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref134" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref134'] = new Array();
        </script></li><li id="bib135" data-t="r"><ul class="reference" id="bibsbref135"><li class="referenceLabel"><a href="#bib135" class="label intra_ref">135</a></li><li class="author">E.A. Abbondanzieri, G. Bokinsky, J.W. Rausch, J.X. Zhang, S.F. Le Grice, X. Zhuang</li><li class="title"><p>Dynamic binding orientations direct activity of HIV reverse transcriptase</p></li><li class="source"><p>Nature, 453 (2008), pp. 184–189</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref135" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref135'] = new Array();
        </script></li><li id="bib136" data-t="r"><ul class="reference" id="bibsbref136"><li class="referenceLabel"><a href="#bib136" class="label intra_ref">136</a></li><li class="author">J.M. Whitcomb, R. Kumar, S.H. Hughes</li><li class="title"><p>Sequence of the circle junction of human immunodeficiency virus type 1: implications for reverse transcription and integration</p></li><li class="source"><p>J. Virol., 64 (1990), pp. 4903–4906</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref136" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref136'] = new Array();
        </script></li><li id="bib137" data-t="r"><ul class="reference" id="bibsbref137"><li class="referenceLabel"><a href="#bib137" class="label intra_ref">137</a></li><li class="author">R.A. Katz, G. Merkel, J. Kulkosky, J. Leis, A.M. Skalka</li><li class="title"><p>The avian retroviral IN protein is both necessary and sufficient for integrative recombination in vitro</p></li><li class="source"><p>Cell, 63 (1990), pp. 87–95</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref137" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref137'] = new Array();
        </script></li><li id="bib138" data-t="r"><ul class="reference" id="bibsbref138"><li class="referenceLabel"><a href="#bib138" class="label intra_ref">138</a></li><li class="author">K. Klumpp, T. Mirzadegan</li><li class="title"><p>Recent progress in the design of small molecule inhibitors of HIV RNase H</p></li><li class="source"><p>Curr. Pharm. Des., 12 (2006), pp. 1909–1922</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref138" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref138'] = new Array();
        </script></li><li id="bib139" data-t="r"><ul class="reference" id="bibsbref139"><li class="referenceLabel"><a href="#bib139" class="label intra_ref">139</a></li><li class="author">E. Tramontano</li><li class="title"><p>HIV-1 RNase H: recent progress in an exciting, yet little explored, drug target</p></li><li class="source"><p>Mini Rev. Med. Chem., 6 (2006), pp. 727–737</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref139" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref139'] = new Array();
        </script></li><li id="bib140" data-t="r"><ul class="reference" id="bibsbref140"><li class="referenceLabel"><a href="#bib140" class="label intra_ref">140</a></li><li class="author">G. Borkow, R.S. Fletcher, J. Barnard, D. Arion, D. Motakis, G.I. Dmitrienko, M.A. Parniak</li><li class="title"><p>Inhibition of the ribonuclease H and DNA polymerase activities of HIV-1 reverse transcriptase by N-(4-tert-butylbenzoyl)-2-hydroxy-1-naphthaldehyde hydrazone</p></li><li class="source"><p>Biochemistry, 36 (1997), pp. 3179–3185</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref140" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref140'] = new Array();
        </script></li><li id="bib141" data-t="r"><ul class="reference" id="bibsbref141"><li class="referenceLabel"><a href="#bib141" class="label intra_ref">141</a></li><li class="author">D.M. Himmel, S.G. Sarafianos, S. Dharmasena, M.M. Hossain, K. McCoy-Simandle <em>et al.</em></li><li class="title"><p>HIV-1 reverse transcriptase structure with RNase H inhibitor dihydroxy benzoyl naphthyl hydrazone bound at a novel site</p></li><li class="source"><p>ACS Chem. Biol., 1 (2006), pp. 702–712</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref141" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref141'] = new Array();
        </script></li><li id="bib142" data-t="r"><ul class="reference" id="bibsbref142"><li class="referenceLabel"><a href="#bib142" class="label intra_ref">142</a></li><li class="author">D. Arion, N. Sluis-Cremer, K.L. Min, M.E. Abram, R.S. Fletcher, M.A. Parniak</li><li class="title"><p>Mutational analysis of Tyr-501 of HIV-1 reverse transcriptase. Effects on ribonuclease H activity and inhibition of this activity by N-acylhydrazones</p></li><li class="source"><p>J. Biol. Chem., 277 (2002), pp. 1370–1374</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref142" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref142'] = new Array();
        </script></li><li id="bib143" data-t="r"><ul class="reference" id="bibsbref143"><li class="referenceLabel"><a href="#bib143" class="label intra_ref">143</a></li><li class="author">W.R. Davis, J. Tomsho, S. Nikam, E.M. Cook, D. Somand, J.A. Peliska</li><li class="title"><p>Inhibition of HIV-1 reverse transcriptase-catalyzed DNA strand transfer reactions by 4-chlorophenylhydrazone of mesoxalic acid</p></li><li class="source"><p>Biochemistry, 39 (2000), pp. 14279–14291</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref143" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref143'] = new Array();
        </script></li><li id="bib144" data-t="r"><ul class="reference" id="bibsbref144"><li class="referenceLabel"><a href="#bib144" class="label intra_ref">144</a></li><li class="author">C.A. Shaw-Reid, V. Munshi, P. Graham, A. Wolfe, M. Witmer, R. Danzeisen <em>et al.</em></li><li class="title"><p>Inhibition of HIV-1 ribonuclease H by a novel diketo acid, 4-[5-(benzoylamino)thien-2-yl]-2,4-dioxobutanoic acid</p></li><li class="source"><p>J. Biol. Chem., 278 (2003), pp. 2777–2780</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref144" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref144'] = new Array();
        </script></li><li id="bib145" data-t="r"><ul class="reference" id="bibsbref145"><li class="referenceLabel"><a href="#bib145" class="label intra_ref">145</a></li><li class="author">J.Q. Hang, S. Rajendran, Y. Yang, Y. Li, P.W. In, H. Overton <em>et al.</em></li><li class="title"><p>Activity of the isolated HIV RNase H domain and specific inhibition by N-hydroxyimides</p></li><li class="source"><p>Biochem. Biophys. Res. Commun., 317 (2004), pp. 321–329</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref145" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref145'] = new Array();
        </script></li><li id="bib146" data-t="r"><ul class="reference" id="bibsbref146"><li class="referenceLabel"><a href="#bib146" class="label intra_ref">146</a></li><li class="author">S.R. Budihas, I. Gorshkova, S. Gaidamakov, A. Wamiru, M.K. Bona, M.A. Parniak <em>et al.</em></li><li class="title"><p>Selective inhibition of HIV-1 reverse transcriptase-associated ribonuclease H activity by hydroxylated tropolones</p></li><li class="source"><p>Nucleic Acids Res., 33 (2005), pp. 1249–1256</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref146" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref146'] = new Array();
        </script></li><li id="bib147" data-t="r"><ul class="reference" id="bibsbref147"><li class="referenceLabel"><a href="#bib147" class="label intra_ref">147</a></li><li class="author">V.E. Marquez, T. Ben-Kasus, J.J. Barchi Jr, K.M. Green, M.C. Nicklaus, R. Agbaria</li><li class="title"><p>Experimental and structural evidence that herpes 1 kinase and cellular DNA polymerase(s) discriminate on the basis of sugar pucker</p></li><li class="source"><p>J. Am. Chem. Soc., 126 (2004), pp. 543–549</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref147" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref147'] = new Array();
        </script></li><li id="bib148" data-t="r"><ul class="reference" id="bibsbref148"><li class="referenceLabel"><a href="#bib148" class="label intra_ref">148</a></li><li class="author">V.E. Marquez, S.H. Hughes, S. Sei, R. Agbaria</li><li class="title"><p>The history of N-methanocarbathymidine: the investigation of a conformational concept leads to the discovery of a potent and selective nucleoside antiviral agent</p></li><li class="source"><p>Antiviral Res., 71 (2006), pp. 268–275</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref148" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref148'] = new Array();
        </script></li><li id="bib149" data-t="r"><ul class="reference" id="bibsbref149"><li class="referenceLabel"><a href="#bib149" class="label intra_ref">149</a></li><li class="author">P.L. Boyer, J.G. Julias, V.E. Marquez, S.H. Hughes</li><li class="title"><p>Fixed conformation nucleoside analogs effectively inhibit excision-proficient HIV-1 reverse transcriptases</p></li><li class="source"><p>J. Mol. Biol., 345 (2005), pp. 441–450</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref149" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref149'] = new Array();
        </script></li><li id="bib150" data-t="r"><ul class="reference" id="bibsbref150"><li class="referenceLabel"><a href="#bib150" class="label intra_ref">150</a></li><li class="author">S. Doublie, S. Tabor, A.M. Long, C.C. Richardson, T. Ellenberger</li><li class="title"><p>Crystal structure of a bacteriophage T7 DNA replication complex at 2.2&nbsp;Å resolution</p></li><li class="source"><p>Nature, 391 (1998), pp. 251–258</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref150" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref150'] = new Array();
        </script></li><li id="bib151" data-t="r"><ul class="reference" id="bibsbref151"><li class="referenceLabel"><a href="#bib151" class="label intra_ref">151</a></li><li class="author">H. Pelletier, M.R. Sawaya, A. Kumar, S.H. Wilson, J. Kraut</li><li class="title"><p>Structures of ternary complexes of rat DNA polymerase beta, a DNA template-primer, and ddCTP</p></li><li class="source"><p>Science, 264 (1994), pp. 1891–1903</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref151" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref151'] = new Array();
        </script></li><li id="bib152" data-t="r"><ul class="reference" id="bibsbref152"><li class="referenceLabel"><a href="#bib152" class="label intra_ref">152</a></li><li class="author">S.H. Eom, J. Wang, T.A. Steitz</li><li class="title"><p>Structure of Taq polymerase with DNA at the polymerase active site</p></li><li class="source"><p>Nature, 382 (1996), pp. 278–281</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref152" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref152'] = new Array();
        </script></li><li id="bib153" data-t="r"><ul class="reference" id="bibsbref153"><li class="referenceLabel"><a href="#bib153" class="label intra_ref">153</a></li><li class="author">T. Maltseva, E. Usova, S. Eriksson, J. Milecki, A. Foldesi, J. Chattopadhayaya</li><li class="title"><p>The NMR conformation study of the complexes of deoxycytidine kinase (dCK) and 2′-deoxycytidine/2′-deoxyadenosine</p></li><li class="source"><p>Nucleosides Nucleotides Nucleic Acids, 20 (2001), pp. 1225–1228</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref153" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref153'] = new Array();
        </script></li><li id="bib154" data-t="r"><ul class="reference" id="bibsbref154"><li class="referenceLabel"><a href="#bib154" class="label intra_ref">154</a></li><li class="author">K. Lee, Y. Chong, C.K. Chu</li><li class="title"><p>Understanding the mode of action of L-nucleosides as antiviral agents: a molecular modeling approach</p></li><li class="source"><p>Nucleosides Nucleotides Nucleic Acids, 20 (2001), pp. 385–388</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref154" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref154'] = new Array();
        </script></li><li id="bib155" data-t="r"><ul class="reference" id="bibsbref155"><li class="referenceLabel"><a href="#bib155" class="label intra_ref">155</a></li><li class="author">Y. Xu, O. Sellam, S. Morera, S. Sarfati, R. Biondi, M. Veron, J. Janin</li><li class="title"><p>X-ray analysis of azido-thymidine diphosphate binding to nucleoside diphosphate kinase</p></li><li class="source"><p>Proc. Natl Acad. Sci. USA, 94 (1997), pp. 7162–7165</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref155" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref155'] = new Array();
        </script></li><li id="bib156" data-t="r"><ul class="reference" id="bibsbref156"><li class="referenceLabel"><a href="#bib156" class="label intra_ref">156</a></li><li class="author">S. Eriksson, B. Munch-Petersen, K. Johansson, H. Eklund</li><li class="title"><p>Structure and function of cellular deoxyribonucleoside kinases</p></li><li class="source"><p>Cell Mol. Life Sci., 59 (2002), pp. 1327–1346</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref156" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref156'] = new Array();
        </script></li><li id="bib157" data-t="r"><ul class="reference" id="bibsbref157"><li class="referenceLabel"><a href="#bib157" class="label intra_ref">157</a></li><li class="author">H. Ohrui, H. Mitsuya</li><li class="title"><p>4′-C-substituted-2′-deoxynucleosides: a family of antiretroviral agents which are potent against drug-resistant HIV variants</p></li><li class="source"><p>Curr. Drug Targets Infect. Disord., 1 (2001), pp. 1–10</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref157" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref157'] = new Array();
        </script></li><li id="bib158" data-t="r"><ul class="reference" id="bibsbref158"><li class="referenceLabel"><a href="#bib158" class="label intra_ref">158</a></li><li class="author">E.I. Kodama, S. Kohgo, K. Kitano, H. Machida, H. Gatanaga, S. Shigeta <em>et al.</em></li><li class="title"><p>4′-Ethynyl nucleoside analogs: potent inhibitors of multidrug-resistant human immunodeficiency virus variants in vitro</p></li><li class="source"><p>Antimicrob. Agents Chemother., 45 (2001), pp. 1539–1546</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref158" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref158'] = new Array();
        </script></li><li id="bib159" data-t="r"><ul class="reference" id="bibsbref159"><li class="referenceLabel"><a href="#bib159" class="label intra_ref">159</a></li><li class="source">Kawamoto, A., Kodama, E., Sarafianos, S. G., Sakagami, Y., Kohgo, S., Kitano, K. <em>et al.</em> (2008). 2′-Deoxy-4′-C-ethynyl-2-halo-adenosines active against drug-resistant human immunodeficiency virus type 1 variants. <em>Int. J. Biochem. Cell Biol</em>. In the press.</li><li data-sr="N" class="external refPlaceHolder" id="bibsbref159" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref159'] = new Array();
        </script></li><li id="bib160" data-t="r"><ul class="reference" id="bibsbref160"><li class="referenceLabel"><a href="#bib160" class="label intra_ref">160</a></li><li class="author">H. Ohrui, S. Kohgo, H. Hayakawa, E. Kodama, M. Matsuoka, T. Nakata, H. Mitsuya</li><li class="title"><p>2′-Deoxy-4′-C-ethynyl-2-fluoroadenosine: a nucleoside reverse transcriptase inhibitor with highly potent activity against all HIV-1 strains, favorable toxic profiles and stability in plasma</p></li><li class="source"><p>Nucleic Acids Symp. Ser (Oxf) (2006), pp. 1–2</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref160" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref160'] = new Array();
        </script></li><li id="bib161" data-t="r"><ul class="reference" id="bibsbref161"><li class="referenceLabel"><a href="#bib161" class="label intra_ref">161</a></li><li class="author">H. Nakata, M. Amano, Y. Koh, E. Kodama, G. Yang, C.M. Bailey <em>et al.</em></li><li class="title"><p>Activity against human immunodeficiency virus type 1, intracellular metabolism, and effects on human DNA polymerases of 4′-ethynyl-2-fluoro-2′-deoxyadenosine</p></li><li class="source"><p>Antimicrob. Agents Chemother., 51 (2007), pp. 2701–2708</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref161" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref161'] = new Array();
        </script></li><li id="bib162" data-t="r"><ul class="reference" id="bibsbref162"><li class="referenceLabel"><a href="#bib162" class="label intra_ref">162</a></li><li class="author">P.L. Boyer, J.G. Julias, Z. Ambrose, M.A. Siddiqui, V.E. Marquez, S.H. Hughes</li><li class="title"><p>The nucleoside analogs 4′C-methyl thymidine and 4′C-ethyl thymidine block DNA synthesis by wild-type HIV-1 RT and excision proficient NRTI resistant RT variants</p></li><li class="source"><p>J. Mol. Biol., 371 (2007), pp. 873–882</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref162" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref162'] = new Array();
        </script></li><li id="bib163" data-t="r"><ul class="reference" id="bibsbref163"><li class="referenceLabel"><a href="#bib163" class="label intra_ref">163</a></li><li class="author">H. Tanaka, K. Haraguchi, H. Kumamoto, M. Baba, Y.C. Cheng</li><li class="title"><p>4′-Ethynylstavudine (4′-Ed4T) has potent anti-HIV-1 activity with reduced toxicity and shows a unique activity profile against drug-resistant mutants</p></li><li class="source"><p>Antivir. Chem. Chemother., 16 (2005), pp. 217–221</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref163" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref163'] = new Array();
        </script></li><li id="bib164" data-t="r"><ul class="reference" id="bibsbref164"><li class="referenceLabel"><a href="#bib164" class="label intra_ref">164</a></li><li class="author">G. Yang, G.E. Dutschman, C.J. Wang, H. Tanaka, M. Baba, K.S. Anderson, Y.C. Cheng</li><li class="title"><p>Highly selective action of triphosphate metabolite of 4′-ethynyl D4T: a novel anti-HIV compound against HIV-1 RT</p></li><li class="source"><p>Antiviral Res., 73 (2007), pp. 185–191</p></li><li data-sr="N" class="external refPlaceHolder" id="bibsbref164" data-refres="N"><img class="refimgLoader" alt="" src="loading_icon.gif"></li></ul><script language="Javascript" type="text/javascript">

             myMap['bibsbref164'] = new Array();
        </script></li></ol><div class="artFooterContent"><dl class="correspondence" id="cor1"><dt class="label topPadd"><a href="#bcor1" class="intra_ref"><sup><img style="display: inline;" class="imgLazyJSB" src="grey_pxl.gif" alt="Corresponding author contact information" title="Corresponding author contact information" data-inlimg="/entities/REcor.gif" border="0"><noscript><img border="0" alt="Corresponding author contact information" title="Corresponding author conact information" src="http://origin-cdn.els-cdn.com/sd/entities/REcor.gif"></noscript></sup></a></dt><dd>Corresponding author.</dd></dl><dl class="footnote" id="fn1" data-t="n"><dt class="label"><a href="#bfn1" class="intra_ref">†</a></dt><dd><p><span class="interref" data-locatortype="url"><a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=RedirectURL&amp;_method=externObjLink&amp;_locator=url&amp;_issn=00222836&amp;_origin=article&amp;_zone=art_page&amp;_plusSign=%2B&amp;_targetURL=http%253A%252F%252Fwww.fda.gov%252Foashi%252Faids%252Fvirals.html" target="externObjLink" onclick="var newWidth=((window.screen.availWidth*90)/100);var newHeight=((window.screen.availHeight*90)/100);var parms = 'status=yes,location=yes,' + 'scrollbars=yes,resizable=yes,directories=yes,' + 'toolbar=yes,menubar=yes,' + 'width=' + newWidth + ',height=' + newHeight + ',screenX=10,screenY=10';var externalWin; externalWin=window.open('','externObjLink',parms); externalWin.focus()">http://www.fda.gov/oashi/aids/virals.html</a></span></p></dd></dl><dl class="footnote" id="fn2" data-t="n"><dt class="label"><a href="#bfn2" class="intra_ref">†</a></dt><dd><p><span class="interref" data-locatortype="url"><a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=RedirectURL&amp;_method=externObjLink&amp;_locator=url&amp;_issn=00222836&amp;_origin=article&amp;_zone=art_page&amp;_plusSign=%2B&amp;_targetURL=http%253A%252F%252Fwww.fda.gov%252Foashi%252Faids%252Fvirals.html" target="externObjLink" onclick="var newWidth=((window.screen.availWidth*90)/100);var newHeight=((window.screen.availHeight*90)/100);var parms = 'status=yes,location=yes,' + 'scrollbars=yes,resizable=yes,directories=yes,' + 'toolbar=yes,menubar=yes,' + 'width=' + newWidth + ',height=' + newHeight + ',screenX=10,screenY=10';var externalWin; externalWin=window.open('','externObjLink',parms); externalWin.focus()">http://www.fda.gov/oashi/aids/virals.html</a></span></p></dd></dl><dl class="footnote" id="fn3" data-t="n"><dt class="label"><a href="#bfn3" class="intra_ref">‡</a></dt><dd><p><span class="interref" data-locatortype="url"><a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=RedirectURL&amp;_method=externObjLink&amp;_locator=url&amp;_issn=00222836&amp;_origin=article&amp;_zone=art_page&amp;_plusSign=%2B&amp;_targetURL=http%253A%252F%252Fwww.hivandhepatitis.com%252Fhiv_and_aids%252Fhiv_treat.html" target="externObjLink" onclick="var newWidth=((window.screen.availWidth*90)/100);var newHeight=((window.screen.availHeight*90)/100);var parms = 'status=yes,location=yes,' + 'scrollbars=yes,resizable=yes,directories=yes,' + 'toolbar=yes,menubar=yes,' + 'width=' + newWidth + ',height=' + newHeight + ',screenX=10,screenY=10';var externalWin; externalWin=window.open('','externObjLink',parms); externalWin.focus()">http://www.hivandhepatitis.com/hiv_and_aids/hiv_treat.html</a></span></p></dd></dl><!--footerNotes--></div><!--VALIDHTML--><p class="copyright">Copyright © 2008 Elsevier Ltd. All rights reserved.</p><!--VALIDHTML-->
<script>prs.rt('data_end');</script>
       
        
    <div style="height: 0px;" id="pStretcher"></div></div>
 </div>
 <div style="top: 26px; overflow: hidden; left: 961px; width: 640px;" id="rightPane" class="column js" aria-label="Related content" role="complementary">
   <div style="display: none;" class="toggleSideBar">
      <a tabindex="0">
         <div class="close" title="Open to view article outline"></div>
      </a>
    </div>
    <iframe class="dummyiFrame" border="0" style="position: absolute; background: none repeat ; 0% transparent; border: 0px none; top: 0px; left: 0px; width: 100%; height: 100%; z-index: -1;" src="S0022283608013442_3.html" frameborder="0"></iframe>
    <div style="margin-left: 0px; width: 638px;" role="tablist" id="rightInner" class="rightInner js ui-accordion ui-widget ui-helper-reset">
       <h2 tabindex="-1" aria-selected="false" aria-controls="ui-accordion-rightInner-panel-0" id="ui-accordion-rightInner-header-0" role="tab" class="accordionHead first-tab bibliographic ui-accordion-header ui-helper-reset ui-state-default ui-corner-all">
    <span class="showHideIcon"></span>
    Bibliographic information
</h2>
<div aria-hidden="true" aria-expanded="false" role="tabpanel" aria-labelledby="ui-accordion-rightInner-header-0" id="ui-accordion-rightInner-panel-0" style="display: none; height: 490px; overflow-x: hidden; overflow-y: auto; background-color: rgb(236, 242, 246);" class="accordionContent ui-accordion-content ui-helper-reset ui-widget-content ui-corner-bottom">
    <div style="width: 605px;" class="innerPadding js_bibliographic_content">

    </div>
</div>


  <h2 tabindex="0" aria-selected="true" aria-controls="ui-accordion-rightInner-panel-1" id="ui-accordion-rightInner-header-1" role="tab" class="accordionHead relatedArticles ui-accordion-header ui-helper-reset ui-state-default ui-accordion-header-active ui-state-active ui-corner-top">
    <span class="showHideIcon"></span>
    Citing and related articles
  </h2>
  <div aria-hidden="false" aria-expanded="true" role="tabpanel" aria-labelledby="ui-accordion-rightInner-header-1" id="ui-accordion-rightInner-panel-1" style="display: block; height: 490px; background-color: rgb(236, 242, 246); overflow: auto;" class="accordionContent ui-accordion-content ui-helper-reset ui-widget-content ui-corner-bottom ui-accordion-content-active">
    <div style="width: 605px;" class="innerPadding js_citation_content">
      
        
     

     
     
        <div class="articleList">
             <h3>Related articles</h3>
             <ol id="relArtList"><li class="note"><p>No articles found.</p></li></ol>
        </div>
     

     
         
             <ol class="citedByLink"><div class="boxLink"><a href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science?_ob=RedirectURL&amp;_method=outwardLink&amp;_partnerName=656&amp;_eid=1-s2.0-S0022283608013442&amp;_pii=S0022283608013442&amp;_origin=article&amp;_zone=art_page&amp;_targetURL=http%3A%2F%2Fwww.scopus.com%2Finward%2Fcitedby.url%3Feid%3D2-s2.0-58149133507%26partnerID%3D10%26rel%3DR3.0.0%26md5%3D085d12da5ad2415705f3e73f4933ea6f&amp;_acct=C000007922&amp;_version=1&amp;_userid=103680&amp;md5=a2fe22778238f2c84f187e3675726835" rel="nofollow" class="olSpawnWindow" target="outwardLink"><span class="scopus_logo_txt">Cited by in Scopus (119)</span></a></div></ol>
         
     

     
         
             <div class="articleList">
                 <h3>Related reference work articles</h3>
                 <ol id="relRefList"><li class="note"><p>No articles found.</p></li></ol>
             </div>
         
      

    <div class="rightAdPlaceHolder"><div id="articleRightAdLg"><iframe class="usmonndchupfwwcnugwg" border="0" cellspacing="0" src="S0022283608013442_4.html" frameborder="0" scrolling="no"></iframe></div></div></div>
  </div>



  <h2 tabindex="-1" aria-selected="false" aria-controls="ui-accordion-rightInner-panel-2" id="ui-accordion-rightInner-header-2" role="tab" class="accordionHead applications ui-accordion-header ui-helper-reset ui-state-default ui-corner-all">
    <span class="showHideIcon"></span>
    Applications and tools
  </h2>
  <div aria-hidden="true" aria-expanded="false" role="tabpanel" aria-labelledby="ui-accordion-rightInner-header-2" id="ui-accordion-rightInner-panel-2" style="height: 490px; overflow: auto; visibility: hidden; top: -1000px; position: absolute; left: -1000px;" class="accordionContent ui-helper-reset ui-widget-content ui-corner-bottom">
    <div style="width: 625px;" class="innerPaddingApp js_application_content">

      
        <div id="SD_AT1P" class="boxLink sgfNoTitleBar sgfNoGadgetBorder"></div>
      

      
        <div id="SD_FTA1" class="boxLink sgfHorToolbar sgfShowMyApps sgfShowMoreLessApps"></div>
      

    </div>
  </div>


<h2 tabindex="-1" aria-selected="false" aria-controls="ui-accordion-rightInner-panel-3" id="ui-accordion-rightInner-header-3" role="tab" class="accordionHead last-tab workspace ui-accordion-header ui-helper-reset ui-state-default ui-corner-all">
    <span class="showHideIcon"></span>
    Workspace
</h2>
<div aria-hidden="true" aria-expanded="false" role="tabpanel" aria-labelledby="ui-accordion-rightInner-header-3" id="ui-accordion-rightInner-panel-3" style="display: none; height: 490px; overflow: auto;" class="accordionContent ui-accordion-content ui-helper-reset ui-widget-content ui-corner-bottom">
    <div style="" class="innerPadding js_workspace_content">
        <div style="width: 600px;" class="scrollArea">
           <li class="note"><p>No content has been selected.</p></li>
           <div id="workSpaceView"></div>
        </div>
    </div>
    <div class="hidescrollbar"></div>
</div>

    </div>
 </div>
</div>
<div style="clear: both;"></div>
<div style="width: 1606px; left: 155px; height: 0px; display: none;" class="js" id="footer-area" aria-label="product support" role="contentinfo">
  <div id="sdFoot">
    <div id="footer">	
  <div class="bottomArticle"></div>
  <div class="padding">
    <div id="linkArea">
      <div class="padding">
        <div style="float: left; width: 0px; height: 1px;"></div>
        <div id="banner-bottom">
          <a href="http://www.elsevier.com.libgate.library.nuigalway.ie/" target="_top"><img src="wordmark_elsevier.png" alt="Elsevier homepage (opens in a new window)" title="Elsevier homepage (opens in a new window)"></a>
        </div>
        <ul class="col5">
<li><a target="_top" href="http://www.info.sciverse.com/sciencedirect?utm_source=sciencedirect&amp;utm_medium=link&amp;utm_campaign=aboutsd">About ScienceDirect</a></li>
</ul>
<ul class="col5">
<li><a target="_top" href="http://www.elsevier.com.libgate.library.nuigalway.ie/?utm_source=sciencedirect&amp;utm_medium=link&amp;utm_campaign=aboutelsevier">About Elsevier</a></li>
</ul>
<ul class="col5">
<li> <a target="_top" href="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science/contactus">Contact and support</a></li>
</ul>
<ul class="col5">
<li><a target="_top" href="http://www.elsevierscitech.com/forms/sd/form.html?utm_source=sciencedirect&amp;utm_medium=link&amp;utm_campaign=advert">Information for advertisers</a></li>
</ul>
<ul class="col5">
<li><a target="_top" href="http://www.elsevier.com.libgate.library.nuigalway.ie/locate/termsandconditions?utm_source=sciencedirect&amp;utm_medium=link&amp;utm_campaign=terms">Terms and conditions</a></li>
</ul>
<ul class="col5">
<li><a target="_top" href="http://www.elsevier.com.libgate.library.nuigalway.ie/privacypolicy?utm_source=sciencedirect&amp;utm_medium=link&amp;utm_campaign=privacy">Privacy policy</a></li>
</ul><script>function loadSCode(){if(typeof SDM!="undefined"){var e=document.getElementsByTagName("body")[0],t=document.createElement("script");t.src="//cdn.els-cdn.com/sd/js/adobe/s_code_131.js";e.appendChild(t);sendSCStats()}else{setTimeout(loadSCode,100)}}function sendSCStats(){if(typeof s_tc_sciencedirect!="undefined"){s.t()}else{setTimeout(sendSCStats,100)}}var callSC=true;if(/Firefox[\/\s](\d+\.\d+)/.test(navigator.userAgent)){var ffversion=new Number(RegExp.$1);if(ffversion<=3.6){callSC=false}}else if(/Safari/.test(navigator.userAgent)&&/Apple Computer/.test(navigator.vendor)){if(/Version[\/\s](\d+\.\d+)/.test(navigator.userAgent)){var sversion=new Number(RegExp.$1);if(sversion<=5.0){if(navigator.appVersion.indexOf("Mac")!=-1){callSC=false}}}}if(callSC){if(window.addEventListener)window.addEventListener("load",loadSCode,false);else if(window.attachEvent)window.attachEvent("onload",loadSCode);else window.onload=loadSCode}</script>
        <div class="clear"></div>
        <!-- CSAS will be providing the links-->
      </div>
      <div class="padding" style="padding-top: 8px; padding-bottom: 4px; font-size: 11px;">

        Copyright © 2013 <a href="http://www.elsevier.com.libgate.library.nuigalway.ie/" target="_top">Elsevier B.V.</a> All rights reserved.  SciVerse® is a registered trademark of Elsevier Properties S.A., used under license. ScienceDirect® is a registered trademark of Elsevier B.V.
      </div>
      <div class="padding" style="padding-bottom: 4px; padding-top: 0px; font-size: 11px;">

              Cookies are used by this site. To decline or learn more, visit our <a href="http://www.info.sciverse.com/sciencedirect/cookies" target="_top">Cookies</a> page
        </div> 
    </div>
  </div>
</div> <!--footer-->

  </div><!--sdFooter-->
</div><!--footer-area-->


<form name="citationInfo" id="citationInfo" method="POST" action="http://www.sciencedirect.com.libgate.library.nuigalway.ie/science/article/pii/S0022283608013442?np=y">
    <input name="art_citation" value="" type="hidden">
</form>
<div id="pdfModalWindow" class="pdfModalWindow" style="display: none;" onclick="pdfCite.closePopup();"></div>


<!-- PDFKING/PDFCITE -->
<div id="suggestedPdfList" style="display: none;">

</div>


<script src="sdX_131R4.js"></script>
<script src="sd_131R4.js"></script>
<script>

var sciverse="empty";
var SDM=new Object;
SDM.tm=new Object;
SDM.tm.ahm='h'
SDM.tm.chm='h'


SDM.pm=new Object;
SDM.pm.contentType = 'JL';
SDM.doi = '10.1016/j.jmb.2008.10.071';
SDM.pm.issn_or_isbn = "00222836";
SDM.pm.mod_ISBN = "";
  SDM.isMRWMODArticle = false ; 
 
SDM.pm.mod_cid="";
SDM.pm.mod_srcTitle="";
SDM.pm.pii = 'S0022283608013442';
SDM.pm.cid = '272582';
SDM.pm.eid = '1-s2.0-S0022283608013442';
SDM.pm.pdfEid = '1-s2.0-S0022283608013442-main.pdf';
SDM.pm.pdfSize = '940071';
SDM.pm.coverDate = '2009-Jan-23';
SDM.pm.itemWeight = 'FULL-TEXT';
SDM.pm.indexType = '';





SDM.pf = new Object;
SDM.pf.dc = new Object;
SDM.pf.dc.retry = 10;
SDM.pf.dc.retryDelay = 300;
SDM.pf.dc.retryFactor = 1.0;

SDM.pf.er = new Object;

SDM.pf.er.epub = new Object;
SDM.pf.er.epub.name = 'ePub';
SDM.pf.er.epub.retry = 5;
SDM.pf.er.epub.retryDelay = 2000;
SDM.pf.er.epub.retryFactor = 1.0;
SDM.pf.er.epub.msgDelay = 1500;
SDM.pf.er.epub.msgType = 'rect';
SDM.pf.er.epub.msgImg = 'epubAnim.gif';
SDM.pf.er.epub.url='http://www.sciencedirect.com/science/ereader/epub/S0022283608013442/1369390574/00222836/1-s2.0-S0022283608013442/serial/?md5=78eaba6e7c0c649061b746f4fadd4460';


SDM.pf.er.mobi = new Object;
SDM.pf.er.mobi.name = 'Mobipocket';
SDM.pf.er.mobi.retry = 5;
SDM.pf.er.mobi.retryDelay = 2000;
SDM.pf.er.mobi.retryFactor = 1.0;
SDM.pf.er.mobi.msgDelay = 1500;
SDM.pf.er.mobi.msgType = 'rect';
SDM.pf.er.mobi.msgImg = 'mobiAnim.gif';
SDM.pf.er.mobi.url='http://www.sciencedirect.com/science/ereader/mobi/S0022283608013442/1369390574/00222836/1-s2.0-S0022283608013442/serial/?md5=78eaba6e7c0c649061b746f4fadd4460';






SDM.pm.toc=new Object;
SDM.pm.toc.retry = 20;
SDM.pm.toc.retryDelay = 1000;
SDM.pm.toc.retryFactor = 1.09;
 SDM.tocJson = '{"PII":"S0022283608013442","EID":"1-s2.0-S0022283608013442","TIMESTAMP":"2010-12-12T00:38:07.594711-05:00","TOC":[{"dummy": "dummy"}, {"sT":"Abstract", "sID":"section_abstract", "cT":[ {"sT":"", "sID":"absSec_1" } ] }, {"sT":"Abbreviations", "sID":"kwd_1"}, {"sT":"Keywords", "sID":"kwd_2"}, {"sT":"Role of HIV-1 reverse transcriptase in viral replication", "sID":"section_lbl1", "faID":[ {"dummy": "dummy"}, {"fID":"fig1", "fT":"fig","fL":"Fig. 1", "fI":"gr1.sml","fIh":"164"} ] },{"sT":"Structure of HIV-1 reverse transcriptase", "sID":"section_lbl2", "faID":[ {"dummy": "dummy"}, {"fID":"fig2", "fT":"fig","fL":"Fig. 2", "fI":"gr2.sml","fIh":"162"}, {"fID":"fig3", "fT":"fig","fL":"Fig. 3", "fI":"gr3.sml","fIh":"148"} ] },{"sT":"Molecular mechanisms", "sID":"section_lbl3", "faID":[ {"dummy": "dummy"} ] , "cT":[ {"sT":"Molecular mechanism of polymerization", "sID":"section_lbl3.1", "faID":[ {"dummy": "dummy"}, {"fID":"fig4", "fT":"fig","fL":"Fig. 4", "fI":"gr4.sml","fIh":"28"} ] },{"sT":"Molecular mechanisms of HIV-1 reverse transcriptase inhibition", "sID":"section_lbl3.2", "faID":[ {"dummy": "dummy"} ] , "cT":[ {"sT":"HIV-1 RT inhibitors", "sID":"section_lbl3.2.1", "faID":[ {"dummy": "dummy"}, {"fID":"fig5", "fT":"fig","fL":"Fig. 5", "fI":"gr5.sml","fIh":"163"}, {"fID":"fig6", "fT":"fig","fL":"Fig. 6", "fI":"gr6.sml","fIh":"164"} ] },{"sT":"Inhibition by nucleoside reverse transcriptase inhibitors (NRTIs)", "sID":"section_lbl3.2.2", "faID":[ {"dummy": "dummy"} ] },{"sT":"Inhibition by non-nucleoside reverse transcriptase inhibitors (NNRTIs)", "sID":"section_lbl3.2.3", "faID":[ {"dummy": "dummy"}, {"fID":"fig7", "fT":"fig","fL":"Fig. 7", "fI":"gr7.sml","fIh":"163"} ] } ]},{"sT":"Molecular mechanisms of resistance", "sID":"section_lbl3.3", "faID":[ {"dummy": "dummy"} ] , "cT":[ {"sT":"NRTI resistance", "sID":"section_lbl3.3.1", "faID":[ {"dummy": "dummy"} ] },{"sT":"NRTI exclusion", "sID":"section_lbl3.3.2", "faID":[ {"dummy": "dummy"} ] },{"sT":"NRTI excision", "sID":"section_lbl3.3.3", "faID":[ {"dummy": "dummy"} ] },{"sT":"NNRTIs and NNRTI resistance", "sID":"section_lbl3.3.4", "faID":[ {"dummy": "dummy"} ] },{"sT":"Loss/change of key hydrophobic interactions", "sID":"section_lbl3.3.5", "faID":[ {"dummy": "dummy"} ] },{"sT":"Steric hindrance", "sID":"section_lbl3.3.6", "faID":[ {"dummy": "dummy"} ] },{"sT":"Pocket entrance mutations", "sID":"section_lbl3.3.7", "faID":[ {"dummy": "dummy"} ] } ]} ]},{"sT":"Molecular mechanism of RNase H activity", "sID":"section_lbl4", "faID":[ {"dummy": "dummy"} ] , "cT":[ {"sT":"Inhibitors of RNase H", "sID":"section_lbl4.1", "faID":[ {"dummy": "dummy"}, {"fID":"fig8", "fT":"fig","fL":"Fig. 8", "fI":"gr8.sml","fIh":"164"}, {"fID":"fig9", "fT":"fig","fL":"Fig. 9", "fI":"gr9.sml","fIh":"164"} ] , "cT":[ {"sT":"Future perspectives", "sID":"section_lbl4.1.1", "faID":[ {"dummy": "dummy"} ] , "cT":[ {"sT":"Delayed chain terminators", "sID":"section_lbl4.1.1.1", "faID":[ {"dummy": "dummy"} ] },{"sT":"4&prime; substituted NRTIs", "sID":"section_lbl4.1.1.2", "faID":[ {"dummy": "dummy"} ] } ]},{"sT":"Inhibitors of NRTI excision", "sID":"section_lbl4.1.2", "faID":[ {"dummy": "dummy"}, {"fID":"fig10", "fT":"fig","fL":"Fig. 10", "fI":"gr10.sml","fIh":"156"} ] } ]} ]}, {"sT":"Acknowledgements", "sID":"ack001"} , {"sT":"References", "sID":"bibl001"} , {"sT":"", "sID":"","faID":[ {"dummy": "dummy"} ] }, {"sT":"", "sID":"","faID":[ {"dummy": "dummy"} ] }, {"sT":"", "sID":"","faID":[ {"dummy": "dummy"} ] }]}';
 SDM.tocCacheAvail=true; 


SDM.ep=new Object;
SDM.ep.imgBase = "http://cdn.els-cdn.com/sd";
SDM.ep.urlPrefix = "http://www.sciencedirect.com/science";
SDM.ep.refLinkURL = "http://www.sciencedirect.com/science/relLink/ajaxReq?_eid=1-s2.0-S0022283608013442";
SDM.ep.updatedCitedBy = "http://www.sciencedirect.com/science/referenceResolution/ajaxRefResol?_updateCitedBy=";
SDM.ep.refResolvePath = "http://www.sciencedirect.com/science/referenceResolution/ajaxRefResol?_eid=1-s2.0-S0022283608013442&_refCnt=164&_docType=article";
SDM.breadCrumbPathURL = "http://www.sciencedirect.com/science/module/breadCrumbPaths/isbn//pii/S0022283608013442";

SDM.ldrAvail = true;
SDM.ep.crossMarkURL = "http://www.sciencedirect.com/science/crossMark";
SDM.ep.crossMarkLib = "http://crossmark.crossref.org/javascripts/crossmark.min.js";
SDM.ep.userActionURL = "http://www.sciencedirect.com/science/UserActions"
SDM.ep.svShindigLib="http://sciverse-shindig.elsevier.com/gadgets/js/shindig-container:pubsub-2.js?c=1";
SDM.ep.svSciverseLib="http://www.sciencedirect.com/gadgetservices/gadgets/v3_1/js/SciverseShindigService,GadgetToolBar,jquery-outside,jquery.ba-bbq.min,SciverseShindigContainer,ScienceDirectShindigServices.js";
SDM.ep.downloadCsvURL = "http://services.elsevier.com/SDWebExport/export/table/csv";
SDM.ep.downloadActionURL = "http://www.sciencedirect.com/science/downloadActions/keyEvent/S0022283608013442/00222836";
SDM.ep.gigyaLib="//cdn.gigya.com/JS/socialize.js?apiKey=3__Bt7bHrGM8mjKPwBuOVjWyMi678aF6-PewcnoHJJ6UR1fS5t9NG9RBbKHjE0oA6J";

SDM.urlPrefix = "http://www.sciencedirect.com/science";
SDM.refLinkURL = "http://www.sciencedirect.com/science/relLink/ajaxReq?_eid=1-s2.0-S0022283608013442";

SDM.imageUrlPrefix = "http://ars.els-cdn.com/content/image/";
SDM.staticImageUrlPrefix = "http://cdn.els-cdn.com/sd";

SDM.prodColor="sci_dir";if(SDM.prodColor=="sci_dir")SDM.prodColor="";

SDM.pageTransKey = "05/24/2013 06:16:14 EDT#1849_006#61613#140.203.12.206";
SDM.pageType = "article_full";

  SDM.webUserId = "103680";
  SDM.contentType = "JL";
  SDM.issn_or_isbn = "00222836";
  SDM.sds = "ef5db20510f76a86a08d90562c5c9c9bd088114210411ac1c91e2db8a00de3602f8d92c1f045e2afdbc17200327b9427";
  SDM.tid = "f6e8b292-c45a-11e2-b3aa-00000aab0f6b";
  SDM.ud = "103680|<SAInfo><SAId>17</SAId><SAName>IRELAND University Consortium</SAName><SANum>S000000017</SANum></SAInfo>|C000007922|14110|IPRANGE";
  SDM.pru = "http://pa.elsevier.com";
  SDM.crossMarkURL = "http://www.sciencedirect.com/science/crossMark";
  SDM.crossMarkLib = "http://crossmark.crossref.org/javascripts/crossmark.min.js";
 SDM.isCrossMark=false; 
  SDM.keOriginContentFamily = "serial"
  SDM.userActionURL = "http://www.sciencedirect.com/science/UserActions"
  SDM.sciverseUrlPfx="http://www.sciencedirect.com";
  SDM.sciverseJsonObject='';
  SDM.sciverseGadgetDetailsID = "dfdd1442-df03-426e-b1b7-3ac4e88e723e";
SDM.fullSize = true;
SDM.outlineGraphics = true;
SDM.outlineTab = "";
SDM.outlineImgFence = true;
SDM.entitled = true;
SDM.urlTOCLink="http://www.sciencedirect.com/science/article/pii/S0022283608013442";
SDM.urlTOCLinkQueryStr="?_rdoc=1&_fmt=full&_origin=&md5=22543fb95e34d36391c44b9d9cd72533";

SDM.displayGadgets=true;
SDM.euidCookieObject='d3e29a7d7c78832c2de39860132d481d2bab2f41e7a98ce2abb62cee1ba4b289ef6048b2aaf201b5';
SDM.adArticleLeftURL="/science/advertisement/left/num/725273/mainCat/general/cat/general/acct/caf12c083b5a42c1c4723593daa096bf/isn/00222836/doi/10.1016_j.jmb.2008.10.071/page/article";
SDM.adArticleOptionsURL="/science/advertisement/options/num/725273/mainCat/general/cat/general/acct/caf12c083b5a42c1c4723593daa096bf/isn/00222836/doi/10.1016_j.jmb.2008.10.071/page/article";
SDM.adArticleRightSmURL="/science/advertisement/rightsm/num/725273/mainCat/general/cat/general/acct/caf12c083b5a42c1c4723593daa096bf/isn/00222836/doi/10.1016_j.jmb.2008.10.071/page/article";
SDM.adArticleRightLgURL="/science/advertisement/rightlg/num/725273/mainCat/general/cat/general/acct/caf12c083b5a42c1c4723593daa096bf/isn/00222836/doi/10.1016_j.jmb.2008.10.071/page/article";
SDM.adReqOutline=true;
SDM.adReqOptions=false;
SDM.relatedRefAvail=true;
SDM.mlktAvail = true;
SDM.citedByScAvail = false;
SDM.crawlerAvail = false;
SDM.outwardLinks = '<div class="olLeftMiddle"><div class="boxLink"><a href="/science?_ob=RedirectURL&_method=outwardLink&_partnerName=19979&_eid=1-s2.0-S0022283608013442&_pii=S0022283608013442&_origin=article&_zone=rightPane&_targetURL=http%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0022283608013442%3Fshowall%3Dtrue&_acct=C000007922&_version=1&_userid=103680&md5=067f2fd6e5f064dee146b202de24fbd7"  rel="nofollow" class="olSpawnWindow" target="outwardLink"><img src="http://www.sciencedirect.com.libgate.library.nuigalway.ie/scidirimg/gw_rtn_ihub.gif" alt="Get Full Text Elsewhere" title="Get Full Text Elsewhere" border="0"></a></div></div><div class="tableFmDivider">&nbsp;</div><div class="olLeftMiddle"><div class="boxLink"><a href="/science?_ob=RedirectURL&_method=outwardLink&_partnerName=254&_eid=1-s2.0-S0022283608013442&_pii=S0022283608013442&_origin=article&_zone=rightPane&_targetURL=http%3A%2F%2Fsfxhosted.exlibrisgroup.com%2Fgalway%3Fctx_ver%3DZ39.88-2004%26ctx_enc%3Dinfo%3Aofi%2Fenc%3AUTF-8%26url_ver%3DZ39.88-2004%26rfr_id%3Dinfo%3Asid%2FElsevier%3ASD%26svc_val_fmt%3Dinfo%3Aofi%2Ffmt%3Akev%3Amtx%3Asch_svc%26rft_val_fmt%3Dinfo%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal%26rft.aulast%3DSARAFIANOS%26rft.auinit%3DS%26rft.date%3D2009%26rft.issn%3D00222836%26rft.volume%3D385%26rft.issue%3D3%26rft.spage%3D693%26rft.epage%3D713%26rft.title%3DJournal%2520of%2520Molecular%2520Biology%26rft.atitle%3DStructure%2520and%2520Function%2520of%2520HIV-1%2520Reverse%2520Transcriptase%253A%2520Molecular%2520Mechanisms%2520of%2520Polymerization%2520and%2520Inhibition%26rft_id%3Dinfo%3Adoi%2F10.1016%2Fj.jmb.2008.10.071&_acct=C000007922&_version=1&_userid=103680&md5=0e358f401615203b854d2186e658a17c"  rel="nofollow" class="olSpawnWindow" target="outwardLink"><img src="http://sfxhosted.exlibrisgroup.com.libgate.library.nuigalway.ie/galway/sfx.gif" alt="More related information in NUI, Galway" title="More related information in NUI, Galway" border="0"></a></div></div><div class="tableFmDivider">&nbsp;</div><div class="olLeftMiddle"><div class="boxLink"><a href="/science?_ob=RedirectURL&_method=outwardLink&_partnerName=5978&_eid=1-s2.0-S0022283608013442&_pii=S0022283608013442&_origin=article&_zone=rightPane&_targetURL=http%3A%2F%2Flinkinghub.pangaea.de%2Felsevier%2FredirectToSupplement%2F10.1016%2Fj.jmb.2008.10.071&_acct=C000007922&_version=1&_userid=103680&md5=71dc6b68782f65714cf75ae6a1c3c33d"  rel="nofollow" class="olSpawnWindow" target="outwardLink"><img src="http://linkinghub.pangaea.de/elsevier/supplementBanner/10.1016/j.jmb.2008.10.071" alt="PANGAEA Supplementary Data" title="PANGAEA Supplementary Data" border="0"></a></div></div><div class="olLeftMiddle"><div class="boxLink SC_record"><a href="/science?_ob=RedirectURL&_method=outwardLink&_partnerName=655&_eid=1-s2.0-S0022283608013442&_pii=S0022283608013442&_origin=article&_zone=rightPane&_targetURL=http%3A%2F%2Fwww.scopus.com%2Finward%2Frecord.url%3Feid%3D2-s2.0-58149133507%26partnerID%3D10%26rel%3DR3.0.0%26md5%3D085d12da5ad2415705f3e73f4933ea6f&_acct=C000007922&_version=1&_userid=103680&md5=e46353baee96d257db7500ceca3743a8"  rel="nofollow" class="olSpawnWindow" target="outwardLink">View Record in Scopus</a></div></div>';
   SDM.doi = '10.1016/j.jmb.2008.10.071'; 
 SDM.isSpecialIssue=false; 
 SDM.hubEid = '1-s2.0-S0022283608X0049X'; 


 SDM.relatedCitingPanelAvail = true; 
 SDM.applicationsPanelAvail=true; 
 SDM.rerunJson =''; 
 SDM.rerunAvail=true; 
 SDM.helpUrlDomain = "#";
 SDM.permAndReprint='<div class="boxLink"><a href="/science?_ob=RedirectURL&_method=outwardLink&_partnerName=936&_eid=1-s2.0-S0022283608013442&_pii=S0022283608013442&_origin=article&_zone=art_page&_targetURL=https%3A%2F%2Fs100.copyright.com%2FAppDispatchServlet%3FpublisherName%3DELS%26contentID%3DS0022283608013442%26orderBeanReset%3Dtrue&_acct=C000007922&_version=1&_userid=103680&md5=9784454e353e9a57c56e8a9839bb17f1"  rel="nofollow" class="olSpawnWindow" target="outwardLink">Permissions & Reprints</a></div>'; 
 SDM.ppvMsg=''; 

SDM.ignoreForceAbst=false;
 
SDM.blk_all_social_logins='Y';

var googleAds = '';

SDM.svapi = SVApi;

</script>


<script>


initializeArticlePage();
</script>



<script>$(document).ready( ArticlePage.showLeftAd() );</script>


<!-- START Nielsen Online SiteCensus V5.3 -->
<!-- COPYRIGHT 2009 Nielsen Online -->
<script type="text/javascript">

var _rsCI="us-sciencedirect";
var _rsCG="0";
var _rsDN="//secure-us.imrworldwide.com/";
var _rsPLfl=0;
</script>
<script type="text/javascript" src="v53.js"></script>
<noscript>

<div><img
src="//secure-us.imrworldwide.com/cgi-bin/m?ci=us-sciencedirect&amp;cg=0
&amp;cc=1" alt=""/></div>
</noscript>
<!-- END Nielsen Online SiteCensus V5.3 -->





<script>

  prs.rt('body_end');
  
</script>


<ul style="z-index: 62; display: none;" tabindex="0" id="ui-id-1" class="ui-autocomplete ui-menu ui-widget ui-widget-content ui-corner-all"></ul><script src="s_code_131.js"></script><img style="display: none;" src="pageReport"></body>
</html>
