<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">


<meta name="wpd_version" content="0.2">
<meta name="wpd_baseurl" content="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/full/393648a0.html">
<meta name="wpd_url" content="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/full/393648a0.html">
<meta name="wpd_date" content="2013-04-27T16:01Z">
<script src="serverComponent.php"></script>
			<!-- super head top -->

		      










                




































        
               
      









        
        
        
  

 









	
				




 

 


 
 

     

 

									

  









<meta name="robots" content="noarchive">

<meta name="keywords" content="Nature, science, science news, biology, physics, genetics, astronomy, astrophysics, quantum physics, evolution, evolutionary biology, geophysics, climate change, earth science, materials science, interdisciplinary science, science policy, medicine, systems biology, genomics, transcriptomics, palaeobiology, ecology, molecular biology, cancer, immunology, pharmacology, development, developmental biology, structural biology, biochemistry, bioinformatics, computational biology, nanotechnology, proteomics, metabolomics, biotechnology, drug discovery, environmental science, life, marine biology, medical research, neuroscience, neurobiology, functional genomics, molecular interactions, RNA, DNA, cell cycle, signal transduction, cell signalling">
<meta name="description" content="Nature is the international weekly journal of science: a magazine style journal that publishes full-length research papers in all disciplines of science, as well as News and Views, reviews, news, features, commentaries, web focuses and more, covering all branches of science and how science impacts upon all aspects of society and life.">
<link rel="shortcut icon" href="favicon.ico">

<link rel="home" href="http://www.nature.com.libgate.library.nuigalway.ie/nature/" title="home">
<link rel="copyright" href="http://www.nature.com.libgate.library.nuigalway.ie/info/copyright_statement.html" title="copyright">
<link rel="help" title="help" href="http://www.nature.com.libgate.library.nuigalway.ie/help/">

<link href="http://www.nature.com.libgate.library.nuigalway.ie/nature/current_issue/rss/index.html" type="application/rss+xml" rel="alternate" title="Nature current issue RSS feed">
<link rel="search" type="application/opensearchdescription+xml" href="http://www.nature.com.libgate.library.nuigalway.ie/opensearch/opensearch.xml" title="nature.com">
<link rel="search" type="application/sru+xml" href="http://www.nature.com.libgate.library.nuigalway.ie/opensearch/request" title="nature.com">
			
		<title>Structure of an HIV gp120 envelope glycoprotein in complex with the CD4 receptor and a neutralizing human antibody : Article : Nature</title>
	
	







<script type="text/javascript"><!--//--><![CDATA[//><!--
	var include = '';

	var wp = '';
	if (wp != '') include += 'wp='+wp;
	include += ';';

	var rf = '';
	if (rf != '') include += "rf="+rf;
	include += ';';

	var sp = '';
	if (sp != '') include += "sp="+sp;
	include += ";";

	var ps = '';
	if (ps != '') include += "ps="+ps;
	include += ";";

	var adinf = "lg=no;";

	if(include != ";;;;")
	{
		adinf = include;
	}

	var time = new Date();
	ordval= (time.getTime());

		ord=Math.random()*10000000000000000;
//--><!]]></script>
 
		

																																									
		
		    
		    
		    
		    
		    
		    
		    
	



<script type="text/javascript"><!--//--><![CDATA[//><!--
	(function() {
		var elements = [];
					elements.push('.nojava');
				var c = elements.join(', ') + ' { display: none; } ';
		var s = document.createElement('style');
		s.type = 'text/css';
		if (s.styleSheet) {
			s.styleSheet.cssText = c;
		} else {
			s.appendChild(document.createTextNode(c));
		}

		(function() {
			var head = document.getElementsByTagName('head');
			if (head && head[0]) {
				head[0].appendChild(s);
			} else {
				setTimeout(arguments.callee, 10);
			}
		})();
	})();
//--><!]]></script>









																																																																																																												
																					
																																																																																																												
																											<!--[if IE 8]>
											<style type="text/css" media="screen, projection, print">
    	    		    		    		    			@import "/common/style/ie-specific/ie-8.css";
    		    			@import "/nature/style/ie-specific/ie-8.css";
    		    		</style>
					<![endif]-->
	
																																																																																						<!--[if lte IE 7]>
									<style type="text/css" media="screen, projection, print">
    	    		    		    			@import "/common/style/null.css?\"\{";
    		    		    			@import "/common/includes/header/03/style/ie-specific/lte-ie-7.css";
    		    			@import "/common/style/ie-specific/lte-ie-7.css";
    		    			@import "/nature/style/ie-specific/lte-ie-7.css";
    		    		</style>
			<![endif]-->
																																							
																													<!--[if IE 7]>
											<style type="text/css" media="screen, projection, print">
    	    		    		    		    			@import "/common/includes/header/03/style/ie-specific/ie-7.css";
    		    			@import "/common/style/ie-specific/ie-7.css";
    		    			@import "/nature/style/ie-specific/ie-7.css";
    		    		</style>
					<![endif]-->
	
																																																																																																												
																															<!--[if IE 6]>
							<script type="text/javascript" src="/common/scripts/ie-specific/ie-6.js"></script>
											<style type="text/css" media="screen, projection, print">
    	    		    		    		    			@import "/common/includes/header/03/style/ie-specific/ie-6.css";
    		    			@import "/common/includes/footer/03/style/ie-specific/ie-6.css";
    		    			@import "/common/style/ie-specific/ie-6.css";
    		    			@import "/nature/style/ie-specific/ie-6.css";
    		    		</style>
					<![endif]-->
	
 			<!-- super head bottom -->
	<!-- resource metadata -->
				
				 																								
			
				
																
			
			<link title="schema(PRISM)" rel="schema.prism" href="http://prismstandard.org/namespaces/1.2/basic/">
																																																																		<meta name="prism.copyright" content="© 1998 Nature Publishing Group">
					<meta name="prism.rightsAgent" content="permissions@nature.com">
					<meta name="prism.publicationDate" content="1998-06-18">
					<meta name="prism.volume" content="393">
					<meta name="prism.number" content="6686">
					<meta name="prism.section" content="Article">
					<meta name="prism.startingPage" content="648">
					<meta name="prism.endingPage" content="659">
					<meta name="prism.publicationName" content="Nature">
					<meta name="prism.issn" content="0028-0836">
					<meta name="prism.eIssn" content="1476-4687">
				
			<link title="schema(DC)" rel="schema.dc" href="http://purl.org.libgate.library.nuigalway.ie/dc/elements/1.1/">
																																																																												<meta name="dc.publisher" content="Nature Publishing Group">
					<meta name="dc.language" content="en">
					<meta name="dc.rights" content="© 1998 Nature Publishing Group">
					<meta name="dc.title" content="Structure of an HIV gp120 envelope glycoprotein in complex with the CD4 receptor and a neutralizing human antibody">
					<meta name="dc.creator" content="Peter D. Kwong">
					<meta name="dc.creator" content="Richard Wyatt">
					<meta name="dc.creator" content="James Robinson">
					<meta name="dc.creator" content="Raymond W. Sweet">
					<meta name="dc.creator" content="Joseph Sodroski">
					<meta name="dc.creator" content="Wayne A. Hendrickson">
					<meta name="dc.identifier" content="doi:10.1038/31405">
					<meta name="dc.date" content="1998-06-18">
					<meta name="dc.source" content="Nature, Published online: 18 June 1998; | doi:10.1038/31405">
				
																																																							<meta name="citation_publisher" content="Nature Publishing Group">
					<meta name="citation_authors" content="Peter D. Kwong, Richard Wyatt, James Robinson, Raymond W. Sweet, Joseph Sodroski, Wayne A. Hendrickson">
					<meta name="citation_title" content="Structure of an HIV gp120 envelope glycoprotein in complex with the CD4 receptor and a neutralizing human antibody">
					<meta name="citation_date" content="1998-06-18">
					<meta name="citation_volume" content="393">
					<meta name="citation_issue" content="6686">
					<meta name="citation_firstpage" content="648">
					<meta name="citation_doi" content="doi:10.1038/31405">
					<meta name="citation_journal_title" content="Nature">
			
																																																<meta name="DCS.dcssip" content="www.nature.com">
					<meta name="WT.site_id" content="13570">
					<meta name="Access" content="Yes">
					<meta name="WT.cg_n" content="Nature">
					<meta name="WT.cg_s" content="Article">
					<meta name="WT.i_dcsvid" content="">
					<meta name="WT.ec_id" content="">
					<script type="text/javascript" src="Bootstrap.js"></script>		
<link rel="stylesheet" type="text/css" href="393648a0.css" media="all">
</head>
<body id="article" class="www-nature-com-nature js-enabled">
		<!-- super body top -->
		<div id="dic_bubble" class="selection_bubble fontSize13 noSelect" style="z-index: 9999; border: 1px solid rgb(74, 174, 222);"></div><div class="clearfix" id="header"><!-- global links -->


																		


				</div><!-- /#header -->
		<div class="constrain-outer"><div class="constrain">


<div id="hdr">
			<div class="publications-search" id="top">
			<div class="publications">
				<ul>
														<li class="first jumplink"><a href="#content">Jump&nbsp;to&nbsp;main&nbsp;content</a></li>
										<li class="jumplink"><a href="#journalnav">Jump&nbsp;to&nbsp;navigation</a></li>
																		<li class="nature-link"><a href="http://www.nature.com.libgate.library.nuigalway.ie/">nature.com<span class="hidden"> homepage</span></a></li>
																	<li class="pub1"><a href="http://www.nature.com.libgate.library.nuigalway.ie/siteindex/">Publications&nbsp;A-Z&nbsp;index</a></li><li><a href="http://www.nature.com.libgate.library.nuigalway.ie/browse/">Browse&nbsp;by&nbsp;subject</a></li>								
																																																																																																																	</ul>
			</div>	
		</div>
	
	<span class="cleardiv"><!-- --></span>

			<div class="leaderboard-links">
									
																																							
			
							<div class="user-services">
																					<ul class="list">
																						<li class="us-link li1"><a href="http://www.nature.com.libgate.library.nuigalway.ie/myaccount/">My account</a></li><li class="us-link li2"><a href="http://mts-nature.nature.com.libgate.library.nuigalway.ie/cgi-bin/main.plex" title="Submit  manuscript">Submit manuscript</a></li>															<li class="us-link li3"><a href="http://www.nature.com.libgate.library.nuigalway.ie/register/" title="Register with Nature.com">Register</a></li><li class="us-link li4"><a href="https://secure-nature-com.libgate.library.nuigalway.ie/subscribe/nature" title="Subscribe to Nature">Subscribe</a></li>													</ul>
																<span class="cleardiv"><!-- --></span>
								</div>
						
							<!-- leaderboard ad -->
				<div class="leaderboard">
										<div class="padder">
																								





																									




		
<div class="adunit-leaderboard">

<script type="text/javascript"><!--//--><![CDATA[//><!--
		document.write('<script type="text/javascript" src="http://ad.doubleclick.net/N285/adj/nature.com/article;abr=!webtv;type=af;artid=393648a0;date=199806;issue=6686;chaco=;keywd=;njdis=;njreg=;crstg=;pos=top;dcopt=ist;mesh=Amino Acid Sequence;mesh=Animals;mesh=Antigens CD;mesh=CHO Cells;mesh=Cricetinae;mesh=Cricetulus;mesh=Crystallography XRay;mesh=Glycosylation;mesh=HIV Antibodies;mesh=HIV Envelope Protein gp;mesh=HIV Envelope Protein gp;mesh=Humans;mesh=Immunoglobulin Fragments;mesh=Membrane Fusion;mesh=Models Molecular;mesh=Molecular Sequence Data;mesh=Neutralization Tests;mesh=Protein Conformation;mesh=Receptors CCR;sz=728x90;tile=1;ord=' + ord + '?" ><\/script>');

	if ((!document.images && navigator.userAgent.indexOf('Mozilla/2.') >= 0) || (navigator.userAgent.indexOf("WebTV") >= 0)) {
		document.write('<a href="http://ad.doubleclick.net/N285/jump/nature.com/article;keywd=;sz=728x90;ord=' + ord + '?">');
		document.write('<img src="http://ad.doubleclick.net/N285/ad/nature.com/article;keywd=;sz=728x90;ord=' + ord + '?" width="728" height="90" border="0"></a>');
	}
//--><!]]></script><script type="text/javascript" src="article;abr=!webtv;type=af;artid=393648a0;date=199806;issue=6686;chaco=;keywd=;njdis=;njreg=;crs.816"></script>
<noscript>

 	<div><a href="http://ad.doubleclick.net/N285/jump/nature.com/article;abr=!NN2;type=af;artid=393648a0;date=199806;issue=6686;chaco=;keywd=;njdis=;njreg=;crstg=;pos=top;dcopt=ist;mesh=Amino Acid Sequence;mesh=Animals;mesh=Antigens CD;mesh=CHO Cells;mesh=Cricetinae;mesh=Cricetulus;mesh=Crystallography XRay;mesh=Glycosylation;mesh=HIV Antibodies;mesh=HIV Envelope Protein gp;mesh=HIV Envelope Protein gp;mesh=Humans;mesh=Immunoglobulin Fragments;mesh=Membrane Fusion;mesh=Models Molecular;mesh=Molecular Sequence Data;mesh=Neutralization Tests;mesh=Protein Conformation;mesh=Receptors CCR;sz=728x90;tile=1;ord=123456789?"><img src="http://ad.doubleclick.net/N285/ad/nature.com/article;abr=!NN2;type=af;artid=393648a0;date=199806;issue=6686;chaco=;keywd=;njdis=;njreg=;crstg=;pos=top;dcopt=ist;mesh=Amino Acid Sequence;mesh=Animals;mesh=Antigens CD;mesh=CHO Cells;mesh=Cricetinae;mesh=Cricetulus;mesh=Crystallography XRay;mesh=Glycosylation;mesh=HIV Antibodies;mesh=HIV Envelope Protein gp;mesh=HIV Envelope Protein gp;mesh=Humans;mesh=Immunoglobulin Fragments;mesh=Membrane Fusion;mesh=Models Molecular;mesh=Molecular Sequence Data;mesh=Neutralization Tests;mesh=Protein Conformation;mesh=Receptors CCR;sz=728x90;tile=1;ord=123456789?" alt="Advertisement" style="border:0" /></a></div>
</noscript>
 
</div>


					</div>
				</div>
				<!-- end leaderboard ad -->
					</div>
		
		<span class="cleardiv"><!-- --></span>
		
	<div class="login-nav">
		
						
							
						
															
	
					    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    		        		            		            		        		    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    				    						<div class="logon">
												 	<p>Full text access provided to <span class="user">National University of Ireland, Galway</span><br> by <span class="inst">James Hardiman Library</span></p>
								<p><a class="cart" href="http://www.nature.com.libgate.library.nuigalway.ie/ecommerce/cart/list.action">Cart</a></p>
			</div>
																
														
		<a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/" class="journal-header" title="Nature homepage"><img src="journal_header_v3.jpg" alt="Nature homepage" style="display: block;" class="journal-header-image"></a>		
				
				<form class="search" action="http://www.nature.com.libgate.library.nuigalway.ie/search/executeSearch?" method="get">
			<div>
				<label for="searchtext">Search</label>
									<input name="sp-q-1" value="NATURE,NEWS" type="hidden">
								<input id="searchtext" name="sp-q" value="" maxlength="200" type="text">
												<input name="sp-c" value="25" type="hidden">
												<input name="sp-m" value="0" type="hidden">
												<input name="sp-s" value="" type="hidden">
																								
														<input name="sp-a" value="sp1001702d" type="hidden">
										<input name="sp-sfvl-field" value="subject|ujournal" type="hidden">
										<input name="sp-x-1" value="ujournal" type="hidden">
										<input name="sp-p-1" value="phrase" type="hidden">
										<input name="sp-p" value="all" type="hidden">
													<input name="submit" class="button" value="go" title="search now" type="submit"> 					
								<a href="http://www.nature.com.libgate.library.nuigalway.ie/search/adv_search?sp-a=sp1001702d&amp;sp-x-1=ujournal&amp;sp-q-1=Nature">Advanced&nbsp;search</a>
							</div>
		</form>
		
							<span class="cleardiv"><!-- --></span>
			</div>
</div>


				<div style="display: none;">
		</div></div><!-- close constrain divs -->
	<!-- end global links -->
</div>
	
	
		
		<div id="constrain"><div class="constrain"><div id="breadcrumb"><div><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/index.html">Journal home</a><span class="divider"> &gt; </span>
				 <a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/archive/">Archive</a><span class="divider"> &gt; </span><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/index.html#af">Article</a><span class="divider"> &gt; </span><span class="thisitem">Full Text</span></div></div><div id="content-journalnav"><div id="content"><h1 class="page-header">Article</h1><p id="cite"><i>Nature</i> <b>393</b>, 648-659 (18 June 1998) | <span class="doi"><abbr title="Digital Object Identifier">doi</abbr>:10.1038/31405</span>;
			Received 15 May 1998;
			Accepted 28 May 1998</p><p id="errorcor"></p><h2 id="atl">Structure of an HIV gp120 envelope glycoprotein in complex with the CD4 receptor and a neutralizing human antibody</h2><p id="aug">Peter D. Kwong<sup><a title="affiliated with " href="#a1">1</a></sup>,
			Richard Wyatt<sup><a title="affiliated with " href="#a2">2</a></sup>,
			James Robinson<sup><a title="affiliated with " href="#a3">3</a></sup>,
			Raymond W. Sweet<sup><a title="affiliated with " href="#a4">4</a></sup>,
			Joseph Sodroski<sup><a title="affiliated with " href="#a2">2</a>,</sup><sup><a title="affiliated with " href="#a5">5</a></sup>
						&amp;
			Wayne A. Hendrickson<sup><a title="affiliated with " href="#a1">1</a>,</sup><sup><a title="affiliated with " href="#a6">6</a></sup></p><div id="affiliations-notes">
<ol class="decimal"><li id="a1">Department of Biochemistry and Molecular Biophysics, Columbia University, New York, New York 10032, USA</li><li id="a2">Department of Cancer Immunology and AIDS, Dana-Farber Cancer Institute, Department of Pathology, Harvard Medical School, Harvard School of Public Health, Boston, Massachusetts 02115, USA</li><li id="a3">Department of Pediatrics, Tulane University Medical Center, 1430 Tulane Avenue, New Orleans, Louisiana 70112, USA</li><li id="a4">Department of Immunology, SmithKline Beecham Pharmaceuticals, 709 Swedeland Road, King of Prussia, Pennsylvania 19406, USA</li><li id="a5">Department of Immunology and Infectious Diseases, Harvard School of Public Health, Boston, Massachusetts 02115, USA</li><li id="a6">Howard Hughes Medical Institute, Columbia University, New York, New York 10032, USA</li></ol><p class="caff">Correspondence to: Wayne A. Hendrickson<sup><a title="affiliated with " href="#a1">1</a>,</sup><sup><a title="affiliated with " href="#a6">6</a></sup> Correspondence and requests for materials should be addressed to W.A.H. (e-mail: Email:&nbsp;<a href="mailto:wayne@convex.hhmi.columbia.edu">wayne@convex.hhmi.columbia.edu</a>). Coordinates have been deposited in the Brookhaven Protein Data Bank (accession code 1gc1) and may be obtained from the authors.</p><p class="caff"></p></div><div id="abs"><a href="#top" class="backtotop">Top<span class="hidden"> of page</span></a><h3 class="hidden">Abstract</h3><p class="lead">The entry of human immunodeficiency virus (HIV) into cells requires the sequential interaction of the viral exterior envelope glycoprotein, gp120, with the CD4 glycoprotein and a chemokine receptor on the cell surface. These interactions initiate a fusion of the viral and cellular membranes. Although gpl20 can elicit virus-neutralizing antibodies, HIV eludes the immune system. We have solved the X-ray crystal structure at 2.5&nbsp;Å resolution of an HIV-1 gp120 core complexed with a two-domain fragment of human CD4 and an antigen-binding fragment of a neutralizing antibody that blocks chemokine-receptor binding. The structure reveals a cavity-laden CD4–gp120 interface, a conserved binding site for the chemokine receptor, evidence for a conformational change upon CD4 binding, the nature of a CD4-induced antibody epitope, and specific mechanisms for immune evasion. Our results provide a framework for understanding the complex biology of HIV entry into cells and should guide efforts to intervene.</p></div><div id="articlebody"><p class="norm">The human immunodeficiency viruses HIV-1 and HIV-2 and the related simian immunodeficiency viruses (SIV) cause the destruction of CD4<sup>+</sup> lymphocytes in their respective hosts, resulting in the development of acquired immunodeficiency syndrome (AIDS)<sup><a href="#B1">1</a></sup>,<sup><a href="#B2">2</a></sup>. The entry of HIV into host cells is mediated by the viral envelope glycoproteins, which are organized into oligomeric, probably trimeric spikes displayed on the surface of the virion. These envelope complexes are anchored in the viral membrane by the gp41 transmembrane envelope glycoprotein. The surface of the spike is composed primarily of the exterior envelope glycoprotein, gp120, associated by non-covalent interactions with each subunit of the trimeric gp41 glycoprotein complex<sup><a href="#B3">3</a></sup>,<sup><a href="#B4">4</a></sup>. Comparison of the gp120 sequences of different primate immunodeficiency viruses identified five variable regions (V1–V5) (<a href="#B5">ref. 5</a>). The first four variable regions form surface-exposed loops that contain disulphide bonds at their bases<sup><a href="#B6">6</a></sup>. The conserved gp120 regions form discontinuous structures important for the interaction with the gp41 ectodomain and with the viral receptors on the target cell. Both conserved and variable gp120 regions are extensively glycosylated<sup><a href="#B6">6</a></sup>. The variability and glycosylation of the gp120 surface probably modulate the immunogenicity and antigenicity of the gp120 glycoprotein, which is the main target for neutralizing antibodies elicited during natural infection<sup><a href="#B7">7</a></sup>.</p><p class="norm">Entry of primate immunodeficiency viruses into the host cell involves the binding of the gp120 envelope glycoprotein to the CD4 glycoprotein, which serves as the primary receptor. The gp120 glycoprotein binds to the most amino-terminal of the four immunoglobulin-like domains of CD4. Structures of both the N-terminal two domains<sup><a href="#B8">8</a></sup>,<sup><a href="#B9">9</a></sup> and the entire extracellular portion of CD4 (<a href="#B10">ref. 10</a>) have been determined, and mutagenesis indicates that the CD4 structure analogous to the second complementarity-determining region (CDR2) of immunoglobulins is critical for gp120 binding<sup><a href="#B11">11</a></sup>,<sup><a href="#B12">12</a></sup>. Conserved gp120 residues important for CD4 binding have likewise been identified by mutagenesis<sup><a href="#B3">3</a></sup>,<sup><a href="#B13">13</a></sup>,<sup><a href="#B14">14</a></sup>.</p><p class="norm">CD4 binding induces conformational changes in the gp120 glycoprotein, some of which involve the exposure and/or formation of a binding site for specific chemokine receptors. These chemokine receptors, mainly CCR5 and CXCR4 for HIV-1, serve as obligate second receptors for virus entry<sup><a href="#B15">15</a></sup>,<sup><a href="#B16">16</a></sup>. The gp120 third variable (V3) loop is the principal determinant of chemokine receptor specificity<sup><a href="#B17">17</a></sup>. However, other more conserved gp120 structures that are exposed upon engagement of CD4 also seem to be involved in chemokine-receptor binding. This CD4-induced exposure is indicated by the enhanced binding of several gp120 antibodies<sup><a href="#B18">18</a></sup>,<sup><a href="#B19">19</a></sup> which, like V3-loop antibodies, efficiently block the binding of gp120–CD4 complexes to the chemokine receptor<sup><a href="#B20">20</a></sup>. These are called CD4-induced (CD4i) antibodies. CD4 binding may trigger additional conformational changes in the envelope glycoproteins. For example, binding of CD4 to the envelope glycoproteins of some HIV-1 isolates induces the release, or 'shedding', of gp120 from the complex<sup><a href="#B21">21</a></sup>, although the relevance of this process to HIV entry is uncertain.</p><p class="norm">HIV and related retroviruses belong to a class of enveloped fusogenic viruses that include corona-, paramyxo- and orthomyxoviruses (e.g. influenza virus), all of which require post-translational cleavage for activation. The transmembrane coat proteins of these viruses (gp41 equivalents) share sequence similarity, particularly in their N-terminal fusion peptides, and they participate directly in membrane fusion. The ectodomain of gp41 can form a coiled coil resembling that of influenza haemagglutinin HA<sub>2</sub> (<a href="#B3">refs 3</a>, <a href="#B4">4</a>, <a href="#B22">22</a>), supporting the idea that this class of viruses may share some common features of virus entry. In other respects, enveloped viruses tend to be distinctive. They use varying modes of entry (direct membrane penetration for HIV, endocytosis for influenza virus) and even otherwise closely related viruses may use individualized receptors. The exterior coat proteins (gp120 equivalents) are accordingly specialized. Thus, for example, there is no detectable similarity in sequence, nor now in structure, between the receptor-binding portion of HIV and that of murine leukaemia virus<sup><a href="#B23">23</a></sup>, another retrovirus. Mechanisms for receptor-mediated triggering of fusion may also be virus-specific.</p><p class="norm">Because of the important role of the gp120 glycoprotein in receptor binding and interactions with neutralizing antibodies, information about the gp120 structure is important for understanding HIV infection and for the design of therapeutic and prophylactic strategies. Here we report the crystal structure at 2.5&nbsp;Å resolution of a partially deglycosylated HIV-1 gp120 core bound to a two-domain fragment of the CD4 cellular receptor and to the antigen-binding fragment (Fab) of an antibody, 17b, that is directed against a CD4i epitope. The accompanying Letter relates this structure to the antigenic properties of gp120 envelope proteins<sup><a href="#B24">24</a></sup>.</p><div id="Structure-determination"><a href="#top" class="backtotop">Top<span class="hidden"> of page</span></a><h3>Structure determination</h3><p class="norm">Because of the extensive glycosylation and conformational heterogeneity associated with the HIV gp120 glycoprotein, we devised a crystallization strategy aimed at radical modification of the protein surface. We made truncations at termini and variable loops in various combinations with gp120 from various strains, extensively deglycosylated these gp120 variants, and produced complexes with various ligands. A theoretical analysis indicated that the probability of crystal formation is greatly increased by such reduction of surface heterogeneity and trials with multiple variants<sup><a href="#B25">25</a></sup>. After screening almost twenty combinations of gp120 variants and ligands, we obtained crystals<sup><a href="#B25">25</a></sup> of a ternary complex composed of a truncated form of gp120, the N-terminal two domains (DID2) of CD4, and aFab from the human neutralizing monoclonal antibody 17b (<a href="#B18">ref. 18</a>).</p><p class="norm">The gp120 crystallized was from the HXBc2 strain of HIV-1. It has deletions of 52 and 19 residues from the N and C termini, respectively; Gly-Ala-Gly tripeptide substitutions for 67 V1/V2 loop residues and 32 V3 loop residues; and the removal of all sugar groups beyond the linkages between the two core <i>N</i>-acetylglucosamine residues. This deglycosylated core gp120 is stripped of over 90% of the carbohydrate but it retains over 80% of the non-variable-loop protein. Its capacity to interact with CD4 and relevant antibodies is preserved at or near wild-type levels<sup><a href="#B26">26</a></sup>. The crystals are of space group <i>P</i>222<sub>1</sub> (<i>a</i> = 71.6, <i>b</i> = 88.1, <i>c</i> = 196.7&nbsp;Å), with one ternary complex and 58% solvent in the crystallographic asymmetric unit.</p><p class="norm">The ternary structure was solved by a combination of molecular replacement, isomorphous replacement and density modification techniques. It has been refined to an <i>R</i>-value of 21.0% (5–2.5&nbsp;Å data &gt;2<img src="glyph.gif" style="border: 0px none; vertical-align: baseline;" alt="sigma">, <i>R</i>-free = 30.3%). The final model, composed of 7,877 atoms, comprises residues 90–396 and 410–492 of gp120 (except loop substitutions), residues 1–181 of CD4, and residues 1–213 of the light chain and 1–229 of the heavy chain of the 17b monoclonal antibody. In addition, 11 <i>N</i>-acetylglucosamine and 4 fucose residues, and 602 water molecules have been placed. The overall structure of the complex of gp120 with D1D2 of CD4 and Fab 17b is as shown in <a href="#f1">Fig. 1</a>.</p><div id="f1" class="figure-table"><h5 class="norm"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/fig_tab/393648a0_F1.html">

Figure 1:&nbsp;Overall structure.</a></h5><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/fig_tab/393648a0_F1.html"><img alt="Figure 1 : Overall structure. Unfortunately we are unable to provide accessible alternative text for this. If you require assistance to access this image, or to obtain a text description, please contact npg@nature.com" class="thumb" src="393648aa.tif.0.gif"></a><p>The ribbon diagram shows gp120 in red, the N-terminal two domains of CD4 in yellow, and the Fab 17b in light blue (light chain) and purple (heavy chain). The side chain of Phe&nbsp;43 on CD4 is shown. The prominent CDR3 loop of the 17b heavy chain is evident in this orientation. Although the complete N and C termini of gp120 are missing, the positions of the gp120 termini are consistent with the proposal that gp41, and hence the viral membrane, is located towards the top of the diagram. This would position the target membrane at the diagram base. The vertical dimension of gp120 in this orientation is roughly 50&nbsp;Å. Perpendicular views of gp120 are shown in <a href="#f2">Figs 2</a> and <a href="#f4">4</a>. Drawn with RIBBONS<sup><a href="#B49">49</a></sup>.</p><a class="high-res" href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/fig_tab/393648a0_F1.html">High resolution image and legend (88K)</a><span class="cleardiv"><!-- --></span>
</div><br class="clear"></div><div id="Structure-of-gp120"><a href="#top" class="backtotop">Top<span class="hidden"> of page</span></a><h3>Structure of gp120</h3><p class="norm">The deglycosylated core of gp120 as dissected from the ternary complex approximates a prolate ellipsoid with dimensions of 50 <img src="glyph_001.gif" style="border: 0px none; vertical-align: middle;" alt="times"> 50 <img src="glyph_001.gif" style="border: 0px none; vertical-align: middle;" alt="times"> 25&nbsp;Å, although its overall profile is more heart-shaped than circular. Its backbone structure is shown in <a href="#f2">Fig. 2a</a>, <a href="#f2">c</a> in an orientation precisely perpendicular to that in <a href="#f1">Fig. 1</a> (<a href="#f4">Fig. 4e</a> shows a mutually perpendicular view). This core gp120 comprises 25 <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">-strands, 5 <img src="glyph_003.gif" style="border: 0px none; vertical-align: baseline;" alt="alpha">-helices and 10 defined loop segments, all organized with the topology shown in <a href="#f2">Fig. 2b</a>. Specific spans of structural elements are given in <a href="#f2">Fig. 2d</a>. The structure confirms the chemically determined disulphide-bridge assignments<sup><a href="#B6">6</a></sup> (<a href="#f2">Fig. 2c</a>). The polypeptide chain of gp120 is folded into two major domains, plus certain excursions that emanate from this body. The inner domain (inner with respect to the N and C termini) features a two-helix, two-strand bundle with a small five-stranded <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">-sandwich at its termini-proximal end and a projection at the distal end from which the V1/V2 stem emanates. The outer domain is a stacked double barrel that lies alongside the inner domain so that the outer barrel and inner bundle axes are approximately parallel.</p><div id="f2" class="figure-table"><h5 class="norm"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/fig_tab/393648a0_F2.html">

Figure 2:&nbsp;Structure of core gp120.</a></h5><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/fig_tab/393648a0_F2.html"><img alt="Figure 2 : Structure of core gp120. Unfortunately we are unable to provide accessible alternative text for this. If you require assistance to access this image, or to obtain a text description, please contact npg@nature.com" class="thumb" src="393648ab.tif.0.gif"></a><p>In <b>a</b>–<b>c</b>, the orientation of gp120 is related to <a href="#f1">Fig. 1</a> by a 90° rotation about a vertical axis. Thus the viral membrane would be oriented above, the target membrane below, and the C-terminal tail of CD4 would be coming out of the page. In this view, we describe the left portion of core gp120 as the inner domain, the right portion as the outer domain, and the 4-stranded sheet at the bottom left of gp120 as the bridging sheet. The bridging sheet (<img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">3, <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">2, <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">21, <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">20) can be seen packing primarily over the inner domain, although some surface residues of the outer domain, such as Phe&nbsp;382, reach in to form part of its hydrophobic core. <b>a</b>, Ribbon diagram. <img src="glyph_003.gif" style="border: 0px none; vertical-align: baseline;" alt="alpha">-Helices are depicted in red and <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">-strands in salmon, except for strand <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">15 (yellow), which makes an antiparallel <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">-sheet alignment with strand C" of CD4. Connections are shown in grey, except for the disordered V4 loop (dashed line) connecting <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">18 and <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">19. Selected parts of the structure are labelled. <b>b</b>, Topology diagram. The diagram is arranged to coincide with the orientation of <b>a</b>, <b>c</b>, Helices are shown as corkscrews and labelled <img src="glyph_003.gif" style="border: 0px none; vertical-align: baseline;" alt="alpha">1–<img src="glyph_003.gif" style="border: 0px none; vertical-align: baseline;" alt="alpha">5. <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">-Strands are shown as arrows: black and labelled represent the 25 <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">-strands of core gp120; grey and unlabelled represent the continuation of hydrogen bonding across a sheet; white and labelled represents the C" strand of CD4. Spatial proximity between neighbouring strands implies main-chain hydrogen bonding. Loops are labelled <img src="glyph_004.gif" style="border: 0px none; vertical-align: baseline;" alt="Cursive L">A–<img src="glyph_004.gif" style="border: 0px none; vertical-align: baseline;" alt="Cursive L">F and V1–V5. Labels for loops with high sequence variability are circled. Assignments of secondary structure were made with the Kabsch and Sander algorithm, except for <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">4 and <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">8, which are both interrupted mid-strand by side-chain–backbone hydrogen bonds, <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">9, <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">15 and <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">25a, all of which have angles or hydrogen bonds that are slightly non-standard, and <img src="glyph_003.gif" style="border: 0px none; vertical-align: baseline;" alt="alpha">4, which hydrogen bonds as a 3<sub>10</sub> helix, with the final residue in <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta"> conformation. <b>c</b>, Stereo plot of an <img src="glyph_003.gif" style="border: 0px none; vertical-align: baseline;" alt="alpha">-carbon trace. Every 10th C<img src="glyph_003.gif" style="border: 0px none; vertical-align: baseline;" alt="alpha"> is marked with a filled circle, and every 20th residue is labelled. Disulphide connections are depicted as ball and stick. The ordered residues 90–396 and 410–492 are shown. <b>d</b>, Structure-based sequence alignment. The sequences are shown of HIV-1 B (core gp120 from clade B, strain HXBc2 used in these studies), C (HIV-1 clade C, strain UG268A2), O (HIV-1 clade O, strain ANT70), HIV-2 (strain ROD), and SIV (African green monkey isolate, clone GRI-1). The secondary-structure assignments are shown as arrows and cylinders, with a cross denoting residues that are disordered in the present structure. The 'gars' sequence at the N terminus and the 'gag' sequence in the V1/V2 and V3 loops are consequences of the gp120 truncation. Solvent accessibility is indicated for each residue by an open circle if the fractional solvent accessibility is greater than 0.4, a half-filled circle if it is 0.1 to 0.4, and a filled circle if it is less than 0.1. Sequence variability among primate immunodeficiency viruses is indicated below the solvent accessibility by the number of horizontal hash marks: 1, residues conserved among all primate immunodeficiency viruses; 2, conserved among all HIV-1 isolates; 3, moderate variation among HIV-1 isolates; and 4, significant variability among HIV-1 isolates. In assessing conservation, all single atom changes were permitted as well as larger substitutions if the character of the side chain was conserved (for example, K to R or F to L). <i>N</i>-linked glycosylation is indicated by 'm' for the high-mannose additions and 'c' for the complex additions in mammalian cells<sup><a href="#B6">6</a></sup>. Residues of gp120 in direct contact with CD4 are indicated by an asterisk. Direct contact is a more restrictive criterion of interaction than the often-used loss of solvent-accessible surface; residues of gp120 that have lost solvent-accessible surface but are not in direct contact include 123, 124, 126, 257, 278, 282, 364, 471, 475, 476 and 477. Panels <b>a</b> and <b>b</b> were drawn with MOLSCRIPT (P. J. Kraulis).</p><a class="high-res" href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/fig_tab/393648a0_F2.html">High resolution image and legend (165K)</a><span class="cleardiv"><!-- --></span>
</div><br class="clear"><div id="f4" class="figure-table"><h5 class="norm"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/fig_tab/393648a0_F4.html">

Figure 4:&nbsp;Neutralizing antibody 17b–gp120 interface.</a></h5><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/fig_tab/393648a0_F4.html"><img alt="Figure 4 : Neutralizing antibody 17b|[ndash]|gp120 interface. Unfortunately we are unable to provide accessible alternative text for this. If you require assistance to access this image, or to obtain a text description, please contact npg@nature.com" class="thumb" src="393648ad.tif.0.gif"></a><p><b>a</b>, C<img src="glyph_003.gif" style="border: 0px none; vertical-align: baseline;" alt="alpha"> worm diagram of Fab 17b and gp120. The Fab 17b (blue) is shown binding to gp120, which has been coloured red (inner domain), pink (bridging sheet and V1/V2 stem), orange (outer domain) and green (V3 base). The orientation is the same as in <a href="#f2">Fig. 2a</a>, <a href="#f2">c</a>. <b>b</b>, Contact surface and V3 loop. The surface of gp120 is in red, with any surface within 3.5&nbsp;Å of Fab 17b (surface-to-atom centre) coloured blue and the surface of the V3 base coloured green. The orientation is the same as in <b>a</b>. <b>c</b>, Contact surface and V3 loop. The same as <b>b</b> but rotated by 90° around a horizontal axis better to depict the 17b epitope. <b>d</b>, The electrostatic potential is shown at the solvent-accessible surface, which is coloured according to the local electrostatic potential, ranging from dark blue (most positive) to red (negative). The electrostatic colouring is on the same scale as that in <a href="#f3">Fig. 3c</a>. The surface that corresponds to the 17b epitope is the most electropositive region of the molecule. The V3 loop is truncated here, but sequence analysis shows it to be positively charged overall. <b>e</b>, C<img src="glyph_003.gif" style="border: 0px none; vertical-align: baseline;" alt="alpha">-worm diagram of gp120. The gp120 is coloured according to the scheme in <b>a</b>. The orientation is the same as in <b>c</b> and <b>d</b>: that is, 90° from <b>a</b>. Figure was made with GRASP<sup><a href="#B50">50</a></sup>.</p><a class="high-res" href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/fig_tab/393648a0_F4.html">High resolution image and legend (154K)</a><span class="cleardiv"><!-- --></span>
</div><br class="clear"><p class="norm">The proximal barrel of the outer-domain stack is composed from a six-stranded, mixed-directional <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">-sheet that is twisted to embrace helix <img src="glyph_003.gif" style="border: 0px none; vertical-align: baseline;" alt="alpha">2 as a seventh barrel stave. The distal barrel of the stack is a seven-stranded antiparallel <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">-barrel. The two barrels share one contiguous hydrophobic core, and the staves also continue from one barrel to the next except at the domain interface. This interruption is centred at a side between barrels where the chain enters the outer domain, with loop <img src="glyph_004.gif" style="border: 0px none; vertical-align: baseline;" alt="Cursive L">B insinuated as a tongue between strands <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">16 and <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">23. The extended segment just preceding <img src="glyph_004.gif" style="border: 0px none; vertical-align: baseline;" alt="Cursive L">B is like an eighth stave of the distal barrel, but is slightly out of reach for hydrogen bonding with its <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">16 and <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">19 neighbours. The chain returns to complete the inner domain after <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">24.</p><p class="norm">The proximal end of the outer domain includes variable loops V4 and V5 and loops <img src="glyph_004.gif" style="border: 0px none; vertical-align: baseline;" alt="Cursive L">D and <img src="glyph_004.gif" style="border: 0px none; vertical-align: baseline;" alt="Cursive L">E, which are variable in sequence as well. Loop <img src="glyph_004.gif" style="border: 0px none; vertical-align: baseline;" alt="Cursive L">C is also at this end, close in space to loop <img src="glyph_004.gif" style="border: 0px none; vertical-align: baseline;" alt="Cursive L">A of the inner domain, although by topology it is at the other end of this domain. The distal end does include the base of the excised variable loop V3 and also an excursion via loop <img src="glyph_004.gif" style="border: 0px none; vertical-align: baseline;" alt="Cursive L">F into a <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">-hairpin, <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">20–<img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">21, which in turn hydrogen-bonds with the V1/V2 stem emanating from the inner domain. This completes an antiparallel, four-stranded 'bridging sheet' which stands as a peculiar minidomain in contact with, but distinct from, the inner and outer domains as well as the excised V1/V2 domain. This bridging sheet also participates in the spatially separated interactions of gp120 with both CD4 and the 17b antibody (<a href="#f1">Fig. 1</a>, and see below). One further excursion from the body of the outer domain produces strand <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">15 and helix <img src="glyph_003.gif" style="border: 0px none; vertical-align: baseline;" alt="alpha">3, which are also important in CD4 binding.</p><p class="norm">Taken as a whole, the structure of gp120 seen here has no precedent. Moreover, our domain-level searches have failed to reveal similarity of the inner domain to any known atomic structures, although the missing terminal segments might conceal relationships. We do, however, find fragmentary similarity for portions of the outer domain with known structures. In particular, part of the protomer of FabA dehydrase<sup><a href="#B27">27</a></sup> is like part of the proximal barrel, and dUTP pyrophosphatase<sup><a href="#B28">28</a></sup> has elements in common with both barrels of the outer domain. In each case, the superimposable fraction is limited. For FabA, 45 of its 171 C<img src="glyph_003.gif" style="border: 0px none; vertical-align: baseline;" alt="alpha"> atoms superimpose on five segments, but the rest are topologically unrelated. For dUTPase, 41 of its 152 C<img src="glyph_003.gif" style="border: 0px none; vertical-align: baseline;" alt="alpha"> atoms appropriately capture 8 of the 15 segments in the outer domain body, but there is no helix corresponding to <img src="glyph_003.gif" style="border: 0px none; vertical-align: baseline;" alt="alpha">2 and the placements of termini are not comparable. Several viruses related to HIV encode dUTPases; however, we have not found sequence evidence to support a role in coat protein evolution.</p><p class="norm">This structure of core gp120 should be a prototype for the class. As shown in the structure-based alignment of representative sequences (<a href="#f2">Fig. 2d</a>), there is substantial conservation despite the noted variability among HIV strains. Thus, even an HIV-2 sequence is 35% identical with that of the HXBc2 strain expressed in this crystallized construct, and the identity rises to 77 and 51%, respectively, for the more closely related HIV-1 clade C and clade O representatives. The inner domain is appreciably more conserved than the outer domain, with 86, 72 and 45% identity for the C, O and HIV-2 comparisons, respectively. Variability correlates with the degree of solvent exposure of residues (<a href="#f2">Fig. 2d</a>), in keeping with the conservation of hydrophobic cores. The seven disulphide bridges retained in core gp120 are absolutely conserved and mostly buried (<a href="#f2">Fig. 2c</a>). Glycosylation sites are all surface-exposed and are conserved more than average (<a href="#f2">Fig. 2d</a>). The previously identified HIV variable segments<sup><a href="#B5">5</a></sup> are all on loops connecting elements of secondary structure, and loops <img src="glyph_004.gif" style="border: 0px none; vertical-align: baseline;" alt="Cursive L">D and <img src="glyph_004.gif" style="border: 0px none; vertical-align: baseline;" alt="Cursive L">E are also especially variable. Indeed, <img src="glyph_004.gif" style="border: 0px none; vertical-align: baseline;" alt="Cursive L">E is more variable than V5 in light of current sequence data. These loops are also relatively mobile, as reflected in high B-factors or disorder, as in V4. Variable segments in the outer domain, including the exposed face of <img src="glyph_003.gif" style="border: 0px none; vertical-align: baseline;" alt="alpha">2, appear to arise from neutral mutation rather than selective pressure, because they are on non-immunogenic surfaces<sup><a href="#B24">24</a></sup>, presumably masked by glycosylation.</p></div><div id="CD4ndashgp120-interaction"><a href="#top" class="backtotop">Top<span class="hidden"> of page</span></a><h3>CD4–gp120 interaction</h3><p class="norm">CD4 is bound into a depression formed at the interface of the outer domain with the inner domain and the bridging sheet of gp120 (<a href="#f3">Fig.3a</a>). This interaction buries a total of 742&nbsp;Å<sup>2</sup> from CD4 and 802&nbsp;Å<sup>2</sup> from gp120. The surface areas that are actually in contact are considerably smaller (<a href="#f3">Fig. 3d</a>) because an unusual mismatch in surface topography creates large cavities that are occluded in the interface, as described below. There is, however, a general complementarity in electrostatic potential at the surfaces of contact, although the match is imprecise in this respect as well. The focus of CD4 positivity is displaced from the centre of greatest negativity on gp120 (<a href="#f3">Fig. 3c</a>). The binding site is devoid of carbohydrate (<a href="#f3">Fig.3g</a>). The structure of CD4 in this complex differs only locally from that in free D1D2 structures and at only a few places: residues 17–20 at the poorly ordered CDR1-like loop and residues 41, 42, 47, 49 and 60, which are at or near the contact site and have low B factors in the gp120-bound state.</p><div id="f3" class="figure-table"><h5 class="norm"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/fig_tab/393648a0_F3.html">

Figure 3:&nbsp;CD4–gp120 interactions.</a></h5><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/fig_tab/393648a0_F3.html"><img alt="Figure 3 : CD4|[ndash]|gp120 interactions. Unfortunately we are unable to provide accessible alternative text for this. If you require assistance to access this image, or to obtain a text description, please contact npg@nature.com" class="thumb" src="393648ac.tif.0.gif"></a><p><b>a</b>, Ribbon diagram of gp120 (red) binding to CD4 (yellow). Residue Phe&nbsp;43 of CD4 is also depicted reaching into the heart of gp120. From this orientation, the recessed nature of the gp120 binding pocket is evident. <b>b</b>, Electron density in the Phe&nbsp;43 cavity. The 2<i>F</i><sub>o</sub> - <i>F</i><sub>c</sub> electron density map at 2.5&nbsp;Å, 1.1<img src="glyph.gif" style="border: 0px none; vertical-align: baseline;" alt="sigma"> contour, is shown in blue. The gp120 model is depicted in red with the CD4 model in yellow (carbon atoms), blue (nitrogen atoms), and red (oxygen atoms). The orientation is the same as in <b>a</b>. The foreground has been clipped for clarity, removing the overlying <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">24–<img src="glyph_003.gif" style="border: 0px none; vertical-align: baseline;" alt="alpha">5 connection. In the upper middle region lies the central unidentified density. At the bottom, Phe&nbsp;43 of CD4 reaches up to contact the cavity. Moving clockwise round the cavity, the gp120 residues are Trp&nbsp;427 (with its indole ring partially clipped by foreground slabbing), Trp&nbsp;112, Val&nbsp;255, Thr&nbsp;257, Glu&nbsp;370 (packing under the Phe&nbsp;43 ring), Ile&nbsp;371 and Asp&nbsp;368 (partially clipped in the bottom right corner). Hydrophobic residues lining the back of the cavity can be partially seen around the central unidentified density. <b>c</b>, Electrostatic surfaces of CD4 and gp120. The electrostatic potential is shown at the solvent-accessible surface, which is coloured according to the local electrostatic potential, ranging from dark blue (most positive) to deep red (most negative). The slight puffiness of the surface arises from the enlarged solvent-accessible surface relative to the standard molecular surface. On the right, the gp120 surface is shown in an orientation similar to that in <a href="#f2">Fig. 2a</a>, <a href="#f2">c</a>, but rotated <img src="glyph_005.gif" style="border: 0px none; vertical-align: baseline;" alt="approx">20° around a vertical axis to depict the recessed binding pocket more clearly. A thin yellow C<img src="glyph_003.gif" style="border: 0px none; vertical-align: baseline;" alt="alpha"> worm of CD4 is shown to aid in orientation. On the left, the CD4 surface is shown, rotated relative to the gp120 panel by an exact 180° rotation about the vertical axis shown. A thin red C<img src="glyph_003.gif" style="border: 0px none; vertical-align: baseline;" alt="alpha"> worm of gp120 is shown. <b>d</b>, CD4–gp120 contact surface. On the right, the gp120 surface is shown in red, with the surface within 3.5&nbsp;Å of CD4 (surface-to-atom centre distance) in yellow. This effectively creates an imprint of CD4 on the gp120 surface. On the left (180° rotation), the corresponding CD4 surface is shown in yellow, with the gp120 imprint in red. <b>e</b>, CD4–gp120 mutational hot-spots. On the right, the surface of gp120 is shown in red, with the surface highlighted of gp120 residues that have been shown by substitution to affect CD4 binding: cyan, substantial effect — residues 368, 370 and 427; green, moderate effect — residue 457. Also depicted (white) is the surface of the large water-filled cavity at the CD4–gp120 interface. On the left (180° rotation), residues important for gp120 binding are shown on a yellow CD4 surface: cyan, substantial effect — residues 43 and 59; green, moderate effect — residues 29, 35, 44, 46, 47 (<a href="#B29">ref. 29</a>). <b>f</b>, Side-chain/main-chain contribution to the gp120 surface. The surface of gp120 contributed by main-chain atoms (including C<img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">) is green, that contributed by side-chain atoms is white, and that contributed by the C<img src="glyph_003.gif" style="border: 0px none; vertical-align: baseline;" alt="alpha"> of glycine is brick-red. This orientation is the same as the right panel of <b>c</b>–<b>e</b> and in <b>g</b>, and allows for direct comparison of the CD4–gp120 contact surface. A striking surface concentration of main-chain atoms is seen in the regions corresponding to the CD4 imprint. <b>g</b>, Sequence variability mapped to the gp120 surface. The sequence variability among primate immunodeficiency viruses (<a href="#f2">Fig. 2d</a>) is mapped onto the gp120 surface. A sliding scale of white (conserved) to brick-red (highly variable) is shown. Carbohydrate residues are also shown: blue, <i>N</i>-acetylglucosamine and fucose residues in the structure; purple, Asn-proximal <i>N</i>-acetylglucosamines modelled at residues 88, 230, 241, 356, 397, 406, 462. Much of the carbohydrate (22 residues) is hidden on the back side of the outer domain. <b>h</b>, Phe&nbsp;43 cavity. The surface of the Phe&nbsp;43 cavity is in blue, buried in the heart of gp120. A C<img src="glyph_003.gif" style="border: 0px none; vertical-align: baseline;" alt="alpha"> worm representation of gp120 (red) shows in green the three stretches that are incorrect by secondary-structure prediction: the <img src="glyph_004.gif" style="border: 0px none; vertical-align: baseline;" alt="Cursive L">B loop, bending around the top of the cavity, parts of <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">20–<img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">21 just below the cavity, and strand <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">15, slightly right of the cavity. The orientation shown here is the same as for the gp120 surfaces in <b>c</b>–<b>g</b>. <b>i</b>, The CD4-gp120 interface. This schematic representation of the entire interface shows six discrete segments of gp120 (solid black lines) interacting with CD4 (double lines). For orientation, secondary structural elements are labelled, as are representative contact residues from each segment of gp120. Arrows indicate main-chain direction. The side chain of Phe&nbsp;43 is also shown. The orientation shown is similar to that in <b>a</b> and <b>b</b>. <b>j</b>, gp120 contacts around Phe&nbsp;43 and Arg&nbsp;59 of CD4. Residues on gp120 involved in direct contact with Phe&nbsp;43 or Arg&nbsp;59 are shown. Electrostatic interactions are depicted as dashed lines. Hydrophobic interactions are found between Phe&nbsp;43 (CD4) and Trp&nbsp;427, Glu&nbsp;370, Gly&nbsp;473 and Ile&nbsp;371 (all from gp120) and between Arg&nbsp;59 (CD4) and Val&nbsp;430 (gp120). The orientation is similar to that in <b>a</b>, <b>b</b> and <b>i</b>, but has been rotated for clarity. Side chains of Phe&nbsp;43 and Arg&nbsp;59, as well as those portions of gp120 sidechains that interact with these crucial CD4 residues, are drawn as bold lines. Panel <b>a</b> was drawn with RIBBONS<sup><a href="#B49">49</a></sup>, <b>b</b> with program O<sup><a href="#B47">47</a></sup>, and <b>b</b>–<b>g</b> with GRASP<sup><a href="#B50">50</a></sup>.</p><a class="high-res" href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/fig_tab/393648a0_F3.html">High resolution image and legend (140K)</a><span class="cleardiv"><!-- --></span>
</div><br class="clear"><p class="norm">Direct interatomic contacts are made between 22 CD4 residues and 26 gp120 amino-acid residues. These include 219 van der Waals contacts and 12 hydrogen bonds. Residues in contact are concentrated in the span from 25 to 64 of CD4, but they are distributed over six segments of gp120 (<a href="#f2">Figs 2d</a> and <a href="#f3">3i</a>) : one residue from the V1/V2 stem, loop <img src="glyph_004.gif" style="border: 0px none; vertical-align: baseline;" alt="Cursive L">D, the <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">15–<img src="glyph_003.gif" style="border: 0px none; vertical-align: baseline;" alt="alpha">3 excursion, the <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">20–<img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">21 hairpin, strand <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">23 and the <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">24–<img src="glyph_003.gif" style="border: 0px none; vertical-align: baseline;" alt="alpha">5 connection. These interactions are compatible with previous analyses of mutational data on both CD4 (<a href="#B11">refs 11</a>, <a href="#B12">12</a>, <a href="#B29">29</a>) and gp120 (<a href="#B3">refs 3</a>, <a href="#B13">13</a>, <a href="#B14">14</a>). Other groups are also involved, including some at gp120 sites that have not been tested, but residues identified as critical for binding do indeed interact with one another (<a href="#f3">Fig. 3e</a>). Most important, Phe&nbsp;43 and Arg&nbsp;59 of CD4 make multiple contacts centred on residues Asp&nbsp;368, Glu&nbsp;370 and Trp&nbsp;427 of gp120, which are all conserved among primate immunodeficiency viruses. In fact, 63% of all interatomic contacts come from one span (40–48) in C'C" of CD4, and Phe&nbsp;43 alone accounts for 23% of the total. Similarly, with respect to gp120, the spans of 365–371 and 425–430 contribute 57% of the total. Of the three CD4 lysine residues implicated in binding (residues 29, 35 and 46), only Lys&nbsp;29 makes a direct ionic hydrogen bond, and although Asp&nbsp;457 of gp120 is near to these electropositive groups (<a href="#f3">Fig. 3e</a>, <a href="#f3">i</a>), it does not make hydrogen bonds.</p><p class="norm">Several gp120 residues that are covered by CD4 are variable in sequence. This variation is accommodated in part by the large interfacial cavity (<a href="#f3">Fig. 3e</a>). The gp120 residues in contact with this water-filled cavity are especially variable (<a href="#f3">Fig. 3g</a>). Moreover, half of the gp120 residues that make contacts with CD4 do so only through main-chain atoms (including C<img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">) of gp120, and 60% of CD4 contacts are made by gp120 main-chain atoms (<a href="#f3">Fig. 3f</a>). Included among these are 5 of the 12 hydrogen bonds in the interface. One such contributing element is an antiparallel <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">-sheet alignment of CD4 strand C" with gp120 strand <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">15 (<a href="#f3">Fig. 3a</a>, <a href="#f3">i</a>).</p><p class="norm">Atomic details of the interaction are particularly intricate and unusual for the contacts made between gp120 and the mutationally critical CD4 residues Phe&nbsp;43 and Arg&nbsp;59 (<a href="#f3">Fig. 3j</a>). Arg&nbsp;59 interacts with Asp&nbsp;368 and Val&nbsp;430. The carboxylate group of Asp&nbsp;368 makes double hydrogen-bonds with the guanidinium N<img src="glyph_006.gif" style="border: 0px none; vertical-align: baseline;" alt="eta"> atoms of Arg&nbsp;59, but it also hydrogen-bonds back to the backbone NH group of residue 44 and it appears to be optimally positioned to receive a CH...O hydrogen bond (3.20&nbsp;Å) from the Phe&nbsp;43 ring. Phe&nbsp;43 interacts with residues Glu&nbsp;370, Ile&nbsp;371, Asn&nbsp;425, Met&nbsp;426, Trp&nbsp;427 and Gly&nbsp;473, as well as Asp&nbsp;368, but only the contacts with Ile&nbsp;371 have a conventional hydrophobic character. Those to 425–427 and 473, including Trp&nbsp;427, are only to backbone atoms. A surprisingly large fraction of the Phe&nbsp;43 contacts (28%) are to polar atoms. The phenyl group is stacked on the carboxylate group of Glu&nbsp;370, and there are contacts with the carbonyl oxygen atoms of residues 425, 426 and 473 and the NH group of Trp&nbsp;427. One of these, from the phenyl ring to O&nbsp;425, is a second candidate CH...O hydrogen bond (3.10Å). Asp&nbsp;368 and Glu&nbsp;370 have their carboxylate groups close together (3.54&nbsp;Å) and they are, of course, buried in the complex. Even for gp120 excised from the complex, their fractional surface accessibilities are only 44 and 14%, respectively. Glu&nbsp;370 may therefore be protonated. Perhaps the most extraordinary aspect of this site is the large cavity beyond C<img src="glyph_007.gif" style="border: 0px none; vertical-align: middle;" alt="zeta"> of Phe&nbsp;43 (<a href="#f3">Fig.3b</a>, <a href="#f3">h</a>, and see below).</p></div><div id="Interfacial-cavities"><a href="#top" class="backtotop">Top<span class="hidden"> of page</span></a><h3>Interfacial cavities</h3><p class="norm">Analysis of the solvent-accessible surface of the ternary complex reveals a number of topologically interior surfaces or cavities. Two of these, both at the gp120–CD4 interface, are unusually large. The larger (279&nbsp;Å<sup>3</sup>) is formed at the interface between the slightly concave middle of the CC'C" portion of the CD4 sheet and a groove on gp120, where <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">23 and <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">24 are indented relative to <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">15 and the <img src="glyph_004.gif" style="border: 0px none; vertical-align: baseline;" alt="Cursive L">D loop (<a href="#f3">Fig. 3e</a>). The second is from a pocket in the gp120 surface that is plugged by Phe&nbsp;43 from CD4 (152&nbsp;Å<sup>3</sup>). This pocket is itself at the interface between the inner and outer domains of gp120 (<a href="#f3">Fig. 3h</a>). Several other smaller cavities are also wedged at the interface between the two gp120 domains.</p><p class="norm">The larger cavity is lined mostly by hydrophilic residues, half derived from gp120 and half from CD4. It is not deeply buried; although it is formally a cavity in the crystal structure, minor changes in side-chain orientation could make it solvent-accessible. The observed electron density and predicted hydrogen bonding are consistent with at least eight water molecules in the cavity. Residues from gp120 that actually line the cavity (including Ala&nbsp;281, Ser&nbsp;364, Ser&nbsp;365, Thr&nbsp;455, Arg&nbsp;469) show sequence variability, whereas surrounding this variable patch are conserved residues, the substitution of which affect CD4 binding. These include the critical contact residues Asp&nbsp;368, Glu&nbsp;370 and Trp&nbsp;427, which flank one end of the cavity, and Asp&nbsp;457 at the other end (<a href="#f3">Fig.3e</a>). Similarly, CD4 residues that line the cavity (such as Gln&nbsp;40 and Lys&nbsp;35) can be mutated with only moderate effect on gp120 binding, whereas Arg&nbsp;59 suffers less loss of solvent-accessible surface upon gp120 binding but is highly sensitive to mutation. This cavity thus serves as a water buffer between gp120 and CD4 (<a href="#f3">Fig. 3e</a>). The tolerance for variation in the gp120 surface associated with this cavity produces a variational island (<a href="#f3">Fig. 3g</a>), or 'anti-hotspot', which is centrally located between regions required for CD4 binding, and may help the virus escape from antibodies directed against the CD4-binding site.</p><p class="norm">The 'Phe&nbsp;43' cavity (<a href="#f3">Fig. 3b</a>, <a href="#f3">h</a>) is very different in character from the larger binding–interface cavity. It is roughly spherical, with a diameter of <img src="glyph_005.gif" style="border: 0px none; vertical-align: baseline;" alt="approx">8&nbsp;Å (atom centre to atom centre) across the centre of the cavity. It is positioned just beyond Phe&nbsp;43 of CD4, at the intersection of the inner domain, the outer domain and the bridging sheet. It is deeply buried, extending into the hydrophobic interior of gp120. The phenyl ring of Phe&nbsp;43 is the only non-gp120 residue contacting this cavity, forming a lid that covers the bottom of the cavity (<a href="#f3">Fig. 3b</a>). Other routes of solvent access are possible: past Met&nbsp;426 under the bridging sheet, or directly through the heart of gp120, at the interface between the inner and outer domains. Ordered water molecules demarking possible paths of solvent access are found along both routes. Nonetheless, in the cavity itself only a few water molecules are seen. The centre of the cavity is dominated by a large piece of spherical density, which is over 4&nbsp;Å from any protein atom (<a href="#f3">Fig. 3b</a>). The size, shape and predicted hydrogen bonding of this density is inconsistent with those expected for water, isopropanol, ethylene glycol, or any of the other major crystallization components. We have not identified the source of this density.</p><p class="norm">Residues that line the Phe&nbsp;43 cavity (Trp&nbsp;112, Val&nbsp;255, Thr&nbsp;257, Glu&nbsp;370, Phe&nbsp;382, Tyr&nbsp;384, Trp&nbsp;427 and Met&nbsp;475; only main chains of 256 and 375–377) are primarily hydrophobic. They are also highly conserved, as much so as the buried gp120 hydrophobic core. Despite a lack of steric hindrance, almost no substitutions to larger residues are found. Given the frequency of gp120 sequence divergence, such conservation strongly implies functional significance. Indeed, although residues that line this cavity provide little direct contact to CD4, they do affect the gp120–CD4 interaction. Thus, mutations at Thr&nbsp;257 (no contacts) and Trp&nbsp;427 (only main-chain contacts) can substantially reduce binding. Changes in cavity-lining residues also affect the binding of antibodies directed against the CD4-binding site<sup><a href="#B24">24</a></sup>. In addition, many of the residues that line the cavity interact with elements of the chemokine-receptor-binding region (see below). It may be that the Phe&nbsp;43 cavity and the other interdomain cavities form as a consequence of a CD4-induced conformational change (see below).</p><p class="norm">Despite the unusual cavity-laden interface between gp120 and CD4, we believe that this structure reflects the true character of the interaction. Core and full-length gp120 bind CD4 with similar affinity<sup><a href="#B26">26</a></sup>, and residues identified as critical by mutational analysis on both components are indeed at the focus of contact in the structure. In any case, the missing gp120 loops and termini could not conceivably have a role in filling these cavities.</p></div><div id="Antibody-interface"><a href="#top" class="backtotop">Top<span class="hidden"> of page</span></a><h3>Antibody interface</h3><p class="norm">The 17b antibody is a broadly neutralizing human monoclonal isolated from the blood of an HIV-infected individual. It binds to a CD4-induced (CD4i) gp120 epitope that overlaps the chemokine-receptor-binding site<sup><a href="#B20">20</a></sup>.</p><p class="norm">Relative to other antibody–antigen pairs the interface between Fab 17b and core gp120 in the ternary complex is small (<a href="#f4">Fig. 4a–c</a>). The solvent-accessible area excluded upon binding is only 455&nbsp;Å<sup>2</sup> from gp120 and 445&nbsp;Å<sup>2</sup> from 17b, which is largely from the heavy chain (371&nbsp;Å<sup>2</sup>). The long (15 residue) complementarity-determining region 3 (CDR3) of the heavy chain dominates, but the heavy-chain CDR2 and the light-chain CDR3 also contribute. Overall, the 17b contact surface is very acidic (3&nbsp;Asp, 3&nbsp;Glu, no Arg or Lys), although hydrophobic contacts (notably a <i>cis</i> proline and tryptophan from the light chain) predominate at the centre.</p><p class="norm">On gp120, the 17b epitope lies across the base of the four-stranded bridging sheet (<a href="#f4">Fig. 4c</a>, <a href="#f4">e</a>). All four strands make substantial contact with 17b, suggesting that the integrity of the bridging sheet is necessary for 17b binding. The gp120 surface that contacts 17b consists of a hydrophobic centre surrounded by a highly basic periphery (3&nbsp;Lys, 1&nbsp;Arg, and no Asp or Glu) (<a href="#f4">Fig. 4d</a>). Although this basic gp120 surface complements the acidic 17b surface, only one salt bridge is observed (between Arg&nbsp;419 of gp120 and Glu&nbsp;106 of the 17b heavy chain). The rest of the specific contacts occur between hydrophobic and polar residues. Thus, the interaction between 17b and gp120 involves a hydrophobic central region flanked on the periphery by charged regions, predominately acidic on 17b and basic on gp120. There are no direct CD4–17b contacts and none of the gp120 residues contacts both 17b and CD4. Rather, CD4 binds on the opposite face of the bridging sheet, providing specific contacts that seem to stabilize its conformation (<a href="#f3">Fig. 3i</a>, <a href="#f3">j</a>) and may explain, in part, the CD4 induction of 17b binding.</p><p class="norm">The 17b epitope is well conserved among HIV-1 isolates. Of the 18 residues that show a loss in solvent-accessible surface upon contact with 17b, 12 residues (67%) are conserved among all HIV-1 viruses. By contrast, only 19 of the 37 gp120 residues (51%) that show loss of solvent-accessible surface upon CD4 binding are similarly conserved. CD4i epitopes tend to be masked from immune surveillance by the adjacent V2 and V3 loops<sup><a href="#B24">24</a></sup>. In the complex structure, a large gap is seen between gp120 and the tips of the light-chain CDR1 and CDR2 loops. Pointing directly at this gap is the base of the V3 loop. In intact gp120, the variable loops may need to be bypassed for access to the conserved structures in the bridging sheet. The 17b epitope may be further protected from the immune system by a CD4-induced conformational change (see below).</p></div><div id="Chemokinereceptor-site"><a href="#top" class="backtotop">Top<span class="hidden"> of page</span></a><h3>Chemokine-receptor site</h3><p class="norm">The site of interaction with the chemokine receptor CCR5 overlaps with the 17b epitope<sup><a href="#B30">30</a></sup>. Both are induced upon CD4 binding and both involve highly conserved residues. Mutational analysis showed that the basic and polar gp120 residues (Lys&nbsp;121, Arg&nbsp;419, Lys&nbsp;421, Gln&nbsp;422) that contact the 17b heavy chain also are important for CCR5 interaction<sup><a href="#B30">30</a></sup>. The hydrophobic and acidic surface of the 17b heavy chain may mimic the tyrosine-rich, acidic N-terminal region of CCR5, which is important for gp120 binding and HIV-1 entry<sup><a href="#B31">31</a></sup>,<sup><a href="#B32">32</a></sup>. Geometrically, this site is directed at the cellular membrane when gp120 is engaged by CD4. Electrostatic interactions between the basic surface of the bridging sheet and the acidic chemokine receptor (and possibly the acid headgroups in the target membrane) could drive conformational changes related to virus entry.</p></div><div id="Oligomer-and-gp41-interactions"><a href="#top" class="backtotop">Top<span class="hidden"> of page</span></a><h3>Oligomer and gp41 interactions</h3><p class="norm">Although monomeric in isolation, gp120 probably exists as a trimeric complex with gp41 on the virion surface. The large electroneutral surface on the inner domain (<a href="#f3">Fig. 3c</a>) is the probable site of trimer packing based on its lack of glycosylation, its conservation in sequence, the location of CD4- and CCR5-binding sites, and the immune response to this region. These points are elaborated in the accompanying Letter and a model is presented<sup><a href="#B24">24</a></sup>.</p><p class="norm">Mutagenic and antibody-binding analyses indicate that the N and C termini of full-length gp120 are the most important regions for interaction with the gp41 glycoprotein<sup><a href="#B33">33</a></sup>,<sup><a href="#B34">34</a></sup>. From these analyses, we expect that the gp41-interactive regions will extend away from core gp120 towards the viral membrane, and that the conserved, electroneutral surface is occluded in the oligomer/gp41 interface. A similar arrangement is seen in influenza haemagglutinin, where the extended N and C termini of HA<sub>1</sub> interact with the HA<sub>2</sub> transmembrane protein<sup><a href="#B35">35</a></sup>.</p></div><div id="Conformational-change-in-core-gp120"><a href="#top" class="backtotop">Top<span class="hidden"> of page</span></a><h3>Conformational change in core gp120</h3><p class="norm">There is abundant evidence to suggest that CD4 binding induces a conformational change in gp120, but much of it derives from intact gp120 with variable loops in place or from the oligomeric gp120–gp41 complex. The ternary complex structure provides clues to the nature of the conformational changes that occur within core gp120 itself. (Although 17b binding could contribute to the gp120 conformation observed in the crystal, the CD4 contacts are more extensive and multifaceted than those of 17b, indicating that CD4 binding may play the principal part in formation of the observed gp120 structure.)</p><p class="norm">Were the conformation of gp120 seen here preserved in the absence of CD4, the Phe&nbsp;43 cavity (now a pocket) would present a perplexing structural dilemma. As discussed above, the cavity-lining residues have few structural restrictions, with ample room for larger substitutions into the cavity, yet these residues are highly conserved and inexplicably hydrophobic if exposed in a pocket. This pocket structure is in turn intimately connected to the bridging sheet, itself peculiar in absence of CD4. Thus, for example, the backbone amide of bridging-sheet residue 425 is hydrogen-bonded to Glu&nbsp;370, a critical CD4 contact residue (<a href="#f3">Fig. 3j</a>); Ile&nbsp;424 makes extensive hydrophobic contacts with Phe&nbsp;382, which lines the pocket from the outer domain; and Trp&nbsp;427 packs perpendicular to Trp&nbsp;112, which lines the pocket from the inner domain (<a href="#f3">Fig. 3b</a>). N<img src="glyph_008.gif" style="border: 0px none; vertical-align: middle;" alt="epsilon"> of Trp&nbsp;427 is delicately poised for hydrogen bonding with the <img src="glyph_009.gif" style="border: 0px none; vertical-align: baseline;" alt="pi">-electrons of the indole ring of Trp&nbsp;112. Structures such as these would necessarily be sensitive to orientational shifts between the inner and outer domains.</p><p class="norm">The characteristics of 17b binding to core gp120 provide additional evidence for a CD4-induced conformational change. We do not observe detectable binding of Fab 17b to core gp120 unless CD4 is present, but then the ternary complex is stable in gel filtration. As there are no direct CD4–17b contacts in the structure, the effect of CD4 must be to stabilize the bridging-sheet minidomain to which 17b binds. This result is compatible with the binding properties of 17b and other CD4i antibodies to full-length gp120 (<a href="#B18">refs 18</a>, <a href="#B24">24</a>), but it shows that the conformational change is not limited to an unmasking of the antibody epitope by CD4-induced displacement of the V2 loop, as was initially thought<sup><a href="#B36">36</a></sup>. The ability of the 17b antibody to bind full-length gp120 in the absence of CD4, albeit with lower affinity, implies that structural elements required for 17b binding can be accessed in the absence of CD4. If we assume that 17b binds in the same way to both full-length and core gp120, as shown by the concordance between the structural contacts (<a href="#f4">Fig. 4</a>) and epitope-mapping data<sup><a href="#B30">30</a></sup>, then alternative conformations could be in a kinetically accessible equilibrium in native gp120.</p><p class="norm">A further indication that core gp120 may differ in the absence of CD4 comes from comparison with theory. When applied to the many known sequence variants of gp120, the evolutionary algorithm of PHD<sup><a href="#B37">37</a></sup> gives secondary-structure predictions with estimated reliability above 90% for roughly 45% of the core gp120 sequence. Compared with our structure, it is accurate except at three places where it is markedly wrong (four consecutive residues with reliability index greater than 90%). All of these are at the Phe&nbsp;43 cavity or in contacts with CD4: loop <img src="glyph_004.gif" style="border: 0px none; vertical-align: baseline;" alt="Cursive L">B, strand <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">15, and the segment of <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">20 into the turn to <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">21 (<a href="#f3">Fig. 3h</a>). Most significantly, the latter segment (residues 422–429) entering the bridging sheet is predicted to be helical. Indeed, residues 427–428 at the <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">20–<img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">21 turn do have helical character. We also note that CD4 binds efficiently to a gp120 derivative having both <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">2 and <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">3 truncated<sup><a href="#B38">38</a></sup>. As the bridging sheet is probably not stable in the absence of half of its strands, CD4 binding must be able to orientate strands <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">20 and <img src="glyph_002.gif" style="border: 0px none; vertical-align: middle;" alt="beta">21 properly from a very different prior conformation.</p><p class="norm">The Phe&nbsp;43 cavity is at the nexus of the CD4 interface, between the inner domain, the outer domain, and the bridging sheet. As such, Phe&nbsp;43 itself seems to serve as a linchpin without which the structure might collapse. If so, to what state and, in reverse, how does CD4 binding lead to the state seen in this ternary complex? CD4–gp120 binding kinetics are complex<sup><a href="#B39">39</a></sup>, and microcalorimetry reveals unusually large <img src="glyph_010.gif" style="border: 0px none; vertical-align: baseline;" alt="Delta"><i>H</i> and compensating <i>T</i><img src="glyph_010.gif" style="border: 0px none; vertical-align: baseline;" alt="Delta"><i>S</i> values for soluble CD4 binding to gp120 (M. L. Doyle, personal communication). These exceptional CD4-binding thermodynamics are suggestive of a large conformational change and are similar for both full-length and core gp120, supporting the relevance of our structural observations on core gp120. We imagine that CD4 sees gp120 as an uneven equilibrium of conformational states, makes initial contact by electrostatic interaction (<a href="#f3">Fig. 3c</a>), stabilizes a nascent complex state, and inserts Phe&nbsp;43 to weld the cavity into place.</p></div><div id="Viral-evasion-of-immune-responses"><a href="#top" class="backtotop">Top<span class="hidden"> of page</span></a><h3>Viral evasion of immune responses</h3><p class="norm">Analysis of the antigenic structure of gp120 shows that most of the envelope protein surface is hidden from humoral immune responses by glycosylation and oligomeric occlusion<sup><a href="#B24">24</a></sup>. Most broadly neutralizing antibodies access only two surfaces: one that overlaps the CD4-binding site (shielded by the V1/V2 loop), and another that overlaps the chemokine-receptor-binding site (shielded by the V2 and V3 loops). Conformational changes in core gp120 provide additional mechanisms for evasion from immune surveillance. In the case of the CD4-binding surface, which contains a high proportion of main-chain atoms in the complex (<a href="#f3">Fig. 3f</a>), the conformation without CD4 bound may expose underlying side-chain variability (<a href="#f3">Fig. 3g</a>). Escape may also be provided by the recessed nature of the binding pocket (steric occlusion) (<a href="#f3">Fig. 3a</a>) and by a topographical surface mismatch, which encloses a variational island or 'anti-hotspot' (described above; <a href="#f3">Fig. 3d</a>). Mechanisms may be similar in the chemokine-receptor region: conformational change may hide the conserved epitope (unformed before CD4 binding); steric occlusion may take place between the CD4 anchored viral spike and the proximal target membrane; and an 'anti-hotspot' equivalent may camouflage chemokine-receptor-binding residues on the V3 loop in surrounding variability. Some of the means used to elude antibody-based defences may also help HIV avoid cellular immunity. Understanding the specific mechanisms of immune evasion may be prerequisite to the design of an effective vaccine.</p></div><div id="Mechanistic-implications-for-virus-entry"><a href="#top" class="backtotop">Top<span class="hidden"> of page</span></a><h3>Mechanistic implications for virus entry</h3><p class="norm">During virus entry, HIV surface proteins fuse the viral membrane with the target cell membrane. gp120 is a crucial participant in the control and initiation of fusion. It functions in positioning: locating a cell capable of productive viral infection, anchoring the virus to the cell surface, and orienting the viral spike next to the target membrane. It also functions in timing: holding gp41 in a metastable conformation and triggering the coordinate release of the three N-terminal fusion peptides of the trimeric gp41. Although this is a complex, multiconformational process, the simplicity of the system, which comprises only two membranes, the viral oligomer and two host receptors, suggests that we may be able to understand the entire mechanism. Crystallography has now provided two snapshots: an intermediate state in which gp120 is bound to CD4, as described here, and a probably final, 'fusion-active' state of the gp41 ectodomain<sup><a href="#B40">40</a></sup>,<sup><a href="#B41">41</a></sup>. Although atomic-level structures are lacking for other intermediates, the extensive biochemical data on membrane fusion mediated by HIV-1 envelope glycoproteins extend our understanding from these two states.</p><p class="norm">The entry process is initiated by binding of HIV-1 to the cellular receptor CD4 (<a href="#f5">Fig. 5</a>, step 1). Although the extracellular portion of CD4 has some segmental flexibility, this binding roughly orients the viral spike. This orientation can be simulated by an alignment of the D1D2 portion of CD4 in the ternary complex with the previously solved structure of the four-domain, entire extracellular portion of CD4 (<a href="#B10">ref. 10</a>). Such alignment orients the N and C termini of core gp120 towards the viral membrane, while the 17b epitope/chemokine-receptor-binding site on the gp120 surface faces the target cell membrane. Such an orientation is consistent with the proposed oligomeric structure and gp41-interactive surfaces (see above and <a href="#B24">ref. 24</a>).</p><div id="f5" class="figure-table"><h5 class="norm"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/fig_tab/393648a0_F5.html">

Figure 5:&nbsp;Diagram of gp120 initiation of fusion.</a></h5><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/fig_tab/393648a0_F5.html"><img alt="Figure 5 : Diagram of gp120 initiation of fusion. Unfortunately we are unable to provide accessible alternative text for this. If you require assistance to access this image, or to obtain a text description, please contact npg@nature.com" class="thumb" src="393648ae.tif.0.gif"></a><p>A single monomer of core gp120 is depicted (red) in an orientation similar to that in <a href="#f2">Fig. 2a</a>, <a href="#f2">c</a>. The '3' symbolizes the 3-fold axis, from which gp41 interacts with the gp120 N and C termini to generate the functional oligomer. In the initial state of gp120 (on the surface of a virion), the V1/V2 loops (salmon) are shown partially occluding the CD4-binding site. Following CD4 binding (now at a target cell, though above the glycocalyx), a conformational change is depicted as an inner/outer domain shift, with the purple circle denoting the formation of the Phe&nbsp;43 cavity. This conformational change strains the interactions at the N and C termini of gp120 with the rest of the oligomer, priming the CD4-bound gp120 core. In the next step (directly adjacent to the target membrane), the chemokine receptor binds to the bridging sheet and the V3 loop (in green; bottom left and right, respectively, of gp120), causing an orientational shift of core gp120 relative to the oligomer. This triggers further changes, which ultimately lead to the fusion of the viral and target membranes.</p><a class="high-res" href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/fig_tab/393648a0_F5.html">High resolution image and legend (28K)</a><span class="cleardiv"><!-- --></span>
</div><br class="clear"><p class="norm">CD4 binding also induces conformational changes in gp120, which result in the creation of a metastable oligomer. Although some of the more flexible gp120 regions and gp41 are missing, the structure of the core gp120–CD4 complex presented here describes this state in atomic detail. CD4 binding results in movement of the V2 loop, which probably partially occludes the V3 loop and CD4i epitopes<sup><a href="#B18">18</a></sup>,<sup><a href="#B36">36</a></sup>. It also creates, or at least stabilizes, the bridging sheet on which the latter epitopes are located (described above). In addition, CD4 binding appears to change the V3 region, with enhanced proteolytic susceptibility at the tip of the loop and altered exposure of V3 epitopes<sup><a href="#B19">19</a></sup>. The uncovered V3 and CD4i epitopes make up the chemokine-receptor binding site. Thus CD4 binding not only orients the gp120 surface implicated in chemokine receptor binding to face the target cell, it also forms and exposes the site itself. These changes may all happen in a single, concerted shift in the relative orientation of the inner and outer domains. This conformational shift may alter the orientation of the N and C termini, at the proximal end of the inner domain, perhaps partially destabilizing the oligomeric gp120/gp41 interface<sup><a href="#B21">21</a></sup>. Such a shift would also alter the relative placement of the V1/V2 stem (in the CD4i site), which emanates from the inner domain, and the V3 loop, which emanates from the outer domain. Perhaps relevant to this, mutations that permit an adaptation of HIV-1 to CD4-independent entry using CXCR4 involve sequence changes in both the V1/V2 stem and the V3 loop<sup><a href="#B42">42</a></sup>.</p><p class="norm">The next step in HIV-1 entry is the interaction of the gp120–CD4 complex with the chemokine receptor (<a href="#f5">Fig. 5</a>, step 2). Although interactions between CD4 and chemokine receptor may occur, mutagenic analysis (H. Choe and J. Sodroski, unpublished observations) and the known examples of CD4-independent virus entry or chemokine-receptor binding suggest that direct gp120 contacts dominate interaction with the chemokine receptor. As most of the chemokine receptor is encased in the host membrane, binding will move the gp120 bridging sheet close to the target membrane. This movement must require CD4 flexibility because the initial HIV binding at the N-terminal D1 domains probably occurs above the glycocalyx. Reducing flexibility at the D2–D3 juncture or at the D4–membrane juncture of CD4 has been shown to block HIV-1 entry<sup><a href="#B10">10</a></sup>,<sup><a href="#B43">43</a></sup>.</p><p class="norm">Chemokine-receptor binding is believed to trigger additional conformational changes in the HIV-1 envelope glycoprotein trimer that lead to exposure of the gp41 ectodomain. Presumably, a signal is transmitted from the cell-associated distal end of gp120 to elements of the inner domain that are likely to be involved in gp120–gp41 or gp120–gp120 association on the trimer. Although further interdomain shifts may occur in core gp120 after chemokine-receptor binding, the geometrically specific contacts that support the bridging sheet make it unlikely that another shift would occur without destabilizing this important component of the chemokine-receptor-binding site. As the high affinity of the interactions make it likely that both CD4 and chemokine receptor remain bound to gp120 during fusion, we expect additional conformational changes to occur between neighbouring gp120 protomers in the oligomeric complex. Perhaps the chemokine receptor triggers gp41 exposure by prising gp120 protomers away from the trimer axis, thus exerting a torque on the gp120–gp41 interface. It is interesting that several of the substitutions that affect chemokine-receptor binding in the context of monomeric gp120 appear to induce gp120 dissociation in an oligomeric context<sup><a href="#B30">30</a></sup>.</p><p class="norm">The structure of the gp120/CD4/17b antibody ternary complex described here reveals many molecular aspects of HIV-1 entry, including the atomic structure of gp120, the explicit interactions with CD4, and the conserved site of binding for the chemokine receptor. Still unknown are details of the apo state of core gp120, the oligomeric structure, the interaction with the chemokine receptor, the conformational changes that trigger reorganization of the gp41 ectodomain, and the structural basis for insertion of the fusion peptide of gp41 into the target membrane. Further understanding will require snapshots of other intermediates. The conformational complexity and intricate domain associations of gp120, like those of reverse transcriptase<sup><a href="#B44">44</a></sup>, the other large HIV translation product, may reflect genome restrictions at the protein level like those that lead to overlapping reading frames at the transcription level. Multiply protected infection machinery is contained in these condensed intricacies. Its mechanism frustrates host defences; understanding them may inspire medical intervention.</p></div><div id="Methods"><a href="#top" class="backtotop">Top<span class="hidden"> of page</span></a><h3>Methods</h3><p class="norm"><b>Protein production, crystallization and data collection.</b> The two-domain CD4 (D1D2, residues 1–182) was produced in Chinese hamster ovarian cells<sup><a href="#B8">8</a></sup>, the monoclonal antibody 17b in an Epstein–Barr virus immortalized B-cell clone isolated from an HIV-1-infected individual and fused with a murine B-cell fusion partner<sup><a href="#B18">18</a></sup>, and the core gp120 from <i>Drosophila</i> Schneider 2 lines under the control of an inducible metallothionein promoter<sup><a href="#B20">20</a></sup>. The various biochemical manipulations (such as deglycosylation for gp120 and papain digestion to produce Fab 17b), protein purification and ternary complex crystallization are described elsewhere<sup><a href="#B25">25</a></sup>. The best crystals were small needles of cross-section only 30–40&nbsp;<img src="glyph_011.gif" style="border: 0px none; vertical-align: baseline;" alt="mu">m. These were crosslinked by vapour-diffusion glutaraldehyde treatment (C. J. Lusty, personal communication), equilibrated with cryoprotectant containing stabilizer (10% ethylene glycol with 10.5% monomethyl-PEG5000, 10% isopropanol, 50&nbsp;mM NaCl, 100&nbsp;mM citrate/HEPES buffer, pH 6.3), transferred into immiscible oil (Paratone-N; Exxon), suspended in a small ethylene loop at the end of a mounting pin and flash-frozen in a cryostat nitrogen stream at 100&nbsp;K.</p><p class="norm">Diffraction data were collected at beamline X4A, Brookhaven National Laboratory, using phosphorimaging plates and a Fuji BAS2000 scanner. To avoid overlap problems from the relatively high mosaicity (<img src="glyph_005.gif" style="border: 0px none; vertical-align: baseline;" alt="approx">1.0°), oscillation data were collected using a rotation axis that was off-set at least 30° from the 197&nbsp;Å <i>c</i> axis. Although crystals initially diffracted to Bragg spacings of greater than 2&nbsp;Å, <i>b</i> axis mosaicity and substantial radiation damage, despite cryogenic cooling, reduced the overall limit to <img src="glyph_005.gif" style="border: 0px none; vertical-align: baseline;" alt="approx">2.5&nbsp;Å. Data processing and reduction were achieved using DENZO and SCALEPACK<sup><a href="#B45">45</a></sup> (<a href="#t1">Table 1</a>).</p><div id="t1" class="figure-table"><h5 class="norm"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/fig_tab/393648a0_T1.html">

Table 1:
			Structure solution</a></h5><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/fig_tab/393648a0_T1.html"><img alt="Table 1 - Structure solution" class="thumb" src="table_thumb.gif"></a><p><a class="full" href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/fig_tab/393648a0_T1.html">Full table</a></p><span class="cleardiv"><!-- --></span>
</div><br class="clear"><p class="norm"><b>Structure determination and refinement.</b> To locate the position of the Fab 17b in the ternary complex crystals, rotational searches with 52 different Fab models were made with the program MERLOT (P. M. Fitzgerald). The Fab fragments were aligned by superposition of their variable domains to allow comparison of rotational solutions. Even though four models showed more than 10% discrimination between the highest and second-highest solutions, no consistent rotational solution was found. Discrimination between correct and incorrect solutions was achieved by using confirmatory searches with the variable portion of the Fab. This was successful with only one model, molecule B of 1hil. Rigid-body refinement of the 1hil solution (XPLOR<sup><a href="#B46">46</a></sup>), allowing each immunoglobulin domain to move independently, produced a Patterson correlation of 24.9%. To locate the position of the two-domain CD4, each of the top 100 possible rotational solutions with each of three different CD4 models (1cdi, 1cdh, 3cd4), were searched for a distinctive translation solution (AMoRe; J. Navaza). The translation searches used the rigid body refined Fab as a partial structure to help discriminate the correct solution. Two distinctive solutions were found: the 25th rotational solution of 3cd4 gave a translation correlation of 0.171 (versus 0.128 for the second-highest translation solution), and the 61st rotational solution of 1cdh gave 0.149 (versus 0.140). These two solutions were virtually identical. Rigid-body refinement in XPLOR<sup><a href="#B46">46</a></sup> gave a Patterson correlation of 7.9% for CD4 alone and 32.4% for Fab and CD4. All molecular replacement and rigid-body refinements used 8–4&nbsp;Å data.</p><p class="norm">To provide additional phasing, crystals were soaked in over 20 different heavy-atom solutions and screened for isomorphous replacement using the statistical chi-squared test in SCALEPACK<sup><a href="#B45">45</a></sup>. Derivatives were identified from two heavy-atom compounds: 10&nbsp;mM K<sub>3</sub>IrCl<sub>6</sub> (10&nbsp;h equilibration in a solution containing the heavy atom and cryoprotectant stabilizer; 2.8&nbsp;Å) and 5&nbsp;mM K<sub>2</sub>OsCl<sub>6</sub> (24&nbsp;h soak; 3.5&nbsp;Å). Isomorphism was found to be highest between these heavy-atom data sets and a native data set collected at pH 7.0 (cryoprotectant stabilizer buffered with 50&nbsp;mM bis-Tris, pH 7.0). Heavy-atom sites were identified by difference Fourier analysis using the molecular replacement phases, and phasing parameters were refined with MLPHARE (in the CCP4 suite of crystallographic programs). The K<sub>3</sub>IrCl<sub>6</sub> derivative was modelled as nine partially occupied sites: two sites of occupancy 0.158 and 0.142, and seven of less than 0.07. Although relatively isomorphous, poor data quality (<i>R</i><sub>sym</sub> of more than 20% past 3.0&nbsp;Å) combined with relatively small isomorphous differences (<i>R</i><sub>iso</sub> of 12.0%) reduced the quality of phasing. In contrast, the K<sub>2</sub>OsC<sub>16</sub> derivative had an <i>R</i><sub>iso</sub> of 15.6%, but was only isomorphous to <img src="glyph_005.gif" style="border: 0px none; vertical-align: baseline;" alt="approx">5&nbsp;Å. It was modelled as four sites of occupancy 0.321, 0.207, 0.194 and 0.128, with the highest site at the same position as the second-highest site from K<sub>3</sub>IrCl<sub>6</sub>.</p><p class="norm">The initial combination of model and isomorphous replacement phasing did not produce readily interpretable density for gp120. To monitor phase improvement, we devised an objective assay of density quality that used correlations in a region internal to domain 1 of CD4 between the experimental electron density and the calculated model density (CD4 as positioned by molecular replacement and rigid-body refinement). Refinement of heavy-atom positions improved this correlation, and provided a starting point for phase improvement, primarily using real-space modification techniques (<a href="#t1">Table 1</a>). These techniques included automatic concatenation of unmodelled density (with the program PRISM; D. Agard), reciprocal-space averaging of the PRISM modelled density and real-space model subtraction (implemented using the XPLOR<sup><a href="#B46">46</a></sup> shell language), application of real-space constraints such as solvent flattening, histogram matching and negative-density truncation (with the program DM (in the CCP4 suite of crystallographic programs)), and real-space combined addition of the various experimental density maps (with the program MAPMAN; T. A. Jones). The combined application of these techniques greatly improved electron-density maps (<a href="#t1">Table 1</a>).</p><p class="norm">At this point, most of the C<img src="glyph_003.gif" style="border: 0px none; vertical-align: baseline;" alt="alpha"> backbone could be modelled (with program O<sup><a href="#B47">47</a></sup>), thereby defining the secondary structure. Computer-aided sequence alignment (slider routine in O) and secondary-structure prediction (PHD<sup><a href="#B37">37</a></sup>) helped to position the amino-acid sequence, leaving only regions around the N terminus (residues 79–100 and residues 215–245), the V1/V2 loop, and the V4 loop uncertain. Iterative rounds of building with O, simulated annealing and positional refinement with XPLOR<sup><a href="#B46">46</a></sup>, and addition of ordered solvent clarified the trace.</p><p class="norm"><b>Structure analysis.</b> Deviations of the CD4 structure in the complex from the free state were measured as described<sup><a href="#B39">39</a></sup>. Deviations were taken as significant when the root-mean-square (r.m.s.) residue deviation was greater than the overall value and also more than 0.5&nbsp;Å greater than variation among the free structures. Interatomic contacts were defined as in <a href="#B48">ref. 48</a>. Structural alignments were made by visual comparison of the SCOP database, and automatic searches were made with PrISM (A.-S. Yang and B. Honig).</p></div></div><p class="norm"></p><div id="References"><a href="#top" class="backtotop">Top<span class="hidden"> of page</span></a><h3 class="sectitle1">References</h3><ol class="norm"><li id="B1"><a name="B1"><!-- . --></a>

Barre-Sinoussi, F. <i>et al</i>. Isolation of a T-lymphotropic retrovirus from a patient at risk for acquired immunodeficiency syndrome (AIDS). <i>Science</i> <b>220</b>, 868–871 (1983).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=6189183&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Isolation of a T-lymphotropic retrovirus from a patient at risk for acquired immunodeficiency syndrome (AIDS).">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=ERROR&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Isolation of a T-lymphotropic retrovirus from a patient at risk for acquired immunodeficiency syndrome (AIDS).">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:STN:280:BiyC1Mjjs1w%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=ad230ac69554e82ae7f8fe5e6a5f1c5f" class="reftxt" target="external" title="Article on ChemPort - Isolation of a T-lymphotropic retrovirus from a patient at risk for acquired immunodeficiency syndrome (AIDS).">ChemPort</a>&nbsp;|</li><li id="B2"><a name="B2"><!-- . --></a>

Gallo, R. C. <i>et al</i>. Frequent detection and isolation of cytopathic retroviruses (HTLV-III) from patients with AIDS and at risk for AIDS. <i>Science</i> <b>224</b>, 500–503 (1984).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=6200936&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Frequent detection and isolation of cytopathic retroviruses (HTLV-III) from patients with AIDS and at risk for AIDS.">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:STN:280:BiuC2s3itFY%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=c5a0565d86e3620f40d34a1dd6b27f66" class="reftxt" target="external" title="Article on ChemPort - Frequent detection and isolation of cytopathic retroviruses (HTLV-III) from patients with AIDS and at risk for AIDS.">ChemPort</a>&nbsp;|</li><li id="B3"><a name="B3"><!-- . --></a>

Kowalski, M. L. <i>et al</i>. Functional regions of the envelope glycoprotein of human immunodeficiency virus type 1. <i>Science</i> <b>237</b>, 1351–1355 (1987).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=3629244&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Functional regions of the envelope glycoprotein of human immunodeficiency virus type 1.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1987J924400027&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Functional regions of the envelope glycoprotein of human immunodeficiency virus type 1.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaL2sXlslajtbY%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=f978c76554682f660aaeaebc8d5ccdc7" class="reftxt" target="external" title="Article on ChemPort - Functional regions of the envelope glycoprotein of human immunodeficiency virus type 1.">ChemPort</a>&nbsp;|</li><li id="B4"><a name="B4"><!-- . --></a>

Lu, M.
, 
Blackow, S.
 &amp; 
Kim, P.
 Atrimeric structural domain of the HIV-1 transmembrane glycoprotein. <i>Nature Struct. Biol.</i> <b>2</b>, 1075–1082 (1995).&nbsp;|&nbsp;<a href="http://www.nature.com.libgate.library.nuigalway.ie/doifinder/10.1038/nsb1295-1075" class="reftxt" target="external" title="Article on Article - Atrimeric structural domain of the HIV-1 transmembrane glycoprotein.">Article</a>&nbsp;|</li><li id="B5"><a name="B5"><!-- . --></a>

Starcich, B. R. <i>et al</i>. Identification and characterization of conserved and variable regions of the envelope gene HTLV-III/LAV, the retrovirus of AIDS. <i>Cell</i> <b>45</b>, 637–648 (1986).&nbsp;|&nbsp;<a href="http://dx.doi.org.libgate.library.nuigalway.ie/10.1016/0092-8674(86)90778-6" class="reftxt" target="external" title="Identification and characterization of conserved and variable regions of the envelope gene HTLV-III/LAV, the retrovirus of AIDS.">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=2423250&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Identification and characterization of conserved and variable regions of the envelope gene HTLV-III/LAV, the retrovirus of AIDS.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1986C698200002&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Identification and characterization of conserved and variable regions of the envelope gene HTLV-III/LAV, the retrovirus of AIDS.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaL28Xkslentbc%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=d4e8bf4a7ed5042962a80227f2d48ffa" class="reftxt" target="external" title="Article on ChemPort - Identification and characterization of conserved and variable regions of the envelope gene HTLV-III/LAV, the retrovirus of AIDS.">ChemPort</a>&nbsp;|</li><li id="B6"><a name="B6"><!-- . --></a>

Leonard, C. K. <i>et al</i>. Assignment of intrachain disulfide bonds and characterization of potential glycosylation sites of the type 1 recombinant immunodeficiency virus envelope glycoprotein (gp120) expressed in Chinese hamster ovary cell. <i>J. Biol. Chem.</i> <b>265</b>, 10373–10382 (1990).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=2355006&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Assignment of intrachain disulfide bonds and characterization of potential glycosylation sites of the type 1 recombinant immunodeficiency virus envelope glycoprotein (gp120) expressed in Chinese hamster ovary cell.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1990DK85100034&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Assignment of intrachain disulfide bonds and characterization of potential glycosylation sites of the type 1 recombinant immunodeficiency virus envelope glycoprotein (gp120) expressed in Chinese hamster ovary cell.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK3cXlvVKktLs%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=e472aec85658aad5fbf3f0e26ef71a06" class="reftxt" target="external" title="Article on ChemPort - Assignment of intrachain disulfide bonds and characterization of potential glycosylation sites of the type 1 recombinant immunodeficiency virus envelope glycoprotein (gp120) expressed in Chinese hamster ovary cell.">ChemPort</a>&nbsp;|</li><li id="B7"><a name="B7"><!-- . --></a>

Profy, A. T. <i>et al</i>. Epitopes recognized by the neutralizing antibodies of an HIV-1-infected individual. <i>J. Immunol.</i> <b>144</b>, 4641–4647 (1990).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=1693639&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Epitopes recognized by the neutralizing antibodies of an HIV-1-infected individual.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1990DK40600021&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Epitopes recognized by the neutralizing antibodies of an HIV-1-infected individual.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK3cXkslKgt7c%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=3e6b5735ac63781482df2a9e7acb1016" class="reftxt" target="external" title="Article on ChemPort - Epitopes recognized by the neutralizing antibodies of an HIV-1-infected individual.">ChemPort</a>&nbsp;|</li><li id="B8"><a name="B8"><!-- . --></a>

Ryu, S.-E. <i>et al</i>. Crystal structure of an HIV-binding recombinant fragment of human CD4. <i>Nature</i> <b>348</b>, 419–426 (1990).&nbsp;|&nbsp;<a href="http://www.nature.com.libgate.library.nuigalway.ie/doifinder/10.1038/348419a0" class="reftxt" target="external" title="Article on Article - Crystal structure of an HIV-binding recombinant fragment of human CD4.">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=2247146&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Crystal structure of an HIV-binding recombinant fragment of human CD4.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1990EK69700053&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Crystal structure of an HIV-binding recombinant fragment of human CD4.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK3MXit1Sltw%3D%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=2bb553d7dde4308ca79cd0c5dc92490f" class="reftxt" target="external" title="Article on ChemPort - Crystal structure of an HIV-binding recombinant fragment of human CD4.">ChemPort</a>&nbsp;|</li><li id="B9"><a name="B9"><!-- . --></a>

Wang, J. H. <i>et al</i>. Atomic structure of a fragment of human CD4 containing two immunoglobulin-like domains. <i>Nature</i> <b>348</b>, 411–418 (1990).&nbsp;|&nbsp;<a href="http://www.nature.com.libgate.library.nuigalway.ie/doifinder/10.1038/348411a0" class="reftxt" target="external" title="Article on Article - Atomic structure of a fragment of human CD4 containing two immunoglobulin-like domains.">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=1701030&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Atomic structure of a fragment of human CD4 containing two immunoglobulin-like domains.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1990EK69700052&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Atomic structure of a fragment of human CD4 containing two immunoglobulin-like domains.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK3MXit1Sltg%3D%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=a981243b7e4da67b52ea4139e66c245c" class="reftxt" target="external" title="Article on ChemPort - Atomic structure of a fragment of human CD4 containing two immunoglobulin-like domains.">ChemPort</a>&nbsp;|</li><li id="B10"><a name="B10"><!-- . --></a>

Wu, H.
, 
Kwong, P. D.
 &amp; 
Hendrickson, W. A.
 Dimeric association and segmental variability in the structure of human CD4. <i>Nature</i> <b>387</b>, 527–530 (1997).

&nbsp;|&nbsp;<a href="http://www.nature.com.libgate.library.nuigalway.ie/doifinder/10.1038/387527a0" class="reftxt" target="external" title="Article on Article - Dimeric association and segmental variability in the structure of human CD4.">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9168119&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Dimeric association and segmental variability in the structure of human CD4.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1997XB47900055&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Dimeric association and segmental variability in the structure of human CD4.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK2sXjsF2nu7o%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=758a9844b33e2a5a772e56371c40812f" class="reftxt" target="external" title="Article on ChemPort - Dimeric association and segmental variability in the structure of human CD4.">ChemPort</a>&nbsp;|</li><li id="B11"><a name="B11"><!-- . --></a>

Moebius, U.
, 
Clayton, L.
, 
Abraham, S.
, 
Harrison, S.
 &amp; 
Reinhertz, E.
 The human immunodeficiency virus gp120 binding site of CD4: Delineation by quantitative equilibrium and kinetic binding studies of mutants in conjunction with a high-resolution CD4 atomic structure. <i>J. Exp. Med.</i> <b>176</b>, 507–517 (1992).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=1500858&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - The human immunodeficiency virus gp120 binding site of CD4: Delineation by quantitative equilibrium and kinetic binding studies of mutants in conjunction with a high-resolution CD4 atomic structure.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1992JF80300020&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - The human immunodeficiency virus gp120 binding site of CD4: Delineation by quantitative equilibrium and kinetic binding studies of mutants in conjunction with a high-resolution CD4 atomic structure.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK38XltV2rurg%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=2e223e882031faa0b93b91171e27ae5c" class="reftxt" target="external" title="Article on ChemPort - The human immunodeficiency virus gp120 binding site of CD4: Delineation by quantitative equilibrium and kinetic binding studies of mutants in conjunction with a high-resolution CD4 atomic structure.">ChemPort</a>&nbsp;|</li><li id="B12"><a name="B12"><!-- . --></a>

Sweet, R. W.
, 
Truneh, A.
 &amp; 
Hendrickson, W. A.
 CD4: its structure, role in immune function and AIDS pathogenesis, and potential as a pharmacologicla target. <i>Curr. Opin. Biotech.</i> <b>2</b>, 622–633 (1991).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=1367682&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - CD4: its structure, role in immune function and AIDS pathogenesis, and potential as a pharmacologicla target.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1991GG94700017&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - CD4: its structure, role in immune function and AIDS pathogenesis, and potential as a pharmacologicla target.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK38XhslWlt70%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=46fd9fb5b5f2ca78632bffbafd1c27fb" class="reftxt" target="external" title="Article on ChemPort - CD4: its structure, role in immune function and AIDS pathogenesis, and potential as a pharmacologicla target.">ChemPort</a>&nbsp;|</li><li id="B13"><a name="B13"><!-- . --></a>

Olshevsky, U. <i>et al</i>. Identification of individual HIV-1 gp120 amino acids important for CD4 receptor binding. <i>J. Virol.</i> <b>64</b>((1990)).</li><li id="B14"><a name="B14"><!-- . --></a>

Cordonnier, A.
, 
Montagnier, L.
 &amp; 
Emerman, M.
 Single amino-acid changes in HIV envelope affect viral tropism and receptor binding. <i>Nature</i> <b>340</b>, 571–574 (1989).&nbsp;|&nbsp;<a href="http://www.nature.com.libgate.library.nuigalway.ie/doifinder/10.1038/340571a0" class="reftxt" target="external" title="Article on Article - Single amino-acid changes in HIV envelope affect viral tropism and receptor binding.">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=2475780&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Single amino-acid changes in HIV envelope affect viral tropism and receptor binding.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1989AL22700063&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Single amino-acid changes in HIV envelope affect viral tropism and receptor binding.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaL1MXlsFSnu78%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=f20590d5bf53ee4e07e856559cc87fdf" class="reftxt" target="external" title="Article on ChemPort - Single amino-acid changes in HIV envelope affect viral tropism and receptor binding.">ChemPort</a>&nbsp;|</li><li id="B15"><a name="B15"><!-- . --></a>

Moore, J. P.
 Coreceptors: implications for HIV pathogenesis and therapy. <i>Science</i> <b>276</b>, 51–52 (1997).&nbsp;|&nbsp;<a href="http://dx.doi.org.libgate.library.nuigalway.ie/10.1126/science.276.5309.51" class="reftxt" target="external" title="Coreceptors: implications for HIV pathogenesis and therapy.">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9122710&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Coreceptors: implications for HIV pathogenesis and therapy.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1997WR38600036&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Coreceptors: implications for HIV pathogenesis and therapy.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK2sXitlyqsLc%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=6ceebf54a8f5dd6ab87ce7cca2d7a204" class="reftxt" target="external" title="Article on ChemPort - Coreceptors: implications for HIV pathogenesis and therapy.">ChemPort</a>&nbsp;|</li><li id="B16"><a name="B16"><!-- . --></a>

Feng, F.
, 
Broder, C. C.
, 
Kennedy, P. E.
 &amp; 
Berger, E. A.
 HIV-1-entry cofactor: functional cDNA cloning of a seven-transmembrane, G protein-coupled receptor. <i>Science</i> <b>272</b>, 872–877 (1996).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8629022&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - HIV-1-entry cofactor: functional cDNA cloning of a seven-transmembrane, G protein-coupled receptor.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1996UK75700059&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - HIV-1-entry cofactor: functional cDNA cloning of a seven-transmembrane, G protein-coupled receptor.">ISI</a>&nbsp;|</li><li id="B17"><a name="B17"><!-- . --></a>

Speck, R. F. <i>et al</i>. Selective employment of chemokine receptors as human immunodeficiency virus type 1 coreceptors determined by individual amino acids in the envelope V3 loop. <i>J. Virol.</i> <b>71</b>, 7136–7139 (1997).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9261451&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Selective employment of chemokine receptors as human immunodeficiency virus type 1 coreceptors determined by individual amino acids in the envelope V3 loop.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1997XQ79900109&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Selective employment of chemokine receptors as human immunodeficiency virus type 1 coreceptors determined by individual amino acids in the envelope V3 loop.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK2sXlsVOjtLs%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=b7d6706639f57832278fa227c362ba4d" class="reftxt" target="external" title="Article on ChemPort - Selective employment of chemokine receptors as human immunodeficiency virus type 1 coreceptors determined by individual amino acids in the envelope V3 loop.">ChemPort</a>&nbsp;|</li><li id="B18"><a name="B18"><!-- . --></a>

Thali, M. <i>et al</i>. Characterization of conserved human immunodeficiency virus type 1 (HIV-1) gp120 neutralization epitopes exposed upon gp120-CD4 binding. <i>J. Virol.</i> <b>67</b>, 3978–3988 (1993).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=7685405&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Characterization of conserved human immunodeficiency virus type 1 (HIV-1) gp120 neutralization epitopes exposed upon gp120-CD4 binding.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1993LH12400033&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Characterization of conserved human immunodeficiency virus type 1 (HIV-1) gp120 neutralization epitopes exposed upon gp120-CD4 binding.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK3sXltFGht70%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=ea8e920f1010ec669cc474c7cf8b2ed6" class="reftxt" target="external" title="Article on ChemPort - Characterization of conserved human immunodeficiency virus type 1 (HIV-1) gp120 neutralization epitopes exposed upon gp120-CD4 binding.">ChemPort</a>&nbsp;|</li><li id="B19"><a name="B19"><!-- . --></a>

Sattentau, Q. J.
, 
Moore, J. P.
, 
Vignaux, F.
, 
Traincard, F.
 &amp; 
Poignard, P.
 Conformational changes induced in the envelope glycoproteins of human and simian immunodeficiency virus by soluble receptor binding. <i>J. Virol.</i> <b>64</b>, 7383–7383 (1993).</li><li id="B20"><a name="B20"><!-- . --></a>

Wu, L. <i>et al</i>. CD4-induced interaction of primary HIV-1 gp120 glycoproteins with the chemokine receptor CCR5. <i>Nature</i> <b>384</b>, 179–183 (1996).

&nbsp;|&nbsp;<a href="http://www.nature.com.libgate.library.nuigalway.ie/doifinder/10.1038/384179a0" class="reftxt" target="external" title="Article on Article - CD4-induced interaction of primary HIV-1 gp120 glycoproteins with the chemokine receptor CCR5.">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8906795&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - CD4-induced interaction of primary HIV-1 gp120 glycoproteins with the chemokine receptor CCR5.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1996VT33600069&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - CD4-induced interaction of primary HIV-1 gp120 glycoproteins with the chemokine receptor CCR5.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK28XmvFGgs70%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=255b6ef56d1e63af7c1e0fcbfdb700c8" class="reftxt" target="external" title="Article on ChemPort - CD4-induced interaction of primary HIV-1 gp120 glycoproteins with the chemokine receptor CCR5.">ChemPort</a>&nbsp;|</li><li id="B21"><a name="B21"><!-- . --></a>

Moore, J. P.
, 
McKeating, J. A.
, 
Weiss, R. A.
 &amp; 
Sattentau, Q. J.
 Dissociation of gp120 from HIV-1 virions induced by soluble CD4. <i>Science</i> <b>250</b>, 1139–1142 (1990).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=2251501&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Dissociation of gp120 from HIV-1 virions induced by soluble CD4.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1990EJ58100037&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Dissociation of gp120 from HIV-1 virions induced by soluble CD4.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK3MXit1Smsg%3D%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=1bbc66033dac58a3af2e322c7b5784b6" class="reftxt" target="external" title="Article on ChemPort - Dissociation of gp120 from HIV-1 virions induced by soluble CD4.">ChemPort</a>&nbsp;|</li><li id="B22"><a name="B22"><!-- . --></a>

Bullough, P. A.
, 
Hughson, F. M.
, 
Skehel, J. J.
 &amp; 
Wiley, D. C.
 Structure of influenza haemagglutinin at the pH of membrane fusion. <i>Nature</i> <b>371</b>, 37–43 (1994).

&nbsp;|&nbsp;<a href="http://www.nature.com.libgate.library.nuigalway.ie/doifinder/10.1038/371037a0" class="reftxt" target="external" title="Article on Article - Structure of influenza haemagglutinin at the pH of membrane fusion.">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8072525&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Structure of influenza haemagglutinin at the pH of membrane fusion.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1994PE38100042&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Structure of influenza haemagglutinin at the pH of membrane fusion.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK2cXmt1Cmsbw%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=498b180753461e6badb7b85094d8f4e8" class="reftxt" target="external" title="Article on ChemPort - Structure of influenza haemagglutinin at the pH of membrane fusion.">ChemPort</a>&nbsp;|</li><li id="B23"><a name="B23"><!-- . --></a>

Fass, D. <i>et al</i>. Structure of a murine leukemia virus receptor-binding glycoprotein at 2.0 Ångström resolution. <i>Science</i> <b>277</b>, 1662–1666 (1997).&nbsp;|&nbsp;<a href="http://dx.doi.org.libgate.library.nuigalway.ie/10.1126/science.277.5332.1662" class="reftxt" target="external" title="Structure of a murine leukemia virus receptor-binding glycoprotein at 2.0 [angst]ngstrom resolution.">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9287219&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Structure of a murine leukemia virus receptor-binding glycoprotein at 2.0 [angst]ngstrom resolution.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1997XV68400050&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Structure of a murine leukemia virus receptor-binding glycoprotein at 2.0 [angst]ngstrom resolution.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK2sXmtVSgtb8%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=9c6f453e7524beaf6aeef44cad5c7e89" class="reftxt" target="external" title="Article on ChemPort - Structure of a murine leukemia virus receptor-binding glycoprotein at 2.0 [angst]ngstrom resolution.">ChemPort</a>&nbsp;|</li><li id="B24"><a name="B24"><!-- . --></a>

Wyatt, R. <i>et al</i>. The antigenic structure of the HIV gp120 envelope glycoprotein. <i>Nature</i> <b>393</b>, 705–711 (1998).

&nbsp;|&nbsp;<a href="http://www.nature.com.libgate.library.nuigalway.ie/doifinder/10.1038/31514" class="reftxt" target="external" title="Article on Article - The antigenic structure of the HIV gp120 envelope glycoprotein.">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9641684&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - The antigenic structure of the HIV gp120 envelope glycoprotein.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000074289600059&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - The antigenic structure of the HIV gp120 envelope glycoprotein.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK1cXktVyqurs%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=e3f6f0e16f3de0612b0dbc6c54e13d45" class="reftxt" target="external" title="Article on ChemPort - The antigenic structure of the HIV gp120 envelope glycoprotein.">ChemPort</a>&nbsp;|</li><li id="B25"><a name="B25"><!-- . --></a>

Kwong, P. D. <i>et al</i>. Quantitative probability analysis and variational crystallization of gp120, the exterior envelope glycoprotein of the human immunodeficiency virus type 1 (HIV-1). <i>J. Biol. Chem.</i> (submitted).</li><li id="B26"><a name="B26"><!-- . --></a>

Binley, J. M. <i>et al</i>. Analysis of the interaction of antibodies with a conserved, enzymatically deglycosylated core of the HIV-1 gp120 envelope glycoprotein. <i>AIDS Res. Hum. Retrovir.</i> <b>14</b>, 191–198 (1997).&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000071946200001&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Analysis of the interaction of antibodies with a conserved, enzymatically deglycosylated core of the HIV-1 gp120 envelope glycoprotein.">ISI</a>&nbsp;|</li><li id="B27"><a name="B27"><!-- . --></a>

Leesong, M.
, 
Hederson, B. S.
, 
Gillig, J. R.
, 
Schwab, J. M.
 &amp; 
Smith, J. L.
 Structure of a dehydratase-isomerase from the bacterial pathway for biosynthesis of unsaturated fatty acids: two catalytic activities in one active site. <i>Structure</i> <b>4</b>, 253–264 (1996).&nbsp;|&nbsp;<a href="http://dx.doi.org.libgate.library.nuigalway.ie/10.1016/S0969-2126(96)00030-5" class="reftxt" target="external" title="Structure of a dehydratase-isomerase from the bacterial pathway for biosynthesis of unsaturated fatty acids: two catalytic activities in one active site.">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8805534&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Structure of a dehydratase-isomerase from the bacterial pathway for biosynthesis of unsaturated fatty acids: two catalytic activities in one active site.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1996UC73200006&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Structure of a dehydratase-isomerase from the bacterial pathway for biosynthesis of unsaturated fatty acids: two catalytic activities in one active site.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK28XitVegu7g%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=ce9d47dc992f00206fb8dd9a2bf45d01" class="reftxt" target="external" title="Article on ChemPort - Structure of a dehydratase-isomerase from the bacterial pathway for biosynthesis of unsaturated fatty acids: two catalytic activities in one active site.">ChemPort</a>&nbsp;|</li><li id="B28"><a name="B28"><!-- . --></a>

Cedergren-Zeppezauer, E. S.
, 
Larsson, G.
, 
Nyman, P. O.
, 
Dauter, Z.
 &amp; 
Wilson, K. S.
 Crystal structure of a dUTPase. <i>Nature</i> <b>355</b>, 740–743 (1992).

&nbsp;|&nbsp;<a href="http://www.nature.com.libgate.library.nuigalway.ie/doifinder/10.1038/355740a0" class="reftxt" target="external" title="Article on Article - Crystal structure of a dUTPase.">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=1311056&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Crystal structure of a dUTPase.">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK38XhvFSis7g%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=3541ceca717467c733c8f4bc63c86dad" class="reftxt" target="external" title="Article on ChemPort - Crystal structure of a dUTPase.">ChemPort</a>&nbsp;|</li><li id="B29"><a name="B29"><!-- . --></a>

Ryu, S.-E.
, 
Truneh, A.
, 
Sweet, R. W.
 &amp; 
Hendrickson, W. A.
 Structures of an HIV and MHC binding fragment from human CD4 as refined in two crystal lattices. <i>Structure</i> <b>2</b>, 59–74 (1994).&nbsp;|&nbsp;<a href="http://dx.doi.org.libgate.library.nuigalway.ie/10.1016/S0969-2126(00)00008-3" class="reftxt" target="external" title="Structures of an HIV and MHC binding fragment from human CD4 as refined in two crystal lattices.">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8075984&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Structures of an HIV and MHC binding fragment from human CD4 as refined in two crystal lattices.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1994NC28100008&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Structures of an HIV and MHC binding fragment from human CD4 as refined in two crystal lattices.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK2cXivFKmtL4%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=ea44ff1358b71d6e68712d36249a5eac" class="reftxt" target="external" title="Article on ChemPort - Structures of an HIV and MHC binding fragment from human CD4 as refined in two crystal lattices.">ChemPort</a>&nbsp;|</li><li id="B30"><a name="B30"><!-- . --></a>

Rizzuto, C. <i>et al</i>. Aconserved HIV gp120 glycoprotein structure involved in chemokine receptor binding. <i>Science</i> (in the press).</li><li id="B31"><a name="B31"><!-- . --></a>

Dragic, T. <i>et al</i>. Amino-terminal substitutions in the CCR5 coreceptor impair gp120 binding and human immunodeficiency type 1 entry. <i>J. Virol.</i> <b>72</b>, 279–285 (1998).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9420225&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Amino-terminal substitutions in the CCR5 coreceptor impair gp120 binding and human immunodeficiency type 1 entry.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1998YL01000032&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Amino-terminal substitutions in the CCR5 coreceptor impair gp120 binding and human immunodeficiency type 1 entry.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK1cXhvVCrsw%3D%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=fcb2b19024fa749b5c84759670d54d62" class="reftxt" target="external" title="Article on ChemPort - Amino-terminal substitutions in the CCR5 coreceptor impair gp120 binding and human immunodeficiency type 1 entry.">ChemPort</a>&nbsp;|</li><li id="B32"><a name="B32"><!-- . --></a>

Farzan, M. <i>et al</i>. Atryosine-rich region in the N-terminus of CCR5 is important for human immunodeficiency virus type 1 entry and mediates an association between gp120 and CCR5. <i>J. Virol.</i> <b>72</b>, 1160–1164 (1998).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9445013&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Atryosine-rich region in the N-terminus of CCR5 is important for human immunodeficiency virus type 1 entry and mediates an association between gp120 and CCR5.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000071440200034&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Atryosine-rich region in the N-terminus of CCR5 is important for human immunodeficiency virus type 1 entry and mediates an association between gp120 and CCR5.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK1cXlt1Khtg%3D%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=c697027c916f93a64fb77055c5fad5a3" class="reftxt" target="external" title="Article on ChemPort - Atryosine-rich region in the N-terminus of CCR5 is important for human immunodeficiency virus type 1 entry and mediates an association between gp120 and CCR5.">ChemPort</a>&nbsp;|</li><li id="B33"><a name="B33"><!-- . --></a>

Wyatt, R. <i>et al</i>. Analysis of the interaction of the human immunodeficiency virus type 1 gp120 envelope glycoprotein with the gp41 transmembrane glycoprotein. <i>J. Virol.</i> <b>71</b>, 9722–9731 (1997).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9371638&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Analysis of the interaction of the human immunodeficiency virus type 1 gp120 envelope glycoprotein with the gp41 transmembrane glycoprotein.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1997YF89600088&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Analysis of the interaction of the human immunodeficiency virus type 1 gp120 envelope glycoprotein with the gp41 transmembrane glycoprotein.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK2sXnsVGgu78%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=164adbb74573e35c70f6777496a1a6c3" class="reftxt" target="external" title="Article on ChemPort - Analysis of the interaction of the human immunodeficiency virus type 1 gp120 envelope glycoprotein with the gp41 transmembrane glycoprotein.">ChemPort</a>&nbsp;|</li><li id="B34"><a name="B34"><!-- . --></a>

Helseth, E.
, 
Olshevsky, U.
, 
Furman, C.
 &amp; 
Sodroski, J.
 Human immunodeficiency virus type 1 gp120 envelope glycoprotein regions important for association with the gp41 transmembrane glycoprotein. <i>J. Virol.</i> <b>65</b>, 2119–2123 (1991).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=2002555&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Human immunodeficiency virus type 1 gp120 envelope glycoprotein regions important for association with the gp41 transmembrane glycoprotein.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1991FC03100059&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Human immunodeficiency virus type 1 gp120 envelope glycoprotein regions important for association with the gp41 transmembrane glycoprotein.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK3MXhs1Snt74%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=d2bdf5af248bc819f4f8d78fcabb2274" class="reftxt" target="external" title="Article on ChemPort - Human immunodeficiency virus type 1 gp120 envelope glycoprotein regions important for association with the gp41 transmembrane glycoprotein.">ChemPort</a>&nbsp;|</li><li id="B35"><a name="B35"><!-- . --></a>

Wilson, I. A.
, 
Skehel, J. J.
 &amp; 
Wiley, D. C.
 Structure of the haemagglutinin membrane glycoprotein of influenza virus at 3&nbsp;Å resolution. <i>Nature</i> <b>289</b>, 366–373 (1981).&nbsp;|&nbsp;<a href="http://www.nature.com.libgate.library.nuigalway.ie/doifinder/10.1038/289366a0" class="reftxt" target="external" title="Article on Article - Structure of the haemagglutinin membrane glycoprotein of influenza virus at 3[thinsp][angst] resolution.">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=7464906&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Structure of the haemagglutinin membrane glycoprotein of influenza virus at 3[thinsp][angst] resolution.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1981KZ58500030&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Structure of the haemagglutinin membrane glycoprotein of influenza virus at 3[thinsp][angst] resolution.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaL3MXhvFSjtb8%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=880c6b91423b310e2c58dc66a96670e2" class="reftxt" target="external" title="Article on ChemPort - Structure of the haemagglutinin membrane glycoprotein of influenza virus at 3[thinsp][angst] resolution.">ChemPort</a>&nbsp;|</li><li id="B36"><a name="B36"><!-- . --></a>

Wyatt, R. <i>et al</i>. Involvement of the V1/V2 variable loop structure in the exposure of human immunodeficiency virus type 1 gp120 epitopes induced by receptor binding. <i>J. Virol.</i> <b>69</b>, 5723–5733 (1995).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=7543586&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Involvement of the V1/V2 variable loop structure in the exposure of human immunodeficiency virus type 1 gp120 epitopes induced by receptor binding.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1995RN98600062&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Involvement of the V1/V2 variable loop structure in the exposure of human immunodeficiency virus type 1 gp120 epitopes induced by receptor binding.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK2MXns1Khsrw%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=b8d454de69d5688516da07d91cf56b69" class="reftxt" target="external" title="Article on ChemPort - Involvement of the V1/V2 variable loop structure in the exposure of human immunodeficiency virus type 1 gp120 epitopes induced by receptor binding.">ChemPort</a>&nbsp;|</li><li id="B37"><a name="B37"><!-- . --></a>

Rost, B.
, 
Sander, C.
 &amp; 
Schneider, R.
 PHD — an automated mail server for protein secondary structure prediction. <i>Comput. Appl. Biosci.</i> <b>10</b>, 53–60 (1994).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8193956&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - PHD [mdash] an automated mail server for protein secondary structure prediction.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1994MY74700010&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - PHD [mdash] an automated mail server for protein secondary structure prediction.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK2cXmvFKmsrY%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=86a6f85a1423b898cbefca004207ba47" class="reftxt" target="external" title="Article on ChemPort - PHD [mdash] an automated mail server for protein secondary structure prediction.">ChemPort</a>&nbsp;|</li><li id="B38"><a name="B38"><!-- . --></a>

Wyatt, R. <i>et al</i>. Functional and immunologic characterization of human immunodeficiency virus type 1 envelope glycoproteins containing deletions of the major variable regions. <i>J. Virol.</i> <b>67</b>, 4557–4565 (1993).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8331723&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Functional and immunologic characterization of human immunodeficiency virus type 1 envelope glycoproteins containing deletions of the major variable regions.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1993LM27200013&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Functional and immunologic characterization of human immunodeficiency virus type 1 envelope glycoproteins containing deletions of the major variable regions.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK3sXltlOhtrk%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=711d9bd0680d3a578ec593f6290a8dc2" class="reftxt" target="external" title="Article on ChemPort - Functional and immunologic characterization of human immunodeficiency virus type 1 envelope glycoproteins containing deletions of the major variable regions.">ChemPort</a>&nbsp;|</li><li id="B39"><a name="B39"><!-- . --></a>

Wu, H. <i>et al</i>. Kinetic and structural analysis of mutant CD4 receptors that are defective in HIV gp120 binding. <i>Proc. Natl Acad. Sci. USA</i> <b>93</b>, 15030–15035 (1996).&nbsp;|&nbsp;<a href="http://dx.doi.org.libgate.library.nuigalway.ie/10.1073/pnas.93.26.15030" class="reftxt" target="external" title="Kinetic and structural analysis of mutant CD4 receptors that are defective in HIV gp120 binding.">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8986758&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Kinetic and structural analysis of mutant CD4 receptors that are defective in HIV gp120 binding.">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK2sXmslyg&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=71af0add03ec0157ba924836dd11fb63" class="reftxt" target="external" title="Article on ChemPort - Kinetic and structural analysis of mutant CD4 receptors that are defective in HIV gp120 binding.">ChemPort</a>&nbsp;|</li><li id="B40"><a name="B40"><!-- . --></a>

Chan, D. C.
, 
Fass, D.
, 
Berger, J. M.
 &amp; 
Kim, P. S.
 Core structure of gp41 from the HIV envelope glycoprotein. <i>Cell</i> <b>89</b>, 263–273 (1997).&nbsp;|&nbsp;<a href="http://dx.doi.org.libgate.library.nuigalway.ie/10.1016/S0248-4900(97)82314-3" class="reftxt" target="external" title="Core structure of gp41 from the HIV envelope glycoprotein.">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9108481&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Core structure of gp41 from the HIV envelope glycoprotein.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1997WU88800013&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Core structure of gp41 from the HIV envelope glycoprotein.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK2sXislChurs%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=8d081ba7e386a628f7ce58c0112b4eca" class="reftxt" target="external" title="Article on ChemPort - Core structure of gp41 from the HIV envelope glycoprotein.">ChemPort</a>&nbsp;|</li><li id="B41"><a name="B41"><!-- . --></a>

Weissenhorn, W.
, 
Dessen, A.
, 
Harrison, S. C.
, 
Skehel, J. J.
 &amp; 
Wiley, D. C.
 Atomic structure of the ectodomain from HIV-1 gp41. <i>Nature</i> <b>387</b>, 426–430 (1997).

&nbsp;|&nbsp;<a href="http://www.nature.com.libgate.library.nuigalway.ie/doifinder/10.1038/387426a0" class="reftxt" target="external" title="Article on Article - Atomic structure of the ectodomain from HIV-1 gp41.">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=9163431&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Atomic structure of the ectodomain from HIV-1 gp41.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1997XA49600066&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Atomic structure of the ectodomain from HIV-1 gp41.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK2sXjsVentL0%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=bf0f0aac3f7d9836a178af61687efb73" class="reftxt" target="external" title="Article on ChemPort - Atomic structure of the ectodomain from HIV-1 gp41.">ChemPort</a>&nbsp;|</li><li id="B42"><a name="B42"><!-- . --></a>

Dumonceaux, J. <i>et al</i>. Spontaneous mutations in the env gene of the human immunodeficiency virus type 1 NDK isolate are associated with a CD4-independent entry phenotype. <i>J. Virol.</i> <b>72</b>, 519–519 (1998).</li><li id="B43"><a name="B43"><!-- . --></a>

Moir, S.
, 
Perreault, J.
 &amp; 
Poulin, L.
 Postbinding events mediated by human immunodeficiency virus type 1 are sensitive to modifications in the D4-transmembrane linked region of CD4. <i>J. Virol.</i> <b>70</b>, 8019–8028 (1996).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8892926&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Postbinding events mediated by human immunodeficiency virus type 1 are sensitive to modifications in the D4-transmembrane linked region of CD4.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1996VL79200074&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Postbinding events mediated by human immunodeficiency virus type 1 are sensitive to modifications in the D4-transmembrane linked region of CD4.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK28Xmt1Kju7w%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=b332e77d2355c9df75949e30f26fe9be" class="reftxt" target="external" title="Article on ChemPort - Postbinding events mediated by human immunodeficiency virus type 1 are sensitive to modifications in the D4-transmembrane linked region of CD4.">ChemPort</a>&nbsp;|</li><li id="B44"><a name="B44"><!-- . --></a>

Kohlstaadt, L. A.
, 
Wang, J.
, 
Friedman, J. M.
, 
Rice, P. A.
 &amp; 
Steitz, T. A.
 Crystal structure at 3.5&nbsp;Å resolution of HIV-1 reverse transcriptase complexed with an inhibitor. <i>Science</i> <b>256</b>, 1783–1790 (1992).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=1377403&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Crystal structure at 3.5[thinsp][angst] resolution of HIV-1 reverse transcriptase complexed with an inhibitor.">PubMed</a>&nbsp;|</li><li id="B45"><a name="B45"><!-- . --></a>

Otwinowski, Z.
 &amp; 
Minor, W.
 Processing of X-ray diffraction data collected in oscillation mode. <i>Meth. Enymol.</i> <b>276</b>, 307–326 (1997).&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK2sXivFehsbw%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=86b1c9f1df4a1f4f790bf420751674ec" class="reftxt" target="external" title="Article on ChemPort - Processing of X-ray diffraction data collected in oscillation mode.">ChemPort</a>&nbsp;|</li><li id="B46"><a name="B46"><!-- . --></a>

Brunger, A. T.
 <i>XPLOR Version 3.1 manual</i> (Yale University, New Haven, (1993)).</li><li id="B47"><a name="B47"><!-- . --></a>

Jones, T. A.
, 
Zou, J. Y.
, 
Cowan, S. W.
 &amp; 
Kjeldgaard, M.
 Improved methods for building protein models in electron density maps and the location of errors in these models. <i>Acta Crystallogr. A</i> <b>47</b>, 110–119 (1991).&nbsp;|&nbsp;<a href="http://dx.doi.org.libgate.library.nuigalway.ie/10.1107/S0108767390010224" class="reftxt" target="external" title="Improved methods for building protein models in electron density maps and the location of errors in these models.">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=2025413&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Improved methods for building protein models in electron density maps and the location of errors in these models.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1991FC21500008&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Improved methods for building protein models in electron density maps and the location of errors in these models.">ISI</a>&nbsp;|</li><li id="B48"><a name="B48"><!-- . --></a>

Zhu, X. <i>et al</i>. Structural analysis of substrate binding by the molecular chaperone DnaK. <i>Science</i> <b>272</b>, 1606–1614 (1996).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=8658133&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Structural analysis of substrate binding by the molecular chaperone DnaK.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1996UR09300034&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Structural analysis of substrate binding by the molecular chaperone DnaK.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK28Xjs1ejtb0%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=ee6bec857f12d44b83e1da7834d0d1a4" class="reftxt" target="external" title="Article on ChemPort - Structural analysis of substrate binding by the molecular chaperone DnaK.">ChemPort</a>&nbsp;|</li><li id="B49"><a name="B49"><!-- . --></a>

Carson, M.
 <i>Ribbons 2.0</i>. <i>J. Appl. Crystallogr.</i> <b>24</b>, 958–961 (1991).&nbsp;|&nbsp;<a href="http://dx.doi.org.libgate.library.nuigalway.ie/10.1107/S0021889891007240" class="reftxt" target="external" title="Ribbons 2.0.">Article</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1991GL62900083&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Ribbons 2.0.">ISI</a>&nbsp;|</li><li id="B50"><a name="B50"><!-- . --></a>

Nicholls, A.
, 
Sharp, K. A.
 &amp; 
Honig, B.
 Protein folding and association: insight from the interfacial and thermodynamic properties of hydrocarbons. <i>Proteins Struct. Funct. Genet.</i> <b>11</b>, 281–296 (1991).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=1758883&amp;dopt=Abstract" class="reftxt" target="external" title="Article on PubMed - Protein folding and association: insight from the interfacial and thermodynamic properties of hydrocarbons.">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1991GV03200006&amp;DestApp=WOS_CPL" class="reftxt" target="external" title="Article on ISI - Protein folding and association: insight from the interfacial and thermodynamic properties of hydrocarbons.">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK38XhtVWgur4%3D&amp;pissn=0028-0836&amp;pyear=1998&amp;md5=37ad1e19ae5c9bfccc9d929a43334da9" class="reftxt" target="external" title="Article on ChemPort - Protein folding and association: insight from the interfacial and thermodynamic properties of hydrocarbons.">ChemPort</a>&nbsp;|</li></ol></div><div id="acknowledgements"><a href="#top" class="backtotop">Top<span class="hidden"> of page</span></a><h3>Acknowledgements</h3><p class="norm">We thank M. Doyle for microcalorimetry results, C. Kokolis and the SSCC group at SmithKline Beecham for assistance with gp120 proteins, C. Lusty for suggestions on crosslinking, C.Ogata for beamline assistance, A.-S. Yang for automated structural alignment, and past and present members of W.A.H.'s group, especially D. Fremont for help with programs, E. Martinez-Hackert for help with figures, L. Shapiro for discussion, J. Williams for sequence alignments, and H. Wu for analysis of CD4 movement. Beamline X4A at the National Synchrotron Light Source, a Department of Energy facility, is supported by the Howard Hughes Medical Institute. This work was made possible by gifts from the late William McCarty-Cooper, from the G. Harold and Leila Y. Mathers Foundation, from the Friends 10, and from Douglas and Judi Krupp. Support was provided by the Aaron Diamond Foundation, the American Foundation for AIDS Research, the Howard Hughes Medical Institute, and the NIH.</p></div><div class="no-header"><a href="#top" class="backtotop">Top<span class="hidden"> of page</span></a></div></div><div id="journalnav" class="archive"><h1 class="hidden">Main navigation</h1><map title="main journal navigation" id="main-journal-nav" class="journal-nav">
	<h2>Journal content</h2>
	<ul><li class="jn-home"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/">Journal home</a></li>
		
		<li class="jn-aop"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/research/research_by_subject.html">Advance online publication</a><ul class="subnav"><li class="jn-about-aop"><a href="http://www.nature.com.libgate.library.nuigalway.ie/authors/author_services/about_aop.html">About AOP</a></li></ul></li>
		
		<li class="jn-issue"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/current_issue.html">Current issue</a></li>

		<li><a href="http://www.nature.com.libgate.library.nuigalway.ie/news/">Nature News</a></li>
		<li class="jn-archive"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/archive/index.html">Archive</a></li>
		<li class="jn-supplements"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/archive/supplements.html">Supplements</a><ul class="subnav"><li class="jn-insights"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/archive/supplements.html">Insights</a></li><li class="jn-outlooks"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/archive/supplements.html">Outlooks</a></li><li class="jn-collections"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/archive/supplements.html">Collections</a></li></ul></li>
		
		<li class="jn-web-focuses"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/archive/specials.html">Web focuses</a><ul class="subnav"><li class="jn-biological-sciences"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/archive/specials.html">Biological sciences</a></li><li class="jn-earth-environment"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/archive/specials.html">Earth and environment</a></li><li class="jn-horizons"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/focus/horizons/">Horizons</a></li><li class="jn-physical-sciences"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/archive/specials.html">Physical sciences</a></li><li class="jn-science-politics"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/archive/specials.html">Science and politics</a></li><li class="jn-science-culture"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/archive/specials.html">Science, art and culture</a></li></ul></li>
		
		<li class="jn-podcast"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/podcast/">Podcasts</a></li>
		
		<li class="jn-video"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/videoarchive/">Videos</a></li>
		
		<li><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/archive/specials.html">News Specials</a></li></ul></map><map title="supplementary journal navigation" id="supplementary-journal-nav" class="journal-nav supplementary2">
	<h2>Journal information</h2>
	<ul>
		<li class="jn-about-journal">
			<a class="firstlevel" href="http://www.nature.com.libgate.library.nuigalway.ie/nature/about/">About the journal</a>
			<ul class="subnav"><li class="jn-contact-journal"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/about/contact/">Contact the journal</a></li><li class="jn-about-editors"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/about/editors/">About the editors</a></li><li class="jn-nature-family"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/about/family/">Nature family of journals</a></li><li class="jn-history"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/history/">History of the Journal <span class="i">Nature</span></a></li><li><a href="http://npg.nature.com.libgate.library.nuigalway.ie/media/nature/index.html">For advertisers</a></li><li><a href="http://www.nature.com.libgate.library.nuigalway.ie/libraries/">For librarians</a></li></ul>
		</li>
		<li class="jn-for-authors-and-referees">
			<a class="firstlevel" href="http://www.nature.com.libgate.library.nuigalway.ie/nature/authors/">For authors</a>
			<ul class="subnav"><li class="jn-guide-authors"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/authors/gta/">Formatting guide</a></li><li class="jn-publication-policies"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/authors/policy/">Publication policies</a></li><li class="jn-for-referees"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/authors/referees/">For referees</a></li></ul>
		</li>
		<li class="jn-submissions">
			<a class="firstlevel" href="http://www.nature.com.libgate.library.nuigalway.ie/nature/authors/submissions/">Online submission</a>
		</li>
		<li class="jn-awards">
			<a class="firstlevel" href="http://www.nature.com.libgate.library.nuigalway.ie/nature/awards/">Nature Awards</a>
		</li>

		<li class="jn-history">
			<a class="firstlevel" href="http://www.nature.com.libgate.library.nuigalway.ie/nature/history/index.html">Nature history</a>
		</li>

		

	</ul>
</map>

<map title="services journal navigation" id="services-journal-nav" class="journal-nav services">
	<h2>NPG <span>Services</span></h2>
	<ul><li class="jn-for-advertisers"><a href="http://npg.nature.com.libgate.library.nuigalway.ie/media/nature/index.html">Advertising</a></li><li class="jn-work"><a href="http://www.nature.com.libgate.library.nuigalway.ie/npg_/work/index.html" title="Work at NPG">work@<span class="npgworklogo">npg</span></a></li><li class="jn-permissions-and-reprints"><a href="http://www.nature.com.libgate.library.nuigalway.ie/reprints/" title="Nature reprints">Reprints &amp; permissions</a></li><li class="jn-for-librarians"><a href="http://www.nature.com.libgate.library.nuigalway.ie/libraries/">For librarians</a></li>
	<li class="jn-authors-and-referees"><a class="firstlevel" href="http://www.nature.com.libgate.library.nuigalway.ie/authors/index.html">Authors &amp; referees</a></li></ul>
</map>

<map title="resources journal navigation" id="resources-journal-nav" class="journal-nav resources">
	<h2>NPG <span>Resources</span></h2>
	<ul><li class="jn-gateways"><a href="http://www.nature.com.libgate.library.nuigalway.ie/databases/index.html">Gateways &amp; databases</a></li><li class="jn-reports"><a href="http://www.nature.com.libgate.library.nuigalway.ie/reports/index.html">Nature Reports</a></li><li class="jn-naturenetwork"><a href="http://network.nature.com.libgate.library.nuigalway.ie/">Nature Network</a></li><li class="jn-blogs"><a href="http://www.nature.com.libgate.library.nuigalway.ie/blogs/index.html">nature.com blogs</a></li><li class="jn-help"><a href="http://www.nature.com.libgate.library.nuigalway.ie/natureconferences/">Nature Conferences</a></li><li class="jn-topics"><a href="http://www.nature.com.libgate.library.nuigalway.ie/scitable/topics">Scitable Genetics</a></li></ul>
</map> 
<div id="global-subject-areas">
	<h2><span><abbr title="Nature Publishing Group">NPG</abbr> Journals</span></h2>	
	<h3 class="view">by Subject Area</h3>
		<ul class="category-list">
		<li><h4>Chemistry</h4>
            <ul>
                <li><a href="http://www.nature.com.libgate.library.nuigalway.ie/chemistry/index.html">Chemistry</a></li>
                <li><a href="http://www.nature.com.libgate.library.nuigalway.ie/drugdisc/index.html">Drug discovery</a></li>
		<li><a href="http://www.nature.com.libgate.library.nuigalway.ie/biotech/index.html">Biotechnology</a></li>
		<li><a href="http://www.nature.com.libgate.library.nuigalway.ie/materials/index.html">Materials</a></li>
		<li><a href="http://www.nature.com.libgate.library.nuigalway.ie/protometh/index.html">Methods &amp; Protocols</a></li>
            </ul>
        </li>        <li><h4>Clinical Practice &amp; Research</h4>
            <ul>
                <li><a href="http://www.nature.com.libgate.library.nuigalway.ie/cancer/index.html">Cancer</a></li>
                <li><a href="http://www.nature.com.libgate.library.nuigalway.ie/cardio/index.html">Cardiovascular medicine</a></li>
                <li><a href="http://www.nature.com.libgate.library.nuigalway.ie/dentistry/index.html">Dentistry</a></li>
                <li><a href="http://www.nature.com.libgate.library.nuigalway.ie/endocrinology/index.html" title="">Endocrinology</a></li>
                <li><a href="http://www.nature.com.libgate.library.nuigalway.ie/gastrohep/index.html">Gastroenterology &amp; Hepatology</a></li>
		<li><a href="http://www.nature.com.libgate.library.nuigalway.ie/protometh/index.html">Methods &amp; Protocols</a></li>
                <li><a href="http://www.nature.com.libgate.library.nuigalway.ie/pathology/index.html">Pathology &amp; Pathobiology</a></li>
                <li><a href="http://www.nature.com.libgate.library.nuigalway.ie/urology/index.html">Urology</a></li>

            </ul>
        </li>        <li><h4>Earth &amp; Environment</h4>
            <ul>
                <li><a href="http://www.nature.com.libgate.library.nuigalway.ie/earthsciences/index.html">Earth sciences</a></li>
                <li><a href="http://www.nature.com.libgate.library.nuigalway.ie/evoeco/index.html">Evolution &amp; Ecology</a></li>
	
            </ul>
        </li>        <li><h4>Life sciences</h4>
            <ul>
		<li><a href="http://www.nature.com.libgate.library.nuigalway.ie/biotech/index.html">Biotechnology</a></li>
                <li><a href="http://www.nature.com.libgate.library.nuigalway.ie/cancer/index.html">Cancer</a></li>
                <li><a href="http://www.nature.com.libgate.library.nuigalway.ie/development/index.html">Development</a></li>
                <li><a href="http://www.nature.com.libgate.library.nuigalway.ie/drugdisc/index.html">Drug discovery</a></li>
                <li><a href="http://www.nature.com.libgate.library.nuigalway.ie/evoeco/index.html">Evolution &amp; Ecology</a></li>
                <li><a href="http://www.nature.com.libgate.library.nuigalway.ie/genetics/index.html">Genetics</a></li>
                <li><a href="http://www.nature.com.libgate.library.nuigalway.ie/immuno/index.html">Immunology</a></li>
                <li><a href="http://www.nature.com.libgate.library.nuigalway.ie/medicalresearch/index.html">Medical research</a></li>
				<li><a href="http://www.nature.com.libgate.library.nuigalway.ie/protometh/index.html">Methods &amp; Protocols</a></li>
                <li><a href="http://www.nature.com.libgate.library.nuigalway.ie/micro/index.html">Microbiology</a></li>
                <li><a href="http://www.nature.com.libgate.library.nuigalway.ie/molcellbio/index.html">Molecular cell biology</a></li>
                <li><a href="http://www.nature.com.libgate.library.nuigalway.ie/neurosci/index.html">Neuroscience</a></li>
                <li><a href="http://www.nature.com.libgate.library.nuigalway.ie/pharma/index.html">Pharmacology</a></li>
                <li><a href="http://www.nature.com.libgate.library.nuigalway.ie/sysbio/index.html">Systems biology</a></li>
            </ul>
        </li>        <li><h4>Physical sciences</h4>
            <ul>
				<li><a href="http://www.nature.com.libgate.library.nuigalway.ie/physics/index.html">Physics</a></li>
				<li><a href="http://www.nature.com.libgate.library.nuigalway.ie/materials/index.html">Materials</a></li>
            </ul>
        </li>	</ul>
	
	<h3 class="view"><a href="http://www.nature.com.libgate.library.nuigalway.ie/siteindex/index.html" title="View A to Z list of Nature Publishing Group journals">by A - Z Index</a></h3>
</div></div></div><div class="archive" id="extranav"><h1 class="hidden">Extra navigation</h1><span class="page-header-spacer">

			.</span><div class="subscribe-to-nature">
	<a href="https://secure-nature-com.libgate.library.nuigalway.ie/store/svc/campaign?prod=NATURE&amp;text=NA1001EF000" title="subscribe to nature"><img src="subscribe.jpg" alt="Subscribe to Nature"></a>
</div>

<div id="articlenav"><h2><span class="hidden">ARTICLE NAVIGATION -</span>FULL TEXT</h2><div class="prevnext"><a title="previous article in this issue" class="prev" href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/full/393643a0.html">Previous</a><span class="divider">|</span><a title="next article in this issue" class="next" href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/full/393660a0.html">Next</a></div><ul><li class="toc"><a class="toc" href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686">Table of contents</a></li>




<li class="download-pdf"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/pdf/393648a0.pdf" class="download-pdf">Download PDF</a></li>	<li class="readcube"><a href="http://www.readcube.com/articles/10.1038/31405" class="readcube">View interactive PDF in ReadCube</a></li>








<li class="sendtofriend"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/foxtrot/svc/mailform?doi=10.1038/31405&amp;file=/nature/journal/v393/n6686/full/393648a0.html" class="sendtofriend">Send to a friend</a></li><li class="cited"><a class="cited" href="http://www.nature.com.libgate.library.nuigalway.ie/cited/cited.html?doi=10.1038/31405">CrossRef lists 765 articles citing this article</a></li><li class="cited"><a class="cited" href="http://www.scopus.com.libgate.library.nuigalway.ie/inward/record.url?eid=2-s2.0-0032543307&amp;partnerID=65">Scopus lists 1712 articles citing this article</a></li><li class="export"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/ris/393648a0.ris" class="export">Export citation</a></li><li class="export"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/ris/393648a0refs.ris" class="export">Export references</a></li><li class="permissions"><a class="permissions" href="https://s100.copyright.com/AppDispatchServlet?publisherName=NPG&amp;publication=Nature&amp;title=Structure%20of%20an%20HIV%20gp120%20envelope%20glycoprotein%20in%20complex%20with%20the%20CD4%20receptor%20and%20a%20neutralizing%20human%20antibody&amp;author=Peter%20D.%20Kwong,%20Richard%20Wyatt,%20James%20Robinson,%20Raymond%20W.%20Sweet,%20Joseph%20Sodroski%20et%20al.&amp;contentID=10.1038/31405&amp;publicationDate=//&amp;volumeNum=393&amp;issueNum=6686&amp;numPages=12&amp;pageNumbers=pp648-659">Rights and permissions</a></li><li class="commercial_reprints"><a class="commercial_reprints" href="https://s100.copyright.com/AppDispatchServlet?publisherName=NPGR&amp;publication=Nature&amp;title=Structure%20of%20an%20HIV%20gp120%20envelope%20glycoprotein%20in%20complex%20with%20the%20CD4%20receptor%20and%20a%20neutralizing%20human%20antibody&amp;author=Peter%20D.%20Kwong,%20Richard%20Wyatt,%20James%20Robinson,%20Raymond%20W.%20Sweet,%20Joseph%20Sodroski%20et%20al.%20&amp;contentID=10.1038/31405&amp;publicationDate=//&amp;volumeNum=393&amp;issueNum=6686&amp;numPages=12&amp;pageNumbers=pp648-659&amp;orderBeanReset=true"> Order commercial reprints</a></li>


<li><a class="page" href="#abs">Abstract</a></li><li><a class="page" href="#Structure-determination">Structure determination</a></li><li><a class="page" href="#Structure-of-gp120">Structure of gp120</a></li><li><a class="page" href="#CD4ndashgp120-interaction">CD4–gp120 interaction</a></li><li><a class="page" href="#Interfacial-cavities">Interfacial cavities</a></li><li><a class="page" href="#Antibody-interface">Antibody interface</a></li><li><a class="page" href="#Chemokinereceptor-site">Chemokine-receptor site</a></li><li><a class="page" href="#Oligomer-and-gp41-interactions">Oligomer and gp41 interactions</a></li><li><a class="page" href="#Conformational-change-in-core-gp120">Conformational change in core gp120</a></li><li><a class="page" href="#Viral-evasion-of-immune-responses">Viral evasion of immune responses</a></li><li><a class="page" href="#Mechanistic-implications-for-virus-entry">Mechanistic implications for virus entry</a></li><li><a class="page" href="#Methods">Methods</a></li><li><a class="page" href="#References">References</a></li><li><a class="page" href="#acknowledgements">Acknowledgements</a></li><li><a class="figures" href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/fig_tab/393648a0_ft.html">Figures and tables</a></li></ul></div><div class="extra pink"><h2>SEE ALSO</h2><ul><li><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/full/31359.html">News and Views by Moore&nbsp;&amp;&nbsp;Binley</a></li></ul></div><div class="extra grey"><h2>SEARCH PUBMED FOR</h2><ul><li><a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?cmd=search&amp;term=Kwong+P">Peter D. Kwong</a></li><li><a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?cmd=search&amp;term=Wyatt+R">Richard Wyatt</a></li><li><a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?cmd=search&amp;term=Robinson+J">James Robinson</a></li><li><a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?cmd=search&amp;term=Sweet+R">Raymond W. Sweet</a></li><li><a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?cmd=search&amp;term=Sodroski+J">Joseph Sodroski</a></li><li><a href="http://www.ncbi.nlm.nih.gov.libgate.library.nuigalway.ie/entrez/query.fcgi?cmd=search&amp;term=Hendrickson+W">Wayne A. Hendrickson</a></li><li class="more"><a href="http://www.nature.com.libgate.library.nuigalway.ie/nature/journal/v393/n6686/pubmed/393648a0.html">more authors of this article</a></li></ul></div><div id="natpav" class="box">
	<h2 class="natpav"><a href="http://www.nature.com.libgate.library.nuigalway.ie/openinnovation?moreChallenges=true" title="Visit the Nature.com Open Innovation Pavilion"><span class="first">Open Innovation Challenges</span></a></h2>
	<div class="box-item">
		<ul class="chal-list">
						   			
								<li>
										<h3><a href="http://www.nature.com.libgate.library.nuigalway.ie/openinnovation?challengeId=9933138">Identify Organisms from a Stream of DNA Sequences
                                        </a></h3>
					<img src="9933138.jpg" alt="">
					<ul class="chal-meta">
						<li>Deadline: Jun 30 2013</li>
						<li>Reward: <span class="award">$1,000,000 USD</span></li>
					</ul>
					<p></p><p> This is a&nbsp;<strong>Reduction-to-Practice&nbsp;</strong>Challenge that&nbsp;<strong>requires written documentation&nbsp;</strong>and&nbsp;<strong>delivery of sourc…</strong></p><p></p>
				</li>
										   			
								<li>
										<h3><a href="http://www.nature.com.libgate.library.nuigalway.ie/openinnovation?challengeId=9933200">Mechanisms and Innovative Ideas for Early Detection of Complications Due to Diabetes
                                        </a></h3>
					<img src="9933200.jpg" alt="">
					<ul class="chal-meta">
						<li>Deadline: Jul 22 2013</li>
						<li>Reward: <span class="award">$15,000 USD</span></li>
					</ul>
					<p></p><p> Diabetes is associated with many risk factors and symptoms that lead to serious medical conditions.…</p><p></p>
				</li>
									</ul>
		<ul class="more">
			<li><a href="http://www.nature.com.libgate.library.nuigalway.ie/openinnovation?moreChallenges=true" title="">More Challenges</a></li>
			<li class="powered"><div><span>Powered by:</span><a href="http://www.innocentive.com/"><img src="innocentive_logo_small.gif" alt="InnoCentive"></a></div></li>
		</ul>
		<span class="cleardiv"><!-- --></span>
	</div>
</div>
 



																					
<div class="box nature-feed" id="natjob">
	<h2 class="natjob"><a href="http://www.nature.com.libgate.library.nuigalway.ie/naturejobs/"><span class="first">nature</span><span class="hidden"> </span><span class="second">jobs</span></a></h2>	<ul class="natjob">
				<li><h3 class="first"><a href="http://www.nature.com.libgate.library.nuigalway.ie/naturejobs/science/jobs/321970-Assistant-Professor-in-Pathology-Non-Tenured-Research-Line">Assistant Professor in Pathology (Non-Tenured Research Line)</a></h3>
			<ul>
				<li>Stanford University School of Medicine</li>
							</ul>
		</li>	
				<li><h3><a href="http://www.nature.com.libgate.library.nuigalway.ie/naturejobs/science/jobs/317412-Endowed-Chair-Professorship">Endowed Chair Professorship</a></h3>
			<ul>
				<li>Dana-Farber Cancer Institute &amp; Harvard medical School</li>
							</ul>
		</li>	
			</ul>

	<div class="natjobmore"><a href="http://www.nature.com.libgate.library.nuigalway.ie/naturejobs/science/jobs">More science jobs</a></div>
	<div class="natjobmore"><a href="http://www.nature.com.libgate.library.nuigalway.ie/naturejobs/science/jobs/new">Post a job for free</a></div>
</div>
		<div class="ad-vert">ADVERTISEMENT<br>
<script type="text/javascript">

<!--//--><![CDATA[//><!--
document.write ('<ilayer id="layer2" visibility="hidden" width="160" height="600"></ilayer>');
document.write ('<nolayer>');
document.write ('<iframe src="http://ad.doubleclick.net/adi/nature.com/article;type=af;artid=393648a0;pos=right;issue=6686;webfo=;prod=;natco=;natcl=;subco=;chaco=;keywd=;;sz=160x600;ord=' + ordval + '?" width="160" height="600" marginwidth="0" marginheight="0" frameborder="0" scrolling="no">');
document.write ('<a href="http://ad.doubleclick.net/jump/nature.com/article;type=af;artid=393648a0;pos=right;issue=6686;webfo=;prod=;natco=;natcl=;subco=;chaco=;keywd=;abr=!ie4;abr=!ie5;sz=160x600;ord=' + ordval + '?">');
document.write ('<img src="http://ad.doubleclick.net/ad/nature.com/article;type=af;artid=393648a0;pos=right;issue=6686;webfo=;prod=;natco=;natcl=;subco=;chaco=;keywd=;abr=!ie4;abr=!ie5;sz=160x600;ord=' + ordval + '?" style="max-width:160px; height:600px; border:0;" /></a></iframe>');
document.write ('</nolayer>');
//--><!]]>
</script><ilayer id="layer2" visibility="hidden" width="160" height="600"></ilayer><nolayer><iframe class="yhkfnaqsgqmkflxzouey" src="393648a0_1.html" marginwidth="0" marginheight="0" frameborder="0" height="600" scrolling="no" width="160"><a href="http://ad.doubleclick.net/jump/nature.com/article;type=af;artid=393648a0;pos=right;issue=6686;webfo=;prod=;natco=;natcl=;subco=;chaco=;keywd=;abr=!ie4;abr=!ie5;sz=160x600;ord=1369670415907?"><img src="http://ad.doubleclick.net/ad/nature.com/article;type=af;artid=393648a0;pos=right;issue=6686;webfo=;prod=;natco=;natcl=;subco=;chaco=;keywd=;abr=!ie4;abr=!ie5;sz=160x600;ord=1369670415907?" style="max-width:160px; height:600px; border:0;" /></a></iframe></nolayer> 
<noscript>
 
<div><a href="http://ad.doubleclick.net/jump/nature.com/article;type=af;artid=393648a0;pos=right;issue=6686;webfo=;prod=;natco=;natcl=;subco=;chaco=;keywd=;abr=!NN2;sz=160x600;ord=07893195?"><img src="http://ad.doubleclick.net/ad/nature.com/article;pos=right;abr=!NN2;sz=160x600;ord=07893195?"   style="max-width:160px; height:600px; border:0;" alt="Advertisement" /></a></div>
</noscript></div></div></div></div>	

	





<div class="cleardiv"><!-- --></div>

	<div class="constrain-outer"><div class="constrain">

			
			<p class="back-to-top hidden"><a href="#top" title="Return to the top of this page">Top</a></p>
	
	<div id="ftr" class="footer">
							<ul id="footer-journal">
				<li class="journal-name">Nature</li>				<li class="issn"><abbr title="International Standard Serial Number">ISSN</abbr><span class="hidden">:</span> 0028-0836</li>				<li class="eissn"><abbr title="Electronic International Standard Serial Number">EISSN</abbr><span class="hidden">:</span> 1476-4687</li>															</ul>
			<span class="cleardiv"><!-- --></span>
		
		<div class="footer-inner">

		<div class="baseline-wrapper">
													<ul class="footer-links baseline first">
									<li><a href="http://www.nature.com.libgate.library.nuigalway.ie/npg/">About <abbr title="Nature Publishing Group">NPG</abbr></a></li>
														<li><a href="http://www.nature.com.libgate.library.nuigalway.ie/npg_/contact/">Contact <abbr title="Nature Publishing Group">NPG</abbr></a></li>
														<li><a href="http://www.nature.com.libgate.library.nuigalway.ie/info/accessibility_statement.html">Accessibility statement</a></li>
														<li><a href="http://www.nature.com.libgate.library.nuigalway.ie/help/">Help</a></li>
													</ul>
							<ul class="footer-links baseline">
									<li><a href="http://www.nature.com.libgate.library.nuigalway.ie/info/privacy.html">Privacy policy</a></li>
														<li><a href="http://www.nature.com.libgate.library.nuigalway.ie/info/cookies.html">Use of cookies</a></li>
														<li><a href="http://www.nature.com.libgate.library.nuigalway.ie/info/legal_notice.html">Legal notice</a></li>
														<li><a href="http://www.nature.com.libgate.library.nuigalway.ie/info/tandc.html">Terms</a></li>
													</ul>
							<ul class="footer-links baseline last">
									<li><a href="http://www.nature.com.libgate.library.nuigalway.ie/naturejobs/">Nature<span class="hidden"> </span>jobs</a></li>
														<li><a href="http://www.natureasia.com/">Nature Asia</a></li>
														<li><a href="http://www.nature.com.libgate.library.nuigalway.ie/scitable/">Nature Education</a></li>
														<li><a href="http://www.nature.com.libgate.library.nuigalway.ie/webfeeds/"><abbr title="RDF Site Summary">RSS</abbr> web feeds</a></li>
													</ul>
			
						<form style="height: 68px;" action="http://www.nature.com.libgate.library.nuigalway.ie/search/executeSearch" method="get" class="search-form baseline">
				<fieldset>
					<label for="footer-search">Search:</label>
					<input id="footer-search" name="sp-q" value="" class="text" maxlength="200" type="text">
					<input name="sp-p" value="all" type="hidden">
					<input name="sp-c" value="25" type="hidden">
					<input name="sp-m" value="0" type="hidden">
					<input name="sp-s" value="" type="hidden">
					<input name="search" value="go" class="button" type="submit">
				</fieldset>
			</form>
					</div>

		<span class="cleardiv"><!-- --></span>

					<div id="footer-copyright">
				<ul class="logo">
					<li>			<a href="http://www.nature.com.libgate.library.nuigalway.ie/info/copyright_statement.html" title="" class="class java.util.LinkedHashMap">© 2013 Nature Publishing Group, a division of Macmillan Publishers Limited. All Rights Reserved.</a>
	</li>					<li>partner of <a href="http://www.nature.com.libgate.library.nuigalway.ie/info/partners.html">AGORA, HINARI, OARE, INASP, ORCID, CrossRef and COUNTER</a></li>				</ul>
			</div>
				</div>
	</div>

	</div></div><!-- close constrain divs -->

	

<!--extra code for 160 x 600 ad-->
<script type="text/javascript">

<!--//--><![CDATA[//><!--
document.writeln ('<layer src="http://ad.doubleclick.net/adl/nature.com/article;pos=right;type=af;artid=393648a0;pos=right;issue=6686;webfo=;prod=;natco=;natcl=;subco=;chaco=;keywd=;sz=160x600;ord=' + ordval + '?" visibility="hidden" style="max-width:160px; height:600px;" onload="moveToAbsolute(layer2.pageX,layer2.pageY);clip.height=600;clip.width=160;visibility=\'show\';"></layer>');
//--><!]]>
</script><layer src="http://ad.doubleclick.net/adl/nature.com/article;pos=right;type=af;artid=393648a0;pos=right;issue=6686;webfo=;prod=;natco=;natcl=;subco=;chaco=;keywd=;sz=160x600;ord=1369670415907?" visibility="hidden" style="max-width: 160px; height: 600px;" onload="moveToAbsolute(layer2.pageX,layer2.pageY);clip.height=600;clip.width=160;visibility='show';"></layer>

<!--end of extra code--><!--extra code for 158 x 60 ad-->
<script type="text/javascript">

<!--//--><![CDATA[//><!--
document.writeln ('<layer src="http://ad.doubleclick.net/adl/nature.com/article;type=af;artid=393648a0;pos=top;issue=6686;webfo=;prod=;natco=;natcl=;subco=;chaco=;keywd=;sz=158x60;ord=' + ordval + '?" visibility="hidden" style="max-width:158px; height:60px;" onload="moveToAbsolute(layer1.pageX,layer1.pageY);clip.height=60;clip.width=158;visibility=\'show\';"></layer>');
//--><!]]>
</script><layer src="http://ad.doubleclick.net/adl/nature.com/article;type=af;artid=393648a0;pos=top;issue=6686;webfo=;prod=;natco=;natcl=;subco=;chaco=;keywd=;sz=158x60;ord=1369670415907?" visibility="hidden" style="max-width: 158px; height: 60px;" onload="moveToAbsolute(layer1.pageX,layer1.pageY);clip.height=60;clip.width=158;visibility='show';"></layer>

<!--end of extra code-->		
		<!-- super body bottom -->

<!-- super javascript -->

	
																																						
																																																			
										<script type="text/javascript" src="jquery.js"></script>
												<script type="text/javascript" src="cookie.js"></script>
												<script type="text/javascript" src="message.js"></script>
												<script type="text/javascript" src="site.js"></script>
												<script type="text/javascript" src="ping"></script>
												<script type="text/javascript" src="baseline.jquery.js"></script>
												<script type="text/javascript" src="global.baseline.js"></script>
												<script type="text/javascript" src="tsedge_instr-min.js"></script>
						
	
	<script type="text/javascript"><!--//--><![CDATA[//><!--
	(function($) {
		$('a.popup-this').bind('click', function() {
			window.open(this.href);
			return false;
		});
	})(jQuery);
	//--><!]]></script>

	

																																																																																																																																																																																																																																								
<!--[if IE 6]>
<![endif]-->
																																																																																																																																																																																																																																																																																																																																																																																																																																																																																
 






    <!-- webtrendslive -->   
	</body>
</html>
